<?xml version='1.0' encoding='utf-8' ?> 
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd"><html xmlns='http://www.w3.org/1999/xhtml'> 
<head><title>2 Quantifying allosteric communication via both concerted structural changes and conformational
disorder</title> 
<meta http-equiv='Content-Type' content='text/html; charset=utf-8' /> 
<meta name='generator' content='TeX4ht (https://tug.org/tex4ht/)' /> 
<meta name='originator' content='TeX4ht (https://tug.org/tex4ht/)' /> 
<!--  xhtml,charset=utf-8,epub,uni-html4,html  --> 
<meta name='src' content='main.tex' /> 
<link type='text/css' href='main.css' rel='stylesheet' /> 
</head><body>
<!-- l. 3 --><div class='crosslinks'><p class='noindent'></p></div>
<h2 class='chapterHead'><span class='titlemark'>Chapter 2</span><br /><a id='x7-150002'></a>Quantifying allosteric communication via both concerted structural changes and
conformational disorder</h2>
<!-- l. 12 --><p class='noindent'><span class='ptmri7t-x-x-120'>The work in this chapter is published in: Singh, S., and Bowman, G.R., Quantifying allosteric
</span><span class='ptmri7t-x-x-120'>communication via both concerted structural changes and conformational disorder with CARDS.
</span><span class='ptmri7t-x-x-120'>Journal of Chemical Theory and Computation. 13:1507-1517, 2017. PMID: 28282132., Copyright
</span><span class='ptmri7t-x-x-120'>2018 American Chemical Society </span><span class='cite'> [<a href='mainli5.html#XSingh:2017hh'>90</a>]</span>
</p>
<h3 class='sectionHead'><span class='titlemark'>2.1    </span> <a id='x7-160002.1'></a>Abstract</h3>
<!-- l. 15 --><p class='noindent'>Allosteric (i.e. long-range) communication within proteins is crucial for many biological processes,
such as the activation of signaling cascades in response to specific stimuli. However, the physical basis
for this communication remains unclear. Existing computational methods for identifying
allostery focus on the role of concerted structural changes, but recent experimental work
demonstrates that disorder is also an important factor. Here, we introduce the Correlation of All
Rotameric and Dynamical States (CARDS) framework for quantifying correlations between
both the structure and disorder of different regions of a protein. To quantify disorder, we
draw inspiration from methods for quantifying “dynamic heterogeneity” from chemical
physics to classify segments of a dihedral’s time evolution as being in either ordered or
disordered regimes. To demonstrate the utility of this approach, we apply CARDS to the
Catabolite Activator Protein (CAP), a transcriptional activator that is regulated by Cyclic
Adenosine MonoPhosphate (cAMP) binding. We find that CARDS captures allosteric
                                                                                      
                                                                                      
communication between the two cAMP-Binding Domains (CBDs). Importantly, CARDS
reveals that this coupling is dominated by disorder-mediated correlations, consistent with
NMR experiments that establish allosteric coupling between the CBDs occurs without a
concerted structural change. CARDS also recapitulates an enhanced role for disorder in the
communication between the DNA-Binding Domains (DBDs) and CBDs in the S62F variant of CAP.
Finally, we demonstrate that using CARDS to find communication hotspots identifies regions
of CAP that are in allosteric communication without foreknowledge of their identities.
Therefore, we expect CARDS to be of great utility for both understanding and predicting
allostery.
</p><!-- l. 17 --><p class='noindent'>
</p>
<h3 class='sectionHead'><span class='titlemark'>2.2    </span> <a id='x7-170002.2'></a>Introduction</h3>
<!-- l. 18 --><p class='noindent'>Despite its fundamental importance, understanding of the physical mechanisms of allosteric
communication remains incomplete. For example, significant effort has gone into studying how
G-Protein Coupled Receptors (GPCRs) allosterically transmit extracellular signals to intracellular
binding partners<span class='cite'> [<a href='mainli5.html#XRosenbaum:2009id'>91</a>]</span>. However, understanding of this process is still insufficient for the routine
design of drugs that allosterically modulate GPCRs<span class='cite'> [<a href='mainli5.html#XShoichet:2012fm'>92</a>]</span>.
</p><!-- l. 20 --><p class='noindent'>Models of allostery have typically focused on concerted structural changes<span class='cite'> [<a href='mainli5.html#XGuo:2016hv'>93</a>]</span>. For example, the
classic induced fit model postulates that ligand binding to one subunit of a protein causes a
conformational change in other subunits<span class='cite'> [<a href='mainli5.html#XKoshland:1959ft'>94</a>, <a href='mainli5.html#XKoshlandJr:1966ffa'>95</a>]</span>. The conformational selection model also focuses
on structural changes, positing that ligand binding to one subunit stabilizes an alternative
(but pre-existing) structure of other subunits<span class='cite'> [<a href='mainli5.html#XMonod:1965io'>96</a>]</span>. Extensive work establishes there is
often a role for conformational selection<span class='cite'> [<a href='mainli5.html#XChangeux:2011fv'>97</a>]</span>, though there is clearly a continuum between
                                                                                      
                                                                                      
extreme versions of induced-fit and conformational selection<span class='cite'> [<a href='mainli5.html#XCsermely:2010cv'>98</a>, <a href='mainli5.html#XSilva:2011ct'>99</a>, <a href='mainli5.html#XHammes:2009im'>100</a>]</span>. This conclusion
implies allostery can be inferred from proteins’ equilibrium fluctuations even in the absence
of an allosteric perturbation. Inspired by this implication, numerous methods have been
developed to infer allosteric communication from structural fluctuations observed in molecular
simulations<span class='cite'> [<a href='mainli5.html#XIchiye:1991db'>85</a>, <a href='mainli5.html#Xand:1999bo'>101</a>, <a href='mainli5.html#XPratulKAgarwal:2002dp'>102</a>, <a href='mainli5.html#XAgarwal:2002ema'>103</a>, <a href='mainli5.html#XRod:2003vm'>104</a>, <a href='mainli5.html#XLange:2006gv'>105</a>, <a href='mainli5.html#XWhitley:2009uq'>106</a>, <a href='mainli5.html#XWeinkam:2012jf'>107</a>, <a href='mainli5.html#XLenaerts:2008kn'>108</a>, <a href='mainli5.html#XMcClendon:2009gd'>86</a>, <a href='mainli5.html#XDubay:2011ey'>109</a>, <a href='mainli5.html#XFeher:2014cx'>83</a>, <a href='mainli5.html#XFenwick:2014fc'>110</a>, <a href='mainli5.html#XCui:2008cc'>111</a>, <a href='mainli5.html#XElber:2011dj'>112</a>, <a href='mainli5.html#XVanWart:2014dc'>113</a>, <a href='mainli5.html#XMalmstrom:2015kp'>114</a>, <a href='mainli5.html#XChu:2005gh'>115</a>, <a href='mainli5.html#XRavindranathan:2005ff'>116</a>, <a href='mainli5.html#XWeinkam:2013fj'>117</a>, <a href='mainli5.html#XLiu:2012em'>118</a>, <a href='mainli5.html#XZhang2012'>119</a>]</span>.
</p><!-- l. 22 --><p class='noindent'>While concerted structural changes are clearly important for allostery, there is mounting evidence that
conformational disorder has an important role to play, and can even lead to allosteric communication
in the absence of concerted structural ,transitions<span class='cite'> [<a href='mainli5.html#XGuo:2016hv'>93</a>, <a href='mainli5.html#XKern:2003fv'>120</a>, <a href='mainli5.html#XTzeng:2011by'>64</a>, <a href='mainli5.html#XHilser:2007je'>121</a>, <a href='mainli5.html#XBurger:2016gy'>122</a>, <a href='mainli5.html#XHaliloglu:2015dj'>123</a>, <a href='mainli5.html#XHenzlerWildman:2007jb'>124</a>, <a href='mainli5.html#XHilser:2012em'>125</a>]</span>. The
importance of allostery without conformational change first appeared in a model where ligand binding
perturbs the entropy of a distant site rather than its preferred structure<span class='cite'> [<a href='mainli5.html#XCooper:1984tb'>87</a>]</span>. Since then, NMR and ITC
experiments on Catabolite Activator Protein (CAP) have established allosteric communication
without conformational change exists in nature<span class='cite'> [<a href='mainli5.html#XWand:2013dz'>19</a>, <a href='mainli5.html#XPopovych:2006js'>88</a>, <a href='mainli5.html#XJakob:2014cp'>126</a>]</span>. CAP is a homodimeric
transcription factor whose DNA-binding affinity increases upon binding of cAMP to the
cAMP-Binding Domains (CBDs)<span class='cite'> [<a href='mainli5.html#XHarman:2001td'>127</a>, <a href='mainli5.html#XHeyduk:1989wb'>128</a>]</span>. In wild-type CAP, cAMP binding allosterically
induces the DNA-Binding Domains (DBDs) to swivel around the central hinge region into a
DNA-binding conformation (Fig. <a href='#x7-17001r1'>2.1<!-- tex4ht:ref: fig:cap-intro-figure  --></a>)<span class='cite'> [<a href='mainli5.html#XSchultz:1991uh'>129</a>, <a href='mainli5.html#XSeok:2014cs'>130</a>]</span>. There is also negative coupling between the
two CBDs<span class='cite'> [<a href='mainli5.html#XHarman:2001td'>127</a>, <a href='mainli5.html#XHeyduk:1989wb'>128</a>, <a href='mainli5.html#XPopovych:2009iu'>131</a>]</span>. A combination of NMR and ITC measurements reveal that
binding of cAMP to one CBD reduces the cAMP-binding affinity of the second CBD without
changing its structure<span class='cite'> [<a href='mainli5.html#XPopovych:2006js'>88</a>, <a href='mainli5.html#XPopovych:2009iu'>131</a>]</span>. Additional experiments reveal that binding of cAMP
activates the S62F variant of CAP without causing a conformational change in the hinge or
DBDs<span class='cite'> [<a href='mainli5.html#XTzeng:2009im'>63</a>, <a href='mainli5.html#XTzeng:2012cu'>132</a>]</span>.
</p>
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x7-17001r1'></a>
                                                                                      
                                                                                      
<!-- l. 27 --><p class='noindent'><img width='325' height='131' src='Chapter3/cap-intro-fig.png' alt='PIC' />
<a id='x7-17002'></a>
<br /> </p><div class='caption'><span class='id'>Figure 2.1:  </span><span class='content'>Structures  of  CAP  in  apo  and  holo  forms.  <span class='ptmb7t-x-x-120'>A.  </span>Structure  of  apo-CAP,  with
cAMP-binding  domains  (CBDs)  and  DNA-binding  domains  (DBDs)  indicated  by  brackets.
Residues  that  make  up  the  hinge  region  are  colored  in  red.  <span class='ptmb7t-x-x-120'>B.  </span>Structure  of  holo-CAP  with
cAMP ligands (spheres) bound to the CBD regions. The same hinge residues are colored in red.</span></div><!-- tex4ht:label?: x7-17001r2.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<!-- l. 33 --><p class='noindent'>While the importance of disorder is gaining widespread acceptance, the field lacks systematic methods
for identifying allosteric communication in the absence of conformational change. For example, NMR
has yet to uncover how these signals are transmitted. COREX/BEST<span class='cite'> [<a href='mainli5.html#XHilser:2007je'>121</a>, <a href='mainli5.html#XHilser:1998cj'>133</a>, <a href='mainli5.html#XHilser:1996ft'>134</a>]</span>, other
coarse-grained models<span class='cite'> [<a href='mainli5.html#XWeinkam:2012jf'>107</a>]</span>, and normal modes<span class='cite'> [<a href='mainli5.html#XMarcos:2011dr'>135</a>]</span> provide valuable insights but miss essential
subtleties, such as important side-chain motions. Using molecular dynamics simulations to measure
the mutual information between the orientations of different dihedral angles captures the reduction in
uncertainty (measured with an entropy metric) about the structure of one dihedral given knowledge of
another<span class='cite'> [<a href='mainli5.html#XLenaerts:2008kn'>108</a>, <a href='mainli5.html#XMcClendon:2009gd'>86</a>, <a href='mainli5.html#XDubay:2011ey'>109</a>]</span>, but does not capture phenomena like changes in the rotameric state of one
dihedral increasing the conformational heterogeneity of a distant site. Approaches for inferring
allosteric coupling from sequence covariation<span class='cite'> [<a href='mainli5.html#XLockless:1999uf'>82</a>, <a href='mainli5.html#XSuel:2003ca'>136</a>]</span> are agnostic to the mechanism underlying
this communication and cannot explain how it occurs. A method for identifying timing correlations
has promise for capturing disorder-mediated coupling<span class='cite'> [<a href='mainli5.html#XLin:2016gi'>89</a>]</span>. For example, application of
this approach to side-chain degrees of freedom in CAP successfully identified hotspots
for allosteric communication. It also demonstrated that disorder-mediated correlations
give rise to long-range communication, while purely structural correlations are limited
to short-ranged communication. However, timing correlations do not integrate structural
and dynamical correlations into a holistic measure of communication that can capture
the continuum of possibilities between purely structural and purely disorder-mediated
coupling.
</p><!-- l. 35 --><p class='noindent'>Here, we introduce the CARDS (Correlation of all Rotameric and Dynamical States) methodology for
quantifying the roles of both concerted structural changes and conformational disorder. CARDS is
based on our observation that a single degree of freedom (e.g. a dihedral angle) can transition between
“ordered” regimes, wherein it undergoes small fluctuations within a single structural state, and
“disordered” regimes wherein it undergoes bursts of transitions between different structural states
(Fig. <a href='#x7-18001r2'>2.2<!-- tex4ht:ref: fig:cards-workflow-fig  --></a>). Similar “dynamic heterogeneity”<span class='cite'> [<a href='mainli5.html#XEdiger:2000ed'>137</a>, <a href='mainli5.html#XGlotzer:2000fy'>138</a>, <a href='mainli5.html#XRichert:2002fa'>139</a>]</span> has been observed in the physics of
glasses, where it has been shown that a single degree of freedom’s local environment can either
                                                                                      
                                                                                      
facilitate dynamics by flattening out the effective free energy surface that degree of freedom
experiences or freeze out dynamics<span class='cite'> [<a href='mainli5.html#XMerolle:2005kx'>140</a>, <a href='mainli5.html#XHedges:2007fw'>141</a>, <a href='mainli5.html#XKeys:2011ut'>142</a>]</span>. CARDS identifies ordered and disordered
regimes based on two kinetic signatures: the average time a degree of freedom persists within a
structural state and the typical timescale for transitions between structural states. For many dihedrals,
we observe that the typical time that elapses between structural transitions, which is dominated by
segments of a trajectory in disordered regimes, is orders of magnitude smaller than the typical
persistence time in a state. Based on these kinetic signatures, CARDS assigns segments of
trajectories to dynamical states (i.e. ordered and disordered regimes). CARDS then quantifies
correlations between the structural and dynamical states of different dihedrals. Specifically, we
employ the mutual information to assess how much better one can predict the structural
and dynamical states of one dihedral given knowledge of the structural and dynamical
states of another dihedral. To demonstrate the utility of CARDS, we assess whether it can
identify allosteric communication in the absence of concerted structural changes observed in
CAP.
</p>
<h3 class='sectionHead'><span class='titlemark'>2.3    </span> <a id='x7-180002.3'></a>Theory and methods</h3>
<!-- l. 39 --><p class='noindent'>CARDS captures all forms of correlated fluctuations, including concerted structural changes,
correlations between the conformational disorder of different degrees of freedom, and correlations
between the structure of one degree of freedom and the conformational disorder of another. As in
other recent work, we focus on dihedral angles, as they are natural degrees of freedom for describing
protein structure and dynamics and are easily decomposed into a small number of rotameric
states<span class='cite'> [<a href='mainli5.html#XMcClendon:2009gd'>86</a>, <a href='mainli5.html#XDubay:2011ey'>109</a>]</span>.
</p>
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x7-18001r2'></a>
                                                                                      
                                                                                      
<!-- l. 43 --><p class='noindent'><img width='361' height='526' src='Chapter3/cards-workflow-fig.png' alt='PIC' />
<a id='x7-18002'></a>
<br /> </p><div class='caption'><span class='id'>Figure 2.2: </span><span class='content'>Workflow for identifying ordered and disordered regimes. <span class='ptmb7t-x-x-120'>A. </span>An example of the
time-evolution of a <span class='cmmi-12'>χ</span><sub><span class='cmr-8'>1</span></sub> angle. The core of each rotameric state is shaded green and the buffer
zones between them are white. <span class='ptmb7t-x-x-120'>B. </span>Assignment of each snapshot in the trajectory to rotameric
states. <span class='ptmb7t-x-x-120'>C. </span>Transition indicator function identifying where the dihedral changes states. Examples
of ordered (<span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>ord</span></sub>) and disordered (<span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>dis</span></sub>) times are labeled. <span class='ptmb7t-x-x-120'>D. </span>Assignment of each snapshot in the
trajectory to dynamical states (e.g. ordered and disordered regimes).</span></div><!-- tex4ht:label?: x7-18001r2.3  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<h4 class='subsectionHead'><span class='titlemark'>2.3.1    </span> <a id='x7-190002.3.1'></a>Molecular Dynamics simulations</h4>
<!-- l. 50 --><p class='noindent'>We ran three 500 ns simulations with our previously published protocol<span class='cite'> [<a href='mainli5.html#XHart:2016kb'>9</a>]</span> (see SI for details).
Briefly, we placed PDB ID 4N9H<span class='cite'> [<a href='mainli5.html#XSeok:2014cs'>130</a>]</span> in a dodecahedron box and solvated it with TIP3P explicit
solvent<span class='cite'> [<a href='mainli5.html#XJorgensen:1983fl'>143</a>]</span> extending 1 nm beyond the protein in any dimension. We used PyMOL<span class='cite'> [<a href='mainli5.html#XDeLano:2010wf'>144</a>]</span> to mutate
Ser62 to Phe to create a starting structure for the S62F variant. For each system, we ran
simulations with the Gromacs software package<span class='cite'> [<a href='mainli5.html#XVanDerSpoel:2005hz'>145</a>]</span> using the Amber03 force field<span class='cite'> [<a href='mainli5.html#XDuan:2003gt'>146</a>]</span>.
This combination of software and parameters was selected because it has proven reliable
in our past studies of both protein folding<span class='cite'> [<a href='mainli5.html#XBowman:2011bc'>55</a>]</span> and structural fluctuations within folded
proteins<span class='cite'> [<a href='mainli5.html#XHart:2016kb'>9</a>, <a href='mainli5.html#XBowman:2015dh'>147</a>, <a href='mainli5.html#XBowman:2015da'>56</a>]</span>. As described in the Results section, our microsecond timescale simulations
are sufficient to recapitulate much of what is known from experiments about allostery
in CAP. The correlation coefficient between the couplings obtained from any individual
simulation and the combination of three simulations is 0.64<span class='cmsy-10x-x-120'>±</span>0.02, suggesting that hundreds of
nanoseconds of simulation may be sufficient to discover gross patterns of communication but are
insufficient to obtain converged results. The correlation coefficient between the coupling
obtained from any pair of simulations and the combination of three simulations is 0.79<span class='cmsy-10x-x-120'>±</span>0.02,
suggesting that 1-1.5 microseconds of simulation give more reproducible results. Taken together
with our past work, we conclude that a few microseconds of simulation are adequate to
demonstrate the utility of our new method and, likely, to gain valuable insights into many
systems.
</p><!-- l. 53 --><p class='noindent'>
</p>
                                                                                      
                                                                                      
<h4 class='subsectionHead'><span class='titlemark'>2.3.2    </span> <a id='x7-200002.3.2'></a>Assignment of dihedrals to rotameric states</h4>
<!-- l. 54 --><p class='noindent'>Dihedral angles were calculated with MDTraj <span class='cite'> [<a href='mainli5.html#XMcGibbon:2015fv'>148</a>]</span> and assigned to discrete rotameric states
(e.g. gauche+, gauche-, and trans for most <span class='cmmi-12'>χ </span>angles and cis or trans states for backbone
dihedrals). Transition-Based Assignment (TBA) is used to distinguish lasting transitions from
transient fluctuations<span class='cite'> [<a href='mainli5.html#XHummer:2004kb'>149</a>, <a href='mainli5.html#XBolhuis:2002ew'>150</a>, <a href='mainli5.html#XBuchete:2008hb'>151</a>, <a href='mainli5.html#XSchutte:2011gs'>152</a>]</span>. TBA prevents over-counting of transitions (e.g.
due to fluctuations at a barrier peak where a simulation repeatedly crosses a hard cutoff
between rotameric states) by defining a core region within each rotameric state and buffer
zones between them. A dihedral is only considered to have changed rotameric states if
it transitions from the core of one state to the core of another state, passing completely
through the buffer zone between cores (Fig. <a href='#x7-18001r2'>2.2<!-- tex4ht:ref: fig:cards-workflow-fig  --></a>A and <a href='#x7-18001r2'>2.2<!-- tex4ht:ref: fig:cards-workflow-fig  --></a>B). A dihedral that starts in one
core, enters a buffer zone, and then returns to its initial core is said to have remained in
the initial rotameric state. We define the core of a rotameric state as a region of width W
centered between the boundaries between rotamers. We present results using a core width of
90<span class='ptmr8c-x-x-120'>°</span>, but our results are robust to variations in the core width ranging from 60<span class='ptmr8c-x-x-120'>° </span>to 90<span class='ptmr8c-x-x-120'>° </span>(Fig.
<a href='mainap1.html#x14-138001r1'>A.1<!-- tex4ht:ref: fig:cards-pearsonr-fig  --></a>A)
</p><!-- l. 56 --><p class='noindent'>
</p>
<h4 class='subsectionHead'><span class='titlemark'>2.3.3    </span> <a id='x7-210002.3.3'></a>Assignment of snapshots to dynamical states</h4>
<!-- l. 57 --><p class='noindent'>CARDS assigns snapshots to ordered and disordered regimes based on two variables that describe the
dynamics of the trajectory: the mean ordered time (<span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>ord</span></sub>) and mean disordered time (<span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>dis</span></sub>). An
ordered time (<span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>ord</span></sub>) is the time from any time-point to the next time where a transition occurs
and a disordered time (<span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>dis</span></sub>) is the time between two consecutive transitions (Fig. <a href='#x7-18001r2'>2.2<!-- tex4ht:ref: fig:cards-workflow-fig  --></a>C).
These times are called persistence and exchange times in the condensed matter physics
literature<span class='cite'> [<a href='mainli5.html#XHedges:2007fw'>141</a>, <a href='mainli5.html#XKeys:2011ut'>142</a>, <a href='mainli5.html#XJung:2005jk'>153</a>]</span>. For many dihedrals, we observe that <span class='cmsy-10x-x-120'>⟨</span><span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>dis</span></sub><span class='cmsy-10x-x-120'>⟩≪⟨</span><span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>ord</span></sub><span class='cmsy-10x-x-120'>⟩ </span>because (<span class='tcrm-1200'>〈</span><span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>dis</span></sub><span class='tcrm-1200'>〉</span>) is
                                                                                      
                                                                                      
dominated by the short times between transitions in disordered regimes, whereas <span class='tcrm-1200'>〈</span><span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>ord</span></sub><span class='tcrm-1200'>〉 </span>is heavily
influenced by the lengthy times without any transitions in ordered regimes. To calculate these
times, CARDS first identifies the time points where a dihedral transitions between two
different rotameric states (Fig. <a href='#x7-18001r2'>2.2<!-- tex4ht:ref: fig:cards-workflow-fig  --></a>C), referred to as the transition indicator function. The
method then extract the complete set of <span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>dis</span></sub> and <span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>ord</span></sub> values in the trajectory. Next, CARDS
classifies each segment of a trajectory between two consecutive transitions as being in an
ordered or disordered regime based on whether the length of the segment <span class='cmmi-12'>t </span>between the
transitions is more consistent with the distribution of ordered or disordered times (Fig.
<a href='#x7-18001r2'>2.2<!-- tex4ht:ref: fig:cards-workflow-fig  --></a>D). We find that transitions within ordered and disordered regimes are roughly Poisson
processes with different characteristic times (<span class='cmsy-10x-x-120'>⟨</span><span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>ord</span></sub><span class='cmsy-10x-x-120'>⟩ </span>and<span class='tcrm-1200'>〈</span><span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>dis</span></sub><span class='tcrm-1200'>〉</span>, respectively), see Fig. <a href='mainap1.html#x14-138003r2'>A.2<!-- tex4ht:ref: fig:cards-timescale-dists-fig  --></a>.
Therefore, CARDS determines if a segment of a trajectory of length <span class='cmmi-12'>t </span>between two consecutive
transitions is more consistent with an ordered or disordered regime using the likelihood ratio
(<span class='cmmi-12'>L</span>):
</p>
<table class='equation'><tr><td><a id='x7-21001r1'></a>
<center class='math-display'>
<img src='main6x.png' alt='                          -t--
       P   (t)    (--1-)e− ⟨τdis⟩
L(t) = -dis---=  -⟨τdis⟩------t--
       Pord(t)    (⟨τ1⟩)e− ⟨τord⟩
                   ord
' class='math-display' /></center></td><td class='equation-label'>(2.1)</td></tr></table>
<!-- l. 60 --><p class='nopar'>
where <span class='cmmi-12'>P</span><sub><span class='cmmi-8'>dis</span></sub> is the probability the segment is disordered and <span class='cmmi-12'>P</span><sub><span class='cmmi-8'>ord</span></sub> is the probability it is ordered. Taking
inspiration from the interpretation of Bayes factors<span class='cite'> [<a href='mainli5.html#XKass:2012bb'>154</a>]</span>, CARDS classifies a segment of a
                                                                                      
                                                                                      
trajectory as being disordered if <span class='cmmi-12'>L &gt; </span><span class='cmr-12'>3</span><span class='cmmi-12'>.</span><span class='cmr-12'>0</span>, otherwise the trajectory segment is classified as
being in an ordered regime. Our results are robust to varying this cutoff from 1.5 to 5 (Fig.
<a href='mainap1.html#x14-138001r1'>A.1<!-- tex4ht:ref: fig:cards-pearsonr-fig  --></a>B).
</p><!-- l. 63 --><p class='noindent'>
</p>
<h4 class='subsectionHead'><span class='titlemark'>2.3.4    </span> <a id='x7-220002.3.4'></a>Calculation of structural, disorder-mediated, and holistic correlations</h4>
<!-- l. 64 --><p class='noindent'>The primary objective of CARDS is to calculate the total correlation between different dihedrals,
including both their rotameric state and dynamical state (e.g. whether the dihedral is in an ordered or
disordered regime at a given time). Towards this end, we define the holistic correlation (<span class='cmmi-12'>I</span><sub><span class='cmmi-8'>H</span></sub>) between
dihedrals <span class='cmmi-12'>X </span>and <span class='cmmi-12'>Y </span>as
</p>
<table class='equation'><tr><td><a id='x7-22001r2'></a>
<center class='math-display'>
<img src='main7x.png' alt='            ---------- ----------  ----------  ----------
IH(X, Y ) = Iss(X, Y ) + Idd(X, Y ) + Ids(X, Y ) + Isd(X, Y)
' class='math-display' /></center></td><td class='equation-label'>(2.2)</td></tr></table>
<!-- l. 68 --><p class='nopar'>
where <span class='overline'><span class='cmmi-12'>I</span><sub><span class='cmmi-8'>ss</span></sub><span class='cmr-12'>(</span><span class='cmmi-12'>X,Y </span><span class='cmr-12'>)</span></span> is the normalized mutual information between the structure (i.e. rotameric state) of
                                                                                      
                                                                                      
dihedral <span class='cmmi-12'>X </span>and the structure of dihedral <span class='cmmi-12'>Y </span>, <span class='overline'><span class='cmmi-12'>I</span><sub><span class='cmmi-8'>sd</span></sub><span class='cmr-12'>(</span><span class='cmmi-12'>X,Y </span><span class='cmr-12'>)</span></span> is the normalized mutual information
between the structure of dihedral <span class='cmmi-12'>X </span>and the dynamical state of dihedral <span class='cmmi-12'>Y </span>, <span class='overline'><span class='cmmi-12'>I</span><sub><span class='cmmi-8'>ds</span></sub><span class='cmr-12'>(</span><span class='cmmi-12'>X,Y </span><span class='cmr-12'>)</span></span> is the
normalized mutual information between the dynamical state of dihedral X and the structure of
dihedral Y, and <span class='overline'><span class='cmmi-12'>I</span><sub><span class='cmmi-8'>dd</span></sub><span class='cmr-12'>(</span><span class='cmmi-12'>X,Y </span><span class='cmr-12'>)</span></span> is the normalized mutual information between the dynamical
state of dihedral <span class='cmmi-12'>X </span>and the dynamical state of dihedral <span class='cmmi-12'>Y </span>. The mutual information (I)
is:
</p>
<table class='equation'><tr><td><a id='x7-22002r3'></a>
<center class='math-display'>
<img src='main8x.png' alt='             ∑  ∑
I(X, Y ) = −        --P(x,-y)-
                    P (x)P (y)
             x∈X y∈Y
' class='math-display' /></center></td><td class='equation-label'>(2.3)</td></tr></table>
<!-- l. 72 --><p class='nopar'>
where <span class='cmmi-12'>x </span><span class='cmsy-10x-x-120'>∈ </span><span class='cmmi-12'>X </span>refers to the set of possible states that dihedral <span class='cmmi-12'>X </span>can adopt, <span class='cmmi-12'>p</span><span class='cmr-12'>(</span><span class='cmmi-12'>x</span><span class='cmr-12'>) </span>is the probability that
dihedral <span class='cmmi-12'>X </span>adopts state <span class='cmmi-12'>x</span>, and <span class='cmmi-12'>p</span><span class='cmr-12'>(</span><span class='cmmi-12'>x,y</span><span class='cmr-12'>) </span>is the joint probability that dihedral <span class='cmmi-12'>X </span>adopts state <span class='cmmi-12'>x</span>
and dihedral <span class='cmmi-12'>Y </span>adopts state <span class='cmmi-12'>y</span>. We define the normalized mutual information (<span class='overline'><span class='cmmi-12'>I</span><span class='cmr-12'>(</span><span class='cmmi-12'>X,Y </span><span class='cmr-12'>)</span></span>)
as
</p>
<table class='equation'><tr><td><a id='x7-22003r4'></a>
                                                                                      
                                                                                      
<center class='math-display'>
<img src='main9x.png' alt='--------   I (X,Y )
I(X, Y ) = ---------
           C (X, Y )
' class='math-display' /></center></td><td class='equation-label'>(2.4)</td></tr></table>
<!-- l. 76 --><p class='nopar'>
where <span class='cmmi-12'>C</span><span class='cmr-12'>(</span><span class='cmmi-12'>X,Y </span><span class='cmr-12'>) </span>is the maximum possible mutual information between two dihedrals, called the
channel capacity<span class='cite'> [<a href='mainli5.html#Xwitten2002data'>155</a>]</span>. Using this normalized mutual information allows for a direct comparison
between the different components of the holistic correlation by correcting for the fact that the largest
possible mutual information between different types of dihedrals will vary based on how many
different states there are. For example, a side-chain dihedral has three possible rotameric states
but only two possible dynamical states, so structural correlations (<span class='cmmi-12'>I</span><sub><span class='cmmi-8'>ss</span></sub>) can be as large as
<span class='cmmi-12'>log</span><span class='cmr-12'>(3) </span>while the correlations between dynamical states (<span class='cmmi-12'>I</span><sub><span class='cmmi-8'>dd</span></sub>) can only reach as high as
<span class='cmmi-12'>log</span><span class='cmr-12'>(2)</span>.
</p><!-- l. 79 --><p class='noindent'>In addition to the above, we define the disorder-mediated correlation (<span class='cmmi-12'>I</span><sub><span class='cmmi-8'>DM</span></sub>) as all forms of correlation
between two dihedrals that rely, at least in part, on disorder (<span class='overline'><span class='cmmi-12'>I</span><sub><span class='cmmi-8'>sd</span></sub> </span><span class='cmr-12'>+</span><span class='overline'> <span class='cmmi-12'>I</span><sub><span class='cmmi-8'>ds</span></sub> </span><span class='cmr-12'>+</span><span class='overline'> <span class='cmmi-12'>I</span><sub><span class='cmmi-8'>dd</span></sub></span>). This construct is
useful for assessing the importance of disorder relative to existing methods based purely on
concerted structural changes (<span class='cmmi-12'>I</span><sub><span class='cmmi-8'>ss</span></sub>). We use bootstrapping to measure the uncertainty in our
estimates of all the components of the holistic correlation to ensure any comparisons we
make are statistically sound. Specifically, we draw 20 random samples of our trajectories,
with replacement, and calculate the structural and disorder-mediated correlations between
all pairs of residues. We conclude that disorder-mediated communication dominates if
the average disorder-mediated communication minus the standard deviation across all
our bootstrap samples is greater than the mean structural correlation plus the standard
deviation.
                                                                                      
                                                                                      
</p><!-- l. 81 --><p class='noindent'>
</p>
<h4 class='subsectionHead'><span class='titlemark'>2.3.5    </span> <a id='x7-230002.3.5'></a>Calculation of net communication to a target site</h4>
<!-- l. 82 --><p class='noindent'>We are often interested in calculating how much influence a particular residue has over another site,
such as an active site or ligand-binding site. To calculate the communication between a reference
residue and some target site, we take the average mutual information between two sets of dihedrals: 1)
all dihedrals in the reference residue and its nearest neighbors and 2) all dihedrals in the target site. We
define the nearest neighbors of a reference residue as all residues with atoms that fall within 3 Å
of any atom in the reference residue. Varying this cutoff does not alter our results (Fig.
<a href='mainap1.html#x14-138001r1'>A.1<!-- tex4ht:ref: fig:cards-pearsonr-fig  --></a>C). Including both a reference residue and its nearest neighbors accounts for the fact that
mutating the reference residue will directly change the environment of all neighboring
residues.
</p><!-- l. 84 --><p class='noindent'>
</p>
<h4 class='subsectionHead'><span class='titlemark'>2.3.6    </span> <a id='x7-240002.3.6'></a>Calculation of global communication</h4>
<!-- l. 85 --><p class='noindent'>In addition to identifying residues that have strong correlations to a specific target site, it would also be
valuable to identify residues that generally appear to play an important role in allosteric
networks. Towards this end, we define the global communication strength of a residue as
the sum of its holistic correlations to all other residues. For these calculations, we also
include neighboring residues, as in our calculation of the net communication to a target
site.
                                                                                      
                                                                                      
</p><!-- l. 87 --><p class='noindent'>
</p>
<h3 class='sectionHead'><span class='titlemark'>2.4    </span> <a id='x7-250002.4'></a>Results and discussion</h3>
<!-- l. 88 --><p class='noindent'>
</p>
<h4 class='subsectionHead'><span class='titlemark'>2.4.1    </span> <a id='x7-260002.4.1'></a>Many dihedrals have the potential for disorder-mediated communication</h4>
<!-- l. 89 --><p class='noindent'>For a dihedral to have ordered and disordered regimes, <span class='tcrm-1200'>〈</span><span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>ord</span></sub><span class='tcrm-1200'>〉 </span>must be significantly larger than
<span class='tcrm-1200'>〈</span><span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>dis</span></sub><span class='tcrm-1200'>〉</span>. We reasoned that determining if <span class='cmsy-10x-x-120'>⟨</span><span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>ord</span></sub><span class='cmsy-10x-x-120'>⟩≥ </span><span class='cmr-12'>3 </span><span class='cmsy-10x-x-120'>×⟨</span><span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>dis</span></sub><span class='cmsy-10x-x-120'>⟩ </span>is a reasonable heuristic for
identifying dihedrals with separable ordered and disordered regimes based on the likelihood ratio
defined in Eq. <a href='#x7-21001r1'>2.1<!-- tex4ht:ref: likelihood-ratio-eq  --></a>. Dihedrals that do not meet this criterion are classified as entirely being in
ordered regimes and, therefore, are only capable of having structural correlations with other
dihedrals.
</p><!-- l. 91 --><p class='noindent'>Based on the criterion defined above, we find that 556 of the 1584 dihedrals in CAP have separable
ordered and disordered regimes and, therefore, are capable of disorder-mediated communication with
other dihedrals (Fig. <a href='mainap1.html#x14-138005r3'>A.3<!-- tex4ht:ref: fig:cards-avgtimes-2dhist-fig  --></a>). Mapping these dihedrals to the apo structure of CAP highlights a number of
interesting patterns (Fig. <a href='#x7-26001r3'>2.3<!-- tex4ht:ref: fig:cap-disordered-resis-fig  --></a>). First of all, CARDS reveals that many side-chain dihedrals buried in
CAP’s core are capable of disorder-mediated communication. This finding helps to rectify the
apparent contradiction between the common physical intuition that proteins’ cores should be rigid due
to their tight packing and the observation that there is substantial conformational heterogeneity within
proteins’ cores<span class='cite'> [<a href='mainli5.html#XFraser:2011jf'>156</a>, <a href='mainli5.html#Xbowman2014extensive'>57</a>]</span>. That is, dihedrals within a protein’s core are commonly locked
in a single rotameric state for extended periods of time but rare fluctuations create room
for conformational changes. Backbone dihedrals that are capable of disorder-mediated
communication tend to reside on the protein’s surface. Notably, a number of these backbone
                                                                                      
                                                                                      
dihedrals are in <span class='cmmi-12'>β</span>-sheets that contact cAMP. However, there are also backbone dihedrals
within the core that are capable of disorder-mediated communication. For example, we find
backbone dihedrals within the central hinge region that are capable of disorder-mediated
communication. This observation is noteworthy because the hinge region undergoes a large
conformational change upon activation of CAP (Fig. <a href='#x7-17001r1'>2.1<!-- tex4ht:ref: fig:cap-intro-figure  --></a>)<span class='cite'> [<a href='mainli5.html#XPopovych:2009iu'>131</a>, <a href='mainli5.html#XTzeng:2009im'>63</a>, <a href='mainli5.html#XTzeng:2012cu'>132</a>]</span>. We find that similar
patterns emerge when we vary the cutoff for determining whether a dihedral has separable
ordered and disordered regimes (Fig. <a href='mainap1.html#x14-138007r4'>A.4<!-- tex4ht:ref: fig:cards-disord-resis-cutoff  --></a>). In the future it will be interesting to examine
whether separate proteins, or homologous members of a family, have similar proportions and
patterns of dihedrals that are separable into ordered and disordered regimes. However,
given the opportunity to only analyze a limited number of sufficiently sampled datasets so
far<span class='cite'> [<span class='ptmb7t-x-x-120'>?</span>, <a href='mainli5.html#XSun:2018kx'>48</a>, <a href='mainli5.html#Xporter_conformational_2020'>10</a>]</span>.
</p>
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x7-26001r3'></a>
                                                                                      
                                                                                      
<!-- l. 95 --><p class='noindent'><img width='325' height='177' src='Chapter3/cap-disordered-residues-fig.png' alt='PIC' />
<a id='x7-26002'></a>
<br /> </p><div class='caption'><span class='id'>Figure 2.3:  </span><span class='content'>Residues  whose  dihedrals  are  capable  of  disorder-mediated  communication.
<span class='ptmb7t-x-x-120'>A.  </span>Residues  with  at  least  one  backbone  dihedral  that  is  capable  of  disorder-mediated
communication (orange). <span class='ptmb7t-x-x-120'>B. </span>Residues with at least one side-chain dihedral that is capable of
disorder-mediated communication (green).</span></div><!-- tex4ht:label?: x7-26001r2.4  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<h4 class='subsectionHead'><span class='titlemark'>2.4.2    </span> <a id='x7-270002.4.2'></a>Disorder-mediated correlations dominate communication between the CBDs</h4>
<!-- l. 102 --><p class='noindent'>Given experimental evidence for allosteric communication between the CBDs without a concerted
structural change<span class='cite'> [<a href='mainli5.html#XTzeng:2009im'>63</a>]</span>, we expect the disorder-mediated component of the holistic correlation
between these sites to be larger than the purely structural component. To test this prediction, we
simulated apo CAP for 1.5 <span class='cmmi-12'>μ</span>s and calculated the net communication of every residue to one of the
cAMP-binding sites. Specifically, we defined the target site as all residues with heavy atoms within
6Åof one of the two cAMP molecules in the holo crystal structure (PDB ID 1CGP)<span class='cite'> [<a href='mainli5.html#XSchultz:1991uh'>129</a>]</span>. The
residues in this target site are 30, 36, 49, 61-62, 64, 69-86, and 99 from chain A and residues 122-129
from chain B.
</p><!-- l. 104 --><p class='noindent'>As predicted, CARDS successfully identifies that there is communication between the two CBDs. Fig.
<a href='#x7-27001r4'>2.4<!-- tex4ht:ref: fig:cap-targetsite-fig  --></a>A shows the holistic correlations to a single cAMP-binding site. Unsurprisingly, the residues with
the strongest correlations to this set of residues reside within the same CBD. However, there is also
strong communication between the target site and residues lining the other cAMP-binding
pocket. There are also strong correlations on the central hinge region and the interface
between the CBDs and DBDs that may be responsible for allosteric coupling between these
domains.
</p><!-- l. 106 --><p class='noindent'>To determine the relative importance of disorder-mediated communication and purely structural
correlations, we broke the holistic communication into structural and disorder-mediated
components. Furthermore, we used bootstrapping to estimate the uncertainty in each of these
components.
</p>
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x7-27001r4'></a>
                                                                                      
                                                                                      
<!-- l. 110 --><p class='noindent'><img width='325' height='151' src='Chapter3/cap-targetsite-fig.png' alt='PIC' />
<a id='x7-27002'></a>
<br /> </p><div class='caption'><span class='id'>Figure 2.4: </span><span class='content'>Communication to a single CBD. <span class='ptmb7t-x-x-120'>A. </span>Holistic mutual information (<span class='cmmi-12'>I</span><sub><span class='cmmi-8'>H</span></sub>) of each
residue to the residues lining a single cAMP-binding pocket (shown in sticks). <span class='ptmb7t-x-x-120'>B. </span>Residues
whose  communication  to  the  cAMP-binding  pocket  is  dominated  by  disorder-mediated
communication (orange sticks). Residues lining the target cAMP-binding pocket are in blue
sticks.</span></div><!-- tex4ht:label?: x7-27001r2.4  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<!-- l. 116 --><p class='noindent'>Identifying residues where disorder-mediated communication to the target cAMP-binding site is larger
than purely structural correlations automatically identifies a number of residues in the second
cAMP-binding site (Fig. <a href='#x7-27001r4'>2.4<!-- tex4ht:ref: fig:cap-targetsite-fig  --></a>B, Fig. <a href='mainap1.html#x14-138009r5'>A.5<!-- tex4ht:ref: fig:cards-top5percentresis  --></a>). This finding demonstrates that CARDS recapitulates the
experimental finding that communication betwen the two CBDs does not primarily occur through
concerted structural changes<span class='cite'> [<a href='mainli5.html#XTzeng:2009im'>63</a>]</span>. CARDS also identifies disorder-dominated communication within
a single CBD. Furthermore, the Pearson correlation coefficient between structural and
disorder-mediated communication is 0.44. This result indicates there are some similarities between
patterns of allosteric coupling that could be observed by focusing entirely on structural
correlations and those identified by CARDS, but that considering disorder provides additional
information.
</p>
<h4 class='subsectionHead'><span class='titlemark'>2.4.3    </span> <a id='x7-280002.4.3'></a>Disorder-mediated communication is enhanced in the S62F variant</h4>
<!-- l. 119 --><p class='noindent'>The S62F variant of CAP is still activated by cAMP binding<span class='cite'> [<a href='mainli5.html#XTzeng:2009im'>63</a>, <a href='mainli5.html#XTzeng:2012cu'>132</a>, <a href='mainli5.html#XDai:2004ea'>157</a>, <a href='mainli5.html#XAiba:1985ui'>158</a>]</span>. However, NMR
studies have revealed that the conformation of the DBDs does not change upon cAMP binding. Rather,
NMR and ITC experiments suggest an important role for conformational entropy, with the DBDs only
changing conformation in the presence of both cAMP and DNA<span class='cite'> [<a href='mainli5.html#XTzeng:2009im'>63</a>, <a href='mainli5.html#XTzeng:2012cu'>132</a>]</span>. Therefore, we expect an
increase in disorder-mediated communication between the CBDs and DBDs in the S62F variant,
compared to wild-type CAP.
</p><!-- l. 121 --><p class='noindent'>To determine the effect of the S62F variant, we also ran 1.5 <span class='cmmi-12'>μ</span>s of simulation of this variant. Then we
calculated the holistic communication to a single cAMP-binding site, as described for wild-type
CAP.
</p><!-- l. 123 --><p class='noindent'>As expected, we observe significant increases in disorder-mediated communication between the target
                                                                                      
                                                                                      
CBD and the DBDs. There are particularly large increases in disorder-mediated correlations in regions
of known importance for CAP activation, such as the central hinge region and along the interfaces
between the CBDs and DBDs (Fig. <a href='#x7-28001r5'>2.5<!-- tex4ht:ref: fig:cap-s62f-deltaMI-fig  --></a>A). At the same time, there are some decreases
in disorder-mediated communication within the CBDs. There are also changes in purely
structural correlations. These changes often follow the same qualitative trends as the changes in
disorder-mediated communication. However, the magnitudes of any increases in purely structural
correlations are considerably smaller than the increases in disorder-mediated correlations.
Furthermore, reductions in structural correlations are often larger than any decreases in
disorder-mediated communication.
</p>
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x7-28001r5'></a>
                                                                                      
                                                                                      
<!-- l. 127 --><p class='noindent'><img width='361' height='167' src='Chapter3/cap-s62f-deltaMI-fig.png' alt='PIC' />
<a id='x7-28002'></a>
<br /> </p><div class='caption'><span class='id'>Figure 2.5: </span><span class='content'>Change in coupling to a single CBD pocket upon the S62F mutation. <span class='ptmb7t-x-x-120'>A. </span>Change in
disorder-mediated communication of each residue to the single cAMP-binding pocket depicted
in Fig. <a href='#x7-27001r4'>2.4<!-- tex4ht:ref: fig:cap-targetsite-fig  --></a>. <span class='ptmb7t-x-x-120'>B. </span>Change in structural communication of each residue to the same cAMP-binding
pocket.  Red  indicates  increased  communication  in  S62F  compared  to  wild-type,  and  blue
indicates decreased communication.</span></div><!-- tex4ht:label?: x7-28001r2.4  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<h4 class='subsectionHead'><span class='titlemark'>2.4.4    </span> <a id='x7-290002.4.4'></a>Side-chain-side-chain and backbone-side-chain correlations dominate allosteric
communication in CAP</h4>
<!-- l. 134 --><p class='noindent'>To begin understanding the relative importance of different types of degrees of freedom, we plotted
the matrix of correlations between every pair of dihedrals (Fig. <a href='#x7-29001r6'>2.6<!-- tex4ht:ref: fig:cap-bb-sc-matrix-fig  --></a>A). The upper triangle represents
purely structural correlations, while the lower triangle represents purely disorder-mediated
correlations. Side-chain and backbone dihedrals are also grouped together to enable comparisons
between the relative strengths of backbone-backbone, backbone-side-chain and side-chain-side-chain
correlations. Inspection of the matrix of all pairwise correlations immediately reveals that
side-chain-side-chain correlations dominate allosteric communication in CAP. This observation is
consistent with previous reports that side-chain degrees of freedom are more variable than the
backbone<span class='cite'> [<a href='mainli5.html#XFraser:2011jf'>156</a>, <a href='mainli5.html#Xbowman2014extensive'>57</a>, <a href='mainli5.html#XKurplus:1983ts'>159</a>, <a href='mainli5.html#Xbrooks2006advances'>160</a>, <a href='mainli5.html#XDepristo:2004tb'>161</a>, <a href='mainli5.html#XWand:1996hr'>162</a>, <a href='mainli5.html#XLindorffLarsen:2005bi'>163</a>, <a href='mainli5.html#XIgumenova:2006hw'>164</a>, <a href='mainli5.html#XDubay:2009gv'>165</a>]</span>. Backbone-backbone correlations are far
rarer, and we find that disorder-mediated correlations between backbone dihedrals are more
common than structural correlations. We also find a small number of backbone dihedrals
that appear to be hubs of communication that are coupled to the side-chains of a large
fraction of the residues in CAP. These hubs appear as lines in the backbone-side-chain
quadrants of the matrix in Fig. <a href='#x7-29001r6'>2.6<!-- tex4ht:ref: fig:cap-bb-sc-matrix-fig  --></a>A. Mapping the strongest (top 5%) backbone hubs to the
structure reveals that they cluster in two regions: the phosphate-binding cassette (PBC) of each
CBD and the interface between the CBDs and DBDs, including the central hinge (Fig.
<a href='#x7-29001r6'>2.6<!-- tex4ht:ref: fig:cap-bb-sc-matrix-fig  --></a>B). Using an even stricter cutoff (top 2%) only identifies residues along the CBD-DBD
interface and within the central hinge (Fig. <a href='mainap1.html#x14-138011r6'>A.6<!-- tex4ht:ref: fig:cards-top2percenthubs  --></a>), further emphasizing their importance in the
allosteric network. This result suggests that perturbations to these functionally important
regions (e.g. cAMP binding) can influence the behavior of the entire protein, and vice
versa.
                                                                                      
                                                                                      
</p><!-- l. 138 --><p class='noindent'>It is also possible that the assumed Poissonian distribution of dihedral timescales may impact the
degree of measured communication. While these are long-tailed distributions, it is not clear if they are
truly Poissonian, and proving this behavior may prove a non-trivial task. While we already observe
that our measurements are robust to the choice of Likelihood Ratio cutoff (Figure <a href='mainap1.html#x14-138001r1'>A.1<!-- tex4ht:ref: fig:cards-pearsonr-fig  --></a>). It will
be worth exploring whether or not other exponential distributions can be used to more
appropriately model the probability of being ordered or disordered when computing a likelihood
ratio.
</p><!-- l. 140 --><p class='noindent'>More importantly, using a different binning strategy than 3-state or 2-state designations
can improve measurements of allosteric coupling. These rotameric state designations are
based on classic alkane stereochemistry, and assume flat prior. Given the chemical diversity
of amino acids, it is possible that some dihedrals will never explore all rotameric states.
For example a phenylalanine <span class='cmmi-12'>χ</span><sub><span class='cmr-8'>2</span></sub> will not explore all three possible rotameric states due to
the aromatic ring on the sidechain. One possible avenue to improve the measurement of
dihedral communication is to incorporate established rotamer libraries into the rotameric state
decomposition<span class='cite'> [<a href='mainli5.html#Xchaudhury_pyrosetta:_2010'>166</a>, <a href='mainli5.html#Xshapovalov_smoothed_2011'>167</a>]</span>. Using library-based designations,and their standard error measurements
to define buffer region widths, would correct the currently-used flat prior. This would increase the
reliability of measurement of dihedral communication while still allowing us to assess the
role of side-chain dihedrals against backbone dihedrals in communication. However, it is
important to note that the current binning strategy being used is robust when assessed with
metrics such as bootstrapping or Excess Mutual Information. Excess Mutual Information has
been shown as a robust way of detecting noise in measuring allosteric communication,
and major dihedral communication patterns are still evident after this correction<span class='cite'> [<a href='mainli5.html#XBowman:2012jd'>47</a>]</span>.
Consistently, we observe that communication measurements remain significant (ie. not within error
of zero) (see Appendix <a href='mainap2.html#x15-139000B'>B<!-- tex4ht:ref: ch:cards-supp-info  --></a> and Figure <a href='mainap1.html#x14-138001r1'>A.1<!-- tex4ht:ref: fig:cards-pearsonr-fig  --></a>). It is worth noting that utilizing this naïve,
residue-independent binning system is already capable of making measurable predictions of protein
allostery<span class='cite'> [<a href='mainli5.html#XSun:2018kx'>48</a>, <a href='mainli5.html#XCruz2020vp35'>54</a>]</span>.
</p>
                                                                                      
                                                                                      
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x7-29001r6'></a>
                                                                                      
                                                                                      
<!-- l. 144 --><p class='noindent'><img width='433' height='175' src='Chapter3/cap-bb-sc-matrix-fig.png' alt='PIC' />
<a id='x7-29002'></a>
<br /> </p><div class='caption'><span class='id'>Figure 2.6:  </span><span class='content'>Hubs  of  backbone-side-chain  communication  in  wild-type  CAP.  <span class='ptmb7t-x-x-120'>A.  </span>Mutual
information between every pair of dihedrals in wild-type CAP. The upper triangle of the matrix
(above the green diagonal) represents structural communication and the lower triangle (below
the green diagonal) represents disorder-mediated communication. The pink and cyan squares
encompass the regions of the matrix that capture backbone-side-chain communication that is
mediated by disorder-mediated and structural coupling, respectively. <span class='ptmb7t-x-x-120'>B. </span>Structure of wild type
CAP  highlighting  the  residues  that  are  the  strongest  hubs  (top  5%)  of  backbone-side-chain
communication (red sticks).</span></div><!-- tex4ht:label?: x7-29001r2.4  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<h4 class='subsectionHead'><span class='titlemark'>2.4.5    </span> <a id='x7-300002.4.5'></a>Locating communication hotspots identifies key functional sites</h4>
<!-- l. 151 --><p class='noindent'>The coincidence of backbone dihedrals that are hubs of communication and key functional sites
suggests that CARDS may be capable of predicting the locations of such sites. Indeed, if
evolution has selected for communication between particular sites, then one might expect
residues in these sites to have stronger coupling to other regions of a protein than typical
residues.
</p><!-- l. 153 --><p class='noindent'>To detect strongly communicating residues, we ranked each residue based on the sum of its
correlations to all other residues in the protein. This measure of global communication will highlight
two types of hotspots: 1) hubs with correlations to many residues and 2) residues that have strong
correlations to a few residues.
</p><!-- l. 155 --><p class='noindent'>Coloring each residue in the <span class='cmmi-12'>apo </span>structure according to its global communication highlights that the
central hinge region and the helices between the two cAMP-binding sites are key mediators
of allosteric communication in CAP (Fig. <a href='#x7-30001r7'>2.7<!-- tex4ht:ref: fig:cap-global-mi-fig  --></a>). There are also hotspot residues in other
parts of the cAMP-binding sites and along the interfaces between the CBDs and DBDs.
These are precisely the regions that were identified by assessing communication to a single
cAMP-binding site, providing evidence that CARDS can indeed identify key functional sites
without foreknowledge of their locations. This conclusion is further supported by the fact
that the central hinge region has even stronger communication in the S62F variant (Fig.
<a href='mainap1.html#x14-138013r7'>A.7<!-- tex4ht:ref: fig:cap-deltaglobalMI  --></a>).
</p>
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x7-30001r7'></a>
                                                                                      
                                                                                      
<!-- l. 159 --><p class='noindent'><img width='228' height='222' src='Chapter3/cap-global-mi-fig.png' alt='PIC' />
<a id='x7-30002'></a>
<br /> </p><div class='caption'><span class='id'>Figure 2.7: </span><span class='content'>Global communication strength of each residue in apo CAP.</span></div><!-- tex4ht:label?: x7-30001r2.4  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<h3 class='sectionHead'><span class='titlemark'>2.5    </span> <a id='x7-310002.5'></a>Conclusion</h3>
<!-- l. 165 --><p class='noindent'>CARDS provides a means to integrate concerted structural changes and disorder-mediated correlations
into a holistic view of allostery. Application of this approach to wild-type CAP and the S62F
variant demonstrates the method’s ability to identify allosteric coupling in the absence
of concerted structural changes. Specifically, we showed that examining the coupling of
every residue to a known cAMP binding site naturally highlights regions of the protein that
are known to be impacted by cAMP binding, such as the second cAMP binding site and
the central hinge region connecting the CBDs and DBDs. Decomposing the correlations
between these sites into disorder-mediated and purely structural components demonstrates an
important role for disorder-mediated coupling in the absence of concerted structural changes.
Our global communication metric also provides a means to identify important functional
sites without foreknowledge of their existence and locations. For example, this metric
identifies the central hinge region—which undergoes the largest conformational change
upon activation—and the cAMP binding pockets as important components of the allosteric
network in CAP. Therefore, CARDS should be a powerful means to identify allosteric
networks in systems that have not been studied as thoroughly as CAP. Taken together, we
expect CARDS to be of great utility for understanding allostery in systems where it is
already known to occur, as well as for predicting allostery in systems where it has yet to be
observed.
</p><!-- l. 167 --><p class='noindent'>
                                                                                      
                                                                                      
</p>
<h3 class='sectionHead'><span class='titlemark'>2.6    </span> <a id='x7-320002.6'></a>Acknowledgements</h3>
<!-- l. 168 --><p class='noindent'>This work was funded by NSF CAREER Award MCB-1552471. G.R.B. holds a Career Award at the
Scientific Interface from the Burroughs Wellcome Fund and a Packard Fellowship for Science and
Engineering from The David <span class='cmr-12'>&amp; </span>Lucile Packard Foundation. We are also grateful to NVIDIA
Corporation for the GTX Titan X used to run preliminary simulations. Finally, thanks to Kelsey C.
Schuster and David Chandler, who, in collaboration with G.R.B., first demonstrated that proteins’
side-chains have dynamic heterogeneity.
                                                                                      
                                                                                      
</p>
<!-- l. 4 --><div class='crosslinks'><p class='noindent'></p></div>
 
</body></html>