<?xml version='1.0' encoding='utf-8' ?> 
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd"><html xmlns='http://www.w3.org/1999/xhtml'> 
<head><title>Bibliography</title> 
<meta http-equiv='Content-Type' content='text/html; charset=utf-8' /> 
<meta name='generator' content='TeX4ht (https://tug.org/tex4ht/)' /> 
<meta name='originator' content='TeX4ht (https://tug.org/tex4ht/)' /> 
<!--  xhtml,charset=utf-8,epub,uni-html4,html  --> 
<meta name='src' content='main.tex' /> 
<link type='text/css' href='main.css' rel='stylesheet' /> 
</head><body>
<!-- l. 1 --><div class='crosslinks'><p class='noindent'></p></div>
<h2 class='likechapterHead'><a id='x19-172000E.2'></a>Bibliography</h2>
      <div class='thebibliography'>
      <p class='bibitem'><span class='biblabel'>
   [1]<span class='bibsp'>   </span></span><a id='Xwaterhouse_swiss-model:_2018'></a>Andrew  Waterhouse,  Martino  Bertoni,  Stefan  Bienert,  Gabriel  Studer,  Gerardo
      Tauriello, Rafal Gumienny, Florian T Heer, Tjaart A P de Beer, Christine Rempfer,
      Lorenza   Bordoli,   Rosalba   Lepore,   and   Torsten   Schwede.        SWISS-MODEL:
      homology  modelling  of  protein  structures  and  complexes.    <span class='ptmri7t-x-x-120'>Nucleic  Acids  Research</span>,
      46(W1):W296–W303, July 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
   [2]<span class='bibsp'>   </span></span><a id='Xcubuk_sars-cov-2_2020'></a>Jasmine Cubuk, Jhullian J. Alston, J. Jeremías Incicco, Sukrit Singh, Melissa D.
      Stuchell-Brereton,  Michael D.  Ward,  Maxwell I.  Zimmerman,  Neha  Vithani,  Daniel
      Griffith, Jason A. Wagoner, Gregory R. Bowman, Kathleen B. Hall, Andrea Soranno,
      and Alex S. Holehouse. The SARS-CoV-2 nucleocapsid protein is dynamic, disordered,
      and phase separates with RNA. preprint, Biophysics, June 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
   [3]<span class='bibsp'>   </span></span><a id='Xsu_electron_2018'></a>Zhaoming  Su,  Chao  Wu,  Liuqing  Shi,  Priya  Luthra,  Grigore D.  Pintilie,  Britney
                                                                                      
                                                                                      
      Johnson,  Justin R.  Porter,  Peng  Ge,  Muyuan  Chen,  Gai  Liu,  Thomas E.  Frederick,
      Jennifer M.  Binning,  Gregory R.  Bowman,  Z. Hong  Zhou,  Christopher F.  Basler,
      Michael L. Gross, Daisy W. Leung, Wah Chiu, and Gaya K. Amarasinghe.  Electron
      cryo-microscopy  structure  of  ebola  virus  nucleoprotein  reveals  a  mechanism  for
      nucleocapsid-like assembly. <span class='ptmri7t-x-x-120'>Cell</span>, 172(5):966–978.e12, February 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
   [4]<span class='bibsp'>   </span></span><a id='XRoth:2017gk'></a>Bryan L  Roth,  John J  Irwin,  and  Brian K  Shoichet.    Discovery  of  new  GPCR
      ligands  to  illuminate  new  biology.     <span class='ptmri7t-x-x-120'>Nature  chemical  biology</span>,  13(11):1143–1151,
      November 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
   [5]<span class='bibsp'>   </span></span><a id='Xweitzner_computational_2019'></a>Brian D.  Weitzner,  Yakov  Kipnis,  A. Gerard  Daniel,  Donald  Hilvert,  and  David
      Baker.  A computational method for design of connected catalytic networks in proteins.
      <span class='ptmri7t-x-x-120'>Protein Science</span>, 28(12):2036–2041, December 2019.
      </p>
      <p class='bibitem'><span class='biblabel'>
   [6]<span class='bibsp'>   </span></span><a id='Xkendrew_three-dimensional_1958'></a>J. C.  Kendrew,  G. Bodo,  H. M.  Dintzis,  R. G.  Parrish,  H. Wyckoff,  and  D. C.
      Phillips.    A  three-dimensional  model  of  the  myoglobin  molecule  obtained  by  x-ray
      analysis. <span class='ptmri7t-x-x-120'>Nature</span>, 181(4610):662–666, March 1958.
      </p>
      <p class='bibitem'><span class='biblabel'>
   [7]<span class='bibsp'>   </span></span><a id='Xkhan_performance_2010'></a>Sofia Khan and Mauno Vihinen. Performance of protein stability predictors. <span class='ptmri7t-x-x-120'>Human
      </span><span class='ptmri7t-x-x-120'>Mutation</span>, 31(6):675–684, March 2010.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
   [8]<span class='bibsp'>   </span></span><a id='Xyin_overview_2017'></a>Jian Yin, Niel M. Henriksen, David R. Slochower, Michael R. Shirts, Michael W.
      Chiu, David L. Mobley, and Michael K. Gilson. Overview of the SAMPL5 host–guest
      challenge:  Are  we  doing  better?     <span class='ptmri7t-x-x-120'>Journal  of  Computer-Aided  Molecular  Design</span>,
      31(1):1–19, January 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
   [9]<span class='bibsp'>   </span></span><a id='XHart:2016kb'></a>Kathryn M Hart, Chris M W Ho, Supratik Dutta, Michael L Gross, and Gregory R
      Bowman.   Modelling  proteins’  hidden  conformations  to  predict  antibiotic  resistance.
      <span class='ptmri7t-x-x-120'>Nature communications</span>, 7:12965, October 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [10]<span class='bibsp'>   </span></span><a id='Xporter_conformational_2020'></a>Justin R Porter, Artur Meller, Maxwell I Zimmerman, Michael J Greenberg, and
      Gregory R Bowman.  Conformational distributions of isolated myosin motor domains
      encode their mechanochemical properties. <span class='ptmri7t-x-x-120'>eLife</span>, 9:e55132, May 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [11]<span class='bibsp'>   </span></span><a id='Xlevinthal_how_1969'></a>C Levinthal.     How  to  Fold  Graciously.     <span class='ptmri7t-x-x-120'>Topics  in  mossbauer  spectroscopy</span>,
      20(1):25–44, October 1969.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [12]<span class='bibsp'>   </span></span><a id='Xbaldwin_early_2011'></a>Robert L. Baldwin. Early days of protein hydrogen exchange: 1954-1972. <span class='ptmri7t-x-x-120'>Proteins:
      </span><span class='ptmri7t-x-x-120'>Structure, Function, and Bioinformatics</span>, 79(7):2021–2026, July 2011.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
  [13]<span class='bibsp'>   </span></span><a id='XHenzlerWildman:2007cz'></a>Katherine A   Henzler-Wildman,   Ming   Lei,   Vu Thai,   S Jordan   Kerns,   Martin
      Karplus, and Dorothee Kern. A hierarchy of timescales in protein dynamics is linked to
      enzyme catalysis. <span class='ptmri7t-x-x-120'>Nature</span>, 450(7171):913–916, December 2007.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [14]<span class='bibsp'>   </span></span><a id='XBoehr:2006jd'></a>David D  Boehr,  Dan  McElheny,  H Jane  Dyson,  and  P E  Wright.   The  dynamic
      energy landscape of dihydrofolate reductase catalysis.  <span class='ptmri7t-x-x-120'>Science</span>, 313(5793):1638–1642,
      September 2006.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [15]<span class='bibsp'>   </span></span><a id='XBoehr:2009ju'></a>David D   Boehr,   Ruth   Nussinov,   and   P E   Wright.       The   role   of   dynamic
      conformational  ensembles  in  biomolecular  recognition.     <span class='ptmri7t-x-x-120'>Nature  chemical  biology</span>,
      5(11):789–796, November 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [16]<span class='bibsp'>   </span></span><a id='Xzorba_molecular_2014'></a>Adelajda Zorba, Vanessa Buosi, Steffen Kutter, Nadja Kern, Francesco Pontiggia,
      Young-Jin  Cho,  and  Dorothee  Kern.     Molecular  mechanism  of  Aurora  A  kinase
      autophosphorylation and its allosteric activation by TPX2. <span class='ptmri7t-x-x-120'>eLife</span>, 3:e02667, May 2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [17]<span class='bibsp'>   </span></span><a id='Xkerns_energy_2015'></a>S Jordan Kerns, Roman V Agafonov, Young-Jin Cho, Francesco Pontiggia, Renee
      Otten, Dimitar V Pachov, Steffen Kutter, Lien A Phung, Padraig N Murphy, Vu Thai,
      Tom Alber, Michael F Hagan, and Dorothee Kern.  The energy landscape of adenylate
      kinase  during  catalysis.     <span class='ptmri7t-x-x-120'>Nature  Structural  &amp;  Molecular  Biology</span>,  22(2):124–131,
      February 2015.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
  [18]<span class='bibsp'>   </span></span><a id='XVanEps:2010by'></a>Ned Van Eps, Lori L Anderson, Oleg G Kisselev, Thomas J Baranski, Wayne L
      Hubbell,  and  Garland R  Marshall.      Electron  paramagnetic  resonance  studies  of
      functionally  active,  nitroxide  spin-labeled  peptide  analogues  of  the  C-terminus  of  a
      G-protein alpha subunit. <span class='ptmri7t-x-x-120'>Biochemistry</span>, 49(32):6877–6886, August 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [19]<span class='bibsp'>   </span></span><a id='XWand:2013dz'></a>A Joshua Wand. The dark energy of proteins comes to light: conformational entropy
      and  its  role  in  protein  function  revealed  by  NMR  relaxation.   23(1):75–81,  February
      2013.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [20]<span class='bibsp'>   </span></span><a id='XKoehl:2018fv'></a>Antoine Koehl, Hongli Hu, Shoji Maeda, Yan Zhang, Qianhui Qu, Joseph M Paggi,
      Naomi R  Latorraca,  Daniel  Hilger,  Roger  Dawson,  Hugues  Matile,  Gebhard  F X
      Schertler, Sébastien Granier, William I Weis, Ron O Dror, Aashish Manglik, Georgios
      Skiniotis, and Brian K Kobilka. Structure of the <span class='ptmr8c-x-x-120'>µ</span>-opioid receptor–G i protein complex.
      <span class='ptmri7t-x-x-120'>Nature</span>, 383:1, June 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [21]<span class='bibsp'>   </span></span><a id='XDraperJoyce:2018il'></a>Christopher J Draper-Joyce, Maryam Khoshouei, David M Thal, Yi-Lynn Liang,
      Anh  T N  Nguyen,  Sebastian  G B  Furness,  Hariprasad  Venugopal,  Jo-Anne  Baltos,
      Jürgen M  Plitzko,  Radostin  Danev,  Wolfgang  Baumeister,  Lauren T  May,  Denise
      Wootten, Patrick M Sexton, Alisa Glukhova, and Arthur Christopoulos. Structure of the
      adenosine-bound human adenosine A1 receptor-Gi complex. <span class='ptmri7t-x-x-120'>Nature</span>, 63:1, June 2018.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
  [22]<span class='bibsp'>   </span></span><a id='XGarciaNafria:2018hu'></a>Javier  García-Nafría,  Rony  Nehmé,  Patricia C  Edwards,  and  Christopher G
      Tate.   Cryo-EM structure of the serotonin 5-HT1B receptor coupled to heterotrimeric
      Go. <span class='ptmri7t-x-x-120'>Nature</span>, 7:118, June 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [23]<span class='bibsp'>   </span></span><a id='XKang:2018kq'></a>Yanyong  Kang,  Oleg  Kuybeda,  Parker W  de Waal,  Somnath  Mukherjee,  Ned
      Van Eps, Przemyslaw Dutka, X Edward Zhou, Alberto Bartesaghi, Satchal Erramilli,
      Takefumi  Morizumi,  Xin  Gu,  Yanting  Yin,  Ping  Liu,  Yi Jiang,  Xing  Meng,  Gongpu
      Zhao, Karsten Melcher, Oliver P Ernst, Anthony A Kossiakoff, Sriram Subramaniam,
      and  H Eric  Xu.    Cryo-EM  structure  of  human  rhodopsin  bound  to  an  inhibitory  G
      protein. <span class='ptmri7t-x-x-120'>Nature</span>, 63(Suppl. 1):1256, June 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [24]<span class='bibsp'>   </span></span><a id='XDror:2012cs'></a>Ron O  Dror,  Robert M  Dirks,  J P  Grossman,  Huafeng  Xu,  and  David E  Shaw.
      Biomolecular   Simulation:   A   Computational   Microscope   for   Molecular   Biology.
      <span class='ptmri7t-x-x-120'>dx.doi.org</span>, 41(1):429–452, May 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [25]<span class='bibsp'>   </span></span><a id='Xlee_discovery_2009'></a>Eric H. Lee, Jen Hsin, Marcos Sotomayor, Gemma Comellas, and Klaus Schulten.
      Discovery  through  the  computational  microscope.      <span class='ptmri7t-x-x-120'>Structure</span>,  17(10):1295–1306,
      October 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [26]<span class='bibsp'>   </span></span><a id='Xkukol_current_2015'></a>Pedro E. M. Lopes, Olgun Guvench, and Alexander D. MacKerell.  Current status
      of protein force fields for molecular dynamics simulations.  In Andreas Kukol, editor,
                                                                                      
                                                                                      
      <span class='ptmri7t-x-x-120'>Molecular Modeling of Proteins</span>, volume 1215, pages 47–71. Springer New York, New
      York, NY, 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [27]<span class='bibsp'>   </span></span><a id='XE:2010hs'></a>Weinan  E  and  Eric  Vanden-Eijnden.    Transition-Path  Theory  and  Path-Finding
      Algorithms for the Study of Rare Events. <span class='ptmri7t-x-x-120'>dx.doi.org</span>, 61(1):391–420, March 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [28]<span class='bibsp'>   </span></span><a id='Xhamelberg_accelerated_2004'></a>Donald  Hamelberg,  John  Mongan,  and  J. Andrew  McCammon.     Accelerated
      molecular dynamics: A promising and efficient simulation method for biomolecules. <span class='ptmri7t-x-x-120'>The
      </span><span class='ptmri7t-x-x-120'>Journal of Chemical Physics</span>, 120(24):11919–11929, June 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [29]<span class='bibsp'>   </span></span><a id='Xpaschek_replica_2007'></a>Dietmar  Paschek,  Hugh  Nymeyer,  and  Angel E.  García.     Replica  exchange
      simulation  of  reversible  folding/unfolding  of  the  Trp-cage  miniprotein  in  explicit
      solvent:  On  the  structure  and  possible  role  of  internal  water.    <span class='ptmri7t-x-x-120'>Journal  of  Structural
      </span><span class='ptmri7t-x-x-120'>Biology</span>, 157(3):524–533, March 2007.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [30]<span class='bibsp'>   </span></span><a id='Xpiana_bias-exchange_2007'></a>Stefano Piana and Alessandro Laio.  A bias-exchange approach to protein folding.
      <span class='ptmri7t-x-x-120'>The Journal of Physical Chemistry B</span>, 111(17):4553–4559, May 2007.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [31]<span class='bibsp'>   </span></span><a id='Xstone_accelerating_2007'></a>John E.                 Stone,                 James C.                 Phillips,                 Peter L.
      Freddolino, David J. Hardy, Leonardo G. Trabuco, and Klaus Schulten.  Accelerating
                                                                                      
                                                                                      
      molecular modeling applications with graphics processors.   <span class='ptmri7t-x-x-120'>Journal of Computational
      </span><span class='ptmri7t-x-x-120'>Chemistry</span>, 28(16):2618–2640, December 2007.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [32]<span class='bibsp'>   </span></span><a id='Xfriedrichs_accelerating_2009'></a>Mark S.  Friedrichs,  Peter  Eastman,  Vishal  Vaidyanathan,  Mike  Houston,  Scott
      Legrand,  Adam L.  Beberg,  Daniel L.  Ensign,  Christopher M.  Bruns,  and  Vijay S.
      Pande.     Accelerating  molecular  dynamic  simulation  on  graphics  processing  units.
      <span class='ptmri7t-x-x-120'>Journal of Computational Chemistry</span>, 30(6):864–872, April 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [33]<span class='bibsp'>   </span></span><a id='Xeastman_openmm:_2010'></a>Peter Eastman and Vijay Pande.  Openmm: a hardware-independent framework for
      molecular simulations. <span class='ptmri7t-x-x-120'>Computing in Science &amp; Engineering</span>, 12(4):34–39, July 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [34]<span class='bibsp'>   </span></span><a id='Xeastman_openmm_2017'></a>Peter Eastman, Jason Swails, John D. Chodera, Robert T. McGibbon, Yutong Zhao,
      Kyle A.  Beauchamp,  Lee-Ping  Wang,  Andrew C.  Simmonett,  Matthew P.  Harrigan,
      Chaya D. Stern, Rafal P. Wiewiora, Bernard R. Brooks, and Vijay S. Pande. OpenMM
      7: Rapid development of high performance algorithms for molecular dynamics.  <span class='ptmri7t-x-x-120'>PLOS
      </span><span class='ptmri7t-x-x-120'>Computational Biology</span>, 13(7):e1005659, July 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [35]<span class='bibsp'>   </span></span><a id='Xzimmerman_fast_2015'></a>Maxwell I. Zimmerman and Gregory R. Bowman.   Fast conformational searches
      by  balancing  exploration/exploitation  trade-offs.    <span class='ptmri7t-x-x-120'>Journal  of  Chemical  Theory  and
      </span><span class='ptmri7t-x-x-120'>Computation</span>, 11(12):5747–5757, December 2015.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
  [36]<span class='bibsp'>   </span></span><a id='Xshaw_anton_2008'></a>David E. Shaw, Martin M. Deneroff, Ron O. Dror, Jeffrey S. Kuskin, Richard H.
      Larson,  John K.  Salmon,  Cliff  Young,  Brannon  Batson,  Kevin J.  Bowers,  Jack C.
      Chao,  Michael P.  Eastwood,  Joseph  Gagliardo,  J. P.  Grossman,  C. Richard  Ho,
      Douglas J. Ierardi, István Kolossváry, John L. Klepeis, Timothy Layman, Christine
      McLeavey,  Mark A.  Moraes,  Rolf  Mueller,  Edward C.  Priest,  Yibing  Shan,  Jochen
      Spengler,   Michael   Theobald,   Brian   Towles,   and   Stanley C.   Wang.       Anton,   a
      special-purpose machine for molecular dynamics simulation.   <span class='ptmri7t-x-x-120'>Communications of the
      </span><span class='ptmri7t-x-x-120'>ACM</span>, 51(7):91–97, July 2008.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [37]<span class='bibsp'>   </span></span><a id='XShirts:2000du'></a>M Shirts  and  V S  Pande.    COMPUTING:  Screen  Savers  of  the  World  Unite!
      <span class='ptmri7t-x-x-120'>Science</span>, 290(5498):1903–1904, December 2000.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [38]<span class='bibsp'>   </span></span><a id='XHusic:2018iz'></a>Brooke E  Husic  and  Vijay S  Pande.    Markov  State  Models:  From  an  Art  to  a
      Science. <span class='ptmri7t-x-x-120'>Journal of the American Chemical Society</span>, page jacs.7b12191, January 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [39]<span class='bibsp'>   </span></span><a id='Xporter2018enspara'></a>J. R. Porter, M. I. Zimmerman, and G. R. Bowman. Enspara: Modeling molecular
      ensembles with scalable data structures and parallel computing. <span class='ptmri7t-x-x-120'>The Journal of Chemical
      </span><span class='ptmri7t-x-x-120'>Physics</span>, 150(4):044108, 2019.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [40]<span class='bibsp'>   </span></span><a id='XChodera:2014gk'></a>John D   Chodera   and   Frank   Noé.       Markov   state   models   of   biomolecular
                                                                                      
                                                                                      
      conformational  dynamics.   <span class='ptmri7t-x-x-120'>Current  opinion  in  structural  biology</span>,  25:135–144,  April
      2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [41]<span class='bibsp'>   </span></span><a id='XKnoverek:2018kk'></a>Catherine R Knoverek, Gaya K Amarasinghe, and Gregory R Bowman. Advanced
      Methods for Accessing Protein Shape-Shifting Present New Therapeutic Opportunities.
      <span class='ptmri7t-x-x-120'>Trends in biochemical sciences</span>, December 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [42]<span class='bibsp'>   </span></span><a id='XZimmerman:2018jn'></a>Maxwell I  Zimmerman,  Justin R  Porter,  Xianqiang  Sun,  Roseane R  Silva,  and
      Gregory R Bowman.  Choice of Adaptive Sampling Strategy Impacts State Discovery,
      Transition  Probabilities,  and  the  Apparent  Mechanism  of  Conformational  Changes.
      <span class='ptmri7t-x-x-120'>Journal  of  Chemical  Theory  and  Computation</span>,  q-bio.BM:acs.jctc.8b00500,  October
      2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [43]<span class='bibsp'>   </span></span><a id='XMolgedey:1994ew'></a>L Molgedey and HG Schuster. Separation of a mixture of independent signals using
      time delayed correlations. <span class='ptmri7t-x-x-120'>Physical review letters</span>, 72(23):3634–3637, June 1994.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [44]<span class='bibsp'>   </span></span><a id='XMSultan:2017du'></a>Mohammad  M Sultan  and  Vijay S  Pande.     tICA-Metadynamics:  Accelerating
      Metadynamics by Using Kinetically Selected Collective Variables. <span class='ptmri7t-x-x-120'>Journal of Chemical
      </span><span class='ptmri7t-x-x-120'>Theory and Computation</span>, 13(6):2440–2447, June 2017.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
  [45]<span class='bibsp'>   </span></span><a id='XPerezHernandez:2013ih'></a>Guillermo  Pérez-Hernández,  Fabian  Paul,  Toni  Giorgino,  Gianni  De Fabritiis,
      and Frank Noé.  Identification of slow molecular order parameters for Markov model
      construction. <span class='ptmri7t-x-x-120'>arXiv.org</span>, (1):015102, February 2013.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [46]<span class='bibsp'>   </span></span><a id='XVoelz:2010hs'></a>Vincent A  Voelz,  Gregory R  Bowman,  Kyle  Beauchamp,  and  Vijay S  Pande.
      Molecular simulation of ab initio protein folding for a millisecond folder NTL9(1-39).
      <span class='ptmri7t-x-x-120'>Journal of the American Chemical Society</span>, 132(5):1526–1528, February 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [47]<span class='bibsp'>   </span></span><a id='XBowman:2012jd'></a>Gregory R Bowman and Phillip L Geissler.   Equilibrium fluctuations of a single
      folded protein reveal a multitude of potential cryptic allosteric sites.  <span class='ptmri7t-x-x-120'>Proceedings of the
      </span><span class='ptmri7t-x-x-120'>National Academy of Sciences</span>, 109(29):11681–11686, July 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [48]<span class='bibsp'>   </span></span><a id='XSun:2018kx'></a>Xianqiang Sun, Sukrit Singh, Kendall Blumer, and Gregory R Bowman. Simulation
      of spontaneous G protein activation reveals a new intermediate driving GDP unbinding.
      <span class='ptmri7t-x-x-120'>eLife</span>, 7, October 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [49]<span class='bibsp'>   </span></span><a id='XPorter:2019hv'></a>Justin R  Porter,  Katelyn E  Moeder,  Carrie A  Sibbald,  Maxwell I  Zimmerman,
      Kathryn M Hart, Michael J Greenberg, and Gregory R Bowman. Cooperative Changes
      in  Solvent  Exposure  Identify  Cryptic  Pockets,  Switches,  and  Allosteric  Coupling.
      <span class='ptmri7t-x-x-120'>Biophysical Journal</span>, 116(5):818–830, March 2019.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
  [50]<span class='bibsp'>   </span></span><a id='Xgonzalez_clustering_1985'></a>Teofilo F.  Gonzalez.   Clustering  to  minimize  the  maximum  intercluster  distance.
      <span class='ptmri7t-x-x-120'>Theoretical Computer Science</span>, 38:293–306, 1985.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [51]<span class='bibsp'>   </span></span><a id='Xpark_simple_2009'></a>Hae-Sang  Park  and  Chi-Hyuck  Jun.   A  simple  and  fast  algorithm  for  K-medoids
      clustering. <span class='ptmri7t-x-x-120'>Expert Systems with Applications</span>, 36(2):3336–3341, March 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [52]<span class='bibsp'>   </span></span><a id='Xprinz_optimal_2011'></a>Jan-Hendrik   Prinz,   John D.   Chodera,   Vijay S.   Pande,   William C.   Swope,
      Jeremy C. Smith, and Frank Noé. Optimal use of data in parallel tempering simulations
      for the construction of discrete-state Markov models of biomolecular dynamics.   <span class='ptmri7t-x-x-120'>The
      </span><span class='ptmri7t-x-x-120'>Journal of Chemical Physics</span>, 134(24):244108, June 2011.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [53]<span class='bibsp'>   </span></span><a id='Xprinz_markov_2011'></a>Jan-Hendrik  Prinz,  Hao  Wu,  Marco  Sarich,  Bettina  Keller,  Martin  Senne,  Martin
      Held,  John D.  Chodera,  Christof  Schütte,  and  Frank  Noé.     Markov  models  of
      molecular  kinetics:  Generation  and  validation.     <span class='ptmri7t-x-x-120'>The  Journal  of  Chemical  Physics</span>,
      134(17):174105, May 2011.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [54]<span class='bibsp'>   </span></span><a id='XCruz2020vp35'></a>Matthew A. Cruz, Thomas E. Frederick, Sukrit Singh, Neha Vithani, Maxwell I.
      Zimmerman,   Justin R.   Porter,   Katelyn E.   Moeder,   Gaya K.   Amarasinghe,   and
      Gregory R. Bowman.   Discovery of a cryptic allosteric site in ebola’s ‘undruggable’
      vp35 protein using simulations and experiments. <span class='ptmri7t-x-x-120'>bioRxiv</span>, 2020.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
  [55]<span class='bibsp'>   </span></span><a id='XBowman:2011bc'></a>Gregory R Bowman, Vincent A Voelz, and Vijay S Pande. Taming the complexity
      of protein folding. 21(1):4–11, February 2011.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [56]<span class='bibsp'>   </span></span><a id='XBowman:2015da'></a>Gregory R Bowman.  Accurately modeling nanosecond protein dynamics requires
      at least microseconds of simulation. <span class='ptmri7t-x-x-120'>Journal of computational chemistry</span>, pages n/a–n/a,
      June 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [57]<span class='bibsp'>   </span></span><a id='Xbowman2014extensive'></a>Gregory R Bowman and Phillip L Geissler. Extensive conformational heterogeneity
      within protein cores. <span class='ptmri7t-x-x-120'>The Journal of Physical Chemistry B</span>, 118(24):6417–6423, 2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [58]<span class='bibsp'>   </span></span><a id='Xge_simulations_2018'></a>Yunhui Ge, Elias Borne, Shannon Stewart, Michael R. Hansen, Emilia C. Arturo,
      Eileen K. Jaffe, and Vincent A. Voelz.  Simulations of the regulatory ACT domain of
      human phenylalanine hydroxylase (Pah) unveil its mechanism of phenylalanine binding.
      <span class='ptmri7t-x-x-120'>Journal of Biological Chemistry</span>, 293(51):19532–19543, December 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [59]<span class='bibsp'>   </span></span><a id='Xchen_dynamic_2019'></a>Shi Chen, Rafal P Wiewiora, Fanwang Meng, Nicolas Babault, Anqi Ma, Wenyu
      Yu, Kun Qian, Hao Hu, Hua Zou, Junyi Wang, Shijie Fan, Gil Blum, Fabio Pittella-Silva,
      Kyle A Beauchamp, Wolfram Tempel, Hualiang Jiang, Kaixian Chen, Robert J Skene,
      Yujun George  Zheng,  Peter J  Brown,  Jian  Jin,  Cheng  Luo,  John D  Chodera,  and
                                                                                      
                                                                                      
      Minkui Luo.  The dynamic conformational landscape of the protein methyltransferase
      SETD8. <span class='ptmri7t-x-x-120'>eLife</span>, 8:e45403, May 2019.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [60]<span class='bibsp'>   </span></span><a id='Xpascolutti_structure_2016'></a>Roberta  Pascolutti,  Xianqiang  Sun,  Joseph  Kao,  Roy L.  Maute,  Aaron M.  Ring,
      Gregory R. Bowman, and Andrew C. Kruse.  Structure and dynamics of pd-l1 and an
      ultra-high-affinity pd-1 receptor mutant. <span class='ptmri7t-x-x-120'>Structure</span>, 24(10):1719–1728, October 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [61]<span class='bibsp'>   </span></span><a id='Xhilser_ensemble_2010'></a>V. J. Hilser. An ensemble view of allostery. <span class='ptmri7t-x-x-120'>Science</span>, 327(5966):653–654, February
      2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [62]<span class='bibsp'>   </span></span><a id='XPerutz:1970vb'></a>M F  Perutz.    Stereochemistry  of  cooperative  effects  in  haemoglobin.    <span class='ptmri7t-x-x-120'>Nature</span>,
      228(5273):726–739, November 1970.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [63]<span class='bibsp'>   </span></span><a id='XTzeng:2009im'></a>Shiou-Ru  Tzeng  and  Charalampos G  Kalodimos.     Dynamic  activation  of  an
      allosteric regulatory protein. <span class='ptmri7t-x-x-120'>Nature</span>, 462(7271):368–372, November 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [64]<span class='bibsp'>   </span></span><a id='XTzeng:2011by'></a>Shiou-Ru Tzeng and Charalampos G Kalodimos.  Protein dynamics and allostery:
      an NMR view. 21(1):62–67, February 2011.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
  [65]<span class='bibsp'>   </span></span><a id='XWeis:2018fj'></a>William I Weis and Brian K Kobilka.  The Molecular Basis of G Protein–Coupled
      Receptor Activation. <span class='ptmri7t-x-x-120'>Annual review of biochemistry</span>, 87(1):897–919, June 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [66]<span class='bibsp'>   </span></span><a id='Xwacker_how_2017'></a>Daniel Wacker, Raymond C. Stevens, and Bryan L. Roth.  How ligands illuminate
      gpcr molecular pharmacology. <span class='ptmri7t-x-x-120'>Cell</span>, 170(3):414–427, July 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [67]<span class='bibsp'>   </span></span><a id='Xrosholm_membrane_2017'></a>Kadla R  Rosholm,  Natascha  Leijnse,  Anna  Mantsiou,  Vadym  Tkach,  Søren L
      Pedersen,  Volker F  Wirth,  Lene B  Oddershede,  Knud J  Jensen,  Karen L  Martinez,
      Nikos S Hatzakis, Poul Martin Bendix, Andrew Callan-Jones, and Dimitrios Stamou.
      Membrane  curvature  regulates  ligand-specific  membrane  sorting  of  GPCRs  in  living
      cells. <span class='ptmri7t-x-x-120'>Nature Chemical Biology</span>, 13(7):724–729, July 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [68]<span class='bibsp'>   </span></span><a id='XRasmussen:2011kp'></a>Søren   G F   Rasmussen,   Brian T   DeVree,   Yaozhong   Zou,   Andrew C   Kruse,
      Ka Young  Chung,  Tong Sun  Kobilka,  Foon Sun  Thian,  Pil Seok  Chae,  Els  Pardon,
      Diane  Calinski,  Jesper M  Mathiesen,  Syed  T A  Shah,  Joseph A  Lyons,  Martin
      Caffrey, Samuel H Gellman, Jan Steyaert, Georgios Skiniotis, William I Weis, Roger K
      Sunahara,  and  Brian K  Kobilka.   Crystal  structure  of  the  <span class='cmmi-12'>β</span>2  adrenergic  receptor-Gs
      protein complex. <span class='ptmri7t-x-x-120'>Nature</span>, 477(7366):549–555, July 2011.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [69]<span class='bibsp'>   </span></span><a id='XGunasekaran:2004kq'></a>K Gunasekaran, Buyong Ma, and Ruth Nussinov.  Is allostery an intrinsic property
      of all dynamic proteins? <span class='ptmri7t-x-x-120'>Proteins</span>, 57(3):433–443, November 2004.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
  [70]<span class='bibsp'>   </span></span><a id='Xromero_exploring_2009'></a>Philip A. Romero and Frances H. Arnold.  Exploring protein fitness landscapes by
      directed evolution. <span class='ptmri7t-x-x-120'>Nature Reviews Molecular Cell Biology</span>, 10(12):866–876, December
      2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [71]<span class='bibsp'>   </span></span><a id='XSalverda:2010db'></a>Merijn  L M  Salverda,  J Arjan  G M  De Visser,  and  Miriam  Barlow.    Natural
      evolution  of  TEM-1  <span class='cmmi-12'>β</span>-lactamase:  experimental  reconstruction  and  clinical  relevance.
      <span class='ptmri7t-x-x-120'>FEMS microbiology reviews</span>, 34(6):1015–1036, November 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [72]<span class='bibsp'>   </span></span><a id='Xarkin_small-molecule_2004'></a>Michelle R.   Arkin   and   James A.   Wells.         Small-molecule   inhibitors   of
      protein–protein  interactions:  progressing  towards  the  dream.    <span class='ptmri7t-x-x-120'>Nature  Reviews  Drug
      </span><span class='ptmri7t-x-x-120'>Discovery</span>, 3(4):301–317, April 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [73]<span class='bibsp'>   </span></span><a id='Xivetac_mapping_2010'></a>Anthony  Ivetac  and  J. Andrew McCammon.    Mapping  the  druggable  allosteric
      space of g-protein coupled receptors: a fragment-based molecular dynamics approach:
      computational mapping of novel druggable sites on gpcrs.  <span class='ptmri7t-x-x-120'>Chemical Biology &amp; Drug
      </span><span class='ptmri7t-x-x-120'>Design</span>, pages no–no, July 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [74]<span class='bibsp'>   </span></span><a id='Xhardy_searching_2004'></a>Jeanne A Hardy and James A Wells. Searching for new allosteric sites in enzymes.
      <span class='ptmri7t-x-x-120'>Current Opinion in Structural Biology</span>, 14(6):706–715, December 2004.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
  [75]<span class='bibsp'>   </span></span><a id='XHorn:2004uu'></a>James R   Horn   and   Brian K   Shoichet.      Allosteric   Inhibition   Through   Core
      Disruption. <span class='ptmri7t-x-x-120'>Journal of Molecular Biology</span>, 336(5):1283–1291, March 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [76]<span class='bibsp'>   </span></span><a id='Xarkin_binding_2003'></a>M. R.  Arkin,  M. Randal,  W. L.  DeLano,  J. Hyde,  T. N.  Luong,  J. D.  Oslob,
      D. R. Raphael, L. Taylor, J. Wang, R. S. McDowell, J. A. Wells, and A. C. Braisted.
      Binding of small molecules to an adaptive protein-protein interface.  <span class='ptmri7t-x-x-120'>Proceedings of the
      </span><span class='ptmri7t-x-x-120'>National Academy of Sciences</span>, 100(4):1603–1608, February 2003.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [77]<span class='bibsp'>   </span></span><a id='Xostrem_k-rasg12c_2013'></a>Jonathan M. Ostrem, Ulf Peters, Martin L. Sos, James A. Wells, and Kevan M.
      Shokat.      K-Ras(G12c)  inhibitors  allosterically  control  GTP  affinity  and  effector
      interactions. <span class='ptmri7t-x-x-120'>Nature</span>, 503(7477):548–551, November 2013.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [78]<span class='bibsp'>   </span></span><a id='XVajda:2018kt'></a>Sandor  Vajda,  Dmitri  Beglov,  Amanda E  Wakefield,  Megan  Egbert,  and  Adrian
      Whitty. Cryptic binding sites on proteins: definition, detection, and druggability. <span class='ptmri7t-x-x-120'>Current
      </span><span class='ptmri7t-x-x-120'>opinion in chemical biology</span>, 44:1–8, May 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [79]<span class='bibsp'>   </span></span><a id='Xschames_discovery_2004'></a>Julie R.  Schames,  Richard H.  Henchman,  Jay S.  Siegel,  Christoph A.  Sotriffer,
      Haihong Ni, and J. Andrew McCammon.  Discovery of a novel binding trench in hiv
      integrase. <span class='ptmri7t-x-x-120'>Journal of Medicinal Chemistry</span>, 47(8):1879–1881, April 2004.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
  [80]<span class='bibsp'>   </span></span><a id='Xfrantom_mapping_2009'></a>Patrick A.   Frantom,   Hui-Min   Zhang,   Mark R.   Emmett,   Alan G.   Marshall,
      and   John S.   Blanchard.       Mapping   of   the   allosteric   network   in   the   regulation
      of   -isopropylmalate   synthase   from   <span class='ptmri7t-x-x-120'>mycobacterium   tuberculosis   </span>by   the   feedback
      inhibitor  <span class='cmmi-12'>&lt;</span>span  style=”font-variant:small-caps;”<span class='cmmi-12'>&gt;</span>l<span class='cmmi-12'>&lt;</span>/span<span class='cmmi-12'>&gt; </span>-leucine:  solution-phase
      h/d exchange monitored by ft-icr mass spectrometry.  <span class='ptmri7t-x-x-120'>Biochemistry</span>, 48(31):7457–7464,
      August 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [81]<span class='bibsp'>   </span></span><a id='Xmanley_nmr_2012'></a>Gregory Manley and J. Patrick Loria. NMR insights into protein allostery. <span class='ptmri7t-x-x-120'>Archives
      </span><span class='ptmri7t-x-x-120'>of Biochemistry and Biophysics</span>, 519(2):223–231, March 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [82]<span class='bibsp'>   </span></span><a id='XLockless:1999uf'></a>S W Lockless and R Ranganathan. Evolutionarily conserved pathways of energetic
      connectivity in protein families. <span class='ptmri7t-x-x-120'>Science</span>, 286(5438):295–299, October 1999.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [83]<span class='bibsp'>   </span></span><a id='XFeher:2014cx'></a>Victoria A  Feher,  Jacob D  Durrant,  Adam T  Van Wart,  and  Rommie E  Amaro.
      Computational approaches to mapping allosteric pathways. 25:98–103, April 2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [84]<span class='bibsp'>   </span></span><a id='XGreener:2017ie'></a>Joe G  Greener  and  Michael Je  Sternberg.   Structure-based  prediction  of  protein
      allostery. 50:1–8, October 2017.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
  [85]<span class='bibsp'>   </span></span><a id='XIchiye:1991db'></a>Toshiko Ichiye and Martin Karplus.   Collective motions in proteins: A covariance
      analysis of atomic fluctuations in molecular dynamics and normal mode simulations.
      <span class='ptmri7t-x-x-120'>Proteins</span>, 11(3):205–217, November 1991.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [86]<span class='bibsp'>   </span></span><a id='XMcClendon:2009gd'></a>Christopher L   McClendon,   Gregory   Friedland,   David L   Mobley,   Homeira
      Amirkhani,  and  Matthew P  Jacobson.    Quantifying  Correlations  Between  Allosteric
      Sites in Thermodynamic Ensembles.   <span class='ptmri7t-x-x-120'>Journal of Chemical Theory and Computation</span>,
      5(9):2486–2502, September 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [87]<span class='bibsp'>   </span></span><a id='XCooper:1984tb'></a>A Cooper and D T Dryden.  Allostery without conformational change. A plausible
      model. <span class='ptmri7t-x-x-120'>European biophysics journal : EBJ</span>, 11(2):103–109, 1984.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [88]<span class='bibsp'>   </span></span><a id='XPopovych:2006js'></a>Nataliya   Popovych,   Shangjin   Sun,   Richard H   Ebright,   and   Charalampos G
      Kalodimos.    Dynamically  driven  protein  allostery.    <span class='ptmri7t-x-x-120'>Nature  Structural  &amp;  Molecular
      </span><span class='ptmri7t-x-x-120'>Biology</span>, 13(9):831–838, September 2006.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [89]<span class='bibsp'>   </span></span><a id='XLin:2016gi'></a>Milo M  Lin.    Timing  Correlations  in  Proteins  Predict  Functional  Modules  and
      Dynamic Allostery. <span class='ptmri7t-x-x-120'>Journal of the American Chemical Society</span>, page jacs.5b08814, April
      2016.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
  [90]<span class='bibsp'>   </span></span><a id='XSingh:2017hh'></a>Sukrit Singh and Gregory R Bowman.   Quantifying allosteric communication via
      both concerted structural changes and conformational disorder with CARDS. <span class='ptmri7t-x-x-120'>Journal of
      </span><span class='ptmri7t-x-x-120'>Chemical Theory and Computation</span>, page acs.jctc.6b01181, March 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [91]<span class='bibsp'>   </span></span><a id='XRosenbaum:2009id'></a>Daniel M Rosenbaum, Søren G F Rasmussen, and Brian K Kobilka. The structure
      and function of G-protein-coupled receptors. <span class='ptmri7t-x-x-120'>Nature</span>, 459(7245):356–363, May 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [92]<span class='bibsp'>   </span></span><a id='XShoichet:2012fm'></a>Brian K  Shoichet  and  Brian K  Kobilka.     Structure-based  drug  screening  for
      G-protein-coupled receptors. <span class='ptmri7t-x-x-120'>Trends in pharmacological sciences</span>, 33(5):268–272, May
      2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [93]<span class='bibsp'>   </span></span><a id='XGuo:2016hv'></a>Jingjing  Guo  and  Huan-Xiang  Zhou.      Protein  Allostery  and  Conformational
      Dynamics. <span class='ptmri7t-x-x-120'>Chemical Reviews</span>, 116(11):6503–6515, February 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [94]<span class='bibsp'>   </span></span><a id='XKoshland:1959ft'></a>D E  Koshland.     Enzyme  flexibility  and  enzyme  action.     <span class='ptmri7t-x-x-120'>Journal  of  Cellular
      </span><span class='ptmri7t-x-x-120'>Physiology</span>, 54(S1):245–258, December 1959.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [95]<span class='bibsp'>   </span></span><a id='XKoshlandJr:1966ffa'></a>D E Koshland Jr, G Nemethy, and D Filmer. Comparison of Experimental Binding
                                                                                      
                                                                                      
      Data  and  Theoretical  Models  in  Proteins  Containing  Subunits  *.      <span class='ptmri7t-x-x-120'>Biochemistry</span>,
      5(1):365–385, January 1966.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [96]<span class='bibsp'>   </span></span><a id='XMonod:1965io'></a>Jacque  Monod,  Jeffries  Wyman,  and  Jean-Pierre  Changeux.    On  the  nature  of
      allosteric transitions: A plausible model.  <span class='ptmri7t-x-x-120'>Journal of Molecular Biology</span>, 12(1):88–118,
      May 1965.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [97]<span class='bibsp'>   </span></span><a id='XChangeux:2011fv'></a>Jean-Pierre Changeux and Stuart Edelstein. Conformational selection or induced fit?
      50 years of debate resolved. <span class='ptmri7t-x-x-120'>F1000 biology reports</span>, 3(19):19, 2011.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [98]<span class='bibsp'>   </span></span><a id='XCsermely:2010cv'></a>Peter  Csermely,  Robin  Palotai,  and  Ruth  Nussinov.    Induced  fit,  conformational
      selection  and  independent  dynamic  segments:  an  extended  view  of  binding  events.
      <span class='ptmri7t-x-x-120'>Trends in biochemical sciences</span>, 35(10):539–546, October 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
  [99]<span class='bibsp'>   </span></span><a id='XSilva:2011ct'></a>Daniel-Adriano  Silva,  Gregory R  Bowman,  Alejandro  Sosa-Peinado,  and  Xuhui
      Huang. A role for both conformational selection and induced fit in ligand binding by the
      LAO protein. <span class='ptmri7t-x-x-120'>PLoS Computational Biology</span>, 7(5):e1002054, May 2011.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [100]<span class='bibsp'>   </span></span><a id='XHammes:2009im'></a>Gordon G Hammes, Yu-Chu Chang, and Terrence G Oas. Conformational selection
      or induced fit: a flux description of reaction mechanism.  <span class='ptmri7t-x-x-120'>Proceedings of the National
                                                                                      
                                                                                      
      </span><span class='ptmri7t-x-x-120'>Academy of Sciences of the United States of America</span>, 106(33):13737–13741, August
      2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [101]<span class='bibsp'>   </span></span><a id='Xand:1999bo'></a>Jennifer  L Radkiewicz  and  and  III  Charles  L Brooks.     Protein  Dynamics  in
      Enzymatic Catalysis: Exploration of Dihydrofolate Reductase. <span class='ptmri7t-x-x-120'>Journal of the American
      </span><span class='ptmri7t-x-x-120'>Chemical Society</span>, 122(2):225–231, December 1999.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [102]<span class='bibsp'>   </span></span><a id='XPratulKAgarwal:2002dp'></a>Pratul  K  Agarwal,  Salomon R  Billeter,  ,  and  Sharon  Hammes-Schiffer.   <span class='ptmri7t-x-x-120'>Nuclear
      </span><span class='ptmri7t-x-x-120'>Quantum Effects and Enzyme Dynamics in Dihydrofolate Reductase Catalysis</span>, volume
      106. American Chemical Society, February 2002.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [103]<span class='bibsp'>   </span></span><a id='XAgarwal:2002ema'></a>Pratul K   Agarwal,   Salomon R   Billeter,   P T Ravi   Rajagopalan,   Stephen J
      Benkovic,  and  Sharon  Hammes-Schiffer.   Network  of  coupled  promoting  motions  in
      enzyme catalysis.  <span class='ptmri7t-x-x-120'>Proceedings of the National Academy of Sciences</span>, 99(5):2794–2799,
      March 2002.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [104]<span class='bibsp'>   </span></span><a id='XRod:2003vm'></a>Thomas H  Rod,  Jennifer L  Radkiewicz,  and  Charles L  Brooks III.    Correlated
      motion and the effect of distal mutations in dihydrofolate reductase.  <span class='ptmri7t-x-x-120'>Proceedings of the
      </span><span class='ptmri7t-x-x-120'>National Academy of Sciences</span>, 100(12):6980–6985, June 2003.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [105]<span class='bibsp'>   </span></span><a id='XLange:2006gv'></a>Oliver F Lange and Helmut Grubmüller. Generalized correlation for biomolecular
      dynamics. <span class='ptmri7t-x-x-120'>Proteins</span>, 62(4):1053–1061, March 2006.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [106]<span class='bibsp'>   </span></span><a id='XWhitley:2009uq'></a>Matthew J Whitley and Andrew L Lee.  Frameworks for understanding long-range
      intra-protein communication.  <span class='ptmri7t-x-x-120'>Current protein &amp; peptide science</span>, 10(2):116–127, April
      2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [107]<span class='bibsp'>   </span></span><a id='XWeinkam:2012jf'></a>Patrick Weinkam, Jaume Pons, and Andrej Sali.  Structure-based model of allostery
      predicts  coupling  between  distant  sites.    <span class='ptmri7t-x-x-120'>Proceedings  of  the  National  Academy  of
      </span><span class='ptmri7t-x-x-120'>Sciences of the United States of America</span>, 109(13):4875–4880, March 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [108]<span class='bibsp'>   </span></span><a id='XLenaerts:2008kn'></a>Tom Lenaerts, Jesper Ferkinghoff-Borg, Francois Stricher, Luis Serrano, Joost W H
      Schymkowitz,  and  Frederic  Rousseau.    Quantifying  information  transfer  by  protein
      domains: analysis of the Fyn SH2 domain structure.  <span class='ptmri7t-x-x-120'>BMC structural biology</span>, 8(1):43,
      2008.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [109]<span class='bibsp'>   </span></span><a id='XDubay:2011ey'></a>Kateri H   Dubay,   Jacques P   Bothma,   and   Phillip L   Geissler.       Long-range
      intra-protein  communication  can  be  transmitted  by  correlated  side-chain  fluctuations
      alone. <span class='ptmri7t-x-x-120'>PLoS Computational Biology</span>, 7(9):e1002168, September 2011.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [110]<span class='bibsp'>   </span></span><a id='XFenwick:2014fc'></a>R Bryn  Fenwick,  Laura  Orellana,  Santi  Esteban-Martín,  Modesto  Orozco,  and
      Xavier Salvatella.  Correlated motions are a fundamental property of <span class='cmmi-12'>β</span>-sheets.  <span class='ptmri7t-x-x-120'>Nature
      </span><span class='ptmri7t-x-x-120'>communications</span>, 5:4070, June 2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [111]<span class='bibsp'>   </span></span><a id='XCui:2008cc'></a>Qiang  Cui  and  Martin  Karplus.    Allostery  and  cooperativity  revisited.    <span class='ptmri7t-x-x-120'>Protein
      </span><span class='ptmri7t-x-x-120'>Science</span>, 17(8):1295–1307, August 2008.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [112]<span class='bibsp'>   </span></span><a id='XElber:2011dj'></a>Ron Elber. Simulations of allosteric transitions. 21(2):167–172, April 2011.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [113]<span class='bibsp'>   </span></span><a id='XVanWart:2014dc'></a>Adam T Van Wart, Jacob Durrant, Lane Votapka, and Rommie E Amaro. Weighted
      Implementation  of  Suboptimal  Paths  (WISP):  An  Optimized  Algorithm  and  Tool
      for  Dynamical  Network  Analysis.    <span class='ptmri7t-x-x-120'>Journal  of  Chemical  Theory  and  Computation</span>,
      10(2):511–517, February 2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [114]<span class='bibsp'>   </span></span><a id='XMalmstrom:2015kp'></a>Robert D Malmstrom, Alexandr P Kornev, Susan S Taylor, and Rommie E Amaro.
      Allostery  through  the  computational  microscope:  cAMP  activation  of  a  canonical
      signalling domain. <span class='ptmri7t-x-x-120'>Nature communications</span>, 6:7588, July 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [115]<span class='bibsp'>   </span></span><a id='XChu:2005gh'></a>Jhih-Wei Chu and Gregory A Voth. Allostery of actin filaments: molecular dynamics
                                                                                      
                                                                                      
      simulations  and  coarse-grained  analysis.    <span class='ptmri7t-x-x-120'>Proceedings  of  the  National  Academy  of
      </span><span class='ptmri7t-x-x-120'>Sciences</span>, 102(37):13111–13116, September 2005.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [116]<span class='bibsp'>   </span></span><a id='XRavindranathan:2005ff'></a>Krishna Pratap
      Ravindranathan, Emilio Gallicchio, and Ronald M Levy. Conformational equilibria and
      free energy profiles for the allosteric transition of the ribose-binding protein. <span class='ptmri7t-x-x-120'>Journal of
      </span><span class='ptmri7t-x-x-120'>Molecular Biology</span>, 353(1):196–210, October 2005.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [117]<span class='bibsp'>   </span></span><a id='XWeinkam:2013fj'></a>Patrick  Weinkam,  Yao Chi  Chen,  Jaume  Pons,  and  Andrej  Sali.     Impact  of
      mutations on the allosteric conformational equilibrium.  <span class='ptmri7t-x-x-120'>Journal of Molecular Biology</span>,
      425(3):647–661, February 2013.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [118]<span class='bibsp'>   </span></span><a id='XLiu:2012em'></a>Ying Liu and Ivet Bahar.  Sequence evolution correlates with structural dynamics.
      <span class='ptmri7t-x-x-120'>Molecular biology and evolution</span>, 29(9):2253–2263, September 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [119]<span class='bibsp'>   </span></span><a id='XZhang2012'></a>Liqun Zhang, Sabine Bouguet-Bonnet, and Matthias Buck.   <span class='ptmri7t-x-x-120'>Combining NMR and
      </span><span class='ptmri7t-x-x-120'>Molecular Dynamics Studies for Insights into the Allostery of Small GTPase–Protein
      </span><span class='ptmri7t-x-x-120'>Interactions</span>, pages 235–259. Springer New York, New York, NY, 2012.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [120]<span class='bibsp'>   </span></span><a id='XKern:2003fv'></a>Dorothee  Kern  and  Erik RP  Zuiderweg.     The  role  of  dynamics  in  allosteric
      regulation. 13(6):748–757, December 2003.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [121]<span class='bibsp'>   </span></span><a id='XHilser:2007je'></a>Vincent J  Hilser  and  E Brad  Thompson.    Intrinsic  disorder  as  a  mechanism  to
      optimize  allosteric  coupling  in  proteins.    <span class='ptmri7t-x-x-120'>Proceedings  of  the  National  Academy  of
      </span><span class='ptmri7t-x-x-120'>Sciences</span>, 104(20):8311–8315, May 2007.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [122]<span class='bibsp'>   </span></span><a id='XBurger:2016gy'></a>Virginia M Burger, Diego O Nolasco, and Collin M Stultz. Expanding the Range of
      Protein Function at the Far End of the Order-Structure Continuum. <span class='ptmri7t-x-x-120'>Journal of Biological
      </span><span class='ptmri7t-x-x-120'>Chemistry</span>, 291(13):6706–6713, March 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [123]<span class='bibsp'>   </span></span><a id='XHaliloglu:2015dj'></a>Turkan  Haliloglu  and  Ivet  Bahar.    Adaptability  of  protein  structures  to  enable
      functional interactions and evolutionary implications. 35:17–23, December 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [124]<span class='bibsp'>   </span></span><a id='XHenzlerWildman:2007jb'></a>Katherine Henzler-Wildman and Dorothee Kern. Dynamic personalities of proteins.
      <span class='ptmri7t-x-x-120'>Nature</span>, 450(7172):964–972, December 2007.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [125]<span class='bibsp'>   </span></span><a id='XHilser:2012em'></a>Vincent J Hilser, James O Wrabl, and Hesam N Motlagh. Structural and energetic
      basis of allostery. <span class='ptmri7t-x-x-120'>Annual review of biophysics</span>, 41(1):585–609, 2012.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [126]<span class='bibsp'>   </span></span><a id='XJakob:2014cp'></a>Ursula  Jakob,  Richard  Kriwacki,  and  Vladimir N  Uversky.    Conditionally  and
      transiently disordered proteins: awakening cryptic disorder to regulate protein function.
      <span class='ptmri7t-x-x-120'>Chemical Reviews</span>, 114(13):6779–6805, July 2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [127]<span class='bibsp'>   </span></span><a id='XHarman:2001td'></a>James G Harman.  Allosteric regulation of the cAMP receptor protein.  <span class='ptmri7t-x-x-120'>Biochimica
      </span><span class='ptmri7t-x-x-120'>et Biophysica Acta (BBA) - Protein Structure and Molecular Enzymology</span>, 1547(1):1–17,
      May 2001.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [128]<span class='bibsp'>   </span></span><a id='XHeyduk:1989wb'></a>T Heyduk and J C Lee. Escherichia coli cAMP receptor protein: evidence for three
      protein conformational states with different promoter binding affinities.  <span class='ptmri7t-x-x-120'>Biochemistry</span>,
      28(17):6914–6924, August 1989.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [129]<span class='bibsp'>   </span></span><a id='XSchultz:1991uh'></a>S C  Schultz,  G C  Shields,  and  T A  Steitz.    Crystal  structure  of  a  CAP-DNA
      complex: the DNA is bent by 90 degrees. <span class='ptmri7t-x-x-120'>Science</span>, 253(5023):1001–1007, August 1991.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [130]<span class='bibsp'>   </span></span><a id='XSeok:2014cs'></a>Seung-Hyeon  Seok,  Hookang  Im,  Hyung-Sik  Won,  Min-Duk  Seo,  Yoo-Sup  Lee,
      Hye-Jin Yoon, Min-Jeong Cha, Jin-Young Park, and Bong-Jin Lee. Structures of inactive
      CRP  species  reveal  the  atomic  details  of  the  allosteric  transition  that  discriminates
      cyclic  nucleotide  second  messengers.   <span class='ptmri7t-x-x-120'>Acta  crystallographica.  Section  D,  Biological
      </span><span class='ptmri7t-x-x-120'>crystallography</span>, 70(Pt 6):1726–1742, June 2014.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [131]<span class='bibsp'>   </span></span><a id='XPopovych:2009iu'></a>Nataliya  Popovych,  Shiou-Ru  Tzeng,  Marco  Tonelli,  Richard H  Ebright,  and
      Charalampos G Kalodimos.  Structural basis for cAMP-mediated allosteric control of
      the catabolite activator protein. <span class='ptmri7t-x-x-120'>Proceedings of the National Academy of Sciences of the
      </span><span class='ptmri7t-x-x-120'>United States of America</span>, 106(17):6927–6932, April 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [132]<span class='bibsp'>   </span></span><a id='XTzeng:2012cu'></a>Shiou-Ru  Tzeng  and  Charalampos G  Kalodimos.    Protein  activity  regulation  by
      conformational entropy. <span class='ptmri7t-x-x-120'>Nature</span>, 488(7410):236–240, August 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [133]<span class='bibsp'>   </span></span><a id='XHilser:1998cj'></a>V J  Hilser,  D Dowdy,  T G  Oas,  and  E Freire.    The  structural  distribution  of
      cooperative interactions in proteins: analysis of the native state ensemble.  <span class='ptmri7t-x-x-120'>Proceedings
      </span><span class='ptmri7t-x-x-120'>of the National Academy of Sciences</span>, 95(17):9903–9908, August 1998.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [134]<span class='bibsp'>   </span></span><a id='XHilser:1996ft'></a>V J  Hilser  and  E Freire.   Structure-based  calculation  of  the  equilibrium  folding
      pathway of proteins. Correlation with hydrogen exchange protection factors. <span class='ptmri7t-x-x-120'>Journal of
      </span><span class='ptmri7t-x-x-120'>Molecular Biology</span>, 262(5):756–772, October 1996.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [135]<span class='bibsp'>   </span></span><a id='XMarcos:2011dr'></a>Enrique  Marcos,  Ramon  Crehuet,  and  Ivet  Bahar.    Changes  in  dynamics  upon
      oligomerization  regulate  substrate  binding  and  allostery  in  amino  acid  kinase  family
      members. <span class='ptmri7t-x-x-120'>PLoS Computational Biology</span>, 7(9):e1002201, September 2011.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [136]<span class='bibsp'>   </span></span><a id='XSuel:2003ca'></a>Gürol M  Süel,  Steve W  Lockless,  Mark A  Wall,  and  Rama  Ranganathan.
      Evolutionarily  conserved  networks  of  residues  mediate  allosteric  communication  in
      proteins. <span class='ptmri7t-x-x-120'>Nature structural biology</span>, 10(1):59–69, January 2003.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [137]<span class='bibsp'>   </span></span><a id='XEdiger:2000ed'></a>M D  Ediger.   Spatially  heterogeneous  dynamics  in  supercooled  liquids.   <span class='ptmri7t-x-x-120'>Annual
      </span><span class='ptmri7t-x-x-120'>review of physical chemistry</span>, 51(1):99–128, 2000.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [138]<span class='bibsp'>   </span></span><a id='XGlotzer:2000fy'></a>Sharon C  Glotzer.    Spatially  heterogeneous  dynamics  in  liquids:  insights  from
      simulation. <span class='ptmri7t-x-x-120'>Journal of Non-Crystalline Solids</span>, 274(1-3):342–355, September 2000.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [139]<span class='bibsp'>   </span></span><a id='XRichert:2002fa'></a>Ranko Richert.  Heterogeneous dynamics in liquids: fluctuations in space and time.
      <span class='ptmri7t-x-x-120'>Journal of Physics: Condensed Matter</span>, 14(23):R703–R738, June 2002.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [140]<span class='bibsp'>   </span></span><a id='XMerolle:2005kx'></a>Mauro Merolle, Juan P Garrahan, and David Chandler. Space-time thermodynamics
      of   the   glass   transition.        <span class='ptmri7t-x-x-120'>Proceedings   of   the   National   Academy   of   Sciences</span>,
      102(31):10837–10840, August 2005.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [141]<span class='bibsp'>   </span></span><a id='XHedges:2007fw'></a>Lester O   Hedges,   Lutz   Maibaum,   David   Chandler,   and   Juan P   Garrahan.
                                                                                      
                                                                                      
      Decoupling  of  exchange  and  persistence  times  in  atomistic  models  of  glass  formers.
      127(21):211101, December 2007.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [142]<span class='bibsp'>   </span></span><a id='XKeys:2011ut'></a>Aaron S Keys, Lester O Hedges, Juan P Garrahan, Sharon C Glotzer, and David
      Chandler.  Excitations Are Localized and Relaxation Is Hierarchical in Glass-Forming
      Liquids. <span class='ptmri7t-x-x-120'>Physical Review X</span>, 1(2):021013, November 2011.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [143]<span class='bibsp'>   </span></span><a id='XJorgensen:1983fl'></a>William L  Jorgensen,   Jayaraman   Chandrasekhar,   Jeffry D  Madura,   Roger W
      Impey, and Michael L Klein.  Comparison of simple potential functions for simulating
      liquid water. <span class='ptmri7t-x-x-120'>The Journal of Chemical Physics</span>, 79(2):926–935, July 1983.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [144]<span class='bibsp'>   </span></span><a id='XDeLano:2010wf'></a>W L DeLano.   <span class='ptmri7t-x-x-120'>The PyMOL Molecular Graphics System, Ver. 1.3</span>.   Schrödinger,
      2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [145]<span class='bibsp'>   </span></span><a id='XVanDerSpoel:2005hz'></a>David Van Der Spoel, Erik Lindahl, Berk Hess, Gerrit Groenhof, Alan E Mark, and
      Herman J C Berendsen.  GROMACS: fast, flexible, and free.  <span class='ptmri7t-x-x-120'>Journal of computational
      </span><span class='ptmri7t-x-x-120'>chemistry</span>, 26(16):1701–1718, December 2005.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [146]<span class='bibsp'>   </span></span><a id='XDuan:2003gt'></a>Yong Duan, Chun Wu, Shibasish Chowdhury, Mathew C Lee, Guoming Xiong, Wei
      Zhang, Rong Yang, Piotr Cieplak, Ray Luo, Taisung Lee, James Caldwell, Junmei Wang,
                                                                                      
                                                                                      
      and  Peter  Kollman.   A  point-charge  force  field  for  molecular  mechanics  simulations
      of  proteins  based  on  condensed-phase  quantum  mechanical  calculations.   <span class='ptmri7t-x-x-120'>Journal  of
      </span><span class='ptmri7t-x-x-120'>computational chemistry</span>, 24(16):1999–2012, December 2003.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [147]<span class='bibsp'>   </span></span><a id='XBowman:2015dh'></a>Gregory R  Bowman,  Eric R  Bolin,  Kathryn M  Hart,  Brendan C  Maguire,  and
      Susan Marqusee.   Discovery of multiple hidden allosteric sites by combining Markov
      state models and experiments.  <span class='ptmri7t-x-x-120'>Proceedings of the National Academy of Sciences of the
      </span><span class='ptmri7t-x-x-120'>United States of America</span>, 112(9):2734–2739, March 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [148]<span class='bibsp'>   </span></span><a id='XMcGibbon:2015fv'></a>Robert T McGibbon, Kyle A Beauchamp, Matthew P Harrigan, Christoph Klein,
      Jason M  Swails,  Carlos X  Hernández,  Christian R  Schwantes,  Lee-Ping  Wang,
      Thomas J Lane, and Vijay S Pande. MDTraj: A Modern Open Library for the Analysis
      of Molecular Dynamics Trajectories.  <span class='ptmri7t-x-x-120'>Biophysical Journal</span>, 109(8):1528–1532, October
      2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [149]<span class='bibsp'>   </span></span><a id='XHummer:2004kb'></a>Gerhard Hummer. From transition paths to transition states and rate coefficients. <span class='ptmri7t-x-x-120'>The
      </span><span class='ptmri7t-x-x-120'>Journal of Chemical Physics</span>, 120(2):516–523, January 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [150]<span class='bibsp'>   </span></span><a id='XBolhuis:2002ew'></a>Peter G  Bolhuis,  David  Chandler,  Christoph  Dellago,  and  Phillip L  Geissler.
      Transition  path  sampling:  throwing  ropes  over  rough  mountain  passes,  in  the  dark.
      <span class='ptmri7t-x-x-120'>Annual review of physical chemistry</span>, 53(1):291–318, 2002.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [151]<span class='bibsp'>   </span></span><a id='XBuchete:2008hb'></a>Nicolae-Viorel Buchete and Gerhard Hummer.  Coarse master equations for peptide
      folding dynamics. <span class='ptmri7t-x-x-120'>The Journal of Physical Chemistry B</span>, 112(19):6057–6069, May 2008.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [152]<span class='bibsp'>   </span></span><a id='XSchutte:2011gs'></a>Christof    Schütte,    Frank    Noé,    Jianfeng    Lu,    Marco    Sarich,    and    Eric
      Vanden-Eijnden.  Markov state models based on milestoning.  <span class='ptmri7t-x-x-120'>The Journal of Chemical
      </span><span class='ptmri7t-x-x-120'>Physics</span>, 134(20):204105, May 2011.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [153]<span class='bibsp'>   </span></span><a id='XJung:2005jk'></a>YounJoon Jung, Juan P Garrahan, and David Chandler.   Dynamical exchanges in
      facilitated models of supercooled liquids. 123(8):084509, August 2005.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [154]<span class='bibsp'>   </span></span><a id='XKass:2012bb'></a>Robert E  Kass  and  Adrian E  Raftery.   Bayes  Factors.   <span class='ptmri7t-x-x-120'>Journal  of  the  American
      </span><span class='ptmri7t-x-x-120'>Statistical Association</span>, February 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [155]<span class='bibsp'>   </span></span><a id='Xwitten2002data'></a>Ian H Witten and Eibe Frank.   Data mining: practical machine learning tools and
      techniques with java implementations. <span class='ptmri7t-x-x-120'>Acm Sigmod Record</span>, 31(1):76–77, 2002.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [156]<span class='bibsp'>   </span></span><a id='XFraser:2011jf'></a>James S   Fraser,   Henry   van den   Bedem,   Avi J   Samelson,   P Therese   Lang,
      James M Holton, Nathaniel Echols, and Tom Alber.  Accessing protein conformational
                                                                                      
                                                                                      
      ensembles using room-temperature X-ray crystallography.  <span class='ptmri7t-x-x-120'>Proceedings of the National
      </span><span class='ptmri7t-x-x-120'>Academy of Sciences of the United States of America</span>, 108(39):16247–16252, September
      2011.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [157]<span class='bibsp'>   </span></span><a id='XDai:2004ea'></a>Jiayin  Dai,  Shwu-Hwa  Lin,  Carly  Kemmis,  Anita J  Chin,  and  J Ching  Lee.
      Interplay   between   site-specific   mutations   and   cyclic   nucleotides   in   modulating
      DNA  recognition  by  Escherichia  coli  cyclic  AMP  receptor  protein.    <span class='ptmri7t-x-x-120'>Biochemistry</span>,
      43(28):8901–8910, July 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [158]<span class='bibsp'>   </span></span><a id='XAiba:1985ui'></a>H Aiba, T Nakamura, H Mitani, and H Mori.   Mutations that alter the allosteric
      nature   of   cAMP   receptor   protein   of   Escherichia   coli.       <span class='ptmri7t-x-x-120'>The   EMBO   Journal</span>,
      4(12):3329–3332, December 1985.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [159]<span class='bibsp'>   </span></span><a id='XKurplus:1983ts'></a>M Kurplus and J A McCammon.   Dynamics of proteins: elements and function.
      <span class='ptmri7t-x-x-120'>Annual review of biochemistry</span>, 1983.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [160]<span class='bibsp'>   </span></span><a id='Xbrooks2006advances'></a>Charles L  Brooks,  Martin  Karplus,  and  B Montgomery  Pettitt.     <span class='ptmri7t-x-x-120'>Advances  in
      </span><span class='ptmri7t-x-x-120'>chemical physics, volume 71: Proteins: A theoretical perspective of dynamics, structure,
      </span><span class='ptmri7t-x-x-120'>and thermodynamics</span>. Wiley-Blackwell, 2006.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [161]<span class='bibsp'>   </span></span><a id='XDepristo:2004tb'></a>Mark A  Depristo,  Paul  I W  de Bakker,  and  Tom L  Blundell.     Heterogeneity
      and  Inaccuracy  in  Protein  Structures  Solved  by  X-Ray  Crystallography.    <span class='ptmri7t-x-x-120'>Structure</span>,
      12(5):831–838, May 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [162]<span class='bibsp'>   </span></span><a id='XWand:1996hr'></a>A J Wand, J L Urbauer, R P McEvoy, and R J Bieber. Internal dynamics of human
      ubiquitin revealed by 13C-relaxation studies of randomly fractionally labeled protein.
      <span class='ptmri7t-x-x-120'>Biochemistry</span>, 35(19):6116–6125, May 1996.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [163]<span class='bibsp'>   </span></span><a id='XLindorffLarsen:2005bi'></a>Kresten Lindorff-Larsen, Robert B Best, Mark A Depristo, Christopher M Dobson,
      and  Michele  Vendruscolo.      Simultaneous  determination  of  protein  structure  and
      dynamics. <span class='ptmri7t-x-x-120'>Nature</span>, 433(7022):128–132, January 2005.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [164]<span class='bibsp'>   </span></span><a id='XIgumenova:2006hw'></a>Tatyana I                     Igumenova,                     Kendra King                     Frederick,
      and A Joshua Wand.  Characterization of the fast dynamics of protein amino acid side
      chains using NMR relaxation in solution.  <span class='ptmri7t-x-x-120'>Chemical Reviews</span>, 106(5):1672–1699, May
      2006.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [165]<span class='bibsp'>   </span></span><a id='XDubay:2009gv'></a>Kateri H Dubay and Phillip L Geissler.   Calculation of proteins’ total side-chain
      torsional entropy and its influence on protein-ligand interactions.  <span class='ptmri7t-x-x-120'>Journal of Molecular
      </span><span class='ptmri7t-x-x-120'>Biology</span>, 391(2):484–497, August 2009.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [166]<span class='bibsp'>   </span></span><a id='Xchaudhury_pyrosetta:_2010'></a>S. Chaudhury,  S. Lyskov,  and  J. J.  Gray.     PyRosetta:  a  script-based  interface
      for  implementing  molecular  modeling  algorithms  using  Rosetta.     <span class='ptmri7t-x-x-120'>Bioinformatics</span>,
      26(5):689–691, March 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [167]<span class='bibsp'>   </span></span><a id='Xshapovalov_smoothed_2011'></a>Maxim V. Shapovalov and Roland L. Dunbrack.  A smoothed backbone-dependent
      rotamer  library  for  proteins  derived  from  adaptive  kernel  density  estimates  and
      regressions. <span class='ptmri7t-x-x-120'>Structure</span>, 19(6):844–858, June 2011.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [168]<span class='bibsp'>   </span></span><a id='XOldham:2006ui'></a>W M  Oldham  and  H E  Hamm.   Structural  basis  of  function  in  heterotrimeric  G
      proteins. <span class='ptmri7t-x-x-120'>Quarterly reviews of biophysics</span>, 2006.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [169]<span class='bibsp'>   </span></span><a id='XOldham:2008cr'></a>William M  Oldham  and  Heidi E  Hamm.     Heterotrimeric  G  protein  activation
      by G-protein-coupled receptors.   <span class='ptmri7t-x-x-120'>Nature reviews. Molecular cell biology</span>, 9(1):60–71,
      January 2008.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [170]<span class='bibsp'>   </span></span><a id='XJohnston:2007ir'></a>Christopher A  Johnston  and  David P  Siderovski.    Receptor-mediated  activation
      of  heterotrimeric  G-proteins:  current  structural  insights.    <span class='ptmri7t-x-x-120'>Molecular  pharmacology</span>,
      72(2):219–230, August 2007.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [171]<span class='bibsp'>   </span></span><a id='XFlock:2015dj'></a>Tilman  Flock,  Charles  N J  Ravarani,  Dawei  Sun,  A J  Venkatakrishnan,  Melis
      Kayikci,  Christopher G  Tate,  Dmitry B  Veprintsev,  and  M Madan  Babu.   Universal
      allosteric mechanism for G[alpha] activation by GPCRs.  <span class='ptmri7t-x-x-120'>Nature</span>, 524(7564):173–179,
      August 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [172]<span class='bibsp'>   </span></span><a id='XSunahara:1997tg'></a>R K Sunahara, J J Tesmer, A G Gilman, and S R Sprang.  Crystal structure of the
      adenylyl cyclase activator Gsalpha. <span class='ptmri7t-x-x-120'>Science</span>, 278(5345):1943–1947, December 1997.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [173]<span class='bibsp'>   </span></span><a id='XLeipe:2002ky'></a>Detlef D  Leipe,  Yuri I  Wolf,  Eugene V  Koonin,  and  L Aravind.   Classification
      and evolution of P-loop GTPases and related ATPases.  <span class='ptmri7t-x-x-120'>Journal of Molecular Biology</span>,
      317(1):41–72, March 2002.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [174]<span class='bibsp'>   </span></span><a id='XWestfield:2011ef'></a>Gerwin H  Westfield,  Søren  G F  Rasmussen,  Min  Su,  Somnath  Dutta,  Brian T
      DeVree, Ka Young Chung, Diane Calinski, Gisselle Velez-Ruiz, Austin N Oleskie, Els
      Pardon, Pil Seok Chae, Tong Liu, Sheng Li, Virgil L Woods, Jan Steyaert, Brian K
      Kobilka,  Roger K  Sunahara,  and  Georgios  Skiniotis.   Structural  flexibility  of  the  G
      alpha s alpha-helical domain in the beta2-adrenoceptor Gs complex. <span class='ptmri7t-x-x-120'>Proceedings of the
      </span><span class='ptmri7t-x-x-120'>National Academy of Sciences of the United States of America</span>, 108(38):16086–16091,
      September 2011.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [175]<span class='bibsp'>   </span></span><a id='XLambright:1994gd'></a>D G Lambright, J P Noel, H E Hamm, and P B Sigler.   Structural determinants
                                                                                      
                                                                                      
      for   activation   of   the   alpha-subunit   of   a   heterotrimeric   G   protein.        <span class='ptmri7t-x-x-120'>Nature</span>,
      369(6482):621–628, June 1994.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [176]<span class='bibsp'>   </span></span><a id='XLiang:2017cz'></a>Yi-Lynn Liang, Maryam Khoshouei, Mazdak Radjainia, Yan Zhang, Alisa Glukhova,
      Jeffrey  Tarrasch,  David M  Thal,  Sebastian  G B  Furness,  George  Christopoulos,
      Thomas  Coudrat,  Radostin  Danev,  Wolfgang  Baumeister,  Laurence J  Miller,  Arthur
      Christopoulos, Brian K Kobilka, Denise Wootten, Georgios Skiniotis, and Patrick M
      Sexton.  Phase-plate cryo-EM structure of a class B GPCR–G-protein complex.  <span class='ptmri7t-x-x-120'>Nature</span>,
      159:986, April 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [177]<span class='bibsp'>   </span></span><a id='XHilger:2018en'></a>Daniel Hilger, Matthieu Masureel, and Brian K Kobilka.   Structure and dynamics
      of GPCR signaling complexes.   <span class='ptmri7t-x-x-120'>Nature Structural &amp; Molecular Biology</span>, 25(1):4–12,
      January 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [178]<span class='bibsp'>   </span></span><a id='XFurness:2016it'></a>Sebastian   George Barton   Furness,   Yi-Lynn   Liang,   Cameron James   Nowell,
      Michelle Louise  Halls,  Peter John  Wookey,  Emma  Dal Maso,  Asuka  Inoue,  Arthur
      Christopoulos,  Denise  Wootten,  and  Patrick Michael  Sexton.      Ligand-Dependent
      Modulation of G Protein Conformation Alters Drug Efficacy. <span class='ptmri7t-x-x-120'>Cell</span>, 167(3):739–749.e11,
      October 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [179]<span class='bibsp'>   </span></span><a id='XToyama:2017gy'></a>Yuki Toyama, Hanaho Kano, Yoko Mase, Mariko Yokogawa, Masanori Osawa, and
      Ichio Shimada. Dynamic regulation of GDP binding to G proteins revealed by magnetic
                                                                                      
                                                                                      
      field-dependent NMR relaxation analyses.  <span class='ptmri7t-x-x-120'>Nature communications</span>, 8:14523, February
      2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [180]<span class='bibsp'>   </span></span><a id='XSun:2015gj'></a>Dawei  Sun,  Tilman  Flock,  Xavier  Deupi,  Shoji  Maeda,  Milos  Matkovic,  Sandro
      Mendieta, Daniel Mayer, Roger J P Dawson, Gebhard F X Schertler, M Madan Babu,
      and  Dmitry B  Veprintsev.     Probing  G<span class='cmmi-12'>α</span>i1  protein  activation  at  single-amino  acid
      resolution. <span class='ptmri7t-x-x-120'>Nature Structural &amp; Molecular Biology</span>, 22(9):686–694, September 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [181]<span class='bibsp'>   </span></span><a id='XGoricanec:2016hl'></a>David Goricanec, Ralf Stehle, Pascal Egloff, Simina Grigoriu, Andreas Plückthun,
      Gerhard Wagner, and Franz Hagn.  Conformational dynamics of a G-protein <span class='cmmi-12'>α </span>subunit
      is  tightly  regulated  by  nucleotide  binding.   <span class='ptmri7t-x-x-120'>Proceedings  of  the  National  Academy  of
      </span><span class='ptmri7t-x-x-120'>Sciences of the United States of America</span>, 113(26):E3629–38, June 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [182]<span class='bibsp'>   </span></span><a id='XLindorffLarsen:2011gl'></a>Kresten  Lindorff-Larsen,  Stefano  Piana,  Ron O  Dror,  and  David E  Shaw.   How
      fast-folding proteins fold. <span class='ptmri7t-x-x-120'>Science</span>, 334(6055):517–520, October 2011.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [183]<span class='bibsp'>   </span></span><a id='XBuch:2011bm'></a>Ignasi  Buch,  Toni  Giorgino,  and  Gianni  De Fabritiis.    Complete  reconstruction
      of   an   enzyme-inhibitor   binding   process   by   molecular   dynamics   simulations.
      <span class='ptmri7t-x-x-120'>Proceedings  of  the  National  Academy  of  Sciences  of  the  United  States  of  America</span>,
      108(25):10184–10189, June 2011.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [184]<span class='bibsp'>   </span></span><a id='XPlattner:2015br'></a>Nuria  Plattner  and  Frank  Noé.    Protein  conformational  plasticity  and  complex
      ligand-binding kinetics explored by atomistic simulations and Markov models.  <span class='ptmri7t-x-x-120'>Nature
      </span><span class='ptmri7t-x-x-120'>communications</span>, 6:7653, July 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [185]<span class='bibsp'>   </span></span><a id='XTiwary:2015hw'></a>Pratyush Tiwary, Vittorio Limongelli, Matteo Salvalaglio, and Michele Parrinello.
      Kinetics of protein-ligand unbinding: Predicting pathways, rates, and rate-limiting steps.
      <span class='ptmri7t-x-x-120'>Proceedings  of  the  National  Academy  of  Sciences  of  the  United  States  of  America</span>,
      112(5):E386–91, February 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [186]<span class='bibsp'>   </span></span><a id='XPlattner:2017fs'></a>Nuria  Plattner,  Stefan  Doerr,  Gianni  De Fabritiis,  and  Frank  Noé.    Complete
      protein–protein  association  kinetics  in  atomic  detail  revealed  by  molecular  dynamics
      simulations and Markov modelling. <span class='ptmri7t-x-x-120'>Nature chemistry</span>, June 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [187]<span class='bibsp'>   </span></span><a id='XZhou:2017bh'></a>Guangfeng  Zhou,  George A  Pantelopulos,  Sudipto  Mukherjee,  and  Vincent A
      Voelz.   Bridging  Microscopic  and  Macroscopic  Mechanisms  of  p53-MDM2  Binding
      with Kinetic Network Models. <span class='ptmri7t-x-x-120'>Biophysical Journal</span>, 113(4):785–793, August 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [188]<span class='bibsp'>   </span></span><a id='XDror:2015bg'></a>Ron O  Dror,  Thomas J  Mildorf,  Daniel  Hilger,  Aashish  Manglik,  David W
      Borhani,  Daniel H  Arlow,  Ansgar  Philippsen,  Nicolas  Villanueva,  Zhongyu  Yang,
      Michael T  Lerch,  Wayne L  Hubbell,  Brian K  Kobilka,  Roger K  Sunahara,  and
                                                                                      
                                                                                      
      David E Shaw.  Structural basis for nucleotide exchange in heterotrimeric G proteins.
      <span class='ptmri7t-x-x-120'>Science</span>, 348(6241):1361–1365, June 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [189]<span class='bibsp'>   </span></span><a id='XYao:2016kw'></a>Xin-Qiu Yao, Rabia U Malik, Nicholas W Griggs, Lars Skjærven, John R Traynor,
      Sivaraj  Sivaramakrishnan,  and  Barry J  Grant.     Dynamic  Coupling  and  Allosteric
      Networks  in  the  <span class='cmmi-12'>α </span>Subunit  of  Heterotrimeric  G  Proteins.     <span class='ptmri7t-x-x-120'>Journal  of  Biological
      </span><span class='ptmri7t-x-x-120'>Chemistry</span>, 291(9):4742–4753, February 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [190]<span class='bibsp'>   </span></span><a id='XChidiac:1999jh'></a>Peter Chidiac, Vladislav S Markin, and Elliott M Ross.  Kinetic control of guanine
      nucleotide binding to soluble G<span class='cmmi-12'>α</span>q. <span class='ptmri7t-x-x-120'>Biochemical pharmacology</span>, 58(1):39–48, July 1999.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [191]<span class='bibsp'>   </span></span><a id='XRoss:2008jm'></a>Elliott M Ross.  Coordinating speed and amplitude in G-protein signaling.  <span class='ptmri7t-x-x-120'>Current
      </span><span class='ptmri7t-x-x-120'>biology : CB</span>, 18(17):R777–R783, September 2008.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [192]<span class='bibsp'>   </span></span><a id='XMukhopadhyay:1999dp'></a>S Mukhopadhyay  and  E M  Ross.    Rapid  GTP  binding  and  hydrolysis  by  G(q)
      promoted  by  receptor  and  GTPase-activating  proteins.    <span class='ptmri7t-x-x-120'>Proceedings  of  the  National
      </span><span class='ptmri7t-x-x-120'>Academy of Sciences</span>, 96(17):9539–9544, August 1999.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [193]<span class='bibsp'>   </span></span><a id='XVanRaamsdonk:2009kv'></a>Catherine D Van Raamsdonk, Vladimir Bezrookove, Gary Green, Jürgen Bauer,
      Lona Gaugler, Joan M O’Brien, Elizabeth M Simpson, Gregory S Barsh, and Boris C
                                                                                      
                                                                                      
      Bastian.   Frequent  somatic  mutations  of  GNAQ  in  uveal  melanoma  and  blue  naevi.
      <span class='ptmri7t-x-x-120'>Nature</span>, 457(7229):599–602, January 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [194]<span class='bibsp'>   </span></span><a id='XVanRaamsdonk:2010iy'></a>Catherine D  Van Raamsdonk,  Klaus G  Griewank,  Michelle B  Crosby,  Maria C
      Garrido,  Swapna  Vemula,  Thomas  Wiesner,  Anna C  Obenauf,  Werner  Wackernagel,
      Gary  Green,  Nancy  Bouvier,  M Mert  Sozen,  Gail  Baimukanova,  Ritu  Roy,  Adriana
      Heguy, Igor Dolgalev, Raya Khanin, Klaus Busam, Michael R Speicher, Joan O’Brien,
      and  Boris C  Bastian.   Mutations  in  GNA11  in  uveal  melanoma.   <span class='ptmri7t-x-x-120'>The  New  England
      </span><span class='ptmri7t-x-x-120'>journal of medicine</span>, 363(23):2191–2199, December 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [195]<span class='bibsp'>   </span></span><a id='XLaio:2002ft'></a>Alessandro Laio and Michele Parrinello. Escaping free-energy minima. <span class='ptmri7t-x-x-120'>Proceedings
      </span><span class='ptmri7t-x-x-120'>of the National Academy of Sciences</span>, 99(20):12562–12566, October 2002.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [196]<span class='bibsp'>   </span></span><a id='XBowman:2013uj'></a>Gregory R Bowman, V S Pande, and F Noé.   <span class='ptmri7t-x-x-120'>An introduction to Markov state
      </span><span class='ptmri7t-x-x-120'>models and their application to long timescale molecular simulation</span>.  Springer Science
      &amp; Business Media, 1 edition, 2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [197]<span class='bibsp'>   </span></span><a id='XFraser:2009gv'></a>James S Fraser, Michael W Clarkson, Sheena C Degnan, Renske Erion, Dorothee
      Kern, and Tom Alber.  Hidden alternative structures of proline isomerase essential for
      catalysis. <span class='ptmri7t-x-x-120'>Nature</span>, 462(7273):669–673, December 2009.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [198]<span class='bibsp'>   </span></span><a id='XCarpenter:2016gr'></a>Byron   Carpenter,   Rony   Nehmé,   Tony   Warne,   Andrew   G W   Leslie,   and
      Christopher G Tate.  Structure of the adenosine A¡sub¿2A¡/sub¿ receptor bound to an
      engineered G protein. <span class='ptmri7t-x-x-120'>Nature</span>, 536(7614):104–107, August 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [199]<span class='bibsp'>   </span></span><a id='XLambright:1996en'></a>D G  Lambright,  J Sondek,  A Bohm,  N P  Skiba,  H E  Hamm,  and  P B  Sigler.
      The 2.0 A crystal structure of a heterotrimeric G protein.  <span class='ptmri7t-x-x-120'>Nature</span>, 379(6563):311–319,
      January 1996.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [200]<span class='bibsp'>   </span></span><a id='XNoel:1993dc'></a>J P   Noel,   H E   Hamm,   and   P B   Sigler.       The   2.2   A   crystal   structure   of
      transducin-alpha   complexed   with   GTP   gamma   S.      <span class='ptmri7t-x-x-120'>Nature</span>,   366(6456):654–663,
      December 1993.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [201]<span class='bibsp'>   </span></span><a id='XDenker:1992uj'></a>B M Denker, C J Schmidt, and E J Neer.  Promotion of the GTP-liganded state of
      the Go alpha protein by deletion of the C terminus.  <span class='ptmri7t-x-x-120'>Journal of Biological Chemistry</span>,
      267(14):9998–10002, May 1992.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [202]<span class='bibsp'>   </span></span><a id='XEthanPMarin:2002fr'></a>Ethan  P  Marin,  A Gopala  Krishna,  ,  and  Thomas  P  Sakmar.     Disruption  of
      the  <span class='cmmi-12'>α</span>5  Helix  of  Transducin  Impairs  Rhodopsin-Catalyzed  Nucleotide  Exchange<span class='ptmr8c-x-x-120'>†</span>.
      <span class='ptmri7t-x-x-120'>Biochemistry</span>, 41(22):6988–6994, May 2002.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [203]<span class='bibsp'>   </span></span><a id='XNishimura:2010gt'></a>Akiyuki  Nishimura,  Ken  Kitano,  Jun  Takasaki,  Masatoshi  Taniguchi,  Norikazu
      Mizuno,  Kenji  Tago,  Toshio  Hakoshima,  and  Hiroshi  Itoh.    Structural  basis  for  the
      specific inhibition of heterotrimeric Gq protein by a small molecule. <span class='ptmri7t-x-x-120'>Proceedings of the
      </span><span class='ptmri7t-x-x-120'>National Academy of Sciences of the United States of America</span>, 107(31):13666–13671,
      August 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [204]<span class='bibsp'>   </span></span><a id='XHuang:2009bx'></a>Xuhui Huang, Gregory R Bowman, Sergio Bacallado, and Vijay S Pande.  Rapid
      equilibrium sampling initiated from nonequilibrium data.  <span class='ptmri7t-x-x-120'>Proceedings of the National
      </span><span class='ptmri7t-x-x-120'>Academy of Sciences of the United States of America</span>, 106(47):19765–19769, November
      2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [205]<span class='bibsp'>   </span></span><a id='XDama:2014ef'></a>James F   Dama,   Michele   Parrinello,   and   Gregory A   Voth.       Well-tempered
      metadynamics converges asymptotically.  <span class='ptmri7t-x-x-120'>Physical review letters</span>, 112(24):240602, June
      2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [206]<span class='bibsp'>   </span></span><a id='XBiswas:2018hc'></a>Mithun  Biswas,  Benjamin  Lickert,  and  Gerhard  Stock.   Metadynamics  Enhanced
      Markov Modeling of Protein Dynamics.   <span class='ptmri7t-x-x-120'>The Journal of Physical Chemistry B</span>, page
      acs.jpcb.7b11800, January 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [207]<span class='bibsp'>   </span></span><a id='XZhang:2016ex'></a>Lu Zhang,  Fátima  Pardo-Avila,  Ilona Christy  Unarta,  Peter  Pak-Hang  Cheung,
      Guo  Wang,  Dong  Wang,  and  Xuhui  Huang.      Elucidation  of  the  Dynamics  of
                                                                                      
                                                                                      
      Transcription  Elongation  by  RNA  Polymerase  II  using  Kinetic  Network  Models.
      <span class='ptmri7t-x-x-120'>Accounts of chemical research</span>, 49(4):687–694, April 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [208]<span class='bibsp'>   </span></span><a id='XNoe:2009en'></a>Frank Noé, Christof Schütte, Eric Vanden-Eijnden, Lothar Reich, and Thomas R
      Weikl.      Constructing  the  equilibrium  ensemble  of  folding  pathways  from  short
      off-equilibrium simulations.   <span class='ptmri7t-x-x-120'>Proceedings of the National Academy of Sciences of the
      </span><span class='ptmri7t-x-x-120'>United States of America</span>, 106(45):19011–19016, November 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [209]<span class='bibsp'>   </span></span><a id='XWeinan:2006vr'></a>E Weinan and E Vanden-Eijnden.  Towards a Theory of Transition Paths.  <span class='ptmri7t-x-x-120'>Journal
      </span><span class='ptmri7t-x-x-120'>of statistical physics</span>, 2006.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [210]<span class='bibsp'>   </span></span><a id='XVanEps:2011hb'></a>Ned Van Eps, Anita M Preininger, Nathan Alexander, Ali I Kaya, Scott Meier, Jens
      Meiler,  Heidi E  Hamm,  and  Wayne L  Hubbell.   Interaction  of  a  G  protein  with  an
      activated receptor opens the interdomain interface in the alpha subunit.  <span class='ptmri7t-x-x-120'>Proceedings of
      </span><span class='ptmri7t-x-x-120'>the National Academy of Sciences of the United States of America</span>, 108(23):9420–9424,
      June 2011.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [211]<span class='bibsp'>   </span></span><a id='XChoe:2011cb'></a>Hui-Woog Choe, Yong Ju Kim, Jung Hee Park, Takefumi Morizumi, Emil F Pai,
      Norbert Krauss, Klaus Peter Hofmann, Patrick Scheerer, and Oliver P Ernst.  Crystal
      structure of metarhodopsin II. <span class='ptmri7t-x-x-120'>Nature</span>, 471(7340):651–655, March 2011.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [212]<span class='bibsp'>   </span></span><a id='XZhang:2017gu'></a>Yan Zhang, Bingfa Sun, Dan Feng, Hongli Hu, Matthew Chu, Qianhui Qu, Jeffrey T
      Tarrasch,  Shane  Li,  Tong  Sun Kobilka,  Brian K  Kobilka,  and  Georgios  Skiniotis.
      Cryo-EM structure of the activated GLP-1 receptor in complex with a G protein. <span class='ptmri7t-x-x-120'>Nature</span>,
      1335:29, May 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [213]<span class='bibsp'>   </span></span><a id='XOldham:2006kn'></a>William M  Oldham,  Ned  Van Eps,  Anita M  Preininger,  Wayne L  Hubbell,  and
      Heidi E Hamm.   Mechanism of the receptor-catalyzed activation of heterotrimeric G
      proteins. <span class='ptmri7t-x-x-120'>Nature Structural &amp; Molecular Biology</span>, 13(9):772–777, September 2006.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [214]<span class='bibsp'>   </span></span><a id='XIiri:1994kw'></a>T Iiri, P Herzmark, J M Nakamoto, C van Dop, and H R Bourne.   Rapid GDP
      release  from  Gs  alpha  in  patients  with  gain  and  loss  of  endocrine  function.   <span class='ptmri7t-x-x-120'>Nature</span>,
      371(6493):164–168, September 1994.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [215]<span class='bibsp'>   </span></span><a id='XPosner:1998gj'></a>B A Posner, M B Mixon, M A Wall, S R Sprang, and A G Gilman.  The A326S
      mutant  of  Gialpha1  as  an  approximation  of  the  receptor-bound  state.     <span class='ptmri7t-x-x-120'>Journal  of
      </span><span class='ptmri7t-x-x-120'>Biological Chemistry</span>, 273(34):21752–21758, August 1998.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [216]<span class='bibsp'>   </span></span><a id='XThomas:1993wv'></a>T C  Thomas,  C J  Schmidt,  and  E J  Neer.   G-protein  alpha  o  subunit:  mutation
      of  conserved  cysteines  identifies  a  subunit  contact  surface  and  alters  GDP  affinity.
      <span class='ptmri7t-x-x-120'>Proceedings  of  the  National  Academy  of  Sciences</span>,  90(21):10295–10299,  November
      1993.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [217]<span class='bibsp'>   </span></span><a id='XFlock:2017bm'></a>Tilman  Flock,  Alexander S  Hauser,  Nadia  Lund,  David E  Gloriam,  Santhanam
      Balaji,  and  M Madan  Babu.    Selectivity  determinants  of  GPCR-G-protein  binding.
      <span class='ptmri7t-x-x-120'>Nature</span>, 88:263, May 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [218]<span class='bibsp'>   </span></span><a id='XHatley:2003co'></a>Mark E   Hatley,   Steve W   Lockless,   Scott K   Gibson,   Alfred G   Gilman,   and
      Rama  Ranganathan.   Allosteric  determinants  in  guanine  nucleotide-binding  proteins.
      <span class='ptmri7t-x-x-120'>Proceedings  of  the  National  Academy  of  Sciences</span>,  100(24):14445–14450,  November
      2003.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [219]<span class='bibsp'>   </span></span><a id='XHerrmann:2004gp'></a>Rolf   Herrmann,   Martin   Heck,   Petra   Henklein,   Peter   Henklein,   P Henklein,
      Christiane Kleuss, Klaus Peter Hofmann, and Oliver P Ernst. Sequence of interactions
      in receptor-G protein coupling. <span class='ptmri7t-x-x-120'>Journal of Biological Chemistry</span>, 279(23):24283–24290,
      June 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [220]<span class='bibsp'>   </span></span><a id='XPreininger:2013if'></a>Anita M Preininger, Jens Meiler, and Heidi E Hamm.   Conformational flexibility
      and structural dynamics in GPCR-mediated G protein activation: a perspective. <span class='ptmri7t-x-x-120'>Journal
      </span><span class='ptmri7t-x-x-120'>of Molecular Biology</span>, 425(13):2288–2298, July 2013.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [221]<span class='bibsp'>   </span></span><a id='XZhang:2000cy'></a>B Zhang,  Y Zhang,  Z Wang,  and  Y Zheng.   The  role  of  Mg2+  cofactor  in  the
                                                                                      
                                                                                      
      guanine nucleotide exchange and GTP hydrolysis reactions of Rho family GTP-binding
      proteins. <span class='ptmri7t-x-x-120'>Journal of Biological Chemistry</span>, 275(33):25299–25307, August 2000.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [222]<span class='bibsp'>   </span></span><a id='XAbraham:2015gj'></a>Mark James Abraham, Teemu Murtola, Roland Schulz, Szilárd Páll, Jeremy C
      Smith,  Berk  Hess,  and  Erik  Lindahl.     GROMACS:  High  performance  molecular
      simulations through multi-level parallelism from laptops to supercomputers. <span class='ptmri7t-x-x-120'>SoftwareX</span>,
      1-2:19–25, September 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [223]<span class='bibsp'>   </span></span><a id='XMeagher:2003be'></a>Kristin L  Meagher,  Luke T  Redman,  and  Heather A  Carlson.     Development
      of  polyphosphate  parameters  for  use  with  the  AMBER  force  field.     <span class='ptmri7t-x-x-120'>Journal  of
      </span><span class='ptmri7t-x-x-120'>computational chemistry</span>, 24(9):1016–1025, July 2003.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [224]<span class='bibsp'>   </span></span><a id='XBussi:2007cs'></a>Giovanni  Bussi,  Davide  Donadio,  and  Michele  Parrinello.    Canonical  sampling
      through velocity rescaling.  <span class='ptmri7t-x-x-120'>The Journal of Chemical Physics</span>, 126(1):014101, January
      2007.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [225]<span class='bibsp'>   </span></span><a id='XKolafa:2006jh'></a>Jiri Kolafa and John W Perram.  Cutoff Errors in the Ewald Summation Formulae
      for Point Charge Systems. <span class='ptmri7t-x-x-120'>Molecular Simulation</span>, 9(5):351–368, October 2006.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [226]<span class='bibsp'>   </span></span><a id='XHess:2008fl'></a>Berk Hess. P-LINCS: A Parallel Linear Constraint Solver for Molecular Simulation.
      <span class='ptmri7t-x-x-120'>Journal of Chemical Theory and Computation</span>, 4(1):116–122, January 2008.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [227]<span class='bibsp'>   </span></span><a id='XParrinello:1981it'></a>M Parrinello  and  A Rahman.   Polymorphic  transitions  in  single  crystals:  A  new
      molecular dynamics method. <span class='ptmri7t-x-x-120'>Journal of Applied physics</span>, 52(12):7182–7190, December
      1981.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [228]<span class='bibsp'>   </span></span><a id='XFeenstra:1999ue'></a>K A  Feenstra,  B Hess,  and  HJC  Berendsen.      Improving  efficiency  of  large
      timescale molecular dynamics simulations of hydrogen-rich systems.  <span class='ptmri7t-x-x-120'>J Comput Chem</span>,
      20(8):786–798, June 1999.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [229]<span class='bibsp'>   </span></span><a id='XTribello:2014eb'></a>Gareth A  Tribello,  Massimiliano  Bonomi,  Davide  Branduardi,  Carlo  Camilloni,
      and  Giovanni  Bussi.   PLUMED 2:  New  feathers  for  an  old  bird.   <span class='ptmri7t-x-x-120'>Computer  Physics
      </span><span class='ptmri7t-x-x-120'>Communications</span>, 185(2):604–613, February 2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [230]<span class='bibsp'>   </span></span><a id='XBeauchamp:2011he'></a>Kyle A   Beauchamp,   Gregory R   Bowman,   Thomas J   Lane,   Lutz   Maibaum,
      Imran S  Haque,  and  Vijay S  Pande.      MSMBuilder2:  Modeling  Conformational
      Dynamics  at  the  Picosecond  to  Millisecond  Scale.   <span class='ptmri7t-x-x-120'>Journal  of  Chemical  Theory  and
      </span><span class='ptmri7t-x-x-120'>Computation</span>, 7(10):3412–3419, October 2011.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [231]<span class='bibsp'>   </span></span><a id='XMaragliano:2006kg'></a>Luca Maragliano, Alexander Fischer, Eric Vanden-Eijnden, and Giovanni Ciccotti.
      String  method  in  collective  variables:  minimum  free  energy  paths  and  isocommittor
      surfaces. <span class='ptmri7t-x-x-120'>The Journal of chemical physics</span>, 125(2):24106, July 2006.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [232]<span class='bibsp'>   </span></span><a id='XMaragliano:2007bz'></a>Luca Maragliano and Eric Vanden-Eijnden.  On-the-fly string method for minimum
      free energy paths calculation.  <span class='ptmri7t-x-x-120'>Chemical physics letters</span>, 446(1-3):182–190, September
      2007.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [233]<span class='bibsp'>   </span></span><a id='XMaragliano:2014jj'></a>Luca Maragliano, Benoît Roux, and Eric Vanden-Eijnden.  Comparison between
      Mean Forces and Swarms-of-Trajectories String Methods.  <span class='ptmri7t-x-x-120'>Journal of Chemical Theory
      </span><span class='ptmri7t-x-x-120'>and Computation</span>, 10(2):524–533, February 2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [234]<span class='bibsp'>   </span></span><a id='XBowman:2009cb'></a>Gregory R  Bowman,  Xuhui  Huang,  and  Vijay S  Pande.      Using  generalized
      ensemble  simulations  and  Markov  state  models  to  identify  conformational  states.
      <span class='ptmri7t-x-x-120'>Methods (San Diego, Calif.)</span>, 49(2):197–201, October 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [235]<span class='bibsp'>   </span></span><a id='XSheong:2015jd'></a>Fu Kit  Sheong,  Daniel-Adriano  Silva,  Luming  Meng,  Yutong  Zhao,  and  Xuhui
      Huang.     Automatic  state  partitioning  for  multibody  systems  (APM):  an  efficient
      algorithm for constructing Markov state models to elucidate conformational dynamics
      of  multibody  systems.   <span class='ptmri7t-x-x-120'>Journal  of  Chemical  Theory  and  Computation</span>,  11(1):17–27,
      January 2015.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [236]<span class='bibsp'>   </span></span><a id='XSwope:2004gx'></a>William C  Swope,  Jed W  Pitera,  and  Frank  Suits.    Describing  Protein  Folding
      Kinetics by Molecular Dynamics Simulations. 1. Theory <span class='ptmr8c-x-x-120'>†</span>.   <span class='ptmri7t-x-x-120'>The Journal of Physical
      </span><span class='ptmri7t-x-x-120'>Chemistry B</span>, 108(21):6571–6581, May 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [237]<span class='bibsp'>   </span></span><a id='XShannon:1948iy'></a>C E Shannon.   A Mathematical Theory of Communication.   <span class='ptmri7t-x-x-120'>Bell Labs Technical
      </span><span class='ptmri7t-x-x-120'>Journal</span>, 27(3):379–423, July 1948.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [238]<span class='bibsp'>   </span></span><a id='XMetzner:2009jl'></a>Philipp Metzner, Christof Sch tte, and Eric Vanden-Eijnden. Transition Path Theory
      for  Markov  Jump  Processes.    <span class='ptmri7t-x-x-120'>Multiscale  Modeling  &amp;  Simulation</span>,  7(3):1192–1219,
      January 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [239]<span class='bibsp'>   </span></span><a id='Xhopkins_druggable_2002'></a>Andrew L. Hopkins and Colin R. Groom. The druggable genome. <span class='ptmri7t-x-x-120'>Nature Reviews
      </span><span class='ptmri7t-x-x-120'>Drug Discovery</span>, 1(9):727–730, September 2002.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [240]<span class='bibsp'>   </span></span><a id='Xjohnson_computational_2017'></a>David K.  Johnson  and  John  Karanicolas.    Computational  screening  and  design
      for compounds that disrupt protein-protein interactions.   <span class='ptmri7t-x-x-120'>Current Topics in Medicinal
      </span><span class='ptmri7t-x-x-120'>Chemistry</span>, 17(23):2703–2714, August 2017.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [241]<span class='bibsp'>   </span></span><a id='Xvajda_cryptic_2018'></a>Sandor  Vajda,  Dmitri  Beglov,  Amanda E  Wakefield,  Megan  Egbert,  and  Adrian
      Whitty. Cryptic binding sites on proteins: definition, detection, and druggability. <span class='ptmri7t-x-x-120'>Current
      </span><span class='ptmri7t-x-x-120'>Opinion in Chemical Biology</span>, 44:1–8, June 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [242]<span class='bibsp'>   </span></span><a id='Xerlanson_site-directed_2000'></a>D. A. Erlanson, A. C. Braisted, D. R. Raphael, M. Randal, R. M. Stroud, E. M.
      Gordon, and J. A. Wells.  Site-directed ligand discovery.  <span class='ptmri7t-x-x-120'>Proceedings of the National
      </span><span class='ptmri7t-x-x-120'>Academy of Sciences</span>, 97(17):9367–9372, August 2000.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [243]<span class='bibsp'>   </span></span><a id='Xkeedy_expanded_2018'></a>Daniel A Keedy, Zachary B Hill, Justin T Biel, Emily Kang, T Justin Rettenmaier,
      José  Brandão-Neto,  Nicholas M  Pearce,  Frank  von  Delft,  James A  Wells,  and
      James S  Fraser.     An  expanded  allosteric  network  in  PTP1B  by  multitemperature
      crystallography, fragment screening, and covalent tethering. <span class='ptmri7t-x-x-120'>eLife</span>, 7:e36307, June 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [244]<span class='bibsp'>   </span></span><a id='Xjohnson_druggable_2013'></a>David K. Johnson and John Karanicolas.   Druggable protein interaction sites are
      more predisposed to surface pocket formation than the rest of the protein surface.  <span class='ptmri7t-x-x-120'>PLoS
      </span><span class='ptmri7t-x-x-120'>Computational Biology</span>, 9(3):e1002951, March 2013.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [245]<span class='bibsp'>   </span></span><a id='Xoleinikovas_understanding_2016'></a>Vladimiras Oleinikovas, Giorgio Saladino, Benjamin P. Cossins, and Francesco L.
      Gervasio.          Understanding    cryptic    pocket    formation    in    protein    targets    by
      enhanced   sampling   simulations.       <span class='ptmri7t-x-x-120'>Journal   of   the   American   Chemical   Society</span>,
      138(43):14257–14263, November 2016.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [246]<span class='bibsp'>   </span></span><a id='Xcimermancic_cryptosite:_2016'></a>Peter  Cimermancic,  Patrick  Weinkam,  T. Justin  Rettenmaier,  Leon  Bichmann,
      Daniel A.    Keedy,    Rahel A.    Woldeyes,    Dina    Schneidman-Duhovny,    Omar N.
      Demerdash,  Julie C.  Mitchell,  James A.  Wells,  James S.  Fraser,  and  Andrej  Sali.
      Cryptosite:  expanding  the  druggable  proteome  by  characterization  and  prediction  of
      cryptic binding sites. <span class='ptmri7t-x-x-120'>Journal of Molecular Biology</span>, 428(4):709–719, February 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [247]<span class='bibsp'>   </span></span><a id='Xschmidt_cosolvent-enhanced_2019'></a>Denis Schmidt, Markus Boehm, Christopher L. McClendon, Rubben Torella, and
      Holger  Gohlke.   Cosolvent-enhanced  sampling  and  unbiased  identification  of  cryptic
      pockets  suitable  for  structure-based  drug  design.    <span class='ptmri7t-x-x-120'>Journal  of  Chemical  Theory  and
      </span><span class='ptmri7t-x-x-120'>Computation</span>, 15(5):3331–3343, May 2019.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [248]<span class='bibsp'>   </span></span><a id='Xcuchillo_collective_2015'></a>Rémi Cuchillo, Kevin Pinto-Gil, and Julien Michel.  A collective variable for the
      rapid exploration of protein druggability. <span class='ptmri7t-x-x-120'>Journal of Chemical Theory and Computation</span>,
      11(3):1292–1307, March 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [249]<span class='bibsp'>   </span></span><a id='Xghanakota_moving_2016'></a>Phani  Ghanakota  and  Heather A.  Carlson.   Moving  beyond  active-site  detection:
      mixmd   applied   to   allosteric   systems.       <span class='ptmri7t-x-x-120'>The   Journal   of   Physical   Chemistry   B</span>,
      120(33):8685–8695, August 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [250]<span class='bibsp'>   </span></span><a id='Xwassman_computational_2013'></a>Christopher D.  Wassman,  Roberta  Baronio,  Özlem  Demir,  Brad D.  Wallentine,
                                                                                      
                                                                                      
      Chiung-Kuang Chen, Linda V. Hall, Faezeh Salehi, Da-Wei Lin, Benjamin P. Chung,
      G. Wesley Hatfield,  A. Richard Chamberlin,  Hartmut  Luecke,  Richard H.  Lathrop,
      Peter Kaiser, and Rommie E. Amaro. Computational identification of a transiently open
      L1/S3 pocket for reactivation of mutant p53.  <span class='ptmri7t-x-x-120'>Nature Communications</span>, 4(1):1407, June
      2013.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [251]<span class='bibsp'>   </span></span><a id='Xkeshwara_toward_2017'></a>Rohan Keshwara, Reed F. Johnson, and Matthias J. Schnell.  Toward an effective
      ebola virus vaccine. <span class='ptmri7t-x-x-120'>Annual Review of Medicine</span>, 68(1):371–386, January 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [252]<span class='bibsp'>   </span></span><a id='Xmulangu_randomized_2019'></a>Sabue Mulangu, Lori E. Dodd, Richard T. Davey, Olivier Tshiani Mbaya, Michael
      Proschan,   Daniel   Mukadi,   Mariano   Lusakibanza Manzo,   Didier   Nzolo,   Antoine
      Tshomba Oloma, Augustin Ibanda, Rosine Ali, Sinaré Coulibaly, Adam C. Levine,
      Rebecca Grais, Janet Diaz, H. Clifford Lane, Jean-Jacques Muyembe-Tamfum, and the
      PALM Writing Group. A randomized, controlled trial of ebola virus disease therapeutics.
      <span class='ptmri7t-x-x-120'>New England Journal of Medicine</span>, 381(24):2293–2303, December 2019.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [253]<span class='bibsp'>   </span></span><a id='Xmessaoudi_filovirus_2015'></a>Ilhem  Messaoudi,  Gaya K.  Amarasinghe,  and  Christopher F.  Basler.    Filovirus
      pathogenesis and immune evasion: insights from Ebola virus and Marburg virus. <span class='ptmri7t-x-x-120'>Nature
      </span><span class='ptmri7t-x-x-120'>Reviews Microbiology</span>, 13(11):663–676, November 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [254]<span class='bibsp'>   </span></span><a id='Xcardenas_ebola_2006'></a>Washington B.   Cardenas,   Yueh-Ming   Loo,   Michael   Gale,   Amy L.   Hartman,
      Christopher R.   Kimberlin,   Luis   Martinez-Sobrido,   Erica Ollmann   Saphire,   and
                                                                                      
                                                                                      
      Christopher F. Basler.  Ebola virus vp35 protein binds double-stranded rna and inhibits
      alpha/beta  interferon  production  induced  by  rig-i  signaling.     <span class='ptmri7t-x-x-120'>Journal  of  Virology</span>,
      80(11):5168–5178, June 2006.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [255]<span class='bibsp'>   </span></span><a id='Xbasler_ebola_2003'></a>Christopher F. Basler, Andrea Mikulasova, Luis Martinez-Sobrido, Jason Paragas,
      Elke   Muhlberger,   Mike   Bray,   Hans-Dieter   Klenk,   Peter   Palese,   and   Adolfo
      Garcia-Sastre.  The ebola virus vp35 protein inhibits activation of interferon regulatory
      factor 3. <span class='ptmri7t-x-x-120'>Journal of Virology</span>, 77(14):7945–7956, July 2003.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [256]<span class='bibsp'>   </span></span><a id='Xhartman_c-terminal_2004'></a>Amy L.  Hartman,  Jonathan S.  Towner,  and  Stuart T.  Nichol.     A  C-terminal
      basic  amino  acid  motif  of  Zaire  ebolavirus  VP35  is  essential  for  type  I  interferon
      antagonism  and  displays  high  identity  with  the  RNA-binding  domain  of  another
      interferon antagonist, the NS1 protein of influenza A virus.  <span class='ptmri7t-x-x-120'>Virology</span>, 328(2):177–184,
      October 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [257]<span class='bibsp'>   </span></span><a id='Xleung_structure_2009'></a>D. W. Leung, N. D. Ginder, D. B. Fulton, J. Nix, C. F. Basler, R. B. Honzatko,
      and  G. K.  Amarasinghe.   Structure  of  the  Ebola  VP35  interferon  inhibitory  domain.
      <span class='ptmri7t-x-x-120'>Proceedings of the National Academy of Sciences</span>, 106(2):411–416, January 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [258]<span class='bibsp'>   </span></span><a id='Xleung_structural_2010'></a>Daisy W   Leung,   Kathleen C   Prins,   Dominika M   Borek,   Mina   Farahbakhsh,
      JoAnn M Tufariello, Parameshwaran Ramanan, Jay C Nix, Luke A Helgeson, Zbyszek
      Otwinowski, Richard B Honzatko, Christopher F Basler, and Gaya K Amarasinghe.
                                                                                      
                                                                                      
      Structural  basis  for  dsRNA  recognition  and  interferon  antagonism  by  Ebola  VP35.
      <span class='ptmri7t-x-x-120'>Nature Structural &amp; Molecular Biology</span>, 17(2):165–172, February 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [259]<span class='bibsp'>   </span></span><a id='Xedwards_differential_2016'></a>Megan R.  Edwards,  Gai  Liu,  Chad E.  Mire,  Suhas  Sureshchandra,  Priya  Luthra,
      Benjamin  Yen,  Reed S.  Shabman,  Daisy W.  Leung,  Ilhem  Messaoudi,  Thomas W.
      Geisbert,  Gaya K.  Amarasinghe,  and  Christopher F.  Basler.    Differential  regulation
      of  interferon  responses  by  ebola  and  marburg  virus  vp35  proteins.    <span class='ptmri7t-x-x-120'>Cell  Reports</span>,
      14(7):1632–1640, February 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [260]<span class='bibsp'>   </span></span><a id='Xhartman_reverse_2006'></a>Amy L.  Hartman,  Jason E.  Dover,  Jonathan S.  Towner,  and  Stuart T.  Nichol.
      Reverse genetic generation of recombinant zaire ebola viruses containing disrupted irf-3
      inhibitory domains results in attenuated virus growth in vitro and higher levels of irf-3
      activation  without  inhibiting  viral  transcription  or  replication.    <span class='ptmri7t-x-x-120'>Journal  of  Virology</span>,
      80(13):6430–6440, July 2006.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [261]<span class='bibsp'>   </span></span><a id='Xprins_mutations_2010'></a>Kathleen C.   Prins,   Sebastien   Delpeut,   Daisy W.   Leung,   Olivier   Reynard,
      Valentina A.  Volchkova,  St. Patrick  Reid,  Parameshwaran  Ramanan,  Washington B.
      Cardenas,  Gaya K.  Amarasinghe,  Viktor E.  Volchkov,  and  Christopher F.  Basler.
      Mutations  abrogating  vp35  interaction  with  double-stranded  rna  render  ebola  virus
      avirulent in guinea pigs. <span class='ptmri7t-x-x-120'>Journal of Virology</span>, 84(6):3004–3015, March 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [262]<span class='bibsp'>   </span></span><a id='Xprins_basic_2010'></a>Kathleen C.  Prins,  Jennifer M.  Binning,  Reed S.  Shabman,  Daisy W.  Leung,
                                                                                      
                                                                                      
      Gaya K. Amarasinghe, and Christopher F. Basler. Basic residues within the ebolavirus
      vp35 protein are required for its viral polymerase cofactor function. <span class='ptmri7t-x-x-120'>Journal of Virology</span>,
      84(20):10581–10591, October 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [263]<span class='bibsp'>   </span></span><a id='Xbrown_silico_2014'></a>Craig S. Brown, Michael S. Lee, Daisy W. Leung, Tianjiao Wang, Wei Xu, Priya
      Luthra, Manu Anantpadma, Reed S. Shabman, Lisa M. Melito, Karen S. MacMillan,
      Dominika M.   Borek,   Zbyszek   Otwinowski,   Parameshwaran   Ramanan,   Alisha J.
      Stubbs,  Dayna S.  Peterson,  Jennifer M.  Binning,  Marco  Tonelli,  Mark A.  Olson,
      Robert A. Davey, Joseph M. Ready, Christopher F. Basler, and Gaya K. Amarasinghe.
      In  silico  derived  small  molecules  bind  the  filovirus  vp35  protein  and  inhibit  its
      polymerase cofactor activity.  <span class='ptmri7t-x-x-120'>Journal of Molecular Biology</span>, 426(10):2045–2058, May
      2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [264]<span class='bibsp'>   </span></span><a id='Xglanzer_silico_2016'></a>Jason G. Glanzer, Brendan M. Byrne, Aaron M. McCoy, Ben J. James, Joshua D.
      Frank,  and  Greg G.  Oakley.    In  silico  and  in  vitro  methods  to  identify  ebola  virus
      VP35-dsRNA  inhibitors.     <span class='ptmri7t-x-x-120'>Bioorganic  &amp;  Medicinal  Chemistry</span>,  24(21):5388–5392,
      November 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [265]<span class='bibsp'>   </span></span><a id='Xpande_everything_2010'></a>Vijay S.  Pande,  Kyle  Beauchamp,  and  Gregory R.  Bowman.     Everything  you
      wanted  to  know  about  Markov  State  Models  but  were  afraid  to  ask.     <span class='ptmri7t-x-x-120'>Methods</span>,
      52(1):99–105, September 2010.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [266]<span class='bibsp'>   </span></span><a id='Xbowman_network_2010'></a>Gregory R  Bowman,  Xuhui  Huang,  and  Vijay S  Pande.    Network  models  for
      molecular  kinetics  and  their  initial  applications  to  human  health.     <span class='ptmri7t-x-x-120'>Cell  Research</span>,
      20(6):622–630, June 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [267]<span class='bibsp'>   </span></span><a id='XHruska:2018ec'></a>Eugen  Hruska,  Jayvee R  Abella,  Feliks  Nüske,  Lydia E  Kavraki,  and  Cecilia
      Clementi. Quantitative comparison of adaptive sampling methods for protein dynamics.
      <span class='ptmri7t-x-x-120'>The Journal of chemical physics</span>, 149(24):244119, December 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [268]<span class='bibsp'>   </span></span><a id='XBeglov:2018dm'></a>Dmitri Beglov, David R Hall, Amanda E Wakefield, Lingqi Luo, Karen N Allen,
      Dima Kozakov, Adrian Whitty, and Sandor Vajda.   Exploring the structural origins of
      cryptic sites on proteins. <span class='ptmri7t-x-x-120'>Proceedings of the National Academy of Sciences of the United
      </span><span class='ptmri7t-x-x-120'>States of America</span>, 115(15):E3416–E3425, March 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [269]<span class='bibsp'>   </span></span><a id='Xngan_ftmap:_2012'></a>C. H. Ngan, T. Bohnuud, S. E. Mottarella, D. Beglov, E. A. Villar, D. R. Hall,
      D. Kozakov, and S. Vajda. FTMAP: extended protein mapping with user-selected probe
      molecules. <span class='ptmri7t-x-x-120'>Nucleic Acids Research</span>, 40(W1):W271–W275, July 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [270]<span class='bibsp'>   </span></span><a id='Xkozakov_ftmap_2015'></a>Dima   Kozakov,   Laurie E   Grove,   David R   Hall,   Tanggis   Bohnuud,   Scott E
      Mottarella, Lingqi Luo, Bing Xia, Dmitri Beglov, and Sandor Vajda. The FTMap family
      of web servers for determining and characterizing ligand-binding hot spots of proteins.
      <span class='ptmri7t-x-x-120'>Nature Protocols</span>, 10(5):733–755, May 2015.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [271]<span class='bibsp'>   </span></span><a id='Xbernstein_structural_2011'></a>R. Bernstein,  K. L.  Schmidt,  P. B.  Harbury,  and  S. Marqusee.    Structural  and
      kinetic mapping of side-chain exposure onto the protein energy landscape. <span class='ptmri7t-x-x-120'>Proceedings
      </span><span class='ptmri7t-x-x-120'>of the National Academy of Sciences</span>, 108(26):10532–10537, June 2011.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [272]<span class='bibsp'>   </span></span><a id='Xliu_sensitive_2017'></a>Gai  Liu,  Peter J.  Nash,  Britney  Johnson,  Colette  Pietzsch,  Ma.  Xenia G.  Ilagan,
      Alexander   Bukreyev,   Christopher F.   Basler,   Terry L.   Bowlin,   Donald T.   Moir,
      Daisy W.  Leung,  and  Gaya K.  Amarasinghe.    A  sensitive  in  vitro  high-throughput
      screen  to  identify  pan-filoviral  replication  inhibitors  targeting  the  vp35–np  interface.
      <span class='ptmri7t-x-x-120'>ACS Infectious Diseases</span>, 3(3):190–198, March 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [273]<span class='bibsp'>   </span></span><a id='Xkeck_ssbdna_2012'></a>Alexander G.  Kozlov,  Roberto  Galletto,  and  Timothy M.  Lohman.     Ssb–dna
      binding monitored by fluorescence intensity and anisotropy.  In James L. Keck, editor,
      <span class='ptmri7t-x-x-120'>Single-Stranded DNA Binding Proteins</span>, pages 55–83. Humana Press, Totowa, NJ, 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [274]<span class='bibsp'>   </span></span><a id='Xramanan_structural_2012'></a>P. Ramanan, M. R. Edwards, R. S. Shabman, D. W. Leung, A. C. Endlich-Frazier,
      D. M. Borek, Z. Otwinowski, G. Liu, J. Huh, C. F. Basler, and G. K. Amarasinghe.
      Structural  basis  for  Marburg  virus  VP35-mediated  immune  evasion  mechanisms.
      <span class='ptmri7t-x-x-120'>Proceedings  of  the  National  Academy  of  Sciences</span>,  109(50):20661–20666,  December
      2012.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [275]<span class='bibsp'>   </span></span><a id='Xshrake_environment_1973'></a>A. Shrake and J.A. Rupley.  Environment and exposure to solvent of protein atoms.
      Lysozyme and insulin. <span class='ptmri7t-x-x-120'>Journal of Molecular Biology</span>, 79(2):351–371, September 1973.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [276]<span class='bibsp'>   </span></span><a id='XFrey:2007hs'></a>Brendan J Frey and Delbert Dueck.  Clustering by Passing Messages Between Data
      Points. <span class='ptmri7t-x-x-120'>Science</span>, 315(5814):972–976, February 2007.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [277]<span class='bibsp'>   </span></span><a id='XDianati:2016bt'></a>Navid  Dianati.    Unwinding  the  hairball  graph:  Pruning  algorithms  for  weighted
      complex networks. <span class='ptmri7t-x-x-120'>Physical review. E</span>, 93(1):012304, January 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [278]<span class='bibsp'>   </span></span><a id='Xzimmerman_prediction_2017'></a>Maxwell I.   Zimmerman,   Kathryn M.   Hart,   Carrie A.   Sibbald,   Thomas E.
      Frederick,  John R.  Jimah,  Catherine R.  Knoverek,  Niraj H.  Tolia,  and  Gregory R.
      Bowman.  Prediction of new stabilizing mutations based on mechanistic insights from
      markov state models. <span class='ptmri7t-x-x-120'>ACS Central Science</span>, 3(12):1311–1321, December 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [279]<span class='bibsp'>   </span></span><a id='XZhu2020-do'></a>Na Zhu, Dingyu Zhang, Wenling Wang, Xingwang Li, Bo Yang, Jingdong Song,
      Xiang  Zhao,  Baoying  Huang,  Weifeng  Shi,  Roujian  Lu,  Peihua  Niu,  Faxian  Zhan,
      Xuejun Ma, Dayan Wang, Wenbo Xu, Guizhen Wu, George F Gao, Wenjie Tan, and
      China Novel Coronavirus Investigating and Research Team.  A novel coronavirus from
      patients with pneumonia in china, 2019.   <span class='ptmri7t-x-x-120'>N. Engl. J. Med.</span>, 382(8):727–733, February
      2020.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [280]<span class='bibsp'>   </span></span><a id='XCorman2018-my'></a>Victor M Corman, Doreen Muth, Daniela Niemeyer, and Christian Drosten. Chapter
      eight  -  hosts  and  sources  of  endemic  human  coronaviruses.    In  Margaret  Kielian,
      Thomas C Mettenleiter, and Marilyn J Roossinck, editors, <span class='ptmri7t-x-x-120'>Advances in Virus Research</span>,
      volume 100, pages 163–188. Academic Press, January 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [281]<span class='bibsp'>   </span></span><a id='XRoser2020-yb'></a>Max  Roser,  Hannah  Ritchie,  Esteban  Ortiz-Ospina,  and  Joe  Hasell.   Coronavirus
      pandemic (COVID-19). <span class='ptmri7t-x-x-120'>Our World in Data</span>, 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [282]<span class='bibsp'>   </span></span><a id='XLurie2020-vk'></a>Nicole  Lurie,  Melanie  Saville,  Richard  Hatchett,  and  Jane  Halton.    Developing
      covid-19 vaccines at pandemic speed. <span class='ptmri7t-x-x-120'>N. Engl. J. Med.</span>, 382(21):1969–1973, May 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [283]<span class='bibsp'>   </span></span><a id='XGordon2020-sg'></a>David E  Gordon,  Gwendolyn M  Jang,  Mehdi  Bouhaddou,  Jiewei  Xu,  Kirsten
      Obernier,  Kris M  White,  Matthew J  O’Meara,  Veronica V  Rezelj,  Jeffrey Z  Guo,
      Danielle L  Swaney,  Tia A  Tummino,  Ruth  Huettenhain,  Robyn M  Kaake,  Alicia L
      Richards, Beril Tutuncuoglu, Helene Foussard, Jyoti Batra, Kelsey Haas, Maya Modak,
      Minkyu Kim, Paige Haas, Benjamin J Polacco, Hannes Braberg, Jacqueline M Fabius,
      Manon  Eckhardt,  Margaret  Soucheray,  Melanie J  Bennett,  Merve  Cakir,  Michael J
      McGregor,  Qiongyu  Li,  Bjoern  Meyer,  Ferdinand  Roesch,  Thomas  Vallet,  Alice
      Mac Kain,  Lisa  Miorin,  Elena  Moreno,  Zun  Zar Chi  Naing,  Yuan  Zhou,  Shiming
      Peng, Ying Shi, Ziyang Zhang, Wenqi Shen, Ilsa T Kirby, James E Melnyk, John S
      Chorba, Kevin Lou, Shizhong A Dai, Inigo Barrio-Hernandez, Danish Memon, Claudia
                                                                                      
                                                                                      
      Hernandez-Armenta, Jiankun Lyu, Christopher J P Mathy, Tina Perica, Kala B Pilla,
      Sai J Ganesan, Daniel J Saltzberg, Ramachandran Rakesh, Xi Liu, Sara B Rosenthal,
      Lorenzo  Calviello,  Srivats  Venkataramanan,  Jose  Liboy-Lugo,  Yizhu  Lin,  Xi-Ping
      Huang,   Yongfeng   Liu,   Stephanie A   Wankowicz,   Markus   Bohn,   Maliheh   Safari,
      Fatima S  Ugur,  Cassandra  Koh,  Nastaran Sadat  Savar,  Quang Dinh  Tran,  Djoshkun
      Shengjuler,  Sabrina J  Fletcher,  Michael C  O’Neal,  Yiming  Cai,  Jason  C J  Chang,
      David J  Broadhurst,  Saker  Klippsten,  Phillip P  Sharp,  Nicole A  Wenzell,  Duygu
      Kuzuoglu, Hao-Yuan Wang, Raphael Trenker, Janet M Young, Devin A Cavero, Joseph
      Hiatt, Theodore L Roth, Ujjwal Rathore, Advait Subramanian, Julia Noack, Mathieu
      Hubert,  Robert M  Stroud,  Alan D  Frankel,  Oren S  Rosenberg,  Kliment A  Verba,
      David A  Agard,  Melanie  Ott,  Michael  Emerman,  Natalia  Jura,  Mark  von  Zastrow,
      Eric Verdin, Alan Ashworth, Olivier Schwartz, Christophe d’Enfert, Shaeri Mukherjee,
      Matt  Jacobson,  Harmit S  Malik,  Danica G  Fujimori,  Trey  Ideker,  Charles S  Craik,
      Stephen N Floor, James S Fraser, John D Gross, Andrej Sali, Bryan L Roth, Davide
      Ruggero,  Jack  Taunton,  Tanja  Kortemme,  Pedro  Beltrao,  Marco  Vignuzzi,  Adolfo
      García-Sastre,  Kevan M  Shokat,  Brian K  Shoichet,  and  Nevan J  Krogan.      A
      SARS-CoV-2 protein interaction map reveals targets for drug repurposing. <span class='ptmri7t-x-x-120'>Nature</span>, April
      2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [284]<span class='bibsp'>   </span></span><a id='XSanders2020-ea'></a>James M  Sanders,  Marguerite L  Monogue,  Tomasz Z  Jodlowski,  and  James B
      Cutrell. Pharmacologic treatments for coronavirus disease 2019 (COVID-19): A review.
      <span class='ptmri7t-x-x-120'>JAMA</span>, April 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [285]<span class='bibsp'>   </span></span><a id='XWalls2020-of'></a>Alexandra C   Walls,   Young-Jun   Park,   M Alejandra   Tortorici,   Abigail   Wall,
                                                                                      
                                                                                      
      Andrew T  McGuire,  and  David  Veesler.   Structure,  function,  and  antigenicity  of  the
      SARS-CoV-2 spike glycoprotein. <span class='ptmri7t-x-x-120'>Cell</span>, 181(2):281–292.e6, April 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [286]<span class='bibsp'>   </span></span><a id='XHoffmann2020-nt'></a>Markus Hoffmann, Hannah Kleine-Weber, Simon Schroeder, Nadine Krüger, Tanja
      Herrler, Sandra Erichsen, Tobias S Schiergens, Georg Herrler, Nai-Huei Wu, Andreas
      Nitsche, Marcel A Müller, Christian Drosten, and Stefan Pöhlmann.   SARS-CoV-2
      cell  entry  depends  on  ACE2  and  TMPRSS2  and  is  blocked  by  a  clinically  proven
      protease inhibitor. <span class='ptmri7t-x-x-120'>Cell</span>, 181(2):271–280.e8, April 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [287]<span class='bibsp'>   </span></span><a id='XShang2020-er'></a>Jian Shang, Gang Ye, Ke Shi, Yushun Wan, Chuming Luo, Hideki Aihara, Qibin
      Geng,  Ashley  Auerbach,  and  Fang  Li.    Structural  basis  of  receptor  recognition  by
      SARS-CoV-2. <span class='ptmri7t-x-x-120'>Nature</span>, 581(7807):221–224, May 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [288]<span class='bibsp'>   </span></span><a id='XLan2020-wh'></a>Jun  Lan,  Jiwan  Ge,  Jinfang  Yu,  Sisi  Shan,  Huan  Zhou,  Shilong  Fan,  Qi Zhang,
      Xuanling  Shi,  Qisheng  Wang,  Linqi  Zhang,  and  Xinquan  Wang.    Structure  of  the
      SARS-CoV-2  spike  receptor-binding  domain  bound  to  the  ACE2  receptor.    <span class='ptmri7t-x-x-120'>Nature</span>,
      581(7807):215–220, May 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [289]<span class='bibsp'>   </span></span><a id='Xmasters_coronavirus_2019'></a>Paul S. Masters.   Coronavirus genomic RNA packaging.   <span class='ptmri7t-x-x-120'>Virology</span>, 537:198–207,
      November 2019.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [290]<span class='bibsp'>   </span></span><a id='XVan_der_Lee2014-vc'></a>Robin van der Lee, Marija Buljan, Benjamin Lang, Robert J Weatheritt, Gary W
      Daughdrill, A Keith Dunker, Monika Fuxreiter, Julian Gough, Joerg Gsponer, David T
      Jones, Philip M Kim, Richard W Kriwacki, Christopher J Oldfield, Rohit V Pappu,
      Peter Tompa, Vladimir N Uversky, Peter E Wright, and M Madan Babu. Classification
      of intrinsically disordered regions and proteins.  <span class='ptmri7t-x-x-120'>Chem. Rev.</span>, 114(13):6589–6631, July
      2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [291]<span class='bibsp'>   </span></span><a id='XLaude1995-bx'></a>Hubert  Laude  and  Paul S  Masters.    The  coronavirus  nucleocapsid  protein.    In
      Stuart G Siddell, editor, <span class='ptmri7t-x-x-120'>The Coronaviridae</span>, pages 141–163. Springer US, Boston, MA,
      1995.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [292]<span class='bibsp'>   </span></span><a id='XHolmstrom2019-ns'></a>Erik D  Holmstrom,  Zhaowei  Liu,  Daniel  Nettels,  Robert B  Best,  and  Benjamin
      Schuler. Disordered RNA chaperones can enhance nucleic acid folding via local charge
      screening. <span class='ptmri7t-x-x-120'>Nat. Commun.</span>, 10(1):2453, June 2019.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [293]<span class='bibsp'>   </span></span><a id='XBorgia2018-yo'></a>Alessandro   Borgia,   Madeleine B   Borgia,   Katrine   Bugge,   Vera M   Kissling,
      Pétur O Heidarsson, Catarina B Fernandes, Andrea Sottini, Andrea Soranno, Karin J
      Buholzer, Daniel Nettels, Birthe B Kragelund, Robert B Best, and Benjamin Schuler.
      Extreme disorder in an ultrahigh-affinity protein complex.   <span class='ptmri7t-x-x-120'>Nature</span>, 555:61, February
      2018.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [294]<span class='bibsp'>   </span></span><a id='XDimura2016-kt'></a>Mykola Dimura, Thomas O Peulen, Christian A Hanke, Aiswaria Prakash, Holger
      Gohlke,  and  Claus Am  Seidel.   Quantitative  FRET  studies  and  integrative  modeling
      unravel the structure and dynamics of biomolecular systems.  <span class='ptmri7t-x-x-120'>Curr. Opin. Struct. Biol.</span>,
      40:163–185, October 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [295]<span class='bibsp'>   </span></span><a id='XFuertes2017-au'></a>Gustavo Fuertes, Niccolò Banterle, Kiersten M Ruff, Aritra Chowdhury, Davide
      Mercadante,  Christine  Koehler,  Michael  Kachala,  Gemma  Estrada Girona,  Sigrid
      Milles,  Ankur  Mishra,  Patrick R  Onck,  Frauke  Gräter,  Santiago  Esteban-Martín,
      Rohit V Pappu, Dmitri I Svergun, and Edward A Lemke. Decoupling of size and shape
      fluctuations in heteropolymeric sequences reconciles discrepancies in SAXS vs. FRET
      measurements. <span class='ptmri7t-x-x-120'>Proc. Natl. Acad. Sci. U. S. A.</span>, 114(31):E6342–E6351, August 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [296]<span class='bibsp'>   </span></span><a id='XWarner2017-zh'></a>John B Warner, 4th, Kiersten M Ruff, Piau Siong Tan, Edward A Lemke, Rohit V
      Pappu, and Hilal A Lashuel. Monomeric huntingtin exon 1 has similar overall structural
      features  for  Wild-Type  and  pathological  polyglutamine  lengths.   <span class='ptmri7t-x-x-120'>J.  Am.  Chem.  Soc.</span>,
      139(41):14456–14469, October 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [297]<span class='bibsp'>   </span></span><a id='XChung2015-uh'></a>Hoi Sung Chung, Stefano Piana-Agostinetti, David E Shaw, and William A Eaton.
      Structural origin of slow diffusion in protein folding.  <span class='ptmri7t-x-x-120'>Science</span>, 349(6255):1504–1510,
      September 2015.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [298]<span class='bibsp'>   </span></span><a id='XIserman2020-xm'></a>Christiane Iserman, Christine Anne Roden, Mark Boerneke, Rachel Sealfon, Grace
      McLaughlin,  Irwin  Jungreis,  Christopher Y  Park,  Avinash  Boppana,  Ethan  Fritch,
      Yixuan  Hou,  Chandra  Theesfeld,  Olga  Troyanskaya,  Ralph  S G  Baric,  Timothy P
      Sheahan, Kevin Weeks, and Amy Gladfelter. Specific viral RNA drives the SARS CoV-2
      nucleocapsid to phase separate. June 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [299]<span class='bibsp'>   </span></span><a id='XPerdikari2020-cx'></a>Theodora Myrto Perdikari, Anastasia C Murthy, Veronica H Ryan, Scott Watters,
      Mandar T Naik, and Nicolas L Fawzi.  SARS-CoV-2 nucleocapsid protein undergoes
      liquid-liquid phase separation stimulated by RNA and partitions into phases of human
      ribonucleoproteins. June 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [300]<span class='bibsp'>   </span></span><a id='XSavastano2020-zf'></a>Adriana  Savastano,  Alain Ibáñez  de Opakua,  Marija  Rankovic,  and  Markus
      Zweckstetter.   Nucleocapsid  protein  of  SARS-CoV-2  phase  separates  into  RNA-rich
      polymerase-containing condensates. June 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [301]<span class='bibsp'>   </span></span><a id='Xmcbride_coronavirus_2014'></a>Ruth   McBride,   Marjorie   van   Zyl,   and   Burtram   Fielding.      The   coronavirus
      nucleocapsid is a multifunctional protein. <span class='ptmri7t-x-x-120'>Viruses</span>, 6(8):2991–3018, August 2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [302]<span class='bibsp'>   </span></span><a id='XBaker2001-ma'></a>N A Baker, D Sept, S Joseph, M J Holst, and J A McCammon.  Electrostatics of
      nanosystems: application to microtubules and the ribosome. <span class='ptmri7t-x-x-120'>Proc. Natl. Acad. Sci. U. S.
      </span><span class='ptmri7t-x-x-120'>A.</span>, 98(18):10037–10041, August 2001.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [303]<span class='bibsp'>   </span></span><a id='XChang2009-vd'></a>Chung-Ke  Chang,  Yen-Lan  Hsu,  Yuan-Hsiang  Chang,  Fa-An  Chao,  Ming-Chya
      Wu,  Yu-Shan  Huang,  Chin-Kun  Hu,  and  Tai-Huang  Huang.    Multiple  nucleic  acid
      binding  sites  and  intrinsic  disorder  of  severe  acute  respiratory  syndrome  coronavirus
      nucleocapsid  protein:  implications  for  ribonucleocapsid  protein  packaging.   <span class='ptmri7t-x-x-120'>J.  Virol.</span>,
      83(5):2255–2264, March 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [304]<span class='bibsp'>   </span></span><a id='XGrossoehme2009-px'></a>Nicholas E Grossoehme, Lichun Li, Sarah C Keane, Pinghua Liu, Charles E Dann,
      3rd,  Julian L  Leibowitz,  and  David P  Giedroc.    Coronavirus  N  protein  n-terminal
      domain  (NTD)  specifically  binds  the  transcriptional  regulatory  sequence  (TRS)  and
      melts TRS-cTRS RNA duplexes. <span class='ptmri7t-x-x-120'>J. Mol. Biol.</span>, 394(3):544–557, December 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [305]<span class='bibsp'>   </span></span><a id='XCui2015-bu'></a>Lei Cui, Haiying Wang, Yanxi Ji, Jie Yang, Shan Xu, Xingyu Huang, Zidao Wang,
      Lei Qin, Po Tien, Xi Zhou, Deyin Guo, and Yu Chen.   The nucleocapsid protein of
      coronaviruses acts as a viral suppressor of RNA silencing in mammalian cells. <span class='ptmri7t-x-x-120'>J. Virol.</span>,
      89(17):9029–9043, September 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [306]<span class='bibsp'>   </span></span><a id='XTakeda2008-wl'></a>Mitsuhiro  Takeda,  Chung-Ke  Chang,  Teppei  Ikeya,  Peter  Güntert,  Yuan-Hsiang
      Chang, Yen-Lan Hsu, Tai-Huang Huang, and Masatsune Kainosho.  Solution structure
      of the c-terminal dimerization domain of SARS coronavirus nucleocapsid protein solved
      by the SAIL-NMR method. <span class='ptmri7t-x-x-120'>J. Mol. Biol.</span>, 380(4):608–622, July 2008.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [307]<span class='bibsp'>   </span></span><a id='XJayaram2006-rd'></a>Hariharan Jayaram, Hui Fan, Brian R Bowman, Amy Ooi, Jyothi Jayaram, Ellen W
      Collisson, Julien Lescar, and B V Venkataram Prasad.  X-ray structures of the N- and
      c-terminal domains of a coronavirus nucleocapsid protein: implications for nucleocapsid
      formation. <span class='ptmri7t-x-x-120'>J. Virol.</span>, 80(13):6612–6620, July 2006.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [308]<span class='bibsp'>   </span></span><a id='XYu2005-ar'></a>I-Mei  Yu,  Christin  L T  Gustafson,  Jianbo  Diao,  John W  Burgner,  2nd,  Zhihong
      Li, Jingqiang Zhang, and Jue Chen.   Recombinant severe acute respiratory syndrome
      (SARS) coronavirus nucleocapsid protein forms a dimer through its c-terminal domain.
      <span class='ptmri7t-x-x-120'>J. Biol. Chem.</span>, 280(24):23280–23286, June 2005.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [309]<span class='bibsp'>   </span></span><a id='XLuo2006-et'></a>Haibin  Luo,  Jing  Chen,  Kaixian  Chen,  Xu Shen,  and  Hualiang  Jiang.   Carboxyl
      terminus  of  severe  acute  respiratory  syndrome  coronavirus  nucleocapsid  protein:
      self-association  analysis  and  nucleic  acid  binding  characterization.     <span class='ptmri7t-x-x-120'>Biochemistry</span>,
      45(39):11827–11835, October 2006.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [310]<span class='bibsp'>   </span></span><a id='XChang2013-th'></a>Chung-Ke Chang, Chia-Min Michael Chen, Ming-Hui Chiang, Yen-Lan Hsu, and
      Tai-Huang Huang.  Transient oligomerization of the SARS-CoV N protein–implication
      for virus ribonucleoprotein packaging. <span class='ptmri7t-x-x-120'>PLoS One</span>, 8(5):e65045, May 2013.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [311]<span class='bibsp'>   </span></span><a id='XRobbins1986-lh'></a>S G   Robbins,   M F   Frana,   J J   McGowan,   J F   Boyle,   and   K V   Holmes.
                                                                                      
                                                                                      
      RNA-binding proteins of coronavirus MHV: detection of monomeric and multimeric N
      protein with an RNA overlay-protein blot assay. <span class='ptmri7t-x-x-120'>Virology</span>, 150(2):402–410, April 1986.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [312]<span class='bibsp'>   </span></span><a id='XHe2004-xy'></a>Runtao He, Frederick Dobie, Melissa Ballantine, Andrew Leeson, Yan Li, Nathalie
      Bastien,  Todd  Cutts,  Anton  Andonov,  Jingxin  Cao,  Timothy F  Booth,  Frank A
      Plummer, Shaun Tyler, Lindsay Baker, and Xuguang Li.   Analysis of multimerization
      of  the  SARS  coronavirus  nucleocapsid  protein.    <span class='ptmri7t-x-x-120'>Biochem.  Biophys.  Res.  Commun.</span>,
      316(2):476–483, April 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [313]<span class='bibsp'>   </span></span><a id='XKang2020-tb'></a>Sisi Kang, Mei Yang, Zhongsi Hong, Liping Zhang, Zhaoxia Huang, Xiaoxue Chen,
      Suhua He, Ziliang Zhou, Zhechong Zhou, Qiuyue Chen, Yan Yan, Changsheng Zhang,
      Hong Shan, and Shoudeng Chen. Crystal structure of SARS-CoV-2 nucleocapsid protein
      RNA binding domain reveals potential unique drug targeting sites.  <span class='ptmri7t-x-x-120'>Acta Pharm Sin B</span>,
      April 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [314]<span class='bibsp'>   </span></span><a id='XZinzula2020-mi'></a>Luca  Zinzula,  Massimiliano Orsini  Nagy,  and  Andreas  Bracher.    1.45  angstrom
      resolution   crystal   structure   of   c-terminal   dimerization   domain   of   nucleocapsid
      phosphoprotein from SARS-CoV-2 (PDB: 6YUN). <span class='ptmri7t-x-x-120'>Protein Data Bank</span>, May 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [315]<span class='bibsp'>   </span></span><a id='XYe2020-fz'></a>Qiaozhen Ye, Alan M V West, Steve Silletti, and Kevin D Corbett.   Architecture
      and self-assembly of the SARS-CoV-2 nucleocapsid protein. June 2020.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [316]<span class='bibsp'>   </span></span><a id='XZeng2020-cw'></a>Weihong  Zeng,  Guangfeng  Liu,  Huan  Ma,  Dan  Zhao,  Yunru  Yang,  Muziying
      Liu,   Ahmed   Mohammed,   Changcheng   Zhao,   Yun   Yang,   Jiajia   Xie,   Chengchao
      Ding,  Xiaoling  Ma,  Jianping  Weng,  Yong  Gao,  Hongliang  He,  and  Tengchuan  Jin.
      Biochemical characterization of SARS-CoV-2 nucleocapsid protein. <span class='ptmri7t-x-x-120'>Biochem. Biophys.
      </span><span class='ptmri7t-x-x-120'>Res. Commun.</span>, 527(3):618–623, June 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [317]<span class='bibsp'>   </span></span><a id='XNettels2009-uq'></a>Daniel  Nettels,  Sonja  Müller-Späth,  Frank  Küster,  Hagen  Hofmann,  Dominik
      Haenni,  Stefan  Rüegger,  Luc  Reymond,  Armin  Hoffmann,  Jan  Kubelka,  Benjamin
      Heinz, Klaus Gast, Robert B Best, and Benjamin Schuler. Single-molecule spectroscopy
      of the temperature-induced collapse of unfolded proteins.  <span class='ptmri7t-x-x-120'>Proc. Natl. Acad. Sci. U. S.
      </span><span class='ptmri7t-x-x-120'>A.</span>, 106(49):20740–20745, December 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [318]<span class='bibsp'>   </span></span><a id='XSoranno2012-jn'></a>Andrea   Soranno,   Brigitte   Buchli,   Daniel   Nettels,   Ryan R   Cheng,   Sonja
      Müller-Späth,  Shawn H  Pfeil,  Armin  Hoffmann,  Everett A  Lipman,  Dmitrii E
      Makarov,  and  Benjamin  Schuler.      Quantifying  internal  friction  in  unfolded  and
      intrinsically disordered proteins with single-molecule spectroscopy.  <span class='ptmri7t-x-x-120'>Proc. Natl. Acad.
      </span><span class='ptmri7t-x-x-120'>Sci. U. S. A.</span>, 109(44):17800–17806, October 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [319]<span class='bibsp'>   </span></span><a id='XBorgia2012-tz'></a>Alessandro Borgia, Beth G Wensley, Andrea Soranno, Daniel Nettels, Madeleine B
      Borgia,  Armin  Hoffmann,  Shawn H  Pfeil,  Everett A  Lipman,  Jane  Clarke,  and
      Benjamin Schuler.  Localizing internal friction along the reaction coordinate of protein
                                                                                      
                                                                                      
      folding by combining ensemble and single-molecule fluorescence spectroscopy.   <span class='ptmri7t-x-x-120'>Nat.
      </span><span class='ptmri7t-x-x-120'>Commun.</span>, 3:1195, 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [320]<span class='bibsp'>   </span></span><a id='XSchuler2016-wh'></a>Benjamin   Schuler,   Andrea   Soranno,   Hagen   Hofmann,   and   Daniel   Nettels.
      Single-Molecule   FRET   spectroscopy   and   the   polymer   physics   of   unfolded   and
      intrinsically disordered proteins. <span class='ptmri7t-x-x-120'>Annu. Rev. Biophys.</span>, 45:207–231, July 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [321]<span class='bibsp'>   </span></span><a id='XSoranno2018-iw'></a>A Soranno, F Cabassi, M E Orselli, and others. Dynamics of structural elements of
      GB1 <span class='cmmi-12'>β</span>-Hairpin revealed by Tryptophan–Cysteine contact formation experiments.  <span class='ptmri7t-x-x-120'>The
      </span><span class='ptmri7t-x-x-120'>Journal of</span>, 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [322]<span class='bibsp'>   </span></span><a id='XSoranno2017-bi'></a>Andrea  Soranno,  Andrea  Holla,  Fabian  Dingfelder,  Daniel  Nettels,  Dmitrii E
      Makarov, and Benjamin Schuler. Integrated view of internal friction in unfolded proteins
      from single-molecule FRET, contact quenching, theory, and simulations.   <span class='ptmri7t-x-x-120'>Proc. Natl.
      </span><span class='ptmri7t-x-x-120'>Acad. Sci. U. S. A.</span>, 114(10):E1833–E1839, March 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [323]<span class='bibsp'>   </span></span><a id='XSchellman1987-tp'></a>J A  Schellman.      Selective  binding  and  solvent  denaturation.      <span class='ptmri7t-x-x-120'>Biopolymers</span>,
      26(4):549–559, April 1987.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [324]<span class='bibsp'>   </span></span><a id='XHofmann2012-pz'></a>Hagen Hofmann, Andrea Soranno, Alessandro Borgia, Klaus Gast, Daniel Nettels,
                                                                                      
                                                                                      
      and Benjamin Schuler.   Polymer scaling laws of unfolded and intrinsically disordered
      proteins quantified with single-molecule spectroscopy.  <span class='ptmri7t-x-x-120'>Proc. Natl. Acad. Sci. U. S. A.</span>,
      109(40):16155–16160, September 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [325]<span class='bibsp'>   </span></span><a id='XBorgia2016-oq'></a>Alessandro  Borgia,  Wenwei  Zheng,  Karin  Buholzer,  Madeleine B  Borgia,  Anja
      Schüler,  Hagen  Hofmann,  Andrea  Soranno,  Daniel  Nettels,  Klaus  Gast,  Alexander
      Grishaev,   and   Others.        Consistent   view   of   polypeptide   chain   expansion   in
      chemical  denaturants  from  multiple  experimental  methods.      <span class='ptmri7t-x-x-120'>J.  Am.  Chem.  Soc.</span>,
      138(36):11714–11726, 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [326]<span class='bibsp'>   </span></span><a id='XZheng2016-ds'></a>Wenwei Zheng, Alessandro Borgia, Karin Buholzer, Alexander Grishaev, Benjamin
      Schuler,  and  Robert B  Best.     Probing  the  action  of  chemical  denaturant  on  an
      intrinsically  disordered  protein  by  simulation  and  experiment.    <span class='ptmri7t-x-x-120'>J.  Am.  Chem.  Soc.</span>,
      138(36):11702–11713, September 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [327]<span class='bibsp'>   </span></span><a id='XAznauryan2016-bp'></a>Mikayel   Aznauryan,   Leonildo   Delgado,   Andrea   Soranno,   Daniel   Nettels,
      Jie-Rong  Huang,  Alexander M  Labhardt,  Stephan  Grzesiek,  and  Benjamin  Schuler.
      Comprehensive  structural  and  dynamical  view  of  an  unfolded  protein  from  the
      combination of single-molecule FRET, NMR, and SAXS.  <span class='ptmri7t-x-x-120'>Proc. Natl. Acad. Sci. U. S.
      </span><span class='ptmri7t-x-x-120'>A.</span>, 113(37):E5389–98, September 2016.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [328]<span class='bibsp'>   </span></span><a id='XTompa2008-tn'></a>Peter Tompa and Monika Fuxreiter. Fuzzy complexes: polymorphism and structural
      disorder in protein–protein interactions. <span class='ptmri7t-x-x-120'>Trends Biochem. Sci.</span>, 33(1):2–8, 2008.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [329]<span class='bibsp'>   </span></span><a id='XHolehouse2015-rg'></a>Alex S  Holehouse,  Kanchan  Garai,  Nicholas  Lyle,  Andreas  Vitalis,  and  Rohit V
      Pappu.  Quantitative assessments of the distinct contributions of polypeptide backbone
      amides versus side chain groups to chain expansion via chemical denaturation.  <span class='ptmri7t-x-x-120'>J. Am.
      </span><span class='ptmri7t-x-x-120'>Chem. Soc.</span>, 137(8):2984–2995, March 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [330]<span class='bibsp'>   </span></span><a id='XNettels2007-lf'></a>Daniel Nettels, Irina V Gopich, Armin Hoffmann, and Benjamin Schuler.  Ultrafast
      dynamics of protein collapse from single-molecule photon statistics.  <span class='ptmri7t-x-x-120'>Proc. Natl. Acad.
      </span><span class='ptmri7t-x-x-120'>Sci. U. S. A.</span>, 104(8):2655–2660, February 2007.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [331]<span class='bibsp'>   </span></span><a id='XSauer2014-ux'></a>Markus Sauer and Hannes Neuweiler. PET-FCS: probing rapid structural fluctuations
      of proteins and nucleic acids by single-molecule fluorescence quenching. <span class='ptmri7t-x-x-120'>Methods Mol.
      </span><span class='ptmri7t-x-x-120'>Biol.</span>, 1076:597–615, 2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [332]<span class='bibsp'>   </span></span><a id='XHaenni2013-xz'></a>Dominik  Haenni,  Franziska  Zosel,  Luc  Reymond,  Daniel  Nettels,  and  Benjamin
      Schuler.   Intramolecular distances and dynamics from the combined photon statistics
      of  single-molecule  FRET  and  photoinduced  electron  transfer.     <span class='ptmri7t-x-x-120'>J.  Phys.  Chem.  B</span>,
      117(42):13015–13028, October 2013.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [333]<span class='bibsp'>   </span></span><a id='XZosel2017-ao'></a>Franziska Zosel, Dominik Haenni, Andrea Soranno, Daniel Nettels, and Benjamin
      Schuler.   Combining short- and long-range fluorescence reporters with simulations to
      explore the intramolecular dynamics of an intrinsically disordered protein.   <span class='ptmri7t-x-x-120'>J. Chem.
      </span><span class='ptmri7t-x-x-120'>Phys.</span>, 147(15):152708, October 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [334]<span class='bibsp'>   </span></span><a id='XBanani2017-iu'></a>Salman F  Banani,  Hyun O  Lee,  Anthony A  Hyman,  and  Michael K  Rosen.
      Biomolecular condensates: organizers of cellular biochemistry. <span class='ptmri7t-x-x-120'>Nat. Rev. Mol. Cell Biol.</span>,
      18(5):285–298, May 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [335]<span class='bibsp'>   </span></span><a id='XShin2017-as'></a>Yongdae  Shin  and  Clifford P  Brangwynne.    Liquid  phase  condensation  in  cell
      physiology and disease. <span class='ptmri7t-x-x-120'>Science</span>, 357(6357), September 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [336]<span class='bibsp'>   </span></span><a id='XBrangwynne2009-xn'></a>Clifford P Brangwynne, Christian R Eckmann, David S Courson, Agata Rybarska,
      Carsten Hoege, Joebin Gharakhani, Frank Juelicher, and Anthony A Hyman. Germline
      P  granules  are  liquid  droplets  that  localize  by  controlled  dissolution/condensation.
      <span class='ptmri7t-x-x-120'>Science</span>, 324(5935):1729–1732, June 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [337]<span class='bibsp'>   </span></span><a id='XLi2012-dd'></a>Pilong Li, Sudeep Banjade, Hui-Chun Cheng, Soyeon Kim, Baoyu Chen, Liang Guo,
      Marc  Llaguno,  Javoris V  Hollingsworth,  David S  King,  Salman F  Banani,  Paul S
                                                                                      
                                                                                      
      Russo, Qiu-Xing Jiang, B Tracy Nixon, and Michael K Rosen. Phase transitions in the
      assembly of multivalent signalling proteins. <span class='ptmri7t-x-x-120'>Nature</span>, 483(7389):336–340, March 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [338]<span class='bibsp'>   </span></span><a id='XMartin2020-yc'></a>Erik W  Martin,  Alex S  Holehouse,  Ivan  Peran,  Mina  Farag,  J Jeremias  Incicco,
      Anne Bremer, Christy R Grace, Andrea Soranno, Rohit V Pappu, and Tanja Mittag.
      Valence and patterning of aromatic residues determine the phase behavior of prion-like
      domains. <span class='ptmri7t-x-x-120'>Science</span>, 367(6478):694–699, February 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [339]<span class='bibsp'>   </span></span><a id='XGuillen-Boixet2020-sn'></a>Jordina   Guillén-Boixet,   Andrii   Kopach,   Alex S   Holehouse,   Sina   Wittmann,
      Marcus  Jahnel,  Raimund  Schlüßler,  Kyoohyun  Kim,  Irmela  R  E,  Jie  Wang,  Daniel
      Mateju, Ina Poser, Shovamayee Maharana, Martine Ruer-Gruß, Doris Richter, Xiaojie
      Zhang, Young-Tae Chang, Jochen Guck, Alf Honigmann, Julia Mahamid, Anthony A
      Hyman,  Rohit V  Pappu,  Simon  Alberti,  and  Titus M  Franzmann.     RNA-Induced
      conformational  switching  and  clustering  of  G3BP  drive  stress  granule  assembly  by
      condensation. <span class='ptmri7t-x-x-120'>Cell</span>, 181(2):346–361.e17, April 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [340]<span class='bibsp'>   </span></span><a id='XWang:2018dl'></a>Jie  Wang,  Jeong-Mo  Choi,  Alex S  Holehouse,  Hyun O  Lee,  Xiaojie  Zhang,
      Marcus Jahnel, Shovamayee Maharana, Régis Lemaitre, Andrei Pozniakovsky, David
      Drechsel,  Ina  Poser,  Rohit V  Pappu,  Simon  Alberti,  and  Anthony A  Hyman.     A
      Molecular Grammar Governing the Driving Forces for Phase Separation of Prion-like
      RNA Binding Proteins. <span class='ptmri7t-x-x-120'>Cell</span>, June 2018.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [341]<span class='bibsp'>   </span></span><a id='XStockmayer1950-ad'></a>W H Stockmayer.  Light scattering in Multi-Component systems.  <span class='ptmri7t-x-x-120'>J. Chem. Phys.</span>,
      18(1):58–61, January 1950.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [342]<span class='bibsp'>   </span></span><a id='XBanerjee2017-jv'></a>Priya R Banerjee, Anthony N Milin, Mahdi Muhammad Moosa, Paulo L Onuchic,
      and Ashok A Deniz.  Reentrant phase transition drives dynamic substructure formation
      in  ribonucleoprotein  droplets.     <span class='ptmri7t-x-x-120'>Angew.  Chem.  Int.  Ed  Engl.</span>,  56(38):11354–11359,
      September 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [343]<span class='bibsp'>   </span></span><a id='XGuseva2020-nt'></a>Serafima   Guseva,   Sigrid   Milles,   Malene Ringkjøbing   Jensen,   Nicola   Salvi,
      Jean-Philippe  Kleman,  Damien  Maurin,  Rob  W H  Ruigrok,  and  Martin  Blackledge.
      Measles    virus    nucleo-    and    phosphoproteins    form    liquid-like    phase-separated
      compartments  that  promote  nucleocapsid  assembly.    <span class='ptmri7t-x-x-120'>Sci  Adv</span>,  6(14):eaaz7095,  April
      2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [344]<span class='bibsp'>   </span></span><a id='XPosey2018-dv'></a>Ammon E  Posey,  Alex S  Holehouse,  and  Rohit V  Pappu.   Chapter  one  -  phase
      separation of intrinsically disordered proteins. In Elizabeth Rhoades, editor, <span class='ptmri7t-x-x-120'>Methods in
      </span><span class='ptmri7t-x-x-120'>Enzymology</span>, volume 611, pages 1–30. Academic Press, January 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [345]<span class='bibsp'>   </span></span><a id='XSanders2020-tl'></a>David W  Sanders,  Nancy  Kedersha,  Daniel  S W  Lee,  Amy R  Strom,  Victoria
      Drake,  Joshua A  Riback,  Dan  Bracha,  Jorine M  Eeftens,  Allana  Iwanicki,  Alicia
      Wang,  Ming-Tzo  Wei,  Gena  Whitney,  Shawn M  Lyons,  Paul  Anderson,  William M
                                                                                      
                                                                                      
      Jacobs, Pavel Ivanov, and Clifford P Brangwynne. Competing Protein-RNA interaction
      networks control multiphase intracellular organization. <span class='ptmri7t-x-x-120'>Cell</span>, 181(2):306–324.e28, April
      2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [346]<span class='bibsp'>   </span></span><a id='XRiback2020-tw'></a>Joshua A  Riback,  Lian  Zhu,  Mylene C  Ferrolino,  Michele  Tolbert,  Diana M
      Mitrea,  David W  Sanders,  Ming-Tzo  Wei,  Richard W  Kriwacki,  and  Clifford P
      Brangwynne.         Composition-dependent   thermodynamics   of   intracellular   phase
      separation. <span class='ptmri7t-x-x-120'>Nature</span>, 581(7807):209–214, May 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [347]<span class='bibsp'>   </span></span><a id='XSemenov1998-rp'></a>Alexander N  Semenov  and  Michael  Rubinstein.     Thermoreversible  gelation  in
      solutions  of  associative  polymers.  1.  statics.     <span class='ptmri7t-x-x-120'>Macromolecules</span>,  31(4):1373–1385,
      February 1998.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [348]<span class='bibsp'>   </span></span><a id='XRubinstein2003-fa'></a>M Rubinstein and Ralph H Colby. <span class='ptmri7t-x-x-120'>Polymer Physics</span>. Oxford University Press, New
      York, 2003.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [349]<span class='bibsp'>   </span></span><a id='XChoi2020-nv'></a>Jeong-Mo  Choi,  Alex S  Holehouse,  and  Rohit V  Pappu.     Physical  principles
      underlying the complex biology of intracellular phase transitions.  <span class='ptmri7t-x-x-120'>Annu. Rev. Biophys.</span>,
      49:107–133, May 2020.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [350]<span class='bibsp'>   </span></span><a id='XChoi2019-wg'></a>Jeong-Mo  Choi,  Furqan  Dar,  and  Rohit V  Pappu.     LASSI:  A  lattice  model
      for   simulating   phase   transitions   of   multivalent   proteins.      <span class='ptmri7t-x-x-120'>PLoS   Comput.   Biol.</span>,
      15(10):e1007028, October 2019.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [351]<span class='bibsp'>   </span></span><a id='XPost1979-lv'></a>Carol Beth  Post  and  Bruno H  Zimm.    Internal  condensation  of  a  single  DNA
      molecule. <span class='ptmri7t-x-x-120'>Biopolymers</span>, 18(6):1487–1501, June 1979.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [352]<span class='bibsp'>   </span></span><a id='XHsieh2005-sp'></a>Ping-Kun  Hsieh,  Shin C  Chang,  Chu-Chun  Huang,  Ting-Ting  Lee,  Ching-Wen
      Hsiao,  Yi-Hen  Kou,  I-Yin  Chen,  Chung-Ke  Chang,  Tai-Huang  Huang,  and  Ming-Fu
      Chang.   Assembly of severe acute respiratory syndrome coronavirus RNA packaging
      signal into virus-like particles is nucleocapsid dependent. <span class='ptmri7t-x-x-120'>J. Virol.</span>, 79(22):13848–13855,
      November 2005.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [353]<span class='bibsp'>   </span></span><a id='XWoo1997-lo'></a>K Woo,  M Joo,  K Narayanan,  K H  Kim,  and  S Makino.    Murine  coronavirus
      packaging signal confers packaging to nonviral RNA.  <span class='ptmri7t-x-x-120'>J. Virol.</span>, 71(1):824–827, January
      1997.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [354]<span class='bibsp'>   </span></span><a id='XCologna2000-ex'></a>R Cologna and B G Hogue. Identification of a bovine coronavirus packaging signal.
      <span class='ptmri7t-x-x-120'>J. Virol.</span>, 74(1):580–583, January 2000.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [355]<span class='bibsp'>   </span></span><a id='XPool2007-jj'></a>René Pool and Peter G Bolhuis. Sampling the kinetic pathways of a micelle fusion
      and fission transition. <span class='ptmri7t-x-x-120'>J. Chem. Phys.</span>, 126(24):244703, June 2007.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [356]<span class='bibsp'>   </span></span><a id='XDenkova2010-ax'></a>Antonia G     Denkova,     Eduardo     Mendes,     and     Marc-Olivier     Coppens.
      Non-equilibrium dynamics of block copolymer micelles in solution: recent insights and
      open questions. <span class='ptmri7t-x-x-120'>Soft Matter</span>, 6(11):2351–2357, 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [357]<span class='bibsp'>   </span></span><a id='XRanganathan2020-oo'></a>Srivastav Ranganathan and Eugene I Shakhnovich. Dynamic metastable long-living
      droplets formed by sticker-spacer proteins. <span class='ptmri7t-x-x-120'>Elife</span>, 9, June 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [358]<span class='bibsp'>   </span></span><a id='XLeyrat2011-di'></a>Cédric  Leyrat,  Malene Ringkjøbing  Jensen,  Euripedes A  Ribeiro,  Jr,  Francine
      C A Gérard, Rob W H Ruigrok, Martin Blackledge, and Marc Jamin. The n0-binding
      region  of  the  vesicular  stomatitis  virus  phosphoprotein  is  globally  disordered  but
      contains transient <span class='cmmi-12'>α</span>-helices. <span class='ptmri7t-x-x-120'>Protein Sci.</span>, 20(3):542–556, 2011.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [359]<span class='bibsp'>   </span></span><a id='XFeuerstein2012-jk'></a>Sophie   Feuerstein,   Zsofia   Solyom,   Amine   Aladag,   Adrien   Favier,   Melanie
      Schwarten,  Silke  Hoffmann,  Dieter  Willbold,  and  Bernhard  Brutscher.     Transient
      structure and SH3 interaction sites in an intrinsically disordered fragment of the hepatitis
      C virus protein NS5A. <span class='ptmri7t-x-x-120'>J. Mol. Biol.</span>, 420(4-5):310–323, July 2012.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [360]<span class='bibsp'>   </span></span><a id='XJensen2008-zu'></a>Malene Ringkjøbing Jensen, Klaartje Houben, Ewen Lescop, Laurence Blanchard,
      Rob  W H  Ruigrok,  and  Martin  Blackledge.     Quantitative  conformational  analysis
      of   partially   folded   proteins   from   residual   dipolar   couplings:   application   to   the
      molecular  recognition  element  of  sendai  virus  nucleoprotein.    <span class='ptmri7t-x-x-120'>J.  Am.  Chem.  Soc.</span>,
      130(25):8055–8061, 2008.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [361]<span class='bibsp'>   </span></span><a id='XBayer2005-tp'></a>Travis S  Bayer,  Lauren N  Booth,  Scott M  Knudsen,  and  Andrew D  Ellington.
      Arginine-rich  motifs  present  multiple  interfaces  for  specific  binding  by  RNA.   <span class='ptmri7t-x-x-120'>RNA</span>,
      11(12):1848–1857, December 2005.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [362]<span class='bibsp'>   </span></span><a id='XBattiste1996-pc'></a>J L Battiste, H Mao, N S Rao, R Tan, D R Muhandiram, L E Kay, A D Frankel,
      and  J R  Williamson.    Alpha  helix-RNA  major  groove  recognition  in  an  HIV-1  rev
      peptide-RRE RNA complex. <span class='ptmri7t-x-x-120'>Science</span>, 273(5281):1547–1551, September 1996.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [363]<span class='bibsp'>   </span></span><a id='XHurst2013-iv'></a>Kelley R  Hurst,  Cheri A  Koetzner,  and  Paul S  Masters.    Characterization  of  a
      critical  interaction  between  the  coronavirus  nucleocapsid  protein  and  nonstructural
      protein  3  of  the  viral  replicase-transcriptase  complex.    <span class='ptmri7t-x-x-120'>J.  Virol.</span>,  87(16):9159–9172,
      August 2013.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [364]<span class='bibsp'>   </span></span><a id='XHurst2010-xi'></a>Kelley R   Hurst,   Rong   Ye,   Scott J   Goebel,   Priya   Jayaraman,   and   Paul S
      Masters.    An  interaction  between  the  nucleocapsid  protein  and  a  component  of  the
                                                                                      
                                                                                      
      Replicase-Transcriptase complex is crucial for the infectivity of coronavirus genomic
      RNA. <span class='ptmri7t-x-x-120'>J. Virol.</span>, 84(19):10276–10288, October 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [365]<span class='bibsp'>   </span></span><a id='XVerheije2010-se'></a>Monique H  Verheije,  Marne C  Hagemeijer,  Mustafa  Ulasli,  Fulvio  Reggiori,
      Peter  J M  Rottier,  Paul S  Masters,  and  Cornelis  A M  de Haan.    The  coronavirus
      nucleocapsid   protein   is   dynamically   associated   with   the   replication-transcription
      complexes. <span class='ptmri7t-x-x-120'>J. Virol.</span>, 84(21):11575–11579, November 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [366]<span class='bibsp'>   </span></span><a id='XSurjit2005-qa'></a>Milan
      Surjit, Ravinder Kumar, Rabi N Mishra, Malireddy K Reddy, Vincent T K Chow, and
      Sunil K Lal.  The severe acute respiratory syndrome coronavirus nucleocapsid protein
      is phosphorylated and localizes in the cytoplasm by 14-3-3-mediated translocation.  <span class='ptmri7t-x-x-120'>J.
      </span><span class='ptmri7t-x-x-120'>Virol.</span>, 79(17):11476–11486, September 2005.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [367]<span class='bibsp'>   </span></span><a id='XTimani2005-xr'></a>Khalid Amine  Timani,  Qingjiao  Liao,  Linbai  Ye,  Yingchun  Zeng,  Jing  Liu,
      Yi Zheng,  Li Ye,  Xiaojun  Yang,  Kong  Lingbao,  Jingrong  Gao,  and  Ying  Zhu.
      Nuclear/nucleolar localization properties of c-terminal nucleocapsid protein of SARS
      coronavirus. <span class='ptmri7t-x-x-120'>Virus Res.</span>, 114(1-2):23–34, December 2005.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [368]<span class='bibsp'>   </span></span><a id='XKuo2002-me'></a>Lili Kuo and Paul S Masters.  Genetic evidence for a structural interaction between
      the carboxy termini of the membrane and nucleocapsid proteins of mouse hepatitis virus.
      <span class='ptmri7t-x-x-120'>J. Virol.</span>, 76(10):4987–4999, May 2002.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [369]<span class='bibsp'>   </span></span><a id='XHurst2005-fx'></a>Kelley R  Hurst,  Lili  Kuo,  Cheri A  Koetzner,  Rong  Ye,  Bilan  Hsue,  and  Paul S
      Masters.     A  major  determinant  for  membrane  protein  interaction  localizes  to  the
      carboxy-terminal  domain  of  the  mouse  coronavirus  nucleocapsid  protein.    <span class='ptmri7t-x-x-120'>J.  Virol.</span>,
      79(21):13285–13297, 2005.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [370]<span class='bibsp'>   </span></span><a id='XVerma2006-cw'></a>Sandhya   Verma,   Valerie   Bednar,   Andrew   Blount,   and   Brenda G   Hogue.
      Identification of functionally important negatively charged residues in the carboxy end
      of mouse hepatitis coronavirus A59 nucleocapsid protein.  <span class='ptmri7t-x-x-120'>J. Virol.</span>, 80(9):4344–4355,
      May 2006.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [371]<span class='bibsp'>   </span></span><a id='XBrass2002-on'></a>Volker Brass, Elke Bieck, Roland Montserret, Benno Wölk, Jan Albert Hellings,
      Hubert E   Blum,   François   Penin,   and   Darius   Moradpour.      An   amino-terminal
      amphipathic   <span class='cmmi-12'>α</span>-Helix   mediates   membrane   association   of   the   hepatitis   C   virus
      nonstructural protein 5A. <span class='ptmri7t-x-x-120'>J. Biol. Chem.</span>, 277(10):8130–8139, March 2002.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [372]<span class='bibsp'>   </span></span><a id='XBraun2017-wi'></a>Anthony R  Braun,  Michael M  Lacy,  Vanessa C  Ducas,  Elizabeth  Rhoades,  and
      Jonathan N  Sachs.     <span class='cmmi-12'>α</span>-Synuclein’s  uniquely  long  amphipathic  helix  enhances  its
      membrane binding and remodeling capacity.   <span class='ptmri7t-x-x-120'>J. Membr. Biol.</span>, 250(2):183–193, April
      2017.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [373]<span class='bibsp'>   </span></span><a id='XWyman1990-qn'></a>Jeffries Wyman and Stanley J Gill.  <span class='ptmri7t-x-x-120'>Binding and Linkage: Functional Chemistry of
      </span><span class='ptmri7t-x-x-120'>Biological Macromolecules</span>. University Science Books, 1990.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [374]<span class='bibsp'>   </span></span><a id='XNikolic2017-qx'></a>Jovan    Nikolic,    Romain    Le Bars,    Zoé    Lama,    Nathalie    Scrima,    Cécile
      Lagaudrière-Gesbert,  Yves  Gaudin,  and  Danielle  Blondel.    Negri  bodies  are  viral
      factories with properties of liquid organelles. <span class='ptmri7t-x-x-120'>Nat. Commun.</span>, 8(1):58, July 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [375]<span class='bibsp'>   </span></span><a id='XMetrick2020-na'></a>Claire M Metrick, Andrea L Koenigsberg, and Ekaterina E Heldwein.  Conserved
      outer  tegument  component  UL11  from  herpes  simplex  virus  1  is  an  intrinsically
      disordered, RNA-Binding protein. <span class='ptmri7t-x-x-120'>MBio</span>, 11(3), May 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [376]<span class='bibsp'>   </span></span><a id='XHeinrich2018-bu'></a>Bianca S Heinrich, Zoltan Maliga, David A Stein, Anthony A Hyman, and Sean
      P J  Whelan.     Phase  transitions  drive  the  formation  of  vesicular  stomatitis  virus
      replication compartments. <span class='ptmri7t-x-x-120'>MBio</span>, 9(5), September 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [377]<span class='bibsp'>   </span></span><a id='XZhou2019-uv'></a>Yuqin Zhou, Justin M Su, Charles E Samuel, and Dzwokai Ma. Measles virus forms
      inclusion bodies with properties of liquid organelles. <span class='ptmri7t-x-x-120'>J. Virol.</span>, 93(21), November 2019.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [378]<span class='bibsp'>   </span></span><a id='XMonette2020-do'></a>Anne  Monette,  Meijuan  Niu,  Lois  Chen,  Shringar  Rao,  Robert James  Gorelick,
                                                                                      
                                                                                      
      and  Andrew John  Mouland.   Pan-retroviral  Nucleocapsid-Mediated  phase  separation
      regulates  genomic  RNA  positioning  and  trafficking.   <span class='ptmri7t-x-x-120'>Cell  Rep.</span>,  31(3):107520,  April
      2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [379]<span class='bibsp'>   </span></span><a id='XKlein2020-vw'></a>Steffen   Klein,   Mirko   Cortese,   Sophie L   Winter,   Moritz   Wachsmuth-Melm,
      Christopher J  Neufeldt,  Berati  Cerikan,  Megan L  Stanifer,  Steeve  Boulant,  Ralf
      Bartenschlager, and Petr Chlanda.  SARS-CoV-2 structure and replication characterized
      by in situ cryo-electron tomography. June 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [380]<span class='bibsp'>   </span></span><a id='XCong2017-hw'></a>Yingying  Cong,  Franziska  Kriegenburg,  Cornelis  A M  de Haan,  and  Fulvio
      Reggiori.  Coronavirus nucleocapsid proteins assemble constitutively in high molecular
      oligomers. <span class='ptmri7t-x-x-120'>Sci. Rep.</span>, 7(1):5740, July 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [381]<span class='bibsp'>   </span></span><a id='XChang2014-fy'></a>Chung-Ke  Chang,  Ming-Hon  Hou,  Chi-Fon  Chang,  Chwan-Deng  Hsiao,  and
      Tai-Huang Huang.  The SARS coronavirus nucleocapsid protein–forms and functions.
      <span class='ptmri7t-x-x-120'>Antiviral Res.</span>, 103:39–50, 2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [382]<span class='bibsp'>   </span></span><a id='XBorodavka2013-xr'></a>Alexander Borodavka, Roman Tuma, and Peter G Stockley. A two-stage mechanism
      of  viral  RNA  compaction  revealed  by  single  molecule  fluorescence.     <span class='ptmri7t-x-x-120'>RNA  Biol.</span>,
      10(4):481–489, April 2013.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [383]<span class='bibsp'>   </span></span><a id='XHe2004-xw'></a>Runtao He, Andrew Leeson, Melissa Ballantine, Anton Andonov, Lindsay Baker,
      Frederick  Dobie,  Yan  Li,  Nathalie  Bastien,  Heinz  Feldmann,  Ute  Strocher,  Steven
      Theriault,  Todd  Cutts,  Jingxin  Cao,  Timothy F  Booth,  Frank A  Plummer,  Shaun
      Tyler, and Xuguang Li.   Characterization of protein–protein interactions between the
      nucleocapsid  protein  and  membrane  protein  of  the  SARS  coronavirus.    <span class='ptmri7t-x-x-120'>Virus  Res.</span>,
      105(2):121–125, October 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [384]<span class='bibsp'>   </span></span><a id='XBergeron-Sandoval2018-oo'></a>Louis-Philippe   Bergeron-Sandoval,   Hossein Khadivi   Heris,   Catherine   Chang,
      Caitlin E  Cornell,  Sarah L  Keller,  Paul  François,  Adam G  Hendricks,  Allen J
      Ehrlicher,   Rohit V   Pappu,   and   Stephen W   Michnick.       Endocytosis   caused   by
      liquid-liquid phase separation of proteins. December 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [385]<span class='bibsp'>   </span></span><a id='XBergeron-Sandoval2018-mb'></a>Louis-Philippe Bergeron-Sandoval and Stephen W Michnick.  Mechanics, structure
      and function of biopolymer condensates.  <span class='ptmri7t-x-x-120'>J. Mol. Biol.</span>, 430(23):4754–4761, November
      2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [386]<span class='bibsp'>   </span></span><a id='XHolmstrom2018-yk'></a>Erik D  Holmstrom,  Daniel  Nettels,  and  Benjamin  Schuler.      Conformational
      plasticity   of   hepatitis   C   virus   core   protein   enables   RNA-Induced   formation   of
      nucleocapsid-like particles. <span class='ptmri7t-x-x-120'>J. Mol. Biol.</span>, 430(16):2453–2467, August 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [387]<span class='bibsp'>   </span></span><a id='XRodriguez2004-dx'></a>Lorena  Rodríguez,  Isabel  Cuesta,  Ana  Asenjo,  and  Nieves  Villanueva.   Human
                                                                                      
                                                                                      
      respiratory syncytial virus matrix protein is an RNA-binding protein: binding properties,
      location and identity of the RNA contact residues.   <span class='ptmri7t-x-x-120'>J. Gen. Virol.</span>, 85(Pt 3):709–719,
      March 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [388]<span class='bibsp'>   </span></span><a id='XLinger2004-lk'></a>Benjamin R   Linger,   Lyudmyla   Kunovska,   Richard J   Kuhn,   and   Barbara L
      Golden.   Sindbis virus nucleocapsid assembly: RNA folding promotes capsid protein
      dimerization. <span class='ptmri7t-x-x-120'>RNA</span>, 10(1):128–138, January 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [389]<span class='bibsp'>   </span></span><a id='XZuniga2007-zc'></a>Sonia Zúñiga, Isabel Sola, Jose L Moreno, Patricia Sabella, Juan Plana-Durán,
      and Luis Enjuanes.  Coronavirus nucleocapsid protein is an RNA chaperone.  <span class='ptmri7t-x-x-120'>Virology</span>,
      357(2):215–227, January 2007.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [390]<span class='bibsp'>   </span></span><a id='XLuo2005-qs'></a>Haibin Luo, Qing Chen, Jing Chen, Kaixian Chen, Xu Shen, and Hualiang Jiang.
      The nucleocapsid protein of SARS coronavirus has a high binding affinity to the human
      cellular heterogeneous nuclear ribonucleoprotein A1.  <span class='ptmri7t-x-x-120'>FEBS Lett.</span>, 579(12):2623–2628,
      May 2005.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [391]<span class='bibsp'>   </span></span><a id='XYang2020-iq'></a>Peiguo   Yang,   Cécile   Mathieu,   Regina-Maria   Kolaitis,   Peipei   Zhang,   James
      Messing,  Ugur  Yurtsever,  Zemin  Yang,  Jinjun  Wu,  Yuxin  Li,  Qingfei  Pan,  Jiyang
      Yu,  Erik W  Martin,  Tanja  Mittag,  Hong Joo  Kim,  and  J Paul Taylor.    G3BP1  is
      a  tunable  switch  that  triggers  phase  separation  to  assemble  stress  granules.    <span class='ptmri7t-x-x-120'>Cell</span>,
      181(2):325–345.e28, April 2020.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [392]<span class='bibsp'>   </span></span><a id='XVan_Rosmalen2020-kk'></a>Mariska G M van Rosmalen, Douwe Kamsma, Andreas S Biebricher, Chenglei Li,
      Adam Zlotnick, Wouter H Roos, and Gijs J L Wuite. Revealing in real-time a multistep
      assembly mechanism for SV40 virus-like particles.  <span class='ptmri7t-x-x-120'>Science Advances</span>, 6(16):eaaz1639,
      April 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [393]<span class='bibsp'>   </span></span><a id='XPatel2015-hh'></a>Avinash  Patel,  Hyun O  Lee,  Louise  Jawerth,  Shovamayee  Maharana,  Marcus
      Jahnel, Marco Y Hein, Stoyno Stoynov, Julia Mahamid, Shambaditya Saha, Titus M
      Franzmann, Andrej Pozniakovski, Ina Poser, Nicola Maghelli, Loic A Royer, Martin
      Weigert,  Eugene W  Myers,  Stephan  Grill,  David  Drechsel,  Anthony A  Hyman,  and
      Simon Alberti. A liquid-to-solid phase transition of the ALS protein FUS accelerated by
      disease mutation. <span class='ptmri7t-x-x-120'>Cell</span>, 162(5):1066–1077, August 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [394]<span class='bibsp'>   </span></span><a id='XAlberti2019-zm'></a>Simon Alberti and Dorothee Dormann.  Liquid-Liquid phase separation in disease.
      <span class='ptmri7t-x-x-120'>Annu. Rev. Genet.</span>, 53:171–194, December 2019.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [395]<span class='bibsp'>   </span></span><a id='XWeber2012-ga'></a>Stephanie C Weber and Clifford P Brangwynne. Getting RNA and protein in phase.
      <span class='ptmri7t-x-x-120'>Cell</span>, 149(6):1188–1191, June 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [396]<span class='bibsp'>   </span></span><a id='XZeng2020-ko'></a>X Zeng,  A S  Holehouse,  T Mittag,  A Chilkoti,  and  R V  Pappu.    Connecting
                                                                                      
                                                                                      
      coil-to-globule transitions to full phase diagrams for intrinsically disordered proteins.
      May 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [397]<span class='bibsp'>   </span></span><a id='XDignon2018-jf'></a>Gregory L  Dignon,  Wenwei  Zheng,  Robert B  Best,  Young C  Kim,  and  Jeetain
      Mittal.  Relation between single-molecule properties and phase behavior of intrinsically
      disordered proteins. <span class='ptmri7t-x-x-120'>Proc. Natl. Acad. Sci. U. S. A.</span>, 115(40):9929–9934, October 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [398]<span class='bibsp'>   </span></span><a id='XVitalis2009-gh'></a>Andreas Vitalis and Rohit V Pappu. ABSINTH: A new continuum solvation model
      for simulations of polypeptides in aqueous solutions. <span class='ptmri7t-x-x-120'>J. Comput. Chem.</span>, 30(5):673–699,
      April 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [399]<span class='bibsp'>   </span></span><a id='XMittal2015-sl'></a>Anuradha  Mittal,  Rahul K  Das,  Andreas  Vitalis,  and  Rohit V  Pappu.      The
      ABSINTH  implicit  solvation  model  and  forcefield  paradigm  for  use  in  simulations
      of  intrinsically  disordered  proteins.      In  Monika  Fuxreiter,  editor,  <span class='ptmri7t-x-x-120'>Computational
      </span><span class='ptmri7t-x-x-120'>Approaches to Protein Dynamics: From Quantum to Coarse-Grained Methods</span>, pages
      188–222. CRC Press, 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [400]<span class='bibsp'>   </span></span><a id='XMao2012-hr'></a>Albert H  Mao  and  Rohit V  Pappu.     Crystal  lattice  properties  fully  determine
      short-range  interaction  parameters  for  alkali  and  halide  ions.      <span class='ptmri7t-x-x-120'>J.  Chem.  Phys.</span>,
      137(6):064104, August 2012.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [401]<span class='bibsp'>   </span></span><a id='XSherry2017-sr'></a>Kathryn P  Sherry,  Rahul K  Das,  Rohit V  Pappu,  and  Doug  Barrick.    Control
      of   transcriptional   activity   by   design   of   charge   patterning   in   the   intrinsically
      disordered  RAM  region  of  the  notch  receptor.     <span class='ptmri7t-x-x-120'>Proc.  Natl.  Acad.  Sci.  U.  S.  A.</span>,
      114(44):E9243–E9252, October 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [402]<span class='bibsp'>   </span></span><a id='XHolehouse2019-cj'></a>Alex S Holehouse and Rohit V Pappu. PIMMS (0.24 pre-beta), December 2019.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [403]<span class='bibsp'>   </span></span><a id='XZimmerman2020covid-pocket'></a>Maxwell I.  Zimmerman,  Justin R.  Porter,  Michael D.  Ward,  Sukrit  Singh,  Neha
      Vithani,  Artur  Meller,  Upasana L.  Mallimadugula,  Catherine E.  Kuhn,  Jonathan H.
      Borowsky,  Rafal P.  Wiewiora,  Matthew  F. D.  Hurley,  Aoife M  Harbison,  Carl A
      Fogarty,  Joseph E.  Coffland,  Elisa  Fadda,  Vincent A.  Voelz,  John D.  Chodera,  and
      Gregory R. Bowman. Citizen scientists create an exascale computer to combat covid-19.
      <span class='ptmri7t-x-x-120'>bioRxiv</span>, 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [404]<span class='bibsp'>   </span></span><a id='Xzhou_pneumonia_2020'></a>Peng Zhou, Xing-Lou Yang, Xian-Guang Wang, Ben Hu, Lei Zhang, Wei Zhang,
      Hao-Rui Si, Yan Zhu, Bei Li, Chao-Lin Huang, Hui-Dong Chen, Jing Chen, Yun Luo,
      Hua Guo, Ren-Di Jiang, Mei-Qin Liu, Ying Chen, Xu-Rui Shen, Xi Wang, Xiao-Shuang
      Zheng, Kai Zhao, Quan-Jiao Chen, Fei Deng, Lin-Lin Liu, Bing Yan, Fa-Xian Zhan,
      Yan-Yi Wang, Geng-Fu Xiao, and Zheng-Li Shi. A pneumonia outbreak associated with
      a new coronavirus of probable bat origin. <span class='ptmri7t-x-x-120'>Nature</span>, 579(7798):270–273, March 2020.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [405]<span class='bibsp'>   </span></span><a id='Xliu_reproductive_2020'></a>Ying  Liu,  Albert A  Gayle,  Annelies  Wilder-Smith,  and  Joacim  Rocklöv.    The
      reproductive number of COVID-19 is higher compared to SARS coronavirus.  <span class='ptmri7t-x-x-120'>Journal
      </span><span class='ptmri7t-x-x-120'>of Travel Medicine</span>, 27(2):taaa021, March 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [406]<span class='bibsp'>   </span></span><a id='Xsorci_why_2020'></a>Gabriele Sorci, Bruno Faivre, and Serge Morand. Why does COVID-19 case fatality
      rate  vary  among  countries?   preprint,  Infectious  Diseases  (except  HIV/AIDS),  April
      2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [407]<span class='bibsp'>   </span></span><a id='Xkhafaie_cross-country_2020'></a>Morteza Abdullatif Khafaie and Fakher Rahim.  Cross-country comparison of case
      fatality rates of covid-19/sars-cov-2.  <span class='ptmri7t-x-x-120'>Osong Public Health and Research Perspectives</span>,
      11(2):74–80, April 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [408]<span class='bibsp'>   </span></span><a id='Xmahase_coronavirus:_2020'></a>Elisabeth Mahase.   Coronavirus: covid-19 has killed more people than SARS and
      MERS combined, despite lower case fatality rate. <span class='ptmri7t-x-x-120'>BMJ</span>, page m641, February 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [409]<span class='bibsp'>   </span></span><a id='Xonder_case-fatality_2020'></a>Graziano  Onder,  Giovanni  Rezza,  and  Silvio  Brusaferro.    Case-fatality  rate  and
      characteristics of patients dying in relation to covid-19 in italy. <span class='ptmri7t-x-x-120'>JAMA</span>, March 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [410]<span class='bibsp'>   </span></span><a id='Xferreira_molecular_2015'></a>Leonardo Ferreira, Ricardo dos Santos, Glaucius Oliva, and Adriano Andricopulo.
                                                                                      
                                                                                      
      Molecular   docking   and   structure-based   drug   design   strategies.         <span class='ptmri7t-x-x-120'>Molecules</span>,
      20(7):13384–13421, July 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [411]<span class='bibsp'>   </span></span><a id='XKohlhoff:2014bo'></a>Kai J Kohlhoff, Diwakar Shukla, Morgan Lawrenz, Gregory R Bowman, David E
      Konerding, Dan Belov, Russ B Altman, and Vijay S Pande.  Cloud-based simulations
      on Google Exacycle reveal ligand modulation of GPCR activation pathways.   <span class='ptmri7t-x-x-120'>Nature
      </span><span class='ptmri7t-x-x-120'>chemistry</span>, 6(1):15–21, January 2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [412]<span class='bibsp'>   </span></span><a id='XShukla:2014jp'></a>Diwakar  Shukla,  Yilin  Meng,  Benoît  Roux,  and  Vijay S  Pande.     Activation
      pathway  of  Src  kinase  reveals  intermediate  states  as  targets  for  drug  design.   <span class='ptmri7t-x-x-120'>Nature
      </span><span class='ptmri7t-x-x-120'>communications</span>, 5(1):3397, March 2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [413]<span class='bibsp'>   </span></span><a id='Xhart_designing_2017'></a>Kathryn M. Hart, Katelyn E. Moeder, Chris M. W. Ho, Maxwell I. Zimmerman,
      Thomas E. Frederick, and Gregory R. Bowman.  Designing small molecules to target
      cryptic  pockets  yields  both  positive  and  negative  allosteric  modulators.   <span class='ptmri7t-x-x-120'>PLOS  ONE</span>,
      12(6):e0178678, June 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [414]<span class='bibsp'>   </span></span><a id='Xwang_cap-snatching_2020'></a>Wenjie Wang, Woo-Jin Shin, Bojie Zhang, Younho Choi, Ji-Seung Yoo, Maxwell I.
      Zimmerman, Thomas E. Frederick, Gregory R. Bowman, Michael L. Gross, Daisy W.
      Leung, Jae U. Jung, and Gaya K. Amarasinghe. The cap-snatching sftsv endonuclease
      domain is an antiviral target. <span class='ptmri7t-x-x-120'>Cell Reports</span>, 30(1):153–163.e5, January 2020.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [415]<span class='bibsp'>   </span></span><a id='Xkirchdoerfer_stabilized_2018'></a>Robert N.   Kirchdoerfer,   Nianshuang   Wang,   Jesper   Pallesen,   Daniel   Wrapp,
      Hannah L. Turner, Christopher A. Cottrell, Kizzmekia S. Corbett, Barney S. Graham,
      Jason S. McLellan, and Andrew B. Ward.  Stabilized coronavirus spikes are resistant
      to  conformational  changes  induced  by  receptor  recognition  or  proteolysis.   <span class='ptmri7t-x-x-120'>Scientific
      </span><span class='ptmri7t-x-x-120'>Reports</span>, 8(1):15701, December 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [416]<span class='bibsp'>   </span></span><a id='Xwalls_structure_2020'></a>Alexandra C.   Walls,   Young-Jun   Park,   M. Alejandra   Tortorici,   Abigail   Wall,
      Andrew T. McGuire, and David Veesler.   Structure, function, and antigenicity of the
      sars-cov-2 spike glycoprotein. <span class='ptmri7t-x-x-120'>Cell</span>, 181(2):281–292.e6, April 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [417]<span class='bibsp'>   </span></span><a id='Xwrapp_cryo-em_2020'></a>Daniel  Wrapp,  Nianshuang  Wang,  Kizzmekia S.  Corbett,  Jory A.  Goldsmith,
      Ching-Lin  Hsieh,  Olubukola  Abiona,  Barney S.  Graham,  and  Jason S.  McLellan.
      Cryo-EM  structure  of  the  2019-nCoV  spike  in  the  prefusion  conformation.   <span class='ptmri7t-x-x-120'>Science</span>,
      367(6483):1260–1263, March 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [418]<span class='bibsp'>   </span></span><a id='Xzhang_angiotensin-converting_2020'></a>Haibo  Zhang,  Josef M.  Penninger,  Yimin  Li,  Nanshan  Zhong,  and  Arthur S.
      Slutsky.      Angiotensin-converting  enzyme  2  (Ace2)  as  a  SARS-CoV-2  receptor:
      molecular  mechanisms  and  potential  therapeutic  target.     <span class='ptmri7t-x-x-120'>Intensive  Care  Medicine</span>,
      46(4):586–590, April 2020.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [419]<span class='bibsp'>   </span></span><a id='Xhoffmann_sars-cov-2_2020'></a>Markus Hoffmann, Hannah Kleine-Weber, Simon Schroeder, Nadine Krüger, Tanja
      Herrler, Sandra Erichsen, Tobias S. Schiergens, Georg Herrler, Nai-Huei Wu, Andreas
      Nitsche,  Marcel A.  Müller,  Christian  Drosten,  and  Stefan  Pöhlmann.    Sars-cov-2
      cell entry depends on ace2 and tmprss2 and is blocked by a clinically proven protease
      inhibitor. <span class='ptmri7t-x-x-120'>Cell</span>, 181(2):271–280.e8, April 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [420]<span class='bibsp'>   </span></span><a id='Xwatanabe_vulnerabilities_2020'></a>Yasunori Watanabe, Zachary T. Berndsen, Jayna Raghwani, Gemma E. Seabright,
      Joel D.  Allen,  Oliver G.  Pybus,  Jason S.  McLellan,  Ian A.  Wilson,  Thomas A.
      Bowden,  Andrew B.  Ward,  and  Max  Crispin.   Vulnerabilities  in  coronavirus  glycan
      shields despite extensive glycosylation. <span class='ptmri7t-x-x-120'>Nature Communications</span>, 11(1):2688, December
      2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [421]<span class='bibsp'>   </span></span><a id='Xzimmerman_how_2016'></a>M.I. Zimmerman and G.R. Bowman.  How to run fast simulations.  In <span class='ptmri7t-x-x-120'>Methods in
      </span><span class='ptmri7t-x-x-120'>Enzymology</span>, volume 578, pages 213–225. Elsevier, 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [422]<span class='bibsp'>   </span></span><a id='Xyuan_cryo-em_2017'></a>Yuan  Yuan,  Duanfang  Cao,  Yanfang  Zhang,  Jun  Ma,  Jianxun  Qi,  Qihui  Wang,
      Guangwen Lu, Ying Wu, Jinghua Yan, Yi Shi, Xinzheng Zhang, and George F. Gao.
      Cryo-EM  structures  of  MERS-CoV  and  SARS-CoV  spike  glycoproteins  reveal  the
      dynamic receptor binding domains. <span class='ptmri7t-x-x-120'>Nature Communications</span>, 8(1):15092, April 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [423]<span class='bibsp'>   </span></span><a id='Xhuo_neutralization_2020'></a>Jiandong   Huo,   Yuguang   Zhao,   Jingshan   Ren,   Daming   Zhou,   Helen M.E.
                                                                                      
                                                                                      
      Duyvesteyn,  Helen M.  Ginn,  Loic  Carrique,  Tomas  Malinauskas,  Reinis R.  Ruza,
      Pranav N.M.   Shah,   Tiong Kit   Tan,   Pramila   Rijal,   Naomi   Coombes,   Kevin R.
      Bewley,  Julia A.  Tree,  Julika  Radecke,  Neil G.  Paterson,  Piyada  Supasa,  Juthathip
      Mongkolsapaya, Gavin R. Screaton, Miles Carroll, Alain Townsend, Elizabeth E. Fry,
      Raymond J. Owens, and David I. Stuart. Neutralization of sars-cov-2 by destruction of
      the prefusion spike. <span class='ptmri7t-x-x-120'>Cell Host &amp; Microbe</span>, page S1931312820303516, June 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [424]<span class='bibsp'>   </span></span><a id='Xzhong_epidemiology_2003'></a>Ns Zhong,  Bj Zheng,  Ym Li,  Llm  Poon,  Zh Xie,  Kh Chan,  Ph Li,  Sy Tan,
      Q Chang,   Jp Xie,   Xq Liu,   J Xu,   Dx Li,   Ky Yuen,   Jsm   Peiris,   and   Y Guan.
      Epidemiology  and  cause  of  severe  acute  respiratory  syndrome  (Sars)  in  Guangdong,
      People’s  Republic  of  China,  in  February,  2003.   <span class='ptmri7t-x-x-120'>The  Lancet</span>,  362(9393):1353–1358,
      October 2003.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [425]<span class='bibsp'>   </span></span><a id='Xvan_der_hoek_identification_2004'></a>Lia van der Hoek, Krzysztof Pyrc, Maarten F Jebbink, Wilma Vermeulen-Oost, Ron
      J M Berkhout, Katja C Wolthers, Pauline M E Wertheim-van Dillen, Jos Kaandorp,
      Joke Spaargaren, and Ben Berkhout. Identification of a new human coronavirus. <span class='ptmri7t-x-x-120'>Nature
      </span><span class='ptmri7t-x-x-120'>Medicine</span>, 10(4):368–373, April 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [426]<span class='bibsp'>   </span></span><a id='Xwu_crystal_2009'></a>K. Wu,  W. Li,  G. Peng,  and  F. Li.      Crystal  structure  of  NL63  respiratory
      coronavirus receptor-binding domain complexed with its human receptor.  <span class='ptmri7t-x-x-120'>Proceedings
      </span><span class='ptmri7t-x-x-120'>of the National Academy of Sciences</span>, 106(47):19970–19974, November 2009.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [427]<span class='bibsp'>   </span></span><a id='Xshang_structural_2020'></a>Jian Shang, Gang Ye, Ke Shi, Yushun Wan, Chuming Luo, Hideki Aihara, Qibin
      Geng,  Ashley  Auerbach,  and  Fang  Li.    Structural  basis  of  receptor  recognition  by
      SARS-CoV-2. <span class='ptmri7t-x-x-120'>Nature</span>, 581(7807):221–224, May 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [428]<span class='bibsp'>   </span></span><a id='Xgraham_structure-based_2019'></a>Barney S. Graham, Morgan S.A. Gilman, and Jason S. McLellan. Structure-based
      vaccine antigen design. <span class='ptmri7t-x-x-120'>Annual Review of Medicine</span>, 70(1):91–104, January 2019.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [429]<span class='bibsp'>   </span></span><a id='Xzhang_crystal_2020'></a>Linlin Zhang, Daizong Lin, Xinyuanyuan Sun, Ute Curth, Christian Drosten, Lucie
      Sauerhering,  Stephan Becker,  Katharina Rox,  and Rolf Hilgenfeld.   Crystal structure
      of SARS-CoV-2 main protease provides a basis for design of improved <span class='cmmi-12'>α</span>-ketoamide
      inhibitors. <span class='ptmri7t-x-x-120'>Science</span>, page eabb3405, March 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [430]<span class='bibsp'>   </span></span><a id='Xgraziano_sars_2006'></a>Vito  Graziano,  William J.  McGrath,  Lin  Yang,  and  Walter F.  Mangel.   Sars  cov
      main proteinase: the monomerdimer equilibrium dissociation constant.   <span class='ptmri7t-x-x-120'>Biochemistry</span>,
      45(49):14632–14641, December 2006.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [431]<span class='bibsp'>   </span></span><a id='Xgoyal_targeting_2020'></a>Bhupesh Goyal and Deepti Goyal.  Targeting the dimerization of the main protease
      of coronaviruses: a potential broad-spectrum therapeutic strategy.  <span class='ptmri7t-x-x-120'>ACS Combinatorial
      </span><span class='ptmri7t-x-x-120'>Science</span>, 22(6):297–305, June 2020.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [432]<span class='bibsp'>   </span></span><a id='Xdinesh_structural_2020'></a>Dhurvas Chandrasekaran Dinesh, Dominika Chalupska, Jan Silhan, Vaclav Veverka,
      and   Evzen   Boura.      Structural   basis   of   RNA   recognition   by   the   SARS-CoV-2
      nucleocapsid phosphoprotein. preprint, Biochemistry, April 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [433]<span class='bibsp'>   </span></span><a id='Xhuang_charmm36_2013'></a>Jing Huang and Alexander D. MacKerell.  CHARMM36 all-atom additive protein
      force field: Validation based on comparison to NMR data.   <span class='ptmri7t-x-x-120'>Journal of Computational
      </span><span class='ptmri7t-x-x-120'>Chemistry</span>, 34(25):2135–2145, September 2013.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [434]<span class='bibsp'>   </span></span><a id='Xhendlich_ligsite:_1997'></a>Manfred  Hendlich,  Friedrich  Rippmann,  and  Gerhard  Barnickel.      LIGSITE:
      automatic and efficient detection of potential small molecule-binding sites in proteins.
      <span class='ptmri7t-x-x-120'>Journal of Molecular Graphics and Modelling</span>, 15(6):359–363, December 1997.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [435]<span class='bibsp'>   </span></span><a id='Xthe_uniprot_consortium_uniprot:_2019'></a>The UniProt Consortium.  UniProt: a worldwide hub of protein knowledge.  <span class='ptmri7t-x-x-120'>Nucleic
      </span><span class='ptmri7t-x-x-120'>Acids Research</span>, 47(D1):D506–D515, January 2019.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [436]<span class='bibsp'>   </span></span><a id='Xedgar_muscle:_2004'></a>R. C. Edgar.  MUSCLE: multiple sequence alignment with high accuracy and high
      throughput. <span class='ptmri7t-x-x-120'>Nucleic Acids Research</span>, 32(5):1792–1797, March 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [437]<span class='bibsp'>   </span></span><a id='Xbrown_antagonism_2020'></a>Cameron A. Brown, Liya Hu, Zhizeng Sun, Meha P. Patel, Sukrit Singh, Justin R.
                                                                                      
                                                                                      
      Porter,  Banumathi  Sankaran,  B. V. Venkataram  Prasad,  Gregory R.  Bowman,  and
      Timothy Palzkill.  Antagonism between substitutions in <span class='cmmi-12'>β</span>-lactamase explains a path not
      taken  in  the  evolution  of  bacterial  drug  resistance.   <span class='ptmri7t-x-x-120'>Journal  of  Biological  Chemistry</span>,
      295(21):7376–7390, May 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [438]<span class='bibsp'>   </span></span><a id='Xknowles_enzyme_1991'></a>Jeremy R.  Knowles.      Enzyme  catalysis:  not  different,  just  better.      <span class='ptmri7t-x-x-120'>Nature</span>,
      350(6314):121–124, March 1991.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [439]<span class='bibsp'>   </span></span><a id='Xbreen_epistasis_2012'></a>Michael S.  Breen,  Carsten  Kemena,  Peter K.  Vlasov,  Cedric  Notredame,  and
      Fyodor A. Kondrashov.  Epistasis as the primary factor in molecular evolution.  <span class='ptmri7t-x-x-120'>Nature</span>,
      490(7421):535–538, October 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [440]<span class='bibsp'>   </span></span><a id='Xde_visser_empirical_2014'></a>J. Arjan G.M. de Visser and Joachim Krug.  Empirical fitness landscapes and the
      predictability of evolution. <span class='ptmri7t-x-x-120'>Nature Reviews Genetics</span>, 15(7):480–490, July 2014.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [441]<span class='bibsp'>   </span></span><a id='Xwells_additivity_1990'></a>James A.  Wells.     Additivity  of  mutational  effects  in  proteins.     <span class='ptmri7t-x-x-120'>Biochemistry</span>,
      29(37):8509–8517, September 1990.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [442]<span class='bibsp'>   </span></span><a id='Xweinreich_darwinian_2006'></a>D. M. Weinreich.  Darwinian evolution can follow only very few mutational paths
      to fitter proteins. <span class='ptmri7t-x-x-120'>Science</span>, 312(5770):111–114, April 2006.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [443]<span class='bibsp'>   </span></span><a id='Xdellus-gur_what_2013'></a>Eynat Dellus-Gur, Agnes Toth-Petroczy, Mikael Elias, and Dan S. Tawfik.   What
      makes  a  protein  fold  amenable  to  functional  innovation?  Fold  polarity  and  stability
      trade-offs. <span class='ptmri7t-x-x-120'>Journal of Molecular Biology</span>, 425(14):2609–2621, July 2013.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [444]<span class='bibsp'>   </span></span><a id='Xdoucet_flexibility_2009'></a>Nicolas  Doucet,  Eric D.  Watt,  and  J. Patrick  Loria.   The  flexibility  of  a  distant
      loop modulates active site motion and product release in ribonuclease a.  <span class='ptmri7t-x-x-120'>Biochemistry</span>,
      48(30):7160–7168, August 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [445]<span class='bibsp'>   </span></span><a id='Xwhittier_conformational_2013'></a>S. K.   Whittier,   A. C.   Hengge,   and   J. P.   Loria.       Conformational   motions
      regulate   phosphoryl   transfer   in   related   protein   tyrosine   phosphatases.      <span class='ptmri7t-x-x-120'>Science</span>,
      341(6148):899–903, August 2013.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [446]<span class='bibsp'>   </span></span><a id='Xjames_conformational_2003'></a>Leo C. James and Dan S. Tawfik. Conformational diversity and protein evolution –
      a 60-year-old hypothesis revisited. <span class='ptmri7t-x-x-120'>Trends in Biochemical Sciences</span>, 28(7):361–368, July
      2003.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [447]<span class='bibsp'>   </span></span><a id='Xtokuriki_protein_2009'></a>N. Tokuriki  and  D. S.  Tawfik.     Protein  dynamism  and  evolvability.     <span class='ptmri7t-x-x-120'>Science</span>,
      324(5924):203–207, April 2009.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [448]<span class='bibsp'>   </span></span><a id='Xpetrovic_conformational_2018'></a>Dušan Petrović, Valeria A. Risso, Shina Caroline Lynn Kamerlin, and Jose M.
      Sanchez-Ruiz.  Conformational dynamics and enzyme evolution.  <span class='ptmri7t-x-x-120'>Journal of The Royal
      </span><span class='ptmri7t-x-x-120'>Society Interface</span>, 15(144):20180330, July 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [449]<span class='bibsp'>   </span></span><a id='Xlivermore_-lactamase_2006'></a>David M.   Livermore   and   Neil   Woodford.         The   <span class='cmmi-12'>β</span>-lactamase   threat   in
      enterobacteriaceae,   pseudomonas   and   acinetobacter.        <span class='ptmri7t-x-x-120'>Trends   in   Microbiology</span>,
      14(9):413–420, September 2006.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [450]<span class='bibsp'>   </span></span><a id='Xfisher_bacterial_2005'></a>Jed F.  Fisher,  Samy O.  Meroueh,  and  Shahriar  Mobashery.   Bacterial  resistance
      to <span class='cmmi-12'>β</span>-lactam antibiotics: compelling opportunism, compelling opportunity <sup><span class='ptmr8c-x-x-120'>†</span></sup>.  <span class='ptmri7t-x-x-120'>Chemical
      </span><span class='ptmri7t-x-x-120'>Reviews</span>, 105(2):395–424, February 2005.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [451]<span class='bibsp'>   </span></span><a id='Xambler_standard_1991'></a>R P Ambler, A F W Coulson, J M Frère, J M Ghuysen, B Joris, M Forsman,
      R C Levesque, G Tiraby, and S G Waley.  A standard numbering scheme for the class
      A <span class='cmmi-12'>β</span>-lactamases. <span class='ptmri7t-x-x-120'>Biochemical Journal</span>, 276(1):269–270, May 1991.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [452]<span class='bibsp'>   </span></span><a id='Xbush_epidemiological_2011'></a>Karen Bush and Jed F. Fisher.  Epidemiological expansion, structural studies, and
      clinical challenges of new <span class='cmmi-12'>β</span>-lactamases from gram-negative bacteria. <span class='ptmri7t-x-x-120'>Annual Review of
      </span><span class='ptmri7t-x-x-120'>Microbiology</span>, 65(1):455–478, October 2011.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [453]<span class='bibsp'>   </span></span><a id='Xpalzkill_structural_2018'></a>Timothy   Palzkill.       Structural   and   mechanistic   basis   for   extended-spectrum
      drug-resistance mutations in altering the specificity of tem, ctx-m, and kpc <span class='cmmi-12'>β</span>-lactamases.
      <span class='ptmri7t-x-x-120'>Frontiers in Molecular Biosciences</span>, 5:16, February 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [454]<span class='bibsp'>   </span></span><a id='Xmobashery_three_2009'></a>Jed F. Fisher and Shahriar Mobashery.   Three decades of the class a <span class='cmmi-12'>β</span>-lactamase
      acyl-enzyme, September 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [455]<span class='bibsp'>   </span></span><a id='Xbonomo_kinetics_2007'></a>Kinetics of <span class='cmmi-12'>β</span>-lactamases and penicillin-binding proteins. In Bonomo and Tolmasky,
      editors, <span class='ptmri7t-x-x-120'>Enzyme-Mediated Resistance to Antibiotics</span>, pages 195–213. American Society
      of Microbiology, January 2007.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [456]<span class='bibsp'>   </span></span><a id='Xbonnet_growing_2004'></a>R. Bonnet. Growing group of extended-spectrum <span class='cmmi-12'>β</span>-lactamases: the ctx-m enzymes.
      <span class='ptmri7t-x-x-120'>Antimicrobial Agents and Chemotherapy</span>, 48(1):1–14, January 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [457]<span class='bibsp'>   </span></span><a id='Xdandrea_ctx-m-type_2013'></a>Marco Maria D’Andrea, Fabio Arena, Lucia Pallecchi, and Gian Maria Rossolini.
      CTX-M-type <span class='cmmi-12'>β</span>-lactamases: A successful story of antibiotic resistance.   <span class='ptmri7t-x-x-120'>International
      </span><span class='ptmri7t-x-x-120'>Journal of Medical Microbiology</span>, 303(6-7):305–317, August 2013.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [458]<span class='bibsp'>   </span></span><a id='Xchen_atomic_2005'></a>Yu Chen,  Julien  Delmas,  Jacques  Sirot,  Brian  Shoichet,  and  Richard  Bonnet.
                                                                                      
                                                                                      
      Atomic  resolution  structures  of  ctx-m  <span class='cmmi-12'>β</span>-lactamases:  extended  spectrum  activities
      from  increased  mobility  and  decreased  stability.     <span class='ptmri7t-x-x-120'>Journal  of  Molecular  Biology</span>,
      348(2):349–362, April 2005.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [459]<span class='bibsp'>   </span></span><a id='Xadamski_molecular_2015'></a>Carolyn J.  Adamski,  Ana Maria  Cardenas,  Nicholas G.  Brown,  Lori B.  Horton,
      Banumathi   Sankaran,   B. V. Venkataram   Prasad,   Hiram F.   Gilbert,   and   Timothy
      Palzkill.   Molecular basis for the catalytic specificity of the ctx-m extended-spectrum
      <span class='cmmi-12'>β</span>-lactamases. <span class='ptmri7t-x-x-120'>Biochemistry</span>, 54(2):447–457, January 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [460]<span class='bibsp'>   </span></span><a id='Xpatel_characterization_2015'></a>Meha P.  Patel,  Bartlomiej G.  Fryszczyn,  and  Timothy  Palzkill.   Characterization
      of  the  global  stabilizing  substitution  a77v  and  its  role  in  the  evolution  of  ctx-m
      <span class='cmmi-12'>β</span>-lactamases.  <span class='ptmri7t-x-x-120'>Antimicrobial Agents and Chemotherapy</span>, 59(11):6741–6748, November
      2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [461]<span class='bibsp'>   </span></span><a id='Xpatel_drug-resistant_2017'></a>Meha P.         Patel,         Liya         Hu,         Vlatko         Stojanoski,         Banumathi
      Sankaran, B. V. Venkataram Prasad, and Timothy Palzkill.  The drug-resistant variant
      p167s expands the substrate profile of ctx-m <span class='cmmi-12'>β</span>-lactamases for oxyimino-cephalosporin
      antibiotics by enlarging the active site upon acylation. <span class='ptmri7t-x-x-120'>Biochemistry</span>, 56(27):3443–3453,
      July 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [462]<span class='bibsp'>   </span></span><a id='Xbonnet_effect_2003'></a>R. Bonnet.    Effect  of  d240g  substitution  in  a  novel  esbl  ctx-m-27.    <span class='ptmri7t-x-x-120'>Journal  of
      </span><span class='ptmri7t-x-x-120'>Antimicrobial Chemotherapy</span>, 52(1):29–35, June 2003.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [463]<span class='bibsp'>   </span></span><a id='Xkimura_role_2004'></a>Soichiro  Kimura,  Masaji  Ishiguro,  Yoshikazu  Ishii,  Jimena  Alba,  and  Keizo
      Yamaguchi.  Role of a mutation at position 167 of ctx-m-19 in ceftazidime hydrolysis.
      <span class='ptmri7t-x-x-120'>Antimicrobial Agents and Chemotherapy</span>, 48(5):1454–1460, May 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [464]<span class='bibsp'>   </span></span><a id='Xcanton_ctx-m_2012'></a>Rafael Cantón, José María González-Alba, and Juan Carlos Galán.  Ctx-m
      enzymes: origin and diffusion. <span class='ptmri7t-x-x-120'>Frontiers in Microbiology</span>, 3, 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [465]<span class='bibsp'>   </span></span><a id='Xstrynadka_molecular_1992'></a>Natalie  C. J.  Strynadka,  Hiroyuki  Adachi,  Susan E.  Jensen,  Kathy  Johns,  Anita
      Sielecki, Christian Betzel, Kazuo Sutoh, and Michael N. G. James. Molecular structure
      of the acyl-enzyme intermediate in <span class='cmmi-12'>β</span>-lactam hydrolysis at 1.7 å resolution.   <span class='ptmri7t-x-x-120'>Nature</span>,
      359(6397):700–705, October 1992.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [466]<span class='bibsp'>   </span></span><a id='Xnovais_mutational_2008'></a>Angela Novais, Rafael Canton, Teresa M. Coque, Andres Moya, Fernando Baquero,
      and   Juan Carlos   Galan.       Mutational   events   in   cefotaximase   extended-spectrum
      <span class='cmmi-12'>β</span>-lactamases of the ctx-m-1 cluster involved in ceftazidime resistance.   <span class='ptmri7t-x-x-120'>Antimicrobial
      </span><span class='ptmri7t-x-x-120'>Agents and Chemotherapy</span>, 52(7):2377–2382, July 2008.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [467]<span class='bibsp'>   </span></span><a id='Xishii_biochemical_2007'></a>Yoshikazu   Ishii,   Moreno   Galleni,   Ling   Ma,   Jean-Marie   Frère,   and   Keizo
      Yamaguchi. Biochemical characterisation of the CTX-M-14 <span class='cmmi-12'>β</span>-lactamase. <span class='ptmri7t-x-x-120'>International
      </span><span class='ptmri7t-x-x-120'>Journal of Antimicrobial Agents</span>, 29(2):159–164, February 2007.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [468]<span class='bibsp'>   </span></span><a id='Xhuang_natural_1997'></a>W. Huang  and  T. Palzkill.    A  natural  polymorphism  in  -lactamase  is  a  global
      suppressor.    <span class='ptmri7t-x-x-120'>Proceedings  of  the  National  Academy  of  Sciences</span>,  94(16):8801–8806,
      August 1997.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [469]<span class='bibsp'>   </span></span><a id='Xbrown_multiple_2010'></a>Nicholas G.  Brown,  Jeanine M.  Pennington,  Wanzhi  Huang,  Tulin  Ayvaz,  and
      Timothy  Palzkill.    Multiple  global  suppressors  of  protein  stability  defects  facilitate
      the evolution of extended-spectrum tem <span class='cmmi-12'>β</span>-lactamases.   <span class='ptmri7t-x-x-120'>Journal of Molecular Biology</span>,
      404(5):832–846, December 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [470]<span class='bibsp'>   </span></span><a id='Xmayer_correlation_2007'></a>Sebastian  Mayer,  Stefan  Rüdiger,  Hwee Ching  Ang,  Andreas C.  Joerger,  and
      Alan R.  Fersht.   Correlation  of  levels  of  folded  recombinant  p53  in  escherichia  coli
      with thermodynamic stability in vitro.  <span class='ptmri7t-x-x-120'>Journal of Molecular Biology</span>, 372(1):268–276,
      September 2007.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [471]<span class='bibsp'>   </span></span><a id='Xyuan_prediction_2005'></a>Zheng  Yuan,  Timothy L.  Bailey,  and  Rohan D.  Teasdale.   Prediction  of  protein
      B-factor  profiles.   <span class='ptmri7t-x-x-120'>Proteins:  Structure,  Function,  and  Bioinformatics</span>,  58(4):905–912,
      January 2005.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [472]<span class='bibsp'>   </span></span><a id='Xpatel_synergistic_2018'></a>Meha P. Patel, Liya Hu, Cameron A. Brown, Zhizeng Sun, Carolyn J. Adamski,
      Vlatko  Stojanoski,  Banumathi  Sankaran,  B. V. Venkataram  Prasad,  and  Timothy
                                                                                      
                                                                                      
      Palzkill. Synergistic effects of functionally distinct substitutions in <span class='cmmi-12'>β</span>-lactamase variants
      shed light on the evolution of bacterial drug resistance. <span class='ptmri7t-x-x-120'>Journal of Biological Chemistry</span>,
      293(46):17971–17984, November 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [473]<span class='bibsp'>   </span></span><a id='Xchen_acylation_2007'></a>Yu Chen,  Richard  Bonnet,  and  Brian K.  Shoichet.   The  acylation  mechanism  of
      ctx-m <span class='cmmi-12'>β</span>-lactamase at 0. 88 å resolution.  <span class='ptmri7t-x-x-120'>Journal of the American Chemical Society</span>,
      129(17):5378–5380, May 2007.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [474]<span class='bibsp'>   </span></span><a id='Xcartelle_high-level_2004'></a>Monica  Cartelle,  Maria  del  Mar Tomas,  Francisca  Molina,  Rita  Moure,  Rosa
      Villanueva,  and  German  Bou.    High-level  resistance  to  ceftazidime  conferred  by  a
      novel enzyme, ctx-m-32, derived from ctx-m-1 through a single asp240-gly substitution.
      <span class='ptmri7t-x-x-120'>Antimicrobial Agents and Chemotherapy</span>, 48(6):2308–2313, June 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [475]<span class='bibsp'>   </span></span><a id='Xpetrosino_-lactamases:_1998'></a>Joseph  Petrosino,  Carlos  Cantu,  and  Timothy  Palzkill.     <span class='cmmi-12'>β</span>-Lactamases:  protein
      evolution in real time. <span class='ptmri7t-x-x-120'>Trends in Microbiology</span>, 6(8):323–327, August 1998.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [476]<span class='bibsp'>   </span></span><a id='Xsalverda_natural_2010'></a>Merijn L.M.  Salverda,  J. Arjan G.M.  De Visser,  and  Miriam  Barlow.    Natural
      evolution  of  TEM-1  <span class='cmmi-12'>β</span>-lactamase:  experimental  reconstruction  and  clinical  relevance.
      <span class='ptmri7t-x-x-120'>FEMS Microbiology Reviews</span>, 34(6):1015–1036, November 2010.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [477]<span class='bibsp'>   </span></span><a id='Xdellus-gur_negative_2015'></a>Eynat Dellus-Gur, Mikael Elias, Emilia Caselli, Fabio Prati, Merijn L.M. Salverda,
      J. Arjan G.M. de Visser, James S. Fraser, and Dan S. Tawfik.  Negative epistasis and
      evolvability in tem-1 <span class='cmmi-12'>β</span>-lactamase—the thin line between an enzyme’s conformational
      freedom and disorder. <span class='ptmri7t-x-x-120'>Journal of Molecular Biology</span>, 427(14):2396–2409, July 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [478]<span class='bibsp'>   </span></span><a id='Xgiakkoupi_detrimental_2000'></a>P. Giakkoupi.    Detrimental  effect  of  the  combination  of  R164S  with  G238S  in
      TEM-1  beta-lactamase  on  the  extended-spectrum  activity  conferred  by  each  single
      mutation. <span class='ptmri7t-x-x-120'>Journal of Antimicrobial Chemotherapy</span>, 45(1):101–104, January 2000.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [479]<span class='bibsp'>   </span></span><a id='Xstojanoski_triple_2015'></a>Vlatko  Stojanoski,  Dar-Chone  Chow,  Liya  Hu,  Banumathi  Sankaran,  Hiram F.
      Gilbert,  B. V. Venkataram  Prasad,  and  Timothy  Palzkill.     A  triple  mutant  in  the
      -loop  of  tem-1  <span class='cmmi-12'>β</span>-lactamase  changes  the  substrate  profile  via  a  large  conformational
      change  and  an  altered  general  base  for  catalysis.    <span class='ptmri7t-x-x-120'>Journal  of  Biological  Chemistry</span>,
      290(16):10382–10394, April 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [480]<span class='bibsp'>   </span></span><a id='Xfrohlich_oxa-48-mediated_2019'></a>Christopher Fröhlich, Vidar Sørum, Ane Molden Thomassen, PÅl Jarle Johnsen,
      Hanna-Kirsti S. Leiros, and Ørjan Samuelsen. Oxa-48-mediated ceftazidime-avibactam
      resistance  is  associated  with  evolutionary  trade-offs.      <span class='ptmri7t-x-x-120'>mSphere</span>,  4(2):e00024–19,
      /msphere/4/2/mSphere024–19.atom, March 2019.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [481]<span class='bibsp'>   </span></span><a id='Xbarnes_deciphering_2018'></a>Melissa D.  Barnes,  Magdalena A.  Taracila,  Joseph D.  Rutter,  Christopher R.
                                                                                      
                                                                                      
      Bethel, Ioannis Galdadas, Andrea M. Hujer, Emilia Caselli, Fabio Prati, John P. Dekker,
      Krisztina M.  Papp-Wallace,  Shozeb  Haider,  and  Robert A.  Bonomo.    Deciphering
      the  evolution  of  cephalosporin  resistance  to  ceftolozane-tazobactam  in  <span class='ptmri7t-x-x-120'>pseudomonas
      </span><span class='ptmri7t-x-x-120'>aeruginosa</span>. <span class='ptmri7t-x-x-120'>mBio</span>, 9(6):e02085–18, /mbio/9/6/mBio.02085–18.atom, December 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [482]<span class='bibsp'>   </span></span><a id='Xpetrosino_contributions_1999'></a>Joseph Petrosino, Gary Rudgers, Hiram Gilbert, and Timothy Palzkill. Contributions
      of aspartate 49 and phenylalanine 142 residues of a tight binding inhibitory protein of
      <span class='cmmi-12'>β</span>-lactamases. <span class='ptmri7t-x-x-120'>Journal of Biological Chemistry</span>, 274(4):2394–2400, January 1999.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [483]<span class='bibsp'>   </span></span><a id='Xamann_vectors_1983'></a>Egon Amann, Jürgen Brosius, and Mark Ptashne.  Vectors bearing a hybrid trp-lac
      promoter  useful  for  regulated  expression  of  cloned  genes  in  Escherichia  coli.   <span class='ptmri7t-x-x-120'>Gene</span>,
      25(2-3):167–178, November 1983.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [484]<span class='bibsp'>   </span></span><a id='Xstudier_use_1986'></a>F.William  Studier  and  Barbara A.  Moffatt.      Use  of  bacteriophage  T7  RNA
      polymerase  to  direct  selective  high-level  expression  of  cloned  genes.     <span class='ptmri7t-x-x-120'>Journal  of
      </span><span class='ptmri7t-x-x-120'>Molecular Biology</span>, 189(1):113–130, May 1986.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [485]<span class='bibsp'>   </span></span><a id='Xmarciano_genetic_2008'></a>David C. Marciano, Jeanine M. Pennington, Xiaohu Wang, Jian Wang, Yu Chen,
      Veena L.  Thomas,  Brian K.  Shoichet,  and  Timothy  Palzkill.   Genetic  and  structural
      characterization  of  an  l201p  global  suppressor  substitution  in  tem-1  <span class='cmmi-12'>β</span>-lactamase.
      <span class='ptmri7t-x-x-120'>Journal of Molecular Biology</span>, 384(1):151–164, December 2008.
                                                                                      
                                                                                      
      </p>
      <p class='bibitem'><span class='biblabel'>
 [486]<span class='bibsp'>   </span></span><a id='Xadams_phenix_2010'></a>Paul D. Adams, Pavel V. Afonine, Gábor Bunkóczi, Vincent B. Chen, Ian W.
      Davis,  Nathaniel  Echols,  Jeffrey J.  Headd,  Li-Wei  Hung,  Gary J.  Kapral,  Ralf W.
      Grosse-Kunstleve,  Airlie J.  McCoy,  Nigel W.  Moriarty,  Robert  Oeffner,  Randy J.
      Read, David C. Richardson, Jane S. Richardson, Thomas C. Terwilliger, and Peter H.
      Zwart.   <span class='ptmri7t-x-x-120'>phenix  </span>: a comprehensive python-based system for macromolecular structure
      solution. <span class='ptmri7t-x-x-120'>Acta Crystallographica Section D Biological Crystallography</span>, 66(2):213–221,
      February 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [487]<span class='bibsp'>   </span></span><a id='Xemsley_features_2010'></a>P. Emsley, B. Lohkamp, W. G. Scott, and K. Cowtan.  Features and development
      of <span class='ptmri7t-x-x-120'>Coot</span>. <span class='ptmri7t-x-x-120'>Acta Crystallographica Section D Biological Crystallography</span>, 66(4):486–501,
      April 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [488]<span class='bibsp'>   </span></span><a id='XBerendsen:1995dd'></a>H J C   Berendsen,   D van der   Spoel,   and   R van   Drunen.       GROMACS:   A
      message-passing  parallel  molecular  dynamics  implementation.     <span class='ptmri7t-x-x-120'>Computer  Physics
      </span><span class='ptmri7t-x-x-120'>Communications</span>, 91(1-3):43–56, September 1995.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [489]<span class='bibsp'>   </span></span><a id='Xsousa_da_silva_acpype_2012'></a>Alan W Sousa da Silva and Wim F Vranken. Acpype - antechamber python parser
      interface. <span class='ptmri7t-x-x-120'>BMC Research Notes</span>, 5(1):367, 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [490]<span class='bibsp'>   </span></span><a id='Xwang_development_2004'></a>Junmei  Wang,  Romain M.  Wolf,  James W.  Caldwell,  Peter A.  Kollman,  and
                                                                                      
                                                                                      
      David A. Case.   Development and testing of a general amber force field.   <span class='ptmri7t-x-x-120'>Journal of
      </span><span class='ptmri7t-x-x-120'>Computational Chemistry</span>, 25(9):1157–1174, July 2004.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [491]<span class='bibsp'>   </span></span><a id='Xwang_automatic_2006'></a>Junmei  Wang,  Wei  Wang,  Peter A.  Kollman,  and  David A.  Case.    Automatic
      atom type and bond type perception in molecular mechanical calculations.  <span class='ptmri7t-x-x-120'>Journal of
      </span><span class='ptmri7t-x-x-120'>Molecular Graphics and Modelling</span>, 25(2):247–260, October 2006.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [492]<span class='bibsp'>   </span></span><a id='XOnken:2018gz'></a>Michael D Onken, Carol M Makepeace, Kevin M Kaltenbronn, Stanley M Kanai,
      Tyson D Todd, Shiqi Wang, Thomas J Broekelmann, Prabakar Kumar Rao, John A
      Cooper, and Kendall J Blumer.  Targeting nucleotide exchange to inhibit constitutively
      active  G  protein  <span class='cmmi-12'>α </span>subunits  in  cancer  cells.    <span class='ptmri7t-x-x-120'>Science  signaling</span>,  11(546):eaao6852,
      September 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [493]<span class='bibsp'>   </span></span><a id='XSingh:2017hha'></a>Sukrit Singh and Gregory R Bowman.   Quantifying allosteric communication via
      both concerted structural changes and conformational disorder with CARDS. <span class='ptmri7t-x-x-120'>Journal of
      </span><span class='ptmri7t-x-x-120'>Chemical Theory and Computation</span>, 13(4):acs.jctc.6b01181–1517, March 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [494]<span class='bibsp'>   </span></span><a id='XEssmann:1995gi'></a>Ulrich  Essmann,  Lalith  Perera,  Max L  Berkowitz,  Tom  Darden,  Hsing  Lee,  and
      Lee G Pedersen.   A smooth particle mesh Ewald method.   <span class='ptmri7t-x-x-120'>The Journal of Chemical
      </span><span class='ptmri7t-x-x-120'>Physics</span>, 103(19):8577–8593, November 1995.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [495]<span class='bibsp'>   </span></span><a id='XBerendsen:1984fm'></a>H J C Berendsen, J P M Postma, W F van Gunsteren, A DiNola, and J R Haak.
      Molecular dynamics with coupling to an external bath. <span class='ptmri7t-x-x-120'>The Journal of Chemical Physics</span>,
      81(8):3684–3690, October 1984.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [496]<span class='bibsp'>   </span></span><a id='XHolehouse2020-rl'></a>Alex S Holehouse and Shahar Sukenik.  Controlling structural bias in intrinsically
      disordered  proteins  using  solution  space  scanning.      <span class='ptmri7t-x-x-120'>J.  Chem.  Theory  Comput.</span>,
      16(3):1794–1805, March 2020.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [497]<span class='bibsp'>   </span></span><a id='XDosztanyi2005-bo'></a>Zsuzsanna  Dosztányi,  Veronika  Csizmok,  Peter  Tompa,  and  István  Simon.
      IUPred: web server for the prediction of intrinsically unstructured regions of proteins
      based on estimated energy content. <span class='ptmri7t-x-x-120'>Bioinformatics</span>, 21(16):3433–3434, August 2005.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [498]<span class='bibsp'>   </span></span><a id='XHolehouse2017-sx'></a>Alex S  Holehouse,  Rahul K  Das,  James N  Ahad,  Mary  O G  Richardson,  and
      Rohit V  Pappu.   CIDER:  Resources  to  analyze  Sequence-Ensemble  relationships  of
      intrinsically disordered proteins. <span class='ptmri7t-x-x-120'>Biophys. J.</span>, 112(1):16–21, January 2017.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [499]<span class='bibsp'>   </span></span><a id='XDas2016-ri'></a>Rahul K Das, Yongqi Huang, Aaron H Phillips, Richard W Kriwacki, and Rohit V
      Pappu.   Cryptic sequence features within the disordered protein p27kip1 regulate cell
      cycle signaling. <span class='ptmri7t-x-x-120'>Proc. Natl. Acad. Sci. U. S. A.</span>, 113(20):5616–5621, May 2016.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [500]<span class='bibsp'>   </span></span><a id='XOrtega2018-oa'></a>Esther Ortega, Srinivasan Rengachari, Ziad Ibrahim, Naghmeh Hoghoughi, Jonathan
      Gaucher, Alex S Holehouse, Saadi Khochbin, and Daniel Panne.  Transcription factor
      dimerization activates the p300 acetyltransferase.  <span class='ptmri7t-x-x-120'>Nature</span>, 562(7728):538–544, October
      2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [501]<span class='bibsp'>   </span></span><a id='XAbraham2015-ea'></a>Mark James Abraham, Teemu Murtola, Roland Schulz, Szilárd Páll, Jeremy C
      Smith,  Berk  Hess,  and  Erik  Lindahl.     GROMACS:  High  performance  molecular
      simulations through multi-level parallelism from laptops to supercomputers. <span class='ptmri7t-x-x-120'>SoftwareX</span>,
      1–2:19–25, 2015.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [502]<span class='bibsp'>   </span></span><a id='XBieler2012-nm'></a>Noah S   Bieler,   Tuomas   P J   Knowles,   Daan   Frenkel,   and   Robert   Vácha.
      Connecting  macroscopic  observables  and  microscopic  assembly  events  in  amyloid
      formation  using  coarse  grained  simulations.    <span class='ptmri7t-x-x-120'>PLoS  Comput.  Biol.</span>,  8(10):e1002692,
      October 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [503]<span class='bibsp'>   </span></span><a id='XBoeynaems2019-ja'></a>Steven  Boeynaems,  Alex S  Holehouse,  Venera  Weinhardt,  Denes  Kovacs,  Joris
      Van Lindt,  Carolyn  Larabell,  Ludo  Van  Den Bosch,  Rhiju  Das,  Peter S  Tompa,
      Rohit V  Pappu,  and  Aaron D  Gitler.      Spontaneous  driving  forces  give  rise  to
      protein-RNA condensates with coexisting phases and complex material properties. <span class='ptmri7t-x-x-120'>Proc.
      </span><span class='ptmri7t-x-x-120'>Natl. Acad. Sci. U. S. A.</span>, 116(16):7889–7898, April 2019.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [504]<span class='bibsp'>   </span></span><a id='XFichthorn1991-wl'></a>Kristen A  Fichthorn  and  W H  Weinberg.   Theoretical  foundations  of  dynamical
      monte carlo simulations. <span class='ptmri7t-x-x-120'>J. Chem. Phys.</span>, 95(2):1090–1096, July 1991.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [505]<span class='bibsp'>   </span></span><a id='XSaric2016-le'></a>An<span class='ptmb7t-x-x-120'>?</span>ela   Šarić,   Alexander K   Buell,   Georg   Meisl,   Thomas   C T   Michaels,
      Christopher M Dobson, Sara Linse, Tuomas P J Knowles, and Daan Frenkel.  Physical
      determinants of the self-replication of protein fibrils.  <span class='ptmri7t-x-x-120'>Nat. Phys.</span>, 12(9):874–880, July
      2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [506]<span class='bibsp'>   </span></span><a id='XSchuler2002-hf'></a>Benjamin   Schuler,   Everett A   Lipman,   and   William A   Eaton.      Probing   the
      free-energy surface for protein folding with single-molecule fluorescence spectroscopy.
      <span class='ptmri7t-x-x-120'>Nature</span>, 419:743, October 2002.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [507]<span class='bibsp'>   </span></span><a id='XChung2016-fp'></a>Hoi Sung Chung, John M Louis, and Irina V Gopich.   Analysis of fluorescence
      lifetime  and  energy  transfer  efficiency  in  Single-Molecule  photon  trajectories  of
      Fast-Folding proteins. <span class='ptmri7t-x-x-120'>J. Phys. Chem. B</span>, 120(4):680–699, February 2016.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [508]<span class='bibsp'>   </span></span><a id='XNettels2008-jk'></a>Daniel  Nettels,  Armin  Hoffmann,  and  Benjamin  Schuler.    Unfolded  protein  and
      peptide dynamics investigated with single-molecule FRET and correlation spectroscopy
      from picoseconds to seconds. <span class='ptmri7t-x-x-120'>J. Phys. Chem. B</span>, 112(19):6137–6146, May 2008.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [509]<span class='bibsp'>   </span></span><a id='XGopich2009-ow'></a>Irina V  Gopich,  Daniel  Nettels,  Benjamin  Schuler,  and  Attila  Szabo.     Protein
      dynamics from single-molecule fluorescence intensity correlation functions.  <span class='ptmri7t-x-x-120'>J. Chem.
      </span><span class='ptmri7t-x-x-120'>Phys.</span>, 131(9):095102, September 2009.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [510]<span class='bibsp'>   </span></span><a id='XSchafer2012-bh'></a>Lothar Schäfer. <span class='ptmri7t-x-x-120'>Excluded Volume Effects in Polymer Solutions: as Explained by the
      </span><span class='ptmri7t-x-x-120'>Renormalization Group</span>. Springer Science &amp; Business Media, December 2012.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [511]<span class='bibsp'>   </span></span><a id='XZheng2018-ej'></a>Wenwei  Zheng,  Gül H  Zerze,  Alessandro  Borgia,  Jeetain  Mittal,  Benjamin
      Schuler,  and  Robert B  Best.    Inferring  properties  of  disordered  chains  from  FRET
      transfer efficiencies. <span class='ptmri7t-x-x-120'>J. Chem. Phys.</span>, 148(12):123329, March 2018.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [512]<span class='bibsp'>   </span></span><a id='XMuller-Spath2010-oy'></a>Sonja Müller-Späth, Andrea Soranno, Verena Hirschfeld, Hagen Hofmann, Stefan
      Rüegger, Luc Reymond, Daniel Nettels, and Benjamin Schuler.   Charge interactions
      can dominate the dimensions of intrinsically disordered proteins.  <span class='ptmri7t-x-x-120'>Proc. Natl. Acad. Sci.
      </span><span class='ptmri7t-x-x-120'>U. S. A.</span>, 107(33):14609–14614, August 2010.
      </p>
      <p class='bibitem'><span class='biblabel'>
 [513]<span class='bibsp'>   </span></span><a id='XHiggs1991-rl'></a>Paul G Higgs and Jean-françois Joanny.   Theory of polyampholyte solutions.   <span class='ptmri7t-x-x-120'>J.
      </span><span class='ptmri7t-x-x-120'>Chem. Phys.</span>, 94(2):1543–1554, January 1991.
      </p>
                                                                                      
                                                                                      
      <p class='bibitem'><span class='biblabel'>
 [514]<span class='bibsp'>   </span></span><a id='XMeng2013-qi'></a>Wenli Meng, Nicholas Lyle, Bowu Luan, Daniel P Raleigh, and Rohit V Pappu.
      Experiments  and  simulations  show  how  long-range  contacts  can  form  in  expanded
      unfolded proteins with negligible secondary structure.  <span class='ptmri7t-x-x-120'>Proc. Natl. Acad. Sci. U. S. A.</span>,
      110(6):2123–2128, February 2013.
</p>
      </div>
                                                                                      
                                                                                      
<!-- l. 246 --><p class='noindent'>
                                                                                      
                                                                                      
</p><!-- l. 246 --><p class='noindent'>-422.52342</p>
                                                                                      
                                                                                      
<!-- l. 248 --><div class='crosslinks'><p class='noindent'></p></div>
 
</body></html>