<?xml version='1.0' encoding='utf-8' ?> 
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd"><html xmlns='http://www.w3.org/1999/xhtml'> 
<head><title>D Supplementary Material on the SARS-CoV-2 nucleocapsid protein</title> 
<meta http-equiv='Content-Type' content='text/html; charset=utf-8' /> 
<meta name='generator' content='TeX4ht (https://tug.org/tex4ht/)' /> 
<meta name='originator' content='TeX4ht (https://tug.org/tex4ht/)' /> 
<!--  xhtml,charset=utf-8,epub,uni-html4,html  --> 
<meta name='src' content='main.tex' /> 
<link type='text/css' href='main.css' rel='stylesheet' /> 
</head><body>
<!-- l. 5 --><div class='crosslinks'><p class='noindent'></p></div>
<h2 class='appendixHead'><span class='titlemark'>Appendix D</span><br /><a id='x17-143000D'></a>Supplementary Material on the SARS-CoV-2 nucleocapsid protein</h2>
<!-- l. 8 --><p class='noindent'><span class='ptmri7t-x-x-120'>This chapter is adapted from the following publication:</span>
</p><!-- l. 10 --><p class='noindent'><span class='ptmri7t-x-x-120'>Cubuk, J., Alston, J.J., Incicco, J.J., Singh, S., Stuchell-Brereton, M.D., Ward, M.D., Zimmerman, M.I.,
</span><span class='ptmri7t-x-x-120'>Vithani, N., Griffith, D., Wagoner, J.A., Bowman, G.R., Hall, K.B., Soranno, A., The SARS-CoV-2
</span><span class='ptmri7t-x-x-120'>nucleocapsid protein is dynamic, disordered, and phase separates with RNA, Available on Biorxiv:
</span><span class='ptmri7t-x-x-120'>https://doi.org/10.1101/2020.06.17.158121</span><span class='cite'> [<a href='mainli5.html#Xcubuk_sars-cov-2_2020'>2</a>]</span>
</p>
<h3 class='sectionHead'><span class='titlemark'>D.1    </span> <a id='x17-144000D.1'></a>Supplementary Methods</h3>
<!-- l. 17 --><p class='noindent'>
</p>
<h4 class='subsectionHead'><span class='titlemark'>D.1.1    </span> <a id='x17-145000D.1.1'></a>Sequence analysis</h4>
<!-- l. 18 --><p class='noindent'>Disorder prediction was performed using IUPred, with additional analysis and sequence parsing done
with localCIDER and protfasta, respectively<span class='cite'> [<a href='mainli5.html#XHolehouse2020-rl'>496</a>, <a href='mainli5.html#XDosztanyi2005-bo'>497</a>, <a href='mainli5.html#XHolehouse2017-sx'>498</a>]</span>
</p><!-- l. 20 --><p class='noindent'>Amino acid sequence of the N protein used in simulations. Highlighted regions delineate folded
domains. Underline bolded residues identify the sites of dyes for single-molecule fluorescence
                                                                                      
                                                                                      
experiments.
</p>
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
                                                                                      
                                                                                      
<!-- l. 24 --><p class='noindent'><img width='397' height='397' src='AppendixE/appendixE-sequence.png' alt='PIC' />
                                                                                      
                                                                                      
</p>
</div><hr class='endfigure' />
<h4 class='subsectionHead'><span class='titlemark'>D.1.2    </span> <a id='x17-146000D.1.2'></a>Simulation Methods</h4>
<!-- l. 29 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x17-147000D.1.2'></a>Monte Carlo simulations</h5>
<!-- l. 30 --><p class='noindent'>All simulations were performed at 330 K and at 15 mM NaCl, as have been used previously in a
variety of systems<span class='cite'> [<a href='mainli5.html#XDas2016-ri'>499</a>, <a href='mainli5.html#XHolehouse2020-rl'>496</a>, <a href='mainli5.html#XDas2016-ri'>499</a>, <a href='mainli5.html#XMartin2020-yc'>338</a>, <a href='mainli5.html#XSherry2017-sr'>401</a>]</span>. Simulation analysis was performed with MDTraj and
camparitraj (http://ctraj.com/)<span class='cite'> [<a href='mainli5.html#XMcGibbon:2015fv'>148</a>, <span class='ptmb7t-x-x-120'>?</span>]</span>. For IDR only simulations all degrees of freedom were fully
sampled (backbone and sidechain dihedral angles and rigid-body positions) as is standard in
CAMPARI Monte Carlo simulations<span class='cite'> [<a href='mainli5.html#XVitalis2009-gh'>398</a>]</span>. For simulations of IDRs in the context of folded domains,
the backbone dihedral angles of the folded domains were held fixed while all sidechains were
fully sampled, as were backbone dihedral angles for the disordered regions, as applied
previously<span class='cite'> [<a href='mainli5.html#XOrtega2018-oa'>500</a>]</span>. The folded state starting structures were obtained from PDB structures (listed
below).
</p><!-- l. 32 --><p class='noindent'>For IDR-only simulations 30-40 independent simulations were run generating final ensembles of
40-60 K conformations. For simulations of IDRs in the context of folded domains, the number of
independent simulations and the length of the simulation varied. For the NTD-RBD simulations 400
independent simulations were run, with 2 independent simulations per starting seed from MD
simulations (see methods below) leading to a final ensemble of <span class='cmsy-10x-x-120'>∼</span>400 K conformations (24 M steps
                                                                                      
                                                                                      
per simulation). For the RBD-LINK-dimerization construct, ten independent simulations were run for
a final ensemble of 32 K conformers (66 M steps per simulation). For the dimerization-CTD construct
40 independent simulations were run providing a final ensemble of 40 K conformations
(66 M steps per simulation). For a complete description of simulation details see Table
<a href='#x17-168007r4'>D.4<!-- tex4ht:ref: tab:appendix-e-tab4  --></a>.
</p><!-- l. 34 --><p class='noindent'>For the NTD-RB construct, we used a sequential sampling approach in which long timescale MD
simulations of the RBD in isolation performed on the Folding@home distributed computing platform
were first used to generate hundreds of starting conformations<span class='cite'> [<a href='mainli5.html#XShirts:2000du'>37</a>, <a href='mainli5.html#XZimmerman2020covid-pocket'>403</a>]</span>. Those RBD conformations
were then used as starting structures for independent all-atom Monte Carlo simulations. Monte Carlo
simulations were performed with the ABSINTH forcefield in which the RBD backbone dihedral
angles are held fixed but the NTD is fully sampled, as are RBD sidechains. The RBD starting structure
used was extracted from the 6VYO PDB crystal structure, which is equivalent to the 6YI3 NMR
structure.
</p><!-- l. 36 --><p class='noindent'>For RBD-Link-dimerization domain simulations, we opted to use a single starting seed structure for
the folded domains based on the NMR and crystal-structure conformations for the RBD and
dimerization domains, respectively. To generate the monomeric starting structure of the
dimerization domain, we first built a homology model of the SARS-CoV-2 dimerization
dimer from the NMR structure of the SARS dimerization structure (PDB: 2jw8) using
SWISS-MODEL<span class='cite'> [<a href='mainli5.html#Xwaterhouse_swiss-model:_2018'>1</a>, <a href='mainli5.html#XTakeda2008-wl'>306</a>]</span>. We chose this strategy because at the time, no dimerization
structure existed, a situation that has since resolved itself<span class='cite'> [<a href='mainli5.html#XZinzula2020-mi'>314</a>]</span>. Nevertheless, the SARS
and SARS-CoV-2 dimerization domains are essentially identical, such that this is a minor
detail.
</p><!-- l. 38 --><p class='noindent'>For dimerization domain-CTD simulations, a single starting structure for the dimerization domain was
again used, selected after MD simulations. Having generated a homology model, we extracted a single
protomer from the dimeric structure and ran molecular dynamics simulations to identify equilibrated
starting structures. In running initial simulations we discovered that as a monomer, the first 21 residues
                                                                                      
                                                                                      
appear disordered, in agreement with sequence predictions (Fig. <a href='#x17-168009r1'>D.1<!-- tex4ht:ref: fig:appendix-e-fig1  --></a>A) but in contrast to their
behavior in the dimeric structure (Fig. <a href='#x17-168009r1'>D.1<!-- tex4ht:ref: fig:appendix-e-fig1  --></a>C). As a result, we choose to also model these residues as
fully disordered. A single starting seed conformation was used for all dimerization-CTD
simulations.
</p><!-- l. 40 --><p class='noindent'>Excluded volume (EV) simulations were performed using the same setup, but with a modified
Hamiltonian under which solvation, attractive Lennard-Jones, and polar (charge) interactions are
scaled to zero, as described previously<span class='cite'> [<a href='mainli5.html#XHolehouse2015-rg'>329</a>]</span>.
</p><!-- l. 42 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x17-148000D.1.2'></a>Molecular dynamics simulations</h5>
<!-- l. 43 --><p class='noindent'>All molecular dynamics simulations of SARS-CoV-2 nucleoprotein were performed with Gromacs
2019 using the AMBER03 force field with explicit TIP3P solvent<span class='cite'> [<a href='mainli5.html#XAbraham2015-ea'>501</a>, <a href='mainli5.html#XDuan:2003gt'>146</a>, <a href='mainli5.html#XJorgensen:1983fl'>143</a>]</span>. Simulations were
prepared by placing the starting structure (PDB ID: 6VYO) in a dodecahedron box that extends 1.0 Å
beyond the protein in any dimension. The system was then solvated (29125 atoms), and energy
minimized with a steepest descents algorithm until the maximum force fell below 100 kJ/mol/nm
using a step size of 0.01 nm and a cutoff distance of 1.2 nm for the neighbor list, Coulomb
interactions, and van der Waals interactions. For production runs, all bonds were constrained with the
LINCS algorithm and virtual sites were used to allow a 4 fs time step <span class='cite'> [<a href='mainli5.html#XFeenstra:1999ue'>228</a>, <a href='mainli5.html#XHess:2008fl'>226</a>]</span>. Cutoffs of 1.1 nm
were used for the neighbor list with 0.9 for Coulomb and van der Waals interactions. The Verlet cutoff
scheme was used for the neighbor list. The stochastic velocity rescaling (v-rescale) thermostat
was used to hold the temperature at 300 K<span class='cite'> [<a href='mainli5.html#XBussi:2007cs'>224</a>]</span>. Conformations were stored every 20
ps.
</p><!-- l. 45 --><p class='noindent'>The FAST algorithm was used to enhance conformational sampling and quickly explore the dominant
                                                                                      
                                                                                      
motions of nucleoprotein <span class='cite'> [<a href='mainli5.html#Xzimmerman_fast_2015'>35</a>, <a href='mainli5.html#XZimmerman:2018jn'>42</a>]</span>. FAST-pocket simulations were run for 6 rounds, with 10
simulations per round, where each simulation was 40 ns in length (2.4 <span class='cmmi-12'>μ</span>s aggregate simulation). The
FAST-pocket ranking function favored restarting simulations from states with large pocket openings.
Additionally, a similarity penalty was added to the ranking to promote conformational diversity in
starting structures, as has been described previously<span class='cite'> [<a href='mainli5.html#Xzimmerman_prediction_2017'>278</a>]</span>. The FAST dataset was clustered using a
k-centers algorithm based on RMSD between frames using backbone heavy atoms (C, C<span class='cmmi-12'>α</span>, C<span class='cmmi-12'>β</span>, N, O)
to generate 1421 discrete states, which were then launched on the distributed computing platform
Folding@home<span class='cite'> [<a href='mainli5.html#XShirts:2000du'>37</a>, <a href='mainli5.html#XZimmerman2020covid-pocket'>403</a>]</span>.
</p><!-- l. 47 --><p class='noindent'>Furthering conformational sampling and enhancing statistics, Folding@home produced 500 <span class='cmmi-12'>μ</span>s of
aggregate simulation. A final k-centers clustering was performed with the combined Folding@home
and FAST data using Enspara (https://github.com/bowman-lab/enspara)<span class='cite'> [<a href='mainli5.html#Xporter2018enspara'>39</a>]</span>. This clustering was
performed the same as described above and generated 200 discrete states that capture maximal
diversity in nucleoproteins’ conformational ensemble. These states were then used as the basis for
CAMPARI simulations.
</p><!-- l. 49 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x17-149000D.1.2'></a>Sequential MD/MC sampling approach</h5>
<!-- l. 50 --><p class='noindent'>The NTD and RBD combined are 173 residues of folded and disordered protein, which raises a
significant challenge for all-atom sampling. To address this we leveraged a novel approach
in which we first ran several microsecond of all-atom molecular dynamics simulations
of RBD alone using the Folding@Home platform and the FAST approach for enhanced
conformational sampling <span class='cite'> [<a href='mainli5.html#XShirts:2000du'>37</a>, <a href='mainli5.html#XZimmerman2020covid-pocket'>403</a>, <a href='mainli5.html#Xzimmerman_fast_2015'>35</a>]</span>. We then identified 200 conformationally distinct states
based on these simulations which we used as “seeds” for the RBD. Using these seeds, we
reconstructed the previously missing NTD and ran all-atom Monte Carlo simulations in
                                                                                      
                                                                                      
which the NTD was fully sampled, the RBD sidechains are fully sampled, but the RBD
backbone dihedral angles are held fixed. Multiple replicas of each starting conformation
were run, giving us a total ensemble of <span class='cmsy-10x-x-120'>∼</span>400 K conformations. In parallel, we also ran
simulations of the NTD in isolation, enabling an assessment of the impact of the folded
domain.
</p><!-- l. 52 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x17-150000D.1.2'></a>Coarse-grained Polymer Simulations</h5>
<!-- l. 53 --><p class='noindent'>Coarse-grained simulations were performed using the PIMMS software package<span class='cite'> [<a href='mainli5.html#XHolehouse2019-cj'>402</a>, <a href='mainli5.html#XMartin2020-yc'>338</a>]</span>.
PIMMS is a Monte Carlo lattice-based simulation engine in which each bead engages
in anisotropic interactions with every adjacent lattice site. Moves used here were cluster
translation/rotation moves and single-bead perturbation moves. Specifically, every simulation step,
each bead in the system is sampled to move to adjacent sites in random order 503 of times
multiplied by a factor that reflects the length of the chain. Every 100 moves (on average) a
cluster of chains is randomly selected and translated or rotated, where a cluster reflects a
collection of two or more chains in direct contact. This moveset provides changes to the
system that reflect physical movements expected in a dynamical system, allowing us to - for
equivalently sized systems - compare the apparent dynamics of assembly, as has been done
previously<span class='cite'> [<a href='mainli5.html#XBieler2012-nm'>502</a>, <a href='mainli5.html#XBoeynaems2019-ja'>503</a>, <a href='mainli5.html#XFichthorn1991-wl'>504</a>, <a href='mainli5.html#XSaric2016-le'>505</a>]</span>. We repeated the simulations presented using a range of different
movesets and, while convergence varied from set-to-set, we always observed analogous
results.
</p><!-- l. 55 --><p class='noindent'>All simulations were performed in a 70 x 70 x 70 lattice-site box using period boundary conditions.
The results reported are averaged over the final 20% of the simulation to give average values after
equivalent numbers of MC steps. The “polymer” is represented as a 61-residue polymer with
                                                                                      
                                                                                      
either a central high-affinity binding site or not. The binder is a 2-bead species. Every
simulation was run for 20 x 10<sup><span class='cmr-8'>9</span></sup> Monte Carlo steps, with four independent replicas. Simulations
were run with 1,2,3,4 or 5 polymers and 50, 75, 100, 125, 150, 175, 200, 250, 300, 400
binders.
</p><!-- l. 57 --><p class='noindent'>If our simulations are run in a way deliberately designed to rapidly reach equilibrium using enhanced
sampling approaches eventually all single-polymer condensates coalesce into one large multi-polymer
condensate. Hence, our simulations are deliberately designed explore a regime in which
single-polymer condensates are metastable.
</p><!-- l. 59 --><p class='noindent'>
</p>
<h4 class='subsectionHead'><span class='titlemark'>D.1.3    </span> <a id='x17-151000D.1.3'></a>Protein expression, purification, and labeling.</h4>
<!-- l. 60 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x17-152000D.1.3'></a>Plasmid construct design.</h5>
<!-- l. 61 --><p class='noindent'>SARS-CoV2 Nucleocapsid protein (NCBI Reference Sequence: YP_009724397.2)
including an N term extension containing His9-HRV 3C protease site –
CATCATCACCATCATCATCATCACCACCTCGAAGTTCTGTTCCAAGGCCCGATGAGTGATAACGGTCCCCAGAATCAACG<br class='newline' />GAATGCGCCCAGAATCACGTTCGGCGGTCCAAGCGACAGTACAGGTTCGAATCAGAATGGTGAACGCTCTGGGGCCCGA<br class='newline' />AGCAAACAGCGTCGTCCACAGGGTTTGCCGAACAATACGGCTAGCTGGTTCACTGCGCTGACGCAGCACGGAAAAGAAG<br class='newline' />ACTTAAAATTTCCGCGAGGCCAGGGGGTCCCGATTAATACTAACTCCTCCCCTGACGATCAAATTGGTTATTATCGTCG<br class='newline' />TGCAACCCGCCGTATCCGCGGCGGAGACGGTAAAATGAAAGATCTGTCACCGCGCTGGTATTTTTACTACCTGGGAACA<br class='newline' />GGTCCTGAAGCAGGCTTGCCGTATGGCGCTAACAAAGATGGCATTATCTGGGTGGCTACCGAGGGTGCCCTTAATACGC<br class='newline' />CGAAAGATCATATTGGAACCCGTAACCCAGCCAATAACGCAGCAATCGTACTGCAGCTGCCGCAGGGGACAACCCTGCC<br class='newline' />GAAAGGCTTTTATGCGGAAGGGAGTCGTGGCGGCAGCCAAGCCAGCTCCCGTAGCTCCTCGCGCTCTCGCAACTCCTCG<br class='newline' />CGGAATAGTACACCGGGTTCATCACGCGGCACCTCGCCGGCACGCATGGCTGGCAACGGGGGGGATGCGGCTTTGGCGT<br class='newline' />TACTTTTACTGGATAGGCTTAACCAGTTGGAAAGTAAAATGAGCGGTAAAGGCCAGCAGCAGCAGGGTCAGACTGTGAC<br class='newline' />CAAAAAGAGCGCGGCAGAGGCGTCGAAAAAACCTAGACAAAAGCGTACTGCGACCAAAGCCTACAATGTTACGCAGGCA<br class='newline' />TTCGGCCGGCGCGGTCCGGAACAAACCCAGGGCAACTTTGGTGACCAGGAGCTGATTCGTCAGGGAACCGATTACAAAC<br class='newline' />ACTGGCCACAGATCGCGCAATTTGCCCCCTCGGCGTCAGCCTTTTTTGGTATGTCTCGCATTGGGATGGAGGTAACCCC<br class='newline' />GTCTGGCACGTGGCTGACGTACACGGGCGCTATAAAGCTGGATGATAAAGATCCGAACTTCAAAGACCAGGTGATCTTA<br class='newline' />CTGAACAAACATATTGACGCCTATAAAACGTTCCCCCCTACTGAACCTAAGAAAGATAAAAAAAAAAAGGCCGATGAAA<br class='newline' />CCCAAGCGCTACCACAACGCCAGAAAAAGCAGCAGACCGTCACCCTCCTGCCGGCAGCGGACCTCGACGATTTTTCTAA<br class='newline' />GCAACTGCAACAAAGCATGTCAAGCGCCGATAGTACACAGGCGTAA - was cloned into the
BamHI EcoRI sites in the MCS of pGEX-6P-1 vector (GE Healthcare) to express the protein product:
<br class='newline' />GST-LEVLFQGPLGS<span class='ptmb7t-x-x-120'>HHHHHHHHH</span><br class='newline' />LEVLFQGPMSDNGPQNQRNAPRITFGGPSDSTGSNQNGERSGARSKQRRPQGLPNNTASWFTALTQHGKEDLKFPRGQGV<br class='newline' />PINTNSSPDDQIGYYRRATRRIRGGDGKMKDLSPRWYFYYLGTGPEAGLPYGANKDGIIWVATEGALNTPKDHIGTRNP<br class='newline' />ANNAAIVLQLPQGTTLPKGFYAEGSRGGSQASSRSSSRSRNSSRNSTPGSSRGTSPARMAGNGGDAALALLLLDRLNQL<br class='newline' />ESKMSGKGQQQQGQTVTKKSAAEASKKPRQKRTATKAYNVTQAFGRRGPEQTQGNFGDQELIRQGTDYKHWPQIAQFAP<br class='newline' />SASAFFGMSRIGMEVTPSGTWLTYTGAIKLDDKDPNFKDQVILLNKHIDAYKTFPPTEPKKDKKKKADETQALPQRQKK<br class='newline' />QQTVTLLPAADLDDFSKQLQQSMSSADSTQA. Site-directed mutagenesis was performed on the
His9-SARS-CoV2 Nucleocapsid pGEX vector to create M1C R68C, Y172C T245C, and F363C
A419C variant N protein constructs. All cloning and site-directed mutagenesis steps were performed
by Genewiz and sequences were verified using sanger sequencing.
                                                                                      
                                                                                      
</p><!-- l. 65 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x17-153000D.1.3'></a>Protein Expression and Purification.</h5>
<!-- l. 66 --><p class='noindent'>Both GST-His9-SARS-CoV2 M1C-R68C and Y172C-T245C Nucleocapsid variants were expressed
recombinantly in BL21 Codon-plus pRIL cells (Agilent). 4L cultures were grown in LB medium
containing carbenicillin (100 ug/mL) to OD600 <span class='cmsy-10x-x-120'>∼ </span>0.6 and induced with 0.2 mM IPTG for 12 hours at
16<span class='ptmr8c-x-x-120'>º</span>C. Harvested cells were lysed with sonication at 4<span class='ptmr8c-x-x-120'>º</span>C in lysis buffer (50mM Tris pH 8, 500 mM
NaCl, 10% glycerol, 10 mg/mL lysozyme, 5 mM BME, cOmplete<span class='ptmr8c-x-x-120'>™ </span>EDTA-free Protease Inhibitor
Cocktail (Roche), DNAse I (NEB), RNAse H (NEB)). The supernatant was cleared by centrifugation
(37000 rpm for 1 hr) and bound to an HisTrap FF column (GE Healthcare) in buffer A (50 mM
Tris pH 8, 500 mM NaCl, 10% glycerol, 20mM imidazole, 5 mM BME). GST-His9-N
protein fusion was eluted with buffer B (buffer A + 500 mM imidazole) and dialyzed into
cleavage buffer (50 mM Tris pH 8, 50 mM NaCl, 10% glycerol, 1 mM DTT) with HRV 3C
protease, thus cleaving the GST-His9-N fusion yielding FL N protein with two additional
N-term residues (GlyPro). FL N protein was then bound to an SP sepharose FF column (GE
Healthcare) and eluted using a gradient of 0-100% buffer B (buffer A: 50mM Tris pH 8, 50mM
NaCl, 10% glycerol, 5 mM BME, buffer B: buffer A + 1 M NaCl) over 100 min. Purified N
protein variants were analyzed using SDS-PAGE and verified by electrospray ionization
mass spectrometry (LC-MS). Concentrations were determined spectroscopically in 50mM
Tris (pH 8.0), 500mM NaCl, 10% (v/v) glycerol using an extinction coefficient = 42530
<span class='cmmi-12'>M</span><sup><span class='cmsy-8'>−</span><span class='cmr-8'>1</span></sup><span class='cmmi-12'>cm</span><sup><span class='cmsy-8'>−</span><span class='cmr-8'>1</span></sup>
</p><!-- l. 68 --><p class='noindent'>Both GST-His9-SARS-CoV2 wild-type and F363C A419C Nucleocapsid variants were expressed
recombinantly in Gold BL21(DE3) cells (Agilent). 4 L cultures were grown in LB medium with
carbenicillin (100 ug/mL) to OD600 <span class='cmsy-10x-x-120'>∼ </span>0.6 and induced with 0.2 mM IPTG for 12 hours at 16<span class='ptmr8c-x-x-120'>º</span>C.
Harvested cells were lysed with sonication at 4<span class='ptmr8c-x-x-120'>º</span>C in lysis buffer (listed above). The supernatant was
cleared by centrifugation (37000 rpm for 1 hr) and the pellet was resuspended in 50 mM
                                                                                      
                                                                                      
Tris pH 8, 500 mM NaCl, 10% glycerol, 6 M Urea, 5 mM BME and incubated at 4<span class='ptmr8c-x-x-120'>º</span>C for
one hour. The resuspension was cleared by centrifugation (37000 rpm for 1hr) and the
GST-His9-N protein in the supernatant was bound to a FF HisTrap column (GE Healthcare) in
buffer A (50 mM Tris pH 8, 500 mM NaCl, 10% glycerol, 20 mM imidazole, 5 mM BME)
containing 6 M Urea. The column was then washed with buffer A allowing the protein to refold
on the column. The GST-His9-N protein fusion was then eluted with buffer B (buffer A
containing 500 mM imidazole) and dialyzed into cleavage buffer (50 mM Tris pH8, 50
mM NaCl, 10% glycerol, 1 mM DTT) containing HRV 3C protease. FL N protein was
then bound to an SP sepharose FF column (GE Healthcare) and eluted using a gradient of
0-100% buffer B (buffer A: 50mM Tris pH 8, 50 mM NaCl, 10% glycerol, 5 mM BME,
buffer B: buffer A + 1 M NaCl) over 100 min. Purified N protein variants were analyzed
using SDS-PAGE and verified by electrospray ionization mass spectrometry (LC-MS).
Protein concentrations of stock solutions were determined spectroscopically in 50mM
Tris (pH 8.0), 500mM NaCl, 10% (v/v) glycerol using an extinction coefficient = 42530
<span class='cmmi-12'>M</span><sup><span class='cmsy-8'>−</span><span class='cmr-8'>1</span></sup><span class='cmmi-12'>cm</span><sup><span class='cmsy-8'>−</span><span class='cmr-8'>1</span></sup>
</p><!-- l. 70 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x17-154000D.1.3'></a>Fluorescent Dye Labeling.</h5>
<!-- l. 71 --><p class='noindent'>All Nucleocapsid variants were labeled with Alexa Fluor 488 maleimide (Molecular Probes) under
denaturing conditions in buffer A (50mM Tris pH8, 50mM NaCl, 10% glycerol, 6M Urea,
1mM DTT) at a dye/protein molar ratio of 0.7/1 for 2 hrs at room temperature. Single
labeled protein was isolated via ion-exchange chromatography (Mono S 5/50 GL, GE
Healthcare - protein bound in buffer A and eluted with 0-100% buffer B (buffer A + 1
M NaCl) gradient over 100 min) and UV-Vis spectroscopic analysis to identify fractions
                                                                                      
                                                                                      
with 1:1 dye:protein labeling. Single labeled Alexa Fluor 488 maleimide labeled N protein
was then subsequently labeled with Alexa Fluor 594 maleimide at a dye/protein molar
ratio of 1.3/1 for 2 hrs at room temperature. Double labeled (488:594) protein was then
further purified via ion-exchange chromatography (Mono S 5/50 GL, GE Healthcare - see
above).
</p><!-- l. 73 --><p class='noindent'>
</p>
<h4 class='subsectionHead'><span class='titlemark'>D.1.4    </span> <a id='x17-155000D.1.4'></a>Single Molecule Spectroscopy</h4>
<!-- l. 74 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x17-156000D.1.4'></a>Experimental setup and procedure.</h5>
<!-- l. 75 --><p class='noindent'>Single-molecule fluorescence measurements were performed with a Picoquant MT200 instrument
(Picoquant, Germany). For single-molecule FRET measurements, a diode laser (LDH-D-C-485,
PicoQuant, Germany) was synchronized with a supercontinuum laser (SuperK Extreme, NKT
Photonics, Denmark), filtered by a z582/15 band pass filter (Chroma) and pulsed at 20 MHz for pulsed
interleaved excitation (PIE) (Müller et al., 2005) of labeled molecules. Emitted photons were
collected with a 60x1.2 UPlanSApo Superapochromat water immersion objective (Olympus, Japan),
passed through a dichroic mirror (ZT568rpc, Chroma, USA), and filtered by a 100 <span class='cmmi-12'>μ</span>m pinhole
(Thorlabs, USA). Photons are counted and accumulated by a HydraHarp 400 TCSPC module
(Picoquant, Germany). For FRET-FCS measurements, the same diode laser was used in
continuous-wave mode to excite the donor dye. Photons emitted from the sample were
                                                                                      
                                                                                      
collected by the objective, and scattered light was suppressed by a filter (HQ500LP, Chroma
Technology) before the emitted photons passed the confocal pinhole (100 mm diameter).
The emitted photons were then distributed into four channels, first by a polarizing beam
splitter and then by a dichroic mirror (585DCXR, Chroma) for each polarization. Donor
and acceptor emission was filtered (ET525/50m or HQ642/80m, respectively, Chroma
Technology) and then focused on SPAD detectors (Excelitas, USA). The arrival time of
every detected photon was recorded with a HydraHarp 400 TCSPC module (PicoQuant,
Germany).
</p><!-- l. 77 --><p class='noindent'>FRET experiments were performed by exciting the donor dye with a laser power of 100 <span class='cmmi-12'>μ</span>W
(measured at the back aperture of the objective). For pulsed interleaved excitation experiments, the
power used for exciting the acceptor dye was adjusted to match a total emission intensity after
acceptor excitation to the one observed upon donor excitation (between 50 and 70 mW).
Single-molecule FRET efficiency histograms were acquired from samples with protein concentrations
between 50 pM and 100 pM. Trigger times for excitation pulses (repetition rate 20 MHz) and photon
detection events were stored with 16 ps resolution.
</p><!-- l. 79 --><p class='noindent'>For fluorescence correlation spectroscopy (FCS) experiments, acceptor-donor labeled samples with a
concentration of 100 pM were excited by either the 485 nm diode laser or the supercontinuum laser at
the powers indicated above. However, in the experiments on protein oligomerization, due to an
increase in the fluorescence background upon addition of unlabeled protein above 1 <span class='cmmi-12'>μ</span>M, only the
correlations corresponding to direct acceptor excitation (582 nm) have been considered reliable for the
analysis.
</p><!-- l. 81 --><p class='noindent'>For nsFCS, FRET samples of acceptor-donor labeled protein with a concentration of 100 pM were
excited by the same diode laser but in continuum wavelength mode.
</p><!-- l. 83 --><p class='noindent'>All measurements were performed in 50 mM Tris pH 7.32, 143 mM <span class='cmmi-12'>β</span>-mercaptoethanol (for
photoprotection), 0.001% Tween 20 (for surface passivation) and GdmCl at the reported
concentrations. All measurements were performed in uncoated polymer coverslip cuvettes (Ibidi,
                                                                                      
                                                                                      
Wisconsin, USA), which significantly decrease the fraction of protein adhering to the surface
(compared to normal glass cuvettes) under native conditions. For comparison, experiments have been
performed also in glass cuvette coated with PEG, which provided analogous results to the polymeric
cuvette.
</p><!-- l. 85 --><p class='noindent'>Each sample was measured for at least 30 min at room temperature (295 <span class='cmsy-10x-x-120'>± </span>0.5 K).
</p><!-- l. 87 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x17-157000D.1.4'></a>FRET efficiency histograms. </h5>
<!-- l. 88 --><p class='noindent'>Fluorescence bursts from individual molecules were identified by time-binning photons in bins of 1
ms and retaining the burst if the total number of photons detected after donor excitation was larger
than 30. Transfer efficiencies for each burst were calculated according to <span class='cmmi-12'>E </span><span class='cmr-12'>= </span><span class='cmmi-12'>nA∕</span><span class='cmr-12'>(</span><span class='cmmi-12'>nA </span><span class='cmr-12'>+ </span><span class='cmmi-12'>nD</span><span class='cmr-12'>) </span>where
<span class='cmmi-12'>nD </span>and <span class='cmmi-12'>nA </span>are the number of donor and and receptor photons, respectively. Corrections for
background, acceptor direct excitation, channel crosstalk, differences in detector efficiencies, and
quantum yields of the dyes were applied (Schuler et al., 2012). The labeling stoichiometry ratio <span class='cmmi-12'>S </span>was
computed accordingly to
</p>
<table class='equation'><tr><td><a id='x17-157001r1'></a>
<center class='math-display'>
<img src='main24x.png' alt='          ID
S = --------------
    γP IE(IA + ID)
' class='math-display' /></center></td><td class='equation-label'>(D.1)</td></tr></table>
<!-- l. 91 --><p class='nopar'>
where <span class='cmmi-12'>I</span><sub><span class='cmmi-8'>D</span></sub>and <span class='cmmi-12'>I</span><sub><span class='cmmi-8'>A</span></sub> represent the total intensities observed after donor and acceptor excitation and <span class='cmmi-12'>γ</span><sub><span class='cmmi-8'>PIE</span></sub>
provides a correction factor to account for differences in the detection efficiency and laser intensities.
Bursts with stoichiometry corresponding to 1:1 donor:acceptor labeling (in contrast to donor and
acceptor only populations) were selected and finally from the selected bursts a histogram of transfer
efficiencies is constructed. Variations in the selection criteria for the stoichiometry ratio
do not impact significantly the observed mean transfer efficiency (within experimental
errors).
</p><!-- l. 94 --><p class='noindent'>To estimate the mean transfer efficiency and deconvolve multiple populations (e.g for the NTD
construct) from the transfer efficiency histograms, each population was approximated with a Gaussian
peak function. For fitting more than one peak, the histogram was analyzed with a sum of Gaussian
peak functions. For the conversion of transfer efficiency to distances, we used the value of the Förster
radius for Alexa488 and Alexa594 previously determined and reported in literature, <span class='cmmi-12'>R</span><sub><span class='cmr-8'>0</span></sub> = 5.4
nm<span class='cite'> [<a href='mainli5.html#XSchuler2002-hf'>506</a>]</span>. We further correct the value accounting for the dependence of the Förster
radius on the solution refractive index. The changes in refractive index caused by increasing
concentrations of GdmCl or KCl were measured with an Abbe refractometer (Bausch and Lomb,
USA).
</p><!-- l. 96 --><p class='noindent'>Finally, we estimated a systematic error on transfer efficiency of <span class='cmsy-10x-x-120'>± </span>0.03, based on the variation of
transfer efficiency of the same reference samples after different calibrations of the instrument over the
last two years. Standard deviation of the transfer efficiency for multiple repeats of the NTD, LINK,
and CTD constructs is equal or less than <span class='cmsy-10x-x-120'>± </span>0.01. Since we aim for a comparison with simulations,
here we consider the systematic error as the larger source of error and we propagate the corresponding
effect on all the calculated distances.
                                                                                      
                                                                                      
</p><!-- l. 98 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x17-158000D.1.4'></a>Fluorescence lifetimes and anisotropies analysis.</h5>
<!-- l. 99 --><p class='noindent'>A quantitative interpretation of this transfer efficiency in terms of distance distribution requires the
investigation of protein dynamics. A first method to assess whether the transfer efficiency reports
about a rigid distance (e.g. structure formation or persistent interaction with the RBD) or is the result
of a dynamic average across multiple conformations is the comparison of transfer efficiency and
fluorescence lifetime.The interdependence of these two factors is expected to be linear if the protein
conformations are identical on both timescales (nanoseconds as detected by the fluorescence lifetime,
milliseconds as computed from the number of photons in each burst). Alternatively, protein dynamics
give rise to a departure from the linear relation and an analytical limit can be computed for
configurations rearranging much faster than the burst duration. The dependence of the
fluorescence lifetimes on transfer efficiencies determined for each burst was compared with
the behavior expected for fixed distances and for a chain sampling a broad distribution of
distances. For a fixed distance, <span class='cmmi-12'>R</span>, the mean donor lifetime in the presence of acceptor is given
by
</p>
<table class='equation'><tr><td><a id='x17-158001r2'></a>
<center class='math-display'>
<img src='main25x.png' alt='tD (R) = tD0(1 − E (R ))
' class='math-display' /></center></td><td class='equation-label'>(D.2)</td></tr></table>
<!-- l. 102 --><p class='nopar'>
where <span class='cmmi-12'>t</span><sub><span class='cmmi-8'>D</span></sub> is the lifetime in the absence of acceptor, and
</p>
<table class='equation'><tr><td><a id='x17-158002r3'></a>
<center class='math-display'>
<img src='main26x.png' alt='E (R ) = ---1---
         1 + R66-
             R0
' class='math-display' /></center></td><td class='equation-label'>(D.3)</td></tr></table>
<!-- l. 106 --><p class='nopar'>
For a chain with a dye-to-dye distance distribution <span class='cmmi-12'>P</span><span class='cmr-12'>(</span><span class='cmmi-12'>R</span><span class='cmr-12'>)</span>, the donor lifetime is
</p>
<table class='equation'><tr><td><a id='x17-158003r4'></a>
<center class='math-display'>
                                                                                      
                                                                                      
<img src='main27x.png' alt='     ∫
       tI(t)dt
tD = -∫-------
        I(t)dt
' class='math-display' /></center></td><td class='equation-label'>(D.4)</td></tr></table>
<!-- l. 110 --><p class='nopar'>
where
</p>
<table class='equation'><tr><td><a id='x17-158004r5'></a>
<center class='math-display'>
<img src='main28x.png' alt='                   --−-t--
I(t) = I0P (R) exp(tD (R ))dR
' class='math-display' /></center></td><td class='equation-label'>(D.5)</td></tr></table>
<!-- l. 114 --><p class='nopar'>
is the time-resolved fluorescence emission intensity following donor excitation. A similar calculation
can be carried out for describing the acceptor lifetime<span class='cite'> [<a href='mainli5.html#XChung2016-fp'>507</a>]</span> delay given by
</p>
<table class='equation'><tr><td><a id='x17-158005r6'></a>
                                                                                      
                                                                                      
<center class='math-display'>
<img src='main29x.png' alt='tA(R ) − tA0
------------
    tD0
' class='math-display' /></center></td><td class='equation-label'>(D.6)</td></tr></table>
<!-- l. 118 --><p class='nopar'>
Donor and acceptor lifetimes at different concentrations of GdmCl were analyzed by fitting
subpopulation-specific time-correlated photon counting histograms after donor and acceptor
excitation, respectively.
</p><!-- l. 121 --><p class='noindent'>Multiparameter detection allows also excluding possible artifacts, such as insufficient rotational
averaging of the fluorophores or quenching of the dyes. Subpopulation-specific anisotropies were
determined for both donor and acceptor of all three constructs for NTD, LINK, and CTD, and values
were found to vary between 0.1 and 0.2 for the donor and between 0.1 and 0.2 for the
acceptor, sufficiently low to assume as a good approximation for the orientational factor
<span class='cmmi-12'>κ</span><sup><span class='cmr-8'>2</span></sup> <span class='cmr-12'>= 2</span><span class='cmmi-12'>∕</span><span class='cmr-12'>3</span>.
</p><!-- l. 123 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x17-159000D.1.4'></a>Fluorescence Correlation Spectroscopy (FCS) analysis.</h5>
<!-- l. 124 --><p class='noindent'>In order to determined changes in the hydrodynamic radius (Rh) of the protein, FCS correlations were
analyzed assuming 3D diffusion of the molecule across a three dimensional Gaussian profile of the
confocal volume (Rigler, Eur Blophys J (1993) 22:169-175). For 1 diffusing species, and in the
absence of photophysical transitions in the time scale of the lag times analyzed, this formalism
amounts to the following time autocorrelation function.
                                                                                      
                                                                                      
</p>
<table class='equation'><tr><td><a id='x17-159001r7'></a>
<center class='math-display'>
<img src='main30x.png' alt='           1      τ           τ
g(τ) = t + --(1 + ---)− 1(1 +  -----)− 1∕2
           N      τD         α2τD
' class='math-display' /></center></td><td class='equation-label'>(D.7)</td></tr></table>
<!-- l. 127 --><p class='nopar'>
where <span class='cmmi-12'>N </span>is the average number of molecules in the confocal volume, <span class='cmmi-12'>τ</span>D is the diffusion time
along the xy plane, <span class='cmmi-12'>α </span>is the eccentricity of the three dimensional Gaussian observational
volume.
</p>
<table class='equation'><tr><td><a id='x17-159002r8'></a>
<center class='math-display'>
<img src='main31x.png' alt='      ω2
τD =  -xy-
      4D
' class='math-display' /></center></td><td class='equation-label'>(D.8)</td></tr></table>
                                                                                      
                                                                                      
<!-- l. 131 --><p class='nopar'>
where D is the 3D translational diffusion coefficient and <span class='cmmi-12'>ω</span>xy is the radius from the center of the laser
beam at which the light intensity decreases e<sup><span class='cmr-8'>2</span></sup> times from its maximum value at the center. <span class='cmmi-12'>α </span>=
<span class='cmmi-12'>ω</span>z/<span class='cmmi-12'>ω</span>xy.
</p><!-- l. 134 --><p class='noindent'>Additionally, in order to account for contributions of the photophysics of the fluorophore to the
correlation observed in the <span class='cmmi-12'>μ</span>s timescale, we added two triplet terms multiplying the diffusion
correlation term (see for example Krichevsky, Rep. Prog. Phys. 65 (2002) 251–297). The overall
equation that we fit to the FCS traces is then
</p>
<table class='equation'><tr><td><a id='x17-159003r9'></a>
<center class='math-display'>
<img src='main32x.png' alt='                                      τ                  τ
g (τ ) = 1 + (gD (τ) − 1)(1 + cT1 exp(−---))(1 + cT2exp (−  ---))
                                     τT1                 τT 2
' class='math-display' /></center></td><td class='equation-label'>(D.9)</td></tr></table>
<!-- l. 137 --><p class='nopar'>
where <span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>T</span><span class='cmr-8'>1</span></sub>, <span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>T</span><span class='cmr-8'>2</span></sub><span class='cmmi-12'>,c</span><sub><span class='cmmi-8'>T</span><span class='cmr-8'>1</span></sub><span class='cmmi-12'>,andc</span><sub><span class='cmmi-8'>T</span><span class='cmr-8'>1</span></sub>, denotes the characteristic times and amplitudes of the contributions of
two triplet states to<span class='cmmi-12'>g</span><span class='cmr-12'>(</span><span class='cmmi-12'>τ</span><span class='cmr-12'>)</span>. Parameters <span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>D</span></sub><span class='cmmi-12'>,τ</span><sub><span class='cmmi-8'>T</span><span class='cmr-8'>1</span></sub><span class='cmmi-12'>,τ</span><sub><span class='cmmi-8'>T</span><span class='cmr-8'>2</span></sub><span class='cmmi-12'>,c</span><sub><span class='cmmi-8'>T</span><span class='cmr-8'>1</span></sub><span class='cmmi-12'>,c</span><sub><span class='cmmi-8'>T</span><span class='cmr-8'>2</span></sub><span class='cmmi-12'>andN </span>were fitted by least square nonlinear
regression analysis for each concentration of unlabeled protein tested (Fig. <a href='#x17-168033r13'>D.13<!-- tex4ht:ref: fig:appendix-e-fig13  --></a>A-B), while <span class='cmmi-12'>α </span>was
fixed at a value of 6 determined independently from analysis of fluorescence intensity profiles of
fluorescent nanobeads.
</p><!-- l. 140 --><p class='noindent'>Making use of the definition of <span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>D</span></sub> and the Stokes-Einstein equation, we have, for each concentration
                                                                                      
                                                                                      
of unlabeled protein
</p>
<table class='equation'><tr><td><a id='x17-159004r10'></a>
<center class='math-display'>
<img src='main33x.png' alt='τD--   Rh--
τD0 =  Rh0
' class='math-display' /></center></td><td class='equation-label'>(D.10)</td></tr></table>
<!-- l. 143 --><p class='nopar'>
where <span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>D</span><span class='cmr-8'>0</span></sub> and <span class='cmmi-12'>R</span><sub><span class='cmmi-8'>h</span><span class='cmr-8'>0</span></sub> are the diffusion time and hydrodynamic radius in the absence of unlabeled
protein, respectively. Error bars in Fig. <a href='#x17-168033r13'>D.13<!-- tex4ht:ref: fig:appendix-e-fig13  --></a> B are the standard errors of Rh / Rh0 estimated from
propagation of the standard errors across multiple measurements of the diffusion times obtained from
the fit.
</p><!-- l. 146 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x17-160000D.1.4'></a>Nanosecond Fluorescence Correlation Spectroscopy.</h5>
<!-- l. 147 --><p class='noindent'>Autocorrelation curves of acceptor and donor channels and cross-correlation curves between acceptor
and donor channels were calculated with the methods described previously<span class='cite'> [<a href='mainli5.html#XNettels2007-lf'>330</a>, <a href='mainli5.html#XNettels2008-jk'>508</a>]</span>. All samples
have been measured at a concentration of 100 pM and bursts with a transfer efficiency between 0.3
                                                                                      
                                                                                      
and 0.8 have been selected to eliminate the contribution of donor only to the correlation amplitude.
Finally, the correlation was computed over a time window of 5 <span class='cmmi-12'>μ</span>s and characteristics timescales were
extracted according to:
</p>
<table class='equation'><tr><td><a id='x17-160001r11'></a>
<center class='math-display'>
<img src='main34x.png' alt='          1               τ − τ                  τ − τ               τ − τ
gij = 1 + ---(1 − cAB exp [− -----0])(1 + cCD exp[− -----0)(1 + cT exp [−-----0)
         N                 τAB                    τCD                  τT
' class='math-display' /></center></td><td class='equation-label'>(D.11)</td></tr></table>
<!-- l. 150 --><p class='nopar'>
where N is the mean number of molecules in the confocal volume and <span class='cmmi-12'>i </span>and <span class='cmmi-12'>j </span>indicate the type of
signal (either from the Acceptor or Donor channels). The three multiplicative terms describe the
contribution to amplitude and timescale of photon antibunching (AB), chain dynamics (CD), and
triplet blinking of the dyes (T). <span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>CD</span></sub> is then converted in the reconfiguration time of the interdye
distance <span class='cmmi-12'>τ</span><sub><span class='cmmi-8'>r</span></sub> correcting for the filtering effect of FRET as described previously<span class='cite'> [<a href='mainli5.html#XGopich2009-ow'>509</a>]</span>. An
additional multiplicative CD term has been added only for the donor-donor correlations
to describe the fast decay observed at very short time. Such a decay is not found in the
correlations of other disordered proteins measured on the instrument and we associate the fast
decay with the rotational motion of the overall protein. A fit to this fast decay is about 2
ns.
                                                                                      
                                                                                      
</p><!-- l. 153 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x17-161000D.1.4'></a>Polymer models of distance distributions.</h5>
<!-- l. 154 --><p class='noindent'>Conversion of mean transfer efficiencies for fast rearranging ensembles requires the assumption of a
distribution of distances. Here, we compared the results of two distinct polymer models: the Gaussian
model and a Self-Avoiding Walk (SAW) model that accounts for changes in the excluded
volume<span class='cite'> [<a href='mainli5.html#XSchafer2012-bh'>510</a>]</span>. This second model has been shown to provide a better description of chain distribution
and scaling exponent when compared to distance distributions from MD simulations<span class='cite'> [<a href='mainli5.html#XZheng2018-ej'>511</a>]</span>.
Importantly, both models rely only on one single fitting parameter, the root mean square
interdye distance <span class='cmmi-12'>r </span><span class='cmr-12'>= </span><span class='cmsy-10x-x-120'>⟨</span><span class='cmmi-12'>R</span><sup><span class='cmr-8'>2</span></sup><span class='cmsy-10x-x-120'>⟩</span><sup><span class='cmr-8'>1</span><span class='cmmi-8'>∕</span><span class='cmr-8'>2</span></sup> for the Gaussian chain and the scaling exponent <span class='cmmi-12'>ν </span>for the SAW
model.
</p><!-- l. 156 --><p class='noindent'>Estimates of these parameters are obtained by numerically solving:
</p>
<table class='equation'><tr><td><a id='x17-161001r12'></a>
<center class='math-display'>
<img src='main35x.png' alt='       ∫ lc
⟨E ⟩ =     P (R)E (R)dr
        0
' class='math-display' /></center></td><td class='equation-label'>(D.12)</td></tr></table>
<!-- l. 159 --><p class='nopar'>
where <span class='cmmi-12'>R </span>is the interdye distance, <span class='cmmi-12'>l</span><sub><span class='cmmi-8'>c</span></sub> is the contour length of the chain, <span class='cmmi-12'>P</span><span class='cmr-12'>(</span><span class='cmmi-12'>r</span><span class='cmr-12'>) </span>represents the chosen
                                                                                      
                                                                                      
distribution, and <span class='cmmi-12'>E</span><span class='cmr-12'>(</span><span class='cmmi-12'>R</span><span class='cmr-12'>)</span>is the Förster equation for the dependence of transfer efficiency on distance <span class='cmmi-12'>R</span>
and Förster radius:
</p>
<table class='equation'><tr><td><a id='x17-161002r13'></a>
<center class='math-display'>
<img src='main36x.png' alt='           R6
E(R ) = --6-0--6-
        R 0 + R
' class='math-display' /></center></td><td class='equation-label'>(D.13)</td></tr></table>
<!-- l. 163 --><p class='nopar'>
</p><!-- l. 165 --><p class='noindent'>The Gaussian chain distribution is given by:
</p>
<table class='equation'><tr><td><a id='x17-161003r14'></a>
<center class='math-display'>
<img src='main37x.png' alt='                      3            3R2
PF JC(R, r) = 4πR2 (----2)3∕2 exp(− --2-)
                    2πr            2r
' class='math-display' /></center></td><td class='equation-label'>(D.14)</td></tr></table>
<!-- l. 168 --><p class='nopar'>
The SAW model can be expressed as:
</p>
<table class='equation'><tr><td><a id='x17-161004r15'></a>
<center class='math-display'>
<img src='main38x.png' alt='P    (R, ν) = A  -4π--(--R--)2+g exp(− A (--R--)δ)
 SAW            1b0N ν b0N ν            2 b0N ν
' class='math-display' /></center></td><td class='equation-label'>(D.15)</td></tr></table>
<!-- l. 172 --><p class='nopar'>
where
</p>
<table class='equation'><tr><td><a id='x17-161005r16'></a>
<center class='math-display'>
                                                                                      
                                                                                      
<img src='main39x.png' alt='                3+g
      δ Γ [5 + g -2-]       Γ [5 + g]      γ − 1        1
A1 = ---------δ-5+g-,A2  = (------δg-)δ,g = ------,δ = ------
     4π Γ [3 + gδ  2 ]       Γ [5 + δ]        ν        1 − ν
' class='math-display' /></center></td><td class='equation-label'>(D.16)</td></tr></table>
<!-- l. 179 --><p class='nopar'>
<span class='cmmi-12'>γ </span><span class='cmr-12'>=</span>1.1615, and <span class='cmr-12'>Γ </span>is the Euler Gamma Function, <span class='cmmi-12'>b</span><sub><span class='cmr-8'>0</span></sub>=0.55nm is an empirical prefactor<span class='cite'> [<a href='mainli5.html#XZheng2018-ej'>511</a>]</span>, <span class='cmmi-12'>N </span>is the
number of residues between the fluorophores, and <span class='cmmi-12'>ν </span>is the scaling exponent.
</p><!-- l. 182 --><p class='noindent'>Finally, when converting the distance from transfer efficiencies, to account for the length of dye
linkers and compare the experimental data with simulations, the root-mean-squared interdye distance r
was rescaled according to
</p>
<table class='equation'><tr><td><a id='x17-161006r17'></a>
<center class='math-display'>
<img src='main40x.png' alt='rm,n = |m  − n|0.5Idye|m  − n + 2Idye|0.5
' class='math-display' /></center></td><td class='equation-label'>(D.17)</td></tr></table>
<!-- l. 185 --><p class='nopar'>
with <span class='cmmi-12'>I</span><sub><span class='cmmi-8'>d</span></sub><span class='cmmi-12'>ye</span>=4.5 (Aznauryan et al., 2016; Hoffmann et al., 2007). Finally, the persistence length is
computed using the Gaussian conversion <span class='cmmi-12'>r</span><sup><span class='cmr-8'>2</span></sup> <span class='cmr-12'>= 2</span><span class='cmmi-12'>l</span><sub>
<span class='cmmi-8'>p</span></sub><span class='cmmi-12'>l</span><sub><span class='cmmi-8'>c</span></sub><span class='cite'> [<a href='mainli5.html#XRubinstein2003-fa'>348</a>]</span>.
                                                                                      
                                                                                      
</p><!-- l. 188 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x17-162000D.1.4'></a>Binding of denaturant and folding.</h5>
<!-- l. 189 --><p class='noindent'>As in previous works<span class='cite'> [<a href='mainli5.html#XMuller-Spath2010-oy'>512</a>, <a href='mainli5.html#XSoranno2012-jn'>318</a>, <a href='mainli5.html#XSoranno2017-bi'>322</a>]</span>, we model the chain expansion with the denaturant in terms of
a simple binding model:
</p>
<table class='equation'><tr><td><a id='x17-162001r18'></a>
<center class='math-display'>
<img src='main41x.png' alt='              Kc
rc = r0(1 + ρ------ )
            1 + Kc
' class='math-display' /></center></td><td class='equation-label'>(D.18)</td></tr></table>
<!-- l. 192 --><p class='nopar'>
Where <span class='cmmi-12'>r</span><sub><span class='cmr-8'>0</span></sub> is the mean square interdye distance at zero denaturant,<span class='cmmi-12'>ρ </span>is a term the captures the extent of
chain expansion with the denaturant compared to <span class='cmmi-12'>r</span><sub><span class='cmr-8'>0</span></sub>, and the <span class='cmmi-12'>K </span>is the binding constant, and <span class='cmmi-12'>c </span>is the
concentration of denaturant.
</p><!-- l. 195 --><p class='noindent'>In presence of folded domains, we can imagine the folding/unfolding of the domains can affect the
overall size of the chain because of an increase or decrease of excluded volume due to the surrounding
folded domains (which screen part of the available conformations) or because of the folding or
unfolding of elements in the region between the fluorophores. To account for this effect, as in the case
of the NTD, we weighed the effect of denaturant on the chain for the fraction folded <span class='cmmi-12'>f</span><sub><span class='cmmi-8'>f</span></sub> and unfolded
                                                                                      
                                                                                      
<span class='cmmi-12'>f</span><sub><span class='cmmi-8'>u</span></sub> accordingly to:
</p>
<table class='equation'><tr><td><a id='x17-162002r19'></a>
<center class='math-display'>
<img src='main42x.png' alt='                          Kc
rc = (r0ff + r0ufu)(1 + ρ-------)
                        1 + Kc
' class='math-display' /></center></td><td class='equation-label'>(D.19)</td></tr></table>
<!-- l. 198 --><p class='nopar'>
where <span class='cmmi-12'>r</span><sub><span class='cmr-8'>0</span><span class='cmmi-8'>f</span></sub> and <span class='cmmi-12'>r</span><sub><span class='cmr-8'>0</span><span class='cmmi-8'>u</span></sub> are the root mean square interdye distance in presence of folded or unfolded
domains in native buffer,
</p>
<table class='equation'><tr><td><a id='x17-162003r20'></a>
<center class='math-display'>
<img src='main43x.png' alt='       exp [− m (c − c )]
ff = ----------------m----
     1 + exp [− m (c − cm)]
' class='math-display' /></center></td><td class='equation-label'>(D.20)</td></tr></table>
                                                                                      
                                                                                      
<!-- l. 202 --><p class='nopar'>
and <span class='cmmi-12'>f</span><sub><span class='cmmi-8'>u</span></sub> <span class='cmr-12'>= 1 </span><span class='cmsy-10x-x-120'>−</span><span class='cmmi-12'>f</span><sub><span class='cmmi-8'>f</span></sub>, where <span class='cmmi-12'>c</span><sub><span class='cmmi-8'>m</span></sub> the midpoint concentration and <span class='cmmi-12'>m </span>the denaturant m value, representing the
dependence of free energy on denaturant concentration. The stability parameter <span class='cmr-12'>Δ</span><span class='cmmi-12'>G</span><sub><span class='cmr-8'>0</span></sub> can be computed
as <span class='cmr-12'>Δ</span><span class='cmmi-12'>G</span><sub><span class='cmr-8'>0</span></sub> <span class='cmr-12'>= </span><span class='cmmi-12'>mc</span><sub><span class='cmmi-8'>m</span></sub>.
</p><!-- l. 205 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x17-163000D.1.4'></a>Polymer model of electrostatic interactions.</h5>
<!-- l. 206 --><p class='noindent'>The disordered regions of the N protein are enriched in positive and negative charges. To provide
a term of comparison in the interpretation of protein conformations as function of salt
concentration, we use the polymer theory for polyampholyte solutions developed by Higgs
and Joanny <span class='cite'> [<a href='mainli5.html#XHiggs1991-rl'>513</a>, <a href='mainli5.html#XMuller-Spath2010-oy'>512</a>]</span>, which has been shown previously to capture quantitatively the
conformational changes of unstructured proteins. Briefly, the root mean square interdye distance is
equal to <span class='cmmi-12'>r </span><span class='cmr-12'>= </span><span class='cmmi-12'>N</span><sup><span class='cmr-8'>0</span><span class='cmmi-8'>.</span><span class='cmr-8'>5</span></sup> <span class='cmsy-10x-x-120'>∗ </span><span class='cmmi-12'>l</span><sub>
<span class='cmr-8'>0</span></sub> <span class='cmsy-10x-x-120'>∗ </span><span class='cmmi-12'>α </span>where <span class='cmmi-12'>N </span>is the number of monomers in the disordered region,
<span class='cmmi-12'>l</span><sub><span class='cmr-8'>0</span></sub> is the length of elementary segment (here 0.36 nm) and <span class='cmmi-12'>α </span>is the ratio between <span class='cmmi-12'>l </span>and
<span class='cmmi-12'>l</span><sub><span class='cmr-8'>0</span></sub>, with <span class='cmmi-12'>l </span>being a rescaled segment that accounts for excluded volume and electrostatic
interactions.
</p><!-- l. 208 --><p class='noindent'><span class='cmmi-12'>α</span>is computed according to the equation proposed by Higgs and Joanny<span class='cite'> [<a href='mainli5.html#XHiggs1991-rl'>513</a>, <a href='mainli5.html#XMuller-Spath2010-oy'>512</a>]</span>:
</p>
<table class='equation'><tr><td><a id='x17-163001r21'></a>
                                                                                      
                                                                                      
<center class='math-display'>
<img src='main44x.png' alt='           4  3
α5 −  α3 = -( --)1.5N 0.5v ∗
           3  2π
' class='math-display' /></center></td><td class='equation-label'>(D.21)</td></tr></table>
<!-- l. 211 --><p class='nopar'>
where <span class='cmmi-12'>v</span><sup><span class='cmsy-8'>∗</span></sup> is an effective excluded volume given by the sum of three terms:
</p>
<table class='equation'><tr><td><a id='x17-163002r22'></a>
<center class='math-display'>
<img src='main45x.png' alt='             4πl (f − g)2    πl2(f − g)2
v∗b3 = vb3 + ---B---------−  --B---------
                  k2              k
' class='math-display' /></center></td><td class='equation-label'>(D.22)</td></tr></table>
<!-- l. 215 --><p class='nopar'>
Here, <span class='cmmi-12'>v </span>is the excluded volume (accounting for physical excluded volume and positive and attractive
interactions that are not due to electrostatics), <span class='cmmi-12'>f </span>and <span class='cmmi-12'>g </span>are the fraction of positive and negative residue
respectively for considered segment of the protein, <span class='cmmi-12'>k </span>is the Debye screening length, and <span class='cmmi-12'>l</span><sub><span class='cmmi-8'>b</span></sub> is the
Bjerrum length.
</p><!-- l. 218 --><p class='noindent'>Importantly, when accounting for the fraction of negative charges, we also account for the contribution
of the -2 net charge of each dye at pH 7.3.
                                                                                      
                                                                                      
</p><!-- l. 220 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x17-164000D.1.4'></a>Salt dependence of NTD, LINK, and CTD conformations.</h5>
<!-- l. 221 --><p class='noindent'>In addition to studying the conformations under native buffer conditions, we investigate how salt
affects the conformations of the three disordered regions. We started by testing the effects of
electrostatic interactions on the NTD conformational ensemble. Moving from buffer conditions and
increasing concentration of KCl, we observed a small but noticeable shift toward lower transfer
efficiencies, which represents an expansion of the NTD due to screening of electrostatic interactions.
This can be rationalized in terms of the polyampholyte theory of Higgs and Joanny<span class='cite'> [<a href='mainli5.html#XHiggs1991-rl'>513</a>, <a href='mainli5.html#XMuller-Spath2010-oy'>512</a>]</span> (see
Table <a href='#x17-168003r2'>D.2<!-- tex4ht:ref: tab:appendix-e-tab2  --></a>), where the increasing concentration of ions screens the interaction between oppositely
charged residues (see Fig. <a href='#x17-168027r10'>D.10<!-- tex4ht:ref: fig:appendix-e-fig10  --></a>).
</p><!-- l. 223 --><p class='noindent'>We then analyzed for comparison the LINK construct. Interestingly, we find a negligible effect of salt
screening on the root mean square distance r172-245 as measured by FRET (see Fig. <a href='#x17-168027r10'>D.10<!-- tex4ht:ref: fig:appendix-e-fig10  --></a>).
Predictions of the Higgs and Joanny theory for the content of negative and positive charges within the
LINK construct indicates a variation of interdye distance dimension that is comparable with the
measurement error. It has to be noted that in this case the excluded volume term in the
Higgs and Joanny theory will empirically account not only for the excluded volume of the
amino acids in the chain, but also for the excluded volume occupied by the two folded
domains.
</p><!-- l. 225 --><p class='noindent'>Finally, we test if the addition of salt can provide similar effects than those obtained by
GdmCl on the conformations of the CTD: interestingly, we do not observe any significant
variation either in transfer efficiency or distribution width (Fig. <a href='#x17-168027r10'>D.10<!-- tex4ht:ref: fig:appendix-e-fig10  --></a>), suggesting that the
broadening of the population observed for the CTD does not originate from electrostatic
interactions.
                                                                                      
                                                                                      
</p><!-- l. 227 --><p class='noindent'>
</p>
<h4 class='subsectionHead'><span class='titlemark'>D.1.5    </span> <a id='x17-165000D.1.5'></a>Testing protein oligomerization.</h4>
<!-- l. 228 --><p class='noindent'>NativePAGE experiments were performed to verify that purified recombinantly expressed
SARS-CoV-2 N protein is capable of forming dimers and oligomers, in analogy to SARS-CoV N
protein, and as shown in more recent work for SARS-CoV-2 <span class='cite'> [<a href='mainli5.html#XChang2009-vd'>303</a>, <a href='mainli5.html#XTakeda2008-wl'>306</a>, <a href='mainli5.html#XZeng2020-cw'>316</a>]</span>. Indeed, NativePAGE
experiments reveal the existence of multiple bands (Fig. <a href='#x17-168033r13'>D.13<!-- tex4ht:ref: fig:appendix-e-fig13  --></a>C-D). However, since the lowest band in
the NativePAGE corresponds to an apparent molecular weight of <span class='cmsy-10x-x-120'>∼</span>70-80 kDa, we wanted to verify the
oligomeric state of this band.
</p><!-- l. 230 --><p class='noindent'>To test whether the apparent mass is due to a slow mobility of the protein because of its high positive
charge, we performed crosslinking experiments. These experiments confirm the formation of dimers,
tetramers, and high oligomeric species, as a function of protein concentration above 500 nM (Fig.
<a href='#x17-168033r13'>D.13<!-- tex4ht:ref: fig:appendix-e-fig13  --></a>E-F). These oligomeric species are in equilibrium with the monomer, the smallest species on the
denaturing SDS PAGE (which has the expected molecular weight of <span class='cmsy-10x-x-120'>∼</span>45 kDa). It has to be noted
that, because of the slow reactivity of the crosslinking agent (see Methods below), the
crosslinking experiments do not represent the population of monomeric and oligomeric species
at equilibrium. However, the comparison between the NativePAGE and the crosslinking
experiments supports the fact that the smallest band in the NativePAGE is indeed the monomer
protein.
</p><!-- l. 232 --><p class='noindent'>We finally turned to Fluorescence Correlation Spectroscopy (FCS) to test whether labeled protein can
form dimers. We measured the CTD construct that carries one labeling position at the end of the
oligomerization domain. When increasing the concentration of unlabeled protein, we observe a
systematic increase in the hydrodynamic radius when compared to the hydrodynamic radius under
native conditions (Fig. <a href='#x17-168033r13'>D.13<!-- tex4ht:ref: fig:appendix-e-fig13  --></a>A-B). This suggests that the labeled protein can form higher oligomeric
                                                                                      
                                                                                      
species in a concentration regime comparable to the one observed in NativePAGE and
SDS PAGE experiments and that at 100 pM (the concentration used in single-molecule
experiments), no oligomer is formed. Caution must be used in the interpretation of the
oligomeric bound species observed in FCS experiments, since labeling mutation may have
affected the affinity of the dimerization domain. Future experiments will address the role of
mutation on dimerization. Finally, all experiments have been performed at two different time
points, after 1 hour and after 24 hours of incubation of the labeled sample with unlabeled
protein to test any kinetic effect on the measured value. No significative difference has been
observed.
</p><!-- l. 234 --><p class='noindent'>Taken together, NativePAGE crosslinking experiments verify that in smFRET and FCS
experiments we are in fact monitoring the behavior of the monomeric SARS-CoV-2 N
protein.
</p><!-- l. 236 --><p class='noindent'>
</p>
<h4 class='subsectionHead'><span class='titlemark'>D.1.6    </span> <a id='x17-166000D.1.6'></a>Protein Crosslinking Methods.</h4>
<!-- l. 237 --><p class='noindent'>50 mM disuccinimidyl suberate (DSS) (Thermo Scientific) stock solution was prepared (10 mg into
540 uL of anhydrous DMSO (Sigma)). All protein samples were prepared in 20 mM NaPi pH 7.4
(with and without 200 mM NaCl) at the following concentrations: 0.1, 0.5, 1, 5, 10 and 20uM. DSS
stock solution was added to each sample to a final concentration of 1.25 mM. Samples were incubated
for 1 hour at room temperature. Samples were then quenched to a final concentration of 200mM Tris
pH 7.4 and allowed to incubate for 15 minutes. Crosslinked proteins were then analyzed using SDS
PAGE and Coomassie staining.
                                                                                      
                                                                                      
</p><!-- l. 239 --><p class='noindent'>
</p>
<h4 class='subsectionHead'><span class='titlemark'>D.1.7    </span> <a id='x17-167000D.1.7'></a>NativePAGE Methods.</h4>
<!-- l. 240 --><p class='noindent'>All protein samples were prepared in 20 mM NaPi pH 7.4 (with and without 200 mM
NaCl) at the following concentrations: 0.05, 0.1, 0.5, 1, 5, 10 and 20 uM. Samples were
subjected to NativePAGE (Invitrogen) and protein mobility was analyzed with Coomassie
staining.
</p><!-- l. 242 --><p class='noindent'>Development of turbidity in solutions of N protein and poly(rU) was followed through measurements
of absorbance at 340 nm in a microvolume spectrophotometer (NanoDrop, Thermo, USA). Mixtures
were prepared in 500 <span class='cmmi-12'>μ</span>l plastic reaction tubes by adding 4 <span class='cmmi-12'>μ</span>l protein solution into 3 <span class='cmmi-12'>μ</span>l of poly(rU) and
absorbance was recorded 45 s – 75 s after mixing. Working solutions were kept at room temperature
during experiments.
</p><!-- l. 244 --><p class='noindent'>Reaction media was 50 mM Tris, pH 7.5 (HCl), 0.002 % v/v Tween20, and NaCl as indicated in
Results.
</p><!-- l. 246 --><p class='noindent'>poly(rU) (Midland Certified Reagent Company, TX, USA, lot number 011805) was reconstituted into
this media from stocks dissolved in RNAse free water. According to the manufacturer, the
size of poly(rU) molecules is mostly less than 250 nucleotides (nt.) and longer than 200
nt.
</p><!-- l. 248 --><p class='noindent'>Protein stocks (in 50 mM Tris pH 8.0, 500 mM NaCl, 10% v/v glycerol) were buffer exchanged into
the desired buffer through size exclusion chromatography in Zeba Spin 7 k MWCO desalting columns
(Thermo, USA). poly(rU) concentrations in working dilutions were assessed through the absorbance
at 260 nm employing an extinction coefficient of 9.4 <span class='cmmi-12'>mM</span><sup><span class='cmsy-8'>−</span><span class='cmr-8'>1</span></sup><span class='cmmi-12'>cm</span><sup><span class='cmsy-8'>−</span><span class='cmr-8'>1</span></sup> (Michelson, 1959). Protein
concentrations were assessed through the absorbance at 280 nm employing an extinction coefficient of
42.53 <span class='cmmi-12'>mM</span><sup><span class='cmsy-8'>−</span><span class='cmr-8'>1</span></sup><span class='cmmi-12'>cm</span><sup><span class='cmsy-8'>−</span><span class='cmr-8'>1</span></sup>, computed according to the method proposed by Pace et al. (Pace et al.,
1995).
                                                                                      
                                                                                      
</p><!-- l. 250 --><p class='noindent'>The limiting concentrations of nucleic acid across which an increase in turbidity was detected were
estimated through interpolation of the data. To this end, an empirical equation, describing the trends
observed at all concentrations, was fitted to the data and then was solved to extract the poly(rU)
concentrations at which turbidity reaches a limit value above the background signal. We used
a limiting absorbance value of 0.005 units (340 nm, 1 mm path length). We found that
an appropriate function for this end is an exponential of a Gaussian distribution function
<span class='cmmi-12'>F</span><span class='cmr-12'>(</span><span class='cmmi-12'>x</span><span class='cmr-12'>)</span>:
</p>
<table class='equation'><tr><td><a id='x17-167001r23'></a>
<center class='math-display'>
<img src='main46x.png' alt='F (x) = A (1 − exp[− β γ(x)])
' class='math-display' /></center></td><td class='equation-label'>(D.23)</td></tr></table>
<!-- l. 253 --><p class='nopar'>
where
</p>
<table class='equation'><tr><td><a id='x17-167002r24'></a>
                                                                                      
                                                                                      
<center class='math-display'>
<img src='main47x.png' alt='           1         − (x − μ)2
γ (x) = ----0.5--exp [----------]
        (2π)  σ          2σ
' class='math-display' /></center></td><td class='equation-label'>(D.24)</td></tr></table>
<!-- l. 257 --><p class='nopar'>
</p><!-- l. 259 --><p class='noindent'>where <span class='cmmi-12'>x </span>denotes poly(rU) concentration and <span class='cmmi-12'>A,μ,σandμ </span>are parameters fitted through weighted
minimum least squares for each protein concentration (solid lines in Fig. <a href='mainch5.html#x10-76001r5'>5.5<!-- tex4ht:ref: fig:ch6-fig5  --></a>A-B and limiting value
points in panels C-D). To characterize the observed global trends of turbidity, as a function of both
RNA and protein concentration, we determined approximate functional forms of the dependence on
protein concentration of the individually fitted parameters (<span class='cmmi-12'>A</span><span class='cmr-12'>(</span><span class='cmmi-12'>p</span><span class='cmr-12'>)</span><span class='cmmi-12'>,β</span><span class='cmr-12'>(</span><span class='cmmi-12'>p</span><span class='cmr-12'>)</span><span class='cmmi-12'>,σ</span><span class='cmr-12'>(</span><span class='cmmi-12'>p</span><span class='cmr-12'>)</span><span class='cmmi-12'>andμ</span><span class='cmr-12'>(</span><span class='cmmi-12'>p</span><span class='cmr-12'>)</span>, where <span class='cmmi-12'>p </span>is
protein concentration). The observed dependencies were increasing linearly for <span class='cmmi-12'>μ</span><span class='cmr-12'>(</span><span class='cmmi-12'>p</span><span class='cmr-12'>) </span>and quadratic
for <span class='cmmi-12'>β</span><span class='cmr-12'>(</span><span class='cmmi-12'>p</span><span class='cmr-12'>) </span>and <span class='cmmi-12'>σ</span><span class='cmr-12'>(</span><span class='cmmi-12'>p</span><span class='cmr-12'>)</span>. A was the worst defined parameter and thus displayed the least clear
trend. For the results in absence of added salt we employed an increasing power function
with exponent as a fitting parameter (best fit value was <span class='cmmi-12'>&lt; </span>1), whereas for the results in
presence of 50 mM NaCl the trend of <span class='cmmi-12'>A</span><span class='cmr-12'>(</span><span class='cmmi-12'>p</span><span class='cmr-12'>) </span>was better described by a decreasing exponential
function.
</p><!-- l. 261 --><p class='noindent'>We thus used the functional forms <span class='cmmi-12'>A</span><span class='cmr-12'>(</span><span class='cmmi-12'>p</span><span class='cmr-12'>)</span><span class='cmmi-12'>,β</span><span class='cmr-12'>(</span><span class='cmmi-12'>p</span><span class='cmr-12'>)</span><span class='cmmi-12'>,σ</span><span class='cmr-12'>(</span><span class='cmmi-12'>p</span><span class='cmr-12'>) </span>and <span class='cmmi-12'>μ</span><span class='cmr-12'>(</span><span class='cmmi-12'>p</span><span class='cmr-12'>) </span>to construct a global function dependent
on both protein and RNA concentration. Global fitting of this equation to the whole set of turbidity
titration curves provided the turbidity contour plots shown in Fig. <a href='mainch5.html#x10-76001r5'>5.5<!-- tex4ht:ref: fig:ch6-fig5  --></a>C-D (solid lines). Contour lines
were computed at 1, 10, 20, 50 and 100 times the limiting value employed (A<sub><span class='cmr-8'>340</span><span class='cmmi-8'>nm,</span><span class='cmr-8'>1</span><span class='cmmi-8'>mm</span></sub>=
0.005).
                                                                                      
                                                                                      
</p><!-- l. 263 --><p class='noindent'>
</p>
<h3 class='sectionHead'><span class='titlemark'>D.2    </span> <a id='x17-168000D.2'></a>Supplementary Figures</h3>
<div class='table'>
                                                                                      
                                                                                      
<!-- l. 264 --><p class='noindent'><a id='x17-168001r1'></a></p><hr class='float' /><div class='float'>
                                                                                      
                                                                                      
<a id='x17-168002'></a>
 <div class='caption'><span class='id'>Table D.1: </span><span class='content'>Fit parameters to denaturant binding model. </span></div><!-- tex4ht:label?: x17-168001rD.2  -->
<div class='tabular'> <table class='tabular' cellspacing='0' cellpadding='0' id='TBL-14'><colgroup id='TBL-14-1g'><col id='TBL-14-1' /></colgroup><colgroup id='TBL-14-2g'><col id='TBL-14-2' /></colgroup><colgroup id='TBL-14-3g'><col id='TBL-14-3' /></colgroup><colgroup id='TBL-14-4g'><col id='TBL-14-4' /></colgroup><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-14-1-'><td style='white-space:nowrap; text-align:left;' id='TBL-14-1-1' class='td11'>          </td><td style='white-space:nowrap; text-align:left;' id='TBL-14-1-2' class='td11'> <span class='cmmi-12'>ρ                    </span></td><td style='white-space:nowrap; text-align:left;' id='TBL-14-1-3' class='td11'> K (M<sup><span class='cmsy-8'>−</span><span class='cmr-8'>1</span></sup>)                  </td><td style='white-space:nowrap; text-align:left;' id='TBL-14-1-4' class='td11'> <span class='cmmi-12'>r</span><sub>
<span class='cmr-8'>0</span></sub> (Å)                                                                                                                                              </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-14-2-'><td style='white-space:nowrap; text-align:left;' id='TBL-14-2-1' class='td11'> NTD (1 pop)  </td><td style='white-space:nowrap; text-align:left;' id='TBL-14-2-2' class='td11'> 1.3<span class='cmsy-10x-x-120'>± </span>0.2                  </td><td style='white-space:nowrap; text-align:left;' id='TBL-14-2-3' class='td11'> 0.36 <span class='cmsy-10x-x-120'>± </span>0.05             </td><td style='white-space:nowrap; text-align:left;' id='TBL-14-2-4' class='td11'> <!-- tex4ht:inline --><div class='tabular'><table class='tabular' cellspacing='0' cellpadding='0' id='TBL-15'><colgroup id='TBL-15-1g'><col id='TBL-15-1' /></colgroup><tr style='vertical-align:baseline;' id='TBL-15-1-'><td style='white-space:nowrap; text-align:left;' id='TBL-15-1-1' class='td00'>50 <span class='cmsy-10x-x-120'>± </span>2</td></tr><tr style='vertical-align:baseline;' id='TBL-15-3-'><td style='white-space:nowrap; text-align:left;' id='TBL-15-3-1' class='td00'>(fixed)  </td></tr></table>                                                                                                                                      </div>  </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-14-3-'><td style='white-space:nowrap; text-align:left;' id='TBL-14-3-1' class='td11'> NTD (2 pop)  </td><td style='white-space:nowrap; text-align:left;' id='TBL-14-3-2' class='td11'> (shared parameter)  </td><td style='white-space:nowrap; text-align:left;' id='TBL-14-3-3' class='td11'> (shared parameter)  </td><td style='white-space:nowrap; text-align:left;' id='TBL-14-3-4' class='td11'> 36 <span class='cmsy-10x-x-120'>± </span>3                                                                                                                                              </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-14-4-'><td style='white-space:nowrap; text-align:left;' id='TBL-14-4-1' class='td11'> LINK             </td><td style='white-space:nowrap; text-align:left;' id='TBL-14-4-2' class='td11'> 1.1 <span class='cmsy-10x-x-120'>± </span>0.03               </td><td style='white-space:nowrap; text-align:left;' id='TBL-14-4-3' class='td11'> 0.06 <span class='cmsy-10x-x-120'>±</span>0.03              </td><td style='white-space:nowrap; text-align:left;' id='TBL-14-4-4' class='td11'> <!-- tex4ht:inline --><div class='tabular'><table class='tabular' cellspacing='0' cellpadding='0' id='TBL-16'><colgroup id='TBL-16-1g'><col id='TBL-16-1' /></colgroup><tr style='vertical-align:baseline;' id='TBL-16-1-'><td style='white-space:nowrap; text-align:left;' id='TBL-16-1-1' class='td00'>57 <span class='cmsy-10x-x-120'>± </span>2</td></tr><tr style='vertical-align:baseline;' id='TBL-16-3-'><td style='white-space:nowrap; text-align:left;' id='TBL-16-3-1' class='td00'>(fixed)  </td></tr></table>                                                                                                                                      </div>  </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-14-5-'><td style='white-space:nowrap; text-align:left;' id='TBL-14-5-1' class='td11'> CTD               </td><td style='white-space:nowrap; text-align:left;' id='TBL-14-5-2' class='td11'> 0.47 <span class='cmsy-10x-x-120'>± </span>0.02             </td><td style='white-space:nowrap; text-align:left;' id='TBL-14-5-3' class='td11'> 0.36 <span class='cmsy-10x-x-120'>± </span>0.1               </td><td style='white-space:nowrap; text-align:left;' id='TBL-14-5-4' class='td11'> <!-- tex4ht:inline --><div class='tabular'><table class='tabular' cellspacing='0' cellpadding='0' id='TBL-17'><colgroup id='TBL-17-1g'><col id='TBL-17-1' /></colgroup><tr style='vertical-align:baseline;' id='TBL-17-1-'><td style='white-space:nowrap; text-align:left;' id='TBL-17-1-1' class='td00'>50 <span class='cmsy-10x-x-120'>± </span>2</td></tr><tr style='vertical-align:baseline;' id='TBL-17-3-'><td style='white-space:nowrap; text-align:left;' id='TBL-17-3-1' class='td00'>(fixed)  </td></tr></table>                                                                                                                                      </div>  </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr></table>
</div>
                                                                                      
                                                                                      
</div><hr class='endfloat' />
</div>
<div class='table'>
                                                                                      
                                                                                      
<!-- l. 278 --><p class='noindent'><a id='x17-168003r2'></a></p><hr class='float' /><div class='float'>
                                                                                      
                                                                                      
<a id='x17-168004'></a>
 <div class='caption'><span class='id'>Table D.2: </span><span class='content'>Fit parameters of Higgs &amp; Joanny theory</span></div><!-- tex4ht:label?: x17-168003rD.2  -->
<div class='tabular'> <table class='tabular' cellspacing='0' cellpadding='0' id='TBL-18'><colgroup id='TBL-18-1g'><col id='TBL-18-1' /></colgroup><colgroup id='TBL-18-2g'><col id='TBL-18-2' /></colgroup><tr class='hline'><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-18-1-'><td style='white-space:nowrap; text-align:left;' id='TBL-18-1-1' class='td11'>     </td><td style='white-space:nowrap; text-align:left;' id='TBL-18-1-2' class='td11'> v               </td></tr><tr class='hline'><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-18-2-'><td style='white-space:nowrap; text-align:left;' id='TBL-18-2-1' class='td11'> NTD </td><td style='white-space:nowrap; text-align:left;' id='TBL-18-2-2' class='td11'> 4.4 <span class='cmsy-10x-x-120'>± </span>0.1</td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-18-3-'><td style='white-space:nowrap; text-align:left;' id='TBL-18-3-1' class='td11'> LINK  </td><td style='white-space:nowrap; text-align:left;' id='TBL-18-3-2' class='td11'> 5.5 <span class='cmsy-10x-x-120'>± </span>0.3  </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-18-4-'><td style='white-space:nowrap; text-align:left;' id='TBL-18-4-1' class='td11'> CTD    </td><td style='white-space:nowrap; text-align:left;' id='TBL-18-4-2' class='td11'> 8.4 <span class='cmsy-10x-x-120'>± </span>0.9  </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td></tr></table>
</div>
                                                                                      
                                                                                      
</div><hr class='endfloat' />
</div>
<div class='table'>
                                                                                      
                                                                                      
<!-- l. 291 --><p class='noindent'><a id='x17-168005r3'></a></p><hr class='float' /><div class='float'>
                                                                                      
                                                                                      
<a id='x17-168006'></a>
 <div class='caption'><span class='id'>Table D.3: </span><span class='content'>Scaling exponents</span></div><!-- tex4ht:label?: x17-168005rD.2  -->
<div class='tabular'> <table class='tabular' cellspacing='0' cellpadding='0' id='TBL-19'><colgroup id='TBL-19-1g'><col id='TBL-19-1' /></colgroup><colgroup id='TBL-19-2g'><col id='TBL-19-2' /></colgroup><colgroup id='TBL-19-3g'><col id='TBL-19-3' /></colgroup><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-19-1-'><td style='white-space:nowrap; text-align:left;' id='TBL-19-1-1' class='td11'>     </td><td style='white-space:nowrap; text-align:left;' id='TBL-19-1-2' class='td11'> <span class='cmmi-12'>ν</span>SAW              </td><td style='white-space:nowrap; text-align:left;' id='TBL-19-1-3' class='td11'> <span class='cmmi-12'>ν</span>simulation  </td></tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-19-2-'><td style='white-space:nowrap; text-align:left;' id='TBL-19-2-1' class='td11'> NTD </td><td style='white-space:nowrap; text-align:left;' id='TBL-19-2-2' class='td11'> 0.520 <span class='cmsy-10x-x-120'>± </span>0.009 </td><td style='white-space:nowrap; text-align:left;' id='TBL-19-2-3' class='td11'> 0.52</td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-19-3-'><td style='white-space:nowrap; text-align:left;' id='TBL-19-3-1' class='td11'> LINK  </td><td style='white-space:nowrap; text-align:left;' id='TBL-19-3-2' class='td11'> 0.538 <span class='cmsy-10x-x-120'>± </span>0.008  </td><td style='white-space:nowrap; text-align:left;' id='TBL-19-3-3' class='td11'> 0.58              </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-19-4-'><td style='white-space:nowrap; text-align:left;' id='TBL-19-4-1' class='td11'> CTD    </td><td style='white-space:nowrap; text-align:left;' id='TBL-19-4-2' class='td11'> 0.546 <span class='cmsy-10x-x-120'>± </span>0.008  </td><td style='white-space:nowrap; text-align:left;' id='TBL-19-4-3' class='td11'> 0.49              </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td></tr></table>
</div>
                                                                                      
                                                                                      
</div><hr class='endfloat' />
</div>
<div class='table'>
                                                                                      
                                                                                      
<!-- l. 304 --><p class='noindent'><a id='x17-168007r4'></a></p><hr class='float' /><div class='float'>
                                                                                      
                                                                                      
<a id='x17-168008'></a>
 <div class='caption'><span class='id'>Table D.4: </span><span class='content'>All-atom simulation summary</span></div><!-- tex4ht:label?: x17-168007rD.2  -->
<div class='tabular'> <table class='tabular' cellspacing='0' cellpadding='0' id='TBL-20'><colgroup id='TBL-20-1g'><col id='TBL-20-1' /></colgroup><colgroup id='TBL-20-2g'><col id='TBL-20-2' /></colgroup><colgroup id='TBL-20-3g'><col id='TBL-20-3' /></colgroup><colgroup id='TBL-20-4g'><col id='TBL-20-4' /></colgroup><colgroup id='TBL-20-5g'><col id='TBL-20-5' /></colgroup><colgroup id='TBL-20-6g'><col id='TBL-20-6' /></colgroup><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-20-1-'><td style='white-space:nowrap; text-align:left;' id='TBL-20-1-1' class='td11'> System                 </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-1-2' class='td11'> No. sims  </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-1-3' class='td11'> Total steps per sim (M).  </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-1-4' class='td11'> <!-- tex4ht:inline --><div class='tabular'><table class='tabular' cellspacing='0' cellpadding='0' id='TBL-21'><colgroup id='TBL-21-1g'><col id='TBL-21-1' /></colgroup><tr style='vertical-align:baseline;' id='TBL-21-1-'><td style='white-space:nowrap; text-align:left;' id='TBL-21-1-1' class='td00'>Prod. steps </td>
</tr><tr style='vertical-align:baseline;' id='TBL-21-3-'><td style='white-space:nowrap; text-align:left;' id='TBL-21-3-1' class='td00'>per sim.(M)</td>
</tr></table>                                                                                                                                                       </div>  </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-1-5' class='td11'> Config. output  </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-1-6' class='td11'> <!-- tex4ht:inline --><div class='tabular'><table class='tabular' cellspacing='0' cellpadding='0' id='TBL-22'><colgroup id='TBL-22-1g'><col id='TBL-22-1' /></colgroup><tr style='vertical-align:baseline;' id='TBL-22-1-'><td style='white-space:nowrap; text-align:left;' id='TBL-22-1-1' class='td00'>Ensemble</td></tr><tr style='vertical-align:baseline;' id='TBL-22-3-'><td style='white-space:nowrap; text-align:left;' id='TBL-22-3-1' class='td00'>size           </td></tr></table>                                                                                                                                 </div>  </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-20-2-'><td style='white-space:nowrap; text-align:left;' id='TBL-20-2-1' class='td11'> NTD-RBD            </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-2-2' class='td11'> 400          </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-2-3' class='td11'> 24                                    </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-2-4' class='td11'> 20                                                                                                                                                     </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-2-5' class='td11'> 20,000              </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-2-6' class='td11'> 399,000                                                                                                                                            </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-20-3-'><td style='white-space:nowrap; text-align:left;' id='TBL-20-3-1' class='td11'> RBD-LINK-DIM  </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-3-2' class='td11'> 10            </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-3-3' class='td11'> 66                                    </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-3-4' class='td11'> 60                                                                                                                                                     </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-3-5' class='td11'> 20,000              </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-3-6' class='td11'> 31,113                                                                                                                                              </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-20-4-'><td style='white-space:nowrap; text-align:left;' id='TBL-20-4-1' class='td11'> DIM-CTD             </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-4-2' class='td11'> 40            </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-4-3' class='td11'> 24                                    </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-4-4' class='td11'> 20                                                                                                                                                     </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-4-5' class='td11'> 20,000              </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-4-6' class='td11'> 40,000                                                                                                                                              </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-20-5-'><td style='white-space:nowrap; text-align:left;' id='TBL-20-5-1' class='td11'> NTD                      </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-5-2' class='td11'> 40            </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-5-3' class='td11'> 71                                    </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-5-4' class='td11'> 66                                                                                                                                                     </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-5-5' class='td11'> 30,000              </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-5-6' class='td11'> 64,000                                                                                                                                              </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-20-6-'><td style='white-space:nowrap; text-align:left;' id='TBL-20-6-1' class='td11'> LINK                     </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-6-2' class='td11'> 30            </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-6-3' class='td11'> 101                                  </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-6-4' class='td11'> 80                                                                                                                                                     </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-6-5' class='td11'> 30,000              </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-6-6' class='td11'> 66,660                                                                                                                                              </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-20-7-'><td style='white-space:nowrap; text-align:left;' id='TBL-20-7-1' class='td11'> CTD                      </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-7-2' class='td11'> 40            </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-7-3' class='td11'> 71                                    </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-7-4' class='td11'> 66                                                                                                                                                     </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-7-5' class='td11'> 30,000              </td><td style='white-space:nowrap; text-align:left;' id='TBL-20-7-6' class='td11'> 64,000                                                                                                                                              </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr></table>
</div>
                                                                                      
                                                                                      
</div><hr class='endfloat' />
</div>
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x17-168009r1'></a>
                                                                                      
                                                                                      
<!-- l. 327 --><p class='noindent'><img width='216' height='216' src='AppendixE/appendix-e-fig1.png' alt='PIC' />
<a id='x17-168010'></a>
<br /> </p><div class='caption'><span class='id'>Figure D.1: </span><span class='content'>Sequence alignment of the coronavirus N-terminal domain (NTD).</span></div><!-- tex4ht:label?: x17-168009rD.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x17-168011r2'></a>
                                                                                      
                                                                                      
<!-- l. 335 --><p class='noindent'><img width='216' height='216' src='AppendixE/appendix-e-fig2.png' alt='PIC' />
<a id='x17-168012'></a>
<br /> </p><div class='caption'><span class='id'>Figure D.2: </span><span class='content'>Sequence alignment of the coronavirus RNA binding domain (RBD).</span></div><!-- tex4ht:label?: x17-168011rD.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x17-168013r3'></a>
                                                                                      
                                                                                      
<!-- l. 343 --><p class='noindent'><img width='216' height='216' src='AppendixE/appendix-e-fig3.png' alt='PIC' />
<a id='x17-168014'></a>
<br /> </p><div class='caption'><span class='id'>Figure D.3: </span><span class='content'>Sequence alignment of the coronavirus linker (LINK).</span></div><!-- tex4ht:label?: x17-168013rD.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x17-168015r4'></a>
                                                                                      
                                                                                      
<!-- l. 352 --><p class='noindent'><img width='216' height='216' src='AppendixE/appendix-e-fig4.png' alt='PIC' />
<a id='x17-168016'></a>
<br /> </p><div class='caption'><span class='id'>Figure D.4: </span><span class='content'>Sequence alignment of the coronavirus dimerization domain.</span></div><!-- tex4ht:label?: x17-168015rD.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x17-168017r5'></a>
                                                                                      
                                                                                      
<!-- l. 360 --><p class='noindent'><img width='216' height='216' src='AppendixE/appendix-e-fig5.png' alt='PIC' />
<a id='x17-168018'></a>
<br /> </p><div class='caption'><span class='id'>Figure D.5: </span><span class='content'>Sequence alignment of the coronavirus C-terminal domain (CTD)</span></div><!-- tex4ht:label?: x17-168017rD.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x17-168019r6'></a>
                                                                                      
                                                                                      
<!-- l. 368 --><p class='noindent'><img width='289' height='289' src='AppendixE/appendix-e-fig6.png' alt='PIC' />
<a id='x17-168020'></a>
<br /> </p><div class='caption'><span class='id'>Figure D.6: </span><span class='content'>Histograms of transfer efficiency distributions across denaturant concentrations
for NTD, LINK, and CTD constructs.</span></div><!-- tex4ht:label?: x17-168019rD.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x17-168021r7'></a>
                                                                                      
                                                                                      
<!-- l. 376 --><p class='noindent'><img width='289' height='289' src='AppendixE/appendix-e-fig7.png' alt='PIC' />
<a id='x17-168022'></a>
<br /> </p><div class='caption'><span class='id'>Figure D.7: </span><span class='content'>Dependence of fluorescence lifetime on transfer efficiency. <span class='ptmb7t-x-x-120'>A. </span>NTD construct. <span class='ptmb7t-x-x-120'>B.</span>
LINK construct. <span class='ptmb7t-x-x-120'>C. </span>CTD construct. Black line: linear dependence expected for a rigid molecule.
Green line: the donor lifetime (normalized by the donor lifetime in absence of FRET: tD/tD0)
in the limit of dynamics much faster than the burst duration but slower than the fluorophore
lifetime. Orange line: the acceptor lifetime delay (normalized by the donor lifetime in absence
of FRET. The green and orange contour plots represent the corresponding distributions of donor
lifetime and acceptor lifetime delay as observed in single-molecule experiments under native
conditions The green and orange dots represent the mean value of the measured distributions.
A larger overlap between donor and acceptor lifetime populations is observed for the NTD and
CTD, hinting to possible static conformations.</span></div><!-- tex4ht:label?: x17-168021rD.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x17-168023r8'></a>
                                                                                      
                                                                                      
<!-- l. 384 --><p class='noindent'><img width='289' height='289' src='AppendixE/appendix-e-fig8.png' alt='PIC' />
<a id='x17-168024'></a>
<br /> </p><div class='caption'><span class='id'>Figure D.8: </span><span class='content'>Mean transfer efficiency and width of NTD, LINK, and CTD across denaturant.
The mean transfer efficiency of the NTD domain exhibits a plateau between 1 and 2 M; at the
same concentration we observe a small but systematic increase in the amplitude of the transfer
efficiency distribution hinting to the coexistence of two populations in slow exchange with very
similar transfer efficiencies. The CTD width also shows a small increase in the width of the
transfer efficiency distribution that may reflect the formation of local structure under native
conditions (e.g. the putative helical binding motif).</span></div><!-- tex4ht:label?: x17-168023rD.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x17-168025r9'></a>
                                                                                      
                                                                                      
<!-- l. 392 --><p class='noindent'><img width='144' height='144' src='AppendixE/appendix-e-fig9.png' alt='PIC' />
<a id='x17-168026'></a>
<br /> </p><div class='caption'><span class='id'>Figure D.9: </span><span class='content'>Fit of NTD construct with two populations. To address the change in amplitude
that occurs from the NTD construct between 1 and 2 M GdmCl, we attempt a fit of the same data
using two populations with a fixed distance equal to average width outside the 1-2 M GdmCl
region (see for comparison Fig. <a href='#x17-168023r8'>D.8<!-- tex4ht:ref: fig:appendix-e-fig8  --></a>). <span class='ptmb7t-x-x-120'>Upper panel: </span>fit of the transfer efficiency histogram at
1.5 M GdmCl. The white- and gray- shaded areas reflect fits to the “folded RBD” population
and to the “unfolded RBD” population. <span class='ptmb7t-x-x-120'>Central panel: </span>Comparison of transfer efficiencies
with a single fit (solid orange circles, compare Fig. <a href='#x17-168023r8'>D.8<!-- tex4ht:ref: fig:appendix-e-fig8  --></a>) and from the two populations: gray
solid  circles  for  the  “unfolded  RBD”  population  and  unfilled  circles  for  the  “folded  RBD”
population. <span class='ptmb7t-x-x-120'>Lower panel: </span>Fraction folded estimated from the fit with Eq. S7 compared to the
fraction of “folded RBD” obtained from computing the ratio between the area under “folded
RBD” species and the total histogram area.</span></div><!-- tex4ht:label?: x17-168025rD.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x17-168027r10'></a>
                                                                                      
                                                                                      
<!-- l. 400 --><p class='noindent'><img width='144' height='144' src='AppendixE/appendix-e-fig10.png' alt='PIC' />
<a id='x17-168028'></a>
<br /> </p><div class='caption'><span class='id'>Figure D.10: </span><span class='content'>Interdye distances of NTD, LINK, CTD in presence of salt (KCl). <span class='ptmb7t-x-x-120'>Upper panel:</span>
root mean square interdye distance between position 1 and 68. Dashed line: fit according to the
Higgs and Joanny model (Eq. <a href='#x17-163001r21'>D.21<!-- tex4ht:ref: appendix-e-eq21  --></a>-<a href='#x17-163002r22'>D.22<!-- tex4ht:ref: appendix-e-eq22  --></a>) predicts a comparable change to the one observed.
Central panel: root mean square interdye distance between position 172 and 245. Dashed line:
fit according to the Higgs and Joanny model (Eq. <a href='#x17-163001r21'>D.21<!-- tex4ht:ref: appendix-e-eq21  --></a>-<a href='#x17-163002r22'>D.22<!-- tex4ht:ref: appendix-e-eq22  --></a>) predicts a comparable change to
the one observed. Solid line and shaded area: average value of the root-mean-square interdye
distance across all salt conditions and corresponding standard deviation. The standard deviation
is  comparable  to  the  measurement  error.  <span class='ptmb7t-x-x-120'>Lower  panel:  </span>root  mean  square  interdye  distance
between position 363 and 419. Dashed line: fit according to the Higgs and Joanny model (Eq.
<a href='#x17-163001r21'>D.21<!-- tex4ht:ref: appendix-e-eq21  --></a>-<a href='#x17-163002r22'>D.22<!-- tex4ht:ref: appendix-e-eq22  --></a>) does not capture the observed trend. This can be possibly explained considering
the significant predicted population of helical conformations in the CTD. Solid line and shaded
area: average value of the root-mean-square interdye distance across all salt conditions and
corresponding  standard  deviation.  Inset:  no  variation  in  the  width  of  the  transfer  efficiency
population is observed upon addition of KCl.</span></div><!-- tex4ht:label?: x17-168027rD.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x17-168029r11'></a>
                                                                                      
                                                                                      
<!-- l. 408 --><p class='noindent'><img width='289' height='289' src='AppendixE/appendix-e-fig11.png' alt='PIC' />
<a id='x17-168030'></a>
<br /> </p><div class='caption'><span class='id'>Figure D.11:  </span><span class='content'>Chain  dynamics  measured  via  ns-FCS.  Nanosecond  FCS  measurements  for
the NTD, LINK, and CTD constructs provide a measure of the dynamics on the nanosecond
timescale.  The  donor-donor  (green),  acceptor-acceptor  (red),  and  donor-acceptor  (orange)
correlation  are  fitted  to  a  global  model  that  accounts  for  antibunching,  FRET  dynamic
populations, and triplet. The acceptor-donor correlation shows a clear anticorrelated change for
NTD and LINK in the signal that reflects the anticorrelated nature of the donor-acceptor energy
transfer  as  a  function  of  distance:  an  increase  in  acceptor  reflects  a  decrease  in  donor.  The
CTD cross-correlation exhibits a flat behavior. Occurrence of a correlation in the donor-donor
and acceptor-acceptor autocorrelations corresponding with a characteristic time <span class='cmmi-12'>t</span><sub><span class='cmmi-8'>b</span></sub>= 190 <span class='cmsy-10x-x-120'>± </span>30
ns suggests the presence of chain dynamics, either through FRET or Photo-induced Electron
Transfer  (PET)  <span class='cite'> [<a href='mainli5.html#XSauer2014-ux'>331</a>, <a href='mainli5.html#XHaenni2013-xz'>332</a>]</span>.  Addition  of  0.2  M  GdmCl,  which  causes  a  small  decrease  in
the transfer efficiency width (Fig. <a href='#x17-168023r8'>D.8<!-- tex4ht:ref: fig:appendix-e-fig8  --></a>) leads to an anticorrelation in the cross-correlation of
CTD. The correlation decay appears also faster with a <span class='cmmi-12'>t</span><sub><span class='cmmi-8'>C</span></sub><span class='cmmi-12'>D</span>= 70 <span class='cmsy-10x-x-120'>± </span>15 ns. All measurements
are normalized to the value measured at 1 <span class='cmmi-12'>μ</span>s for highlighting the amplitude relative to the
reconfiguration term.</span></div><!-- tex4ht:label?: x17-168029rD.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x17-168031r12'></a>
                                                                                      
                                                                                      
<!-- l. 416 --><p class='noindent'><img width='252' height='252' src='AppendixE/appendix-e-fig12.png' alt='PIC' />
<a id='x17-168032'></a>
<br /> </p><div class='caption'><span class='id'>Figure D.12:   </span><span class='content'>Turbidity  experiments  plotted  against  RNA/protein  ratio.   Representative
turbidity  titrations  with  poly(rU)  in  50  mM  Tris,  pH  7.5  (HCl)  at  room  temperature,  in
absence of added salt <span class='ptmb7t-x-x-120'>(A) </span>and in presence of 50 mM NaCl <span class='ptmb7t-x-x-120'>(B)</span>, at the indicated concentrations
of  N  protein.  On  the  x-axis,  the  concentration  of  poly(rU)  is  rescaled  for  the  protein
concentration.  Points  and  error  bars  represent  the  mean  and  standard  deviation  of  2-4
consecutive measurements from the same sample. Solid lines are simulations of an empirical
equation fitted individually to each titration curve. An inset is provided for the titration at 3.1
<span class='cmmi-12'>μ</span>M N protein in 50 mM NaCl to show the small yet detectable change in turbidity on a different
scale. Interestingly, within the experimental error, we observe a clear alignment of the turbidity
curves with a maximum at  20 nucleotides per protein in the absence of added salt <span class='ptmb7t-x-x-120'>(A) </span>and  30
nucleotides per protein in the presence of 50 mM NaCl <span class='ptmb7t-x-x-120'>(B)</span>.</span></div><!-- tex4ht:label?: x17-168031rD.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x17-168033r13'></a>
                                                                                      
                                                                                      
<!-- l. 424 --><p class='noindent'><img width='252' height='252' src='AppendixE/appendix-e-fig13.png' alt='PIC' />
<a id='x17-168034'></a>
<br />  </p><div class='caption'><span class='id'>Figure D.13:   </span><span class='content'>Testing   SARS-CoV-2   N   protein   oligomerization.   <span class='ptmb7t-x-x-120'>(A-B)   </span>Fluorescence
Correlation Spectroscopy (FCS) of full-length SARS-CoV-2 N protein as a function of protein
concentration. (A) FCS traces of 100pM Alexa 488/Alexa 594 N protein labeled at positions
363 and 419 in the absence (blue dots) and the presence (gray dots) of 50 <span class='ptmr8c-x-x-120'>µ</span>M unlabeled N
protein. (B) Hydrodynamic radius of SARS-CoV-2 N protein obtained from FCS trace analysis
(blue dot: 100pM labeled N protein; gray dot: 100pM labeled N protein + 50<span class='ptmr8c-x-x-120'>µ</span>M unlabeled N
protein). <span class='ptmb7t-x-x-120'>(C-D) </span>NativePAGE of full-length SARS-CoV-2 N protein in 20 mM NaPi pH 7.4 as
a function of protein concentration in the presence of 200 mM NaCl (C) and in the absence of
added salt (D). ‘Custom Std’ lane contains Alcohol Dehydrogenase ( * , 150 kDa) and Bovine
Serum Albumin ( ** , 66 kDa). (E-F) SDS PAGE of crosslinked full-length SARS-CoV-2 N
protein in 20 mM NaPi pH 7.4, 1.25mM DSS as a function of protein concentration in the
presence of 200mM NaCl (E) and in the absence of added salt (F). </span></div><!-- tex4ht:label?: x17-168033rD.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x17-168035r14'></a>
                                                                                      
                                                                                      
<!-- l. 432 --><p class='noindent'><img width='289' height='289' src='AppendixE/appendix-e-fig14.png' alt='PIC' />
<a id='x17-168036'></a>
<br /> </p><div class='caption'><span class='id'>Figure D.14:  </span><span class='content'>Distributions  of  inter-residue  distance  from  ABSINTH  simulations  (black)
vs.  excluded  volume  simulations  (red).  Comparison  of  simulations  with  the  full  ABSINTH
Hamiltonian (‘normal’, black) against simulations performed in the excluded volume (EV, red)
limit for <span class='ptmb7t-x-x-120'>A. </span>NTD, <span class='ptmb7t-x-x-120'>B. </span>LINK and <span class='ptmb7t-x-x-120'>C. </span>CTD. In all cases the EV simulations report substantially
larger average distances than the ABSINTH simulations, as expected given the absence of any
attractive intramolecular interactions. The distances reported from the EV simulations are also
slightly more expanded than under fully denatured conditions, consistent with systems studied
previously (see <span class='cite'> [<a href='mainli5.html#XHolehouse2020-rl'>496</a>, <a href='mainli5.html#XMeng2013-qi'>514</a>]</span>.</span></div><!-- tex4ht:label?: x17-168035rD.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x17-168037r15'></a>
                                                                                      
                                                                                      
<!-- l. 440 --><p class='noindent'><img width='289' height='289' src='AppendixE/appendix-e-fig15.png' alt='PIC' />
<a id='x17-168038'></a>
<br /> </p><div class='caption'><span class='id'>Figure D.15: </span><span class='content'>Scaling maps for IDR-only simulations. Scaling maps report on the normalized
distance between pairs of residues, where normalization is done by the distance expected if the
IDRs behaved as self-avoiding chains in the excluded-volume limit. Scaling maps for IDR-only
simulations of the <span class='ptmb7t-x-x-120'>A. </span>NTD, <span class='ptmb7t-x-x-120'>B. </span>LINK and <span class='ptmb7t-x-x-120'>C. </span>CTD. For each sequence, transient helices are
annotated  on  the  scaling  maps.  Note  that  in  the  LINK  we  observe  interaction  between  the
C-terminal  region  of  the  LINK  and  H4,  while  H3  does  not  interact  with  any  parts  of  the
sequence. Similarly, in CTD we see extensive intramolecular interactions between H5 and H6.</span></div><!-- tex4ht:label?: x17-168037rD.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x17-168039r16'></a>
                                                                                      
                                                                                      
<!-- l. 448 --><p class='noindent'><img width='289' height='289' src='AppendixE/appendix-e-fig16.png' alt='PIC' />
<a id='x17-168040'></a>
<br /> </p><div class='caption'><span class='id'>Figure D.16: </span><span class='content'>Distributions for the radius of gyration (<span class='cmmi-12'>R</span><sub><span class='cmmi-8'>g</span></sub>) of for IDR-only simulations. <span class='cmmi-12'>R</span><sub><span class='cmmi-8'>g</span></sub>
distributions for <span class='ptmb7t-x-x-120'>A. </span>NTD, <span class='ptmb7t-x-x-120'>B. </span>LINK and <span class='ptmb7t-x-x-120'>C. </span>CTD. Average <span class='cmmi-12'>R</span><sub><span class='cmmi-8'>g</span></sub> for each IDR in isolation is 19.1
Å (NTD), 21.4 Å (LINK), and 17.1 Å (CTD).</span></div><!-- tex4ht:label?: x17-168039rD.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
                                                                                      
                                                                                      
<!-- l. 5 --><div class='crosslinks'><p class='noindent'></p></div>
 
</body></html>