<?xml version='1.0' encoding='utf-8' ?> 
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd"><html xmlns='http://www.w3.org/1999/xhtml'> 
<head><title>3 Simulation of spontaneous G protein activation reveals a new intermediate driving GDP
unbinding</title> 
<meta http-equiv='Content-Type' content='text/html; charset=utf-8' /> 
<meta name='generator' content='TeX4ht (https://tug.org/tex4ht/)' /> 
<meta name='originator' content='TeX4ht (https://tug.org/tex4ht/)' /> 
<!--  xhtml,charset=utf-8,epub,uni-html4,html  --> 
<meta name='src' content='main.tex' /> 
<link type='text/css' href='main.css' rel='stylesheet' /> 
</head><body>
<!-- l. 4 --><div class='crosslinks'><p class='noindent'></p></div>
<h2 class='chapterHead'><span class='titlemark'>Chapter 3</span><br /><a id='x8-330003'></a>Simulation of spontaneous G protein activation reveals a new intermediate driving GDP
unbinding</h2>
<!-- l. 9 --><p class='noindent'><span class='ptmri7t-x-x-120'>This chapter is adapted from the following publication:</span>
</p><!-- l. 11 --><p class='noindent'><span class='ptmri7t-x-x-120'>Sun, X.</span><sup><span class='cmsy-8'>∗</span></sup> <span class='ptmri7t-x-x-120'>and Singh, S.</span><sup><span class='cmsy-8'>∗</span></sup><span class='ptmri7t-x-x-120'>, Blumer, K.J., and Bowman, G.R., Simulation of spontaneous G
</span><span class='ptmri7t-x-x-120'>protein activation reveals a new intermediate driving GDP unbinding. eLife, 7, October
</span><span class='ptmri7t-x-x-120'>2018, https://doi.org/10.7554/eLife.38465.001 </span><span class='cite'> [<a href='mainli5.html#XSun:2018kx'>48</a>]</span> <sup><span class='cmsy-8'>∗</span></sup><span class='ptmri7t-x-x-120'>Authors contributed equally to this
</span><span class='ptmri7t-x-x-120'>work</span>
</p>
<h3 class='sectionHead'><span class='titlemark'>3.1    </span> <a id='x8-340003.1'></a>Abstract</h3>
<!-- l. 15 --><p class='noindent'>Activation of heterotrimeric G proteins is a key step in many signaling cascades. However, a complete
mechanism for this process, which requires allosteric communication between binding sites that are
<span class='cmsy-10x-x-120'>∼</span>30 Å apart, remains elusive. We construct an atomically detailed model of G protein activation by
combining three powerful computational methods: metadynamics, Markov state models (MSMs), and
CARDS analysis of correlated motions. We uncover a mechanism that is consistent with a wide
variety of structural and biochemical data. Surprisingly, the rate-limiting step for GDP release
correlates with tilting rather than translation of the GPCR-binding helix 5. <span class='cmmi-12'>β</span>-Strands 1-3 and helix 1
emerge as hubs in the allosteric network that links conformational changes in the GPCR-binding site
to disordering of the distal nucleotide-binding site and consequent GDP release. Our approach
and insights provide foundations for understanding disease-implicated G protein mutants,
illuminating slow events in allosteric networks, and examining unbinding processes with slow
                                                                                      
                                                                                      
off-rates.
</p><!-- l. 17 --><p class='noindent'>
</p>
<h3 class='sectionHead'><span class='titlemark'>3.2    </span> <a id='x8-350003.2'></a>Introduction</h3>
<!-- l. 18 --><p class='noindent'>Heterotrimeric G proteins are molecular switches that play pivotal roles in signaling processes from
vision to olfaction and neurotransmission<span class='cite'> [<a href='mainli5.html#XOldham:2006ui'>168</a>, <a href='mainli5.html#XOldham:2008cr'>169</a>, <a href='mainli5.html#XJohnston:2007ir'>170</a>]</span>. By default, a G protein adopts an inactive
state in which guanosine diphosphate (GDP) binds between the Ras-like and helical domains of the
<span class='cmmi-12'>α</span>-subunit (G<span class='cmmi-12'>α</span>, Fig. <a href='#x8-35001r1'>3.1<!-- tex4ht:ref: fig:ch4-fig1  --></a>). A dimer consisting of the <span class='cmmi-12'>β</span>- and <span class='cmmi-12'>γ</span>-subunits (G<span class='cmmi-12'>βγ</span>) also binds G<span class='cmmi-12'>α</span>. G
protein-coupled receptors (GPCRs) trigger G protein activation by binding G<span class='cmmi-12'>α </span>and stimulating GDP
release, followed by GTP binding to G<span class='cmmi-12'>α </span>and dissociation of G<span class='cmmi-12'>α </span>from G<span class='cmmi-12'>βγ</span>. G<span class='cmmi-12'>α </span>and <span class='cmmi-12'>βγ </span>then
interact with effectors that trigger downstream cellular responses. G<span class='cmmi-12'>α </span>returns to the inactive
state by hydrolyzing GTP to GDP and rebinding G<span class='cmmi-12'>βγ</span>. Given the central role G<span class='cmmi-12'>α </span>plays, a
common G<span class='cmmi-12'>α </span>numbering scheme (CGN) has been established to facilitate discussion of the
activation mechanisms of different G<span class='cmmi-12'>α </span>homologs<span class='cite'> [<a href='mainli5.html#XFlock:2015dj'>171</a>]</span>. For example, the notation Lys52<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>1</span></sup>
indicates that Lys52 is the first residue in helix 1 (H1) of the Ras-like domain (also called the
GTPase domain, or G). S6 refers to <span class='cmmi-12'>β</span>-strand 6 and s6h5 refers to the loop between S6 and
H5.
</p>
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x8-35001r1'></a>
                                                                                      
                                                                                      
<!-- l. 22 --><p class='noindent'><img width='252' height='252' src='Chapter4/ch4-fig1.png' alt='PIC' />
<a id='x8-35002'></a>
<br /> </p><div class='caption'><span class='id'>Figure 3.1: </span><span class='content'>Structure of G<span class='cmmi-12'>α</span>q with key secondary structure elements labeled according to the
Common G<span class='cmmi-12'>α </span>Numbering (CGN) system. The coloring scheme highlights the GPCR binding
interface  (gray),  GDP  phosphate-binding  regions  (pink),  GDP  nucleotide-binding  regions
(green), <span class='cmmi-12'>β</span>-sheets (blue), and switch 2 (orange).</span></div><!-- tex4ht:label?: x8-35001r3.2  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<!-- l. 28 --><p class='noindent'>Strikingly, the GPCR- and nucleotide-binding sites of G<span class='cmmi-12'>α </span>are <span class='cmsy-10x-x-120'>∼</span>30 Å apart (Fig. <a href='#x8-35001r1'>3.1<!-- tex4ht:ref: fig:ch4-fig1  --></a>), but the
allosteric mechanism coupling these sites to evoke GDP release remains incompletely
understood<span class='cite'> [<a href='mainli5.html#XOldham:2008cr'>169</a>]</span>. Biochemical and structural studies have elucidated some key steps, but the entire
process has yet to be described in atomic detail. Early studies of G<span class='cmmi-12'>α </span>subunits revealed structures of the
GDP- and GTP-bound states, as well as the transition state for GTP hydrolysis<span class='cite'> [<a href='mainli5.html#XSunahara:1997tg'>172</a>, <a href='mainli5.html#XLeipe:2002ky'>173</a>, <a href='mainli5.html#XWestfield:2011ef'>174</a>]</span>. The
high similarity of these structures and the binding of GDP or GTP deep in the protein’s core suggests
that activation occurs by adoption of other conformational states that allow GDP release<span class='cite'> [<a href='mainli5.html#XLambright:1994gd'>175</a>]</span>. One
intermediate in G protein activation was suggested by the first crystal structure of a GPCR-bound G
protein in which the Ras-like and helical domains of G<span class='cmmi-12'>α </span>are hinged apart and GDP has
dissociated<span class='cite'> [<a href='mainli5.html#XRasmussen:2011kp'>68</a>]</span>. Structural analysis has led to the proposal of a universal mechanism for G protein
activation<span class='cite'> [<a href='mainli5.html#XFlock:2015dj'>171</a>]</span>. In this model, GPCR binding induces translation of H5 away from H1,
which increases disorder in H1 and the P-loop (or Walker A motif<span class='cite'> [<a href='mainli5.html#XLeipe:2002ky'>173</a>]</span>) to facilitate GDP
release. However, there is evidence that additional intermediates may be involved in G<span class='cmmi-12'>α</span>
activation<span class='cite'> [<a href='mainli5.html#XOldham:2008cr'>169</a>, <a href='mainli5.html#XWestfield:2011ef'>174</a>, <a href='mainli5.html#XLiang:2017cz'>176</a>, <a href='mainli5.html#XHilger:2018en'>177</a>]</span>,and the functional importance of this conformational
ensemble has been previously suggested<span class='cite'> [<a href='mainli5.html#XFurness:2016it'>178</a>]</span>. Furthermore, mutagenesis and nuclear
magnetic resonance (NMR) studies have suggested important roles for other structural
elements<span class='cite'> [<a href='mainli5.html#XToyama:2017gy'>179</a>, <a href='mainli5.html#XSun:2015gj'>180</a>, <a href='mainli5.html#XGoricanec:2016hl'>181</a>]</span>.
</p><!-- l. 30 --><p class='noindent'>Molecular dynamics simulations promise to capture the entire mechanism of G protein
activation and synthesize the wealth of experimental data on this process. Methodological
advances now enable simulations to capture millisecond timescale processes for proteins with
less than 100 residues<span class='cite'> [<a href='mainli5.html#XLindorffLarsen:2011gl'>182</a>]</span>. For example, it is now possible to capture the binding or
release of small molecules<span class='cite'> [<a href='mainli5.html#XBuch:2011bm'>183</a>, <a href='mainli5.html#XBowman:2012jd'>47</a>, <a href='mainli5.html#XSilva:2011ct'>99</a>, <a href='mainli5.html#XPlattner:2015br'>184</a>, <a href='mainli5.html#XTiwary:2015hw'>185</a>]</span> and peptides<span class='cite'> [<a href='mainli5.html#XPlattner:2017fs'>186</a>, <a href='mainli5.html#XZhou:2017bh'>187</a>]</span> from small
proteins. Impressive simulations on the ANTON supercomputer have revealed critical
conformational dynamics of G proteins in their inactive and active states, elucidating the role of
domain opening in GDP unbinding<span class='cite'> [<a href='mainli5.html#XDror:2015bg'>188</a>, <a href='mainli5.html#XYao:2016kw'>189</a>]</span>. However, even this specialized hardware
could not capture the entire process of G protein activation and GDP release due to the size
                                                                                      
                                                                                      
of the G<span class='cmmi-12'>α </span>subunit (<span class='cmmi-12'>&gt;</span>300 residues) and the slow kinetics of GDP dissociation (<span class='cmsy-10x-x-120'>∼ </span>10<sup><span class='cmsy-8'>−</span><span class='cmr-8'>3</span></sup>
min<sup><span class='cmsy-8'>−</span><span class='cmr-8'>1</span></sup>)<span class='cite'> [<a href='mainli5.html#XChidiac:1999jh'>190</a>, <a href='mainli5.html#XRoss:2008jm'>191</a>, <a href='mainli5.html#XMukhopadhyay:1999dp'>192</a>]</span>.
</p><!-- l. 32 --><p class='noindent'>Here, we introduce an approach to capture rare or long-timescale events, such as GDP release,
and reveal the mechanism of G<span class='cmmi-12'>α </span>activation. As a test of this methodology, we apply it to
G<span class='cmmi-12'>α</span>q, which has one of the slowest GDP release rates<span class='cite'> [<a href='mainli5.html#XChidiac:1999jh'>190</a>]</span> and is frequently mutated in
uveal melanoma<span class='cite'> [<a href='mainli5.html#XVanRaamsdonk:2009kv'>193</a>, <a href='mainli5.html#XVanRaamsdonk:2010iy'>194</a>]</span>. To highlight aspects of the activation mechanism that we
propose are general to all G proteins, we focus our analysis on the behavior of secondary
structure elements and amino acids that are conserved across G<span class='cmmi-12'>α </span>domains. Our approach
first combines two powerful sampling methods, metadynamics<span class='cite'> [<a href='mainli5.html#XLaio:2002ft'>195</a>]</span> and Markov state
models (MSMs)<span class='cite'> [<a href='mainli5.html#XBowman:2013uj'>196</a>]</span>, to observe GDP release and identify the rate-limiting step for this
slow process. Then we use our recently developed CARDS method<span class='cite'> [<a href='mainli5.html#XSingh:2017hh'>90</a>]</span>, which quantifies
correlations between both the structure and disorder of different regions of a protein, to
identify the allosteric network connecting the GPCR- and nucleotide-binding sites. The
resulting model is consistent with a wealth of experimental data and leads to a number of
predictions, described below. Taken together, our results provide the most comprehensive
model of G protein activation to date. Based on this success, we expect our approach to be
valuable for studying other rare events, including conformational changes and unbinding
processes.
</p>
<h3 class='sectionHead'><span class='titlemark'>3.3    </span> <a id='x8-360003.3'></a>Results and Discussion</h3>
<!-- l. 36 --><p class='noindent'>
                                                                                      
                                                                                      
</p>
<h4 class='subsectionHead'><span class='titlemark'>3.3.1    </span> <a id='x8-370003.3.1'></a>Capturing G-protein activation and GDP release in atomic detail</h4>
<!-- l. 37 --><p class='noindent'>We reasoned that studying the mechanism of spontaneous GDP release from a truncated form of G<span class='cmmi-12'>α</span>q
would be representative of the mechanism of GPCR-mediated activation of the heterotrimeric G
protein while minimizing the computational cost of our simulations. This hypothesis was inspired by
previous work demonstrating that a protein’s spontaneous fluctuations are representative of the
conformational changes required for the protein to perform its function <span class='cite'> [<a href='mainli5.html#XBoehr:2006jd'>14</a>, <a href='mainli5.html#XFraser:2009gv'>197</a>, <a href='mainli5.html#XChangeux:2011fv'>97</a>]</span>. Therefore, we
hypothesized that GPCRs stabilize conformational states that G proteins naturally, albeit infrequently,
adopt in the absence of a receptor. We chose to focus on G<span class='cmmi-12'>α </span>since it forms substantial
interactions with GPCRs, compared to the relatively negligible interactions between GPCRs and
G protein <span class='cmmi-12'>β </span>and <span class='cmmi-12'>γ </span>subunits. This view is supported by the fact that GPCRs and ‘mini’
G proteins, essentially composed of just the Ras-like domain of G<span class='cmmi-12'>α</span>, recapitulate many
features of GPCR-G protein interactions<span class='cite'> [<a href='mainli5.html#XCarpenter:2016gr'>198</a>]</span>. We also reasoned that truncating the last five
residues of G<span class='cmmi-12'>α</span>q would facilitate the activation process. These residues contact G<span class='cmmi-12'>α </span>in some
GDP-bound structures but not in GPCR-bound structures<span class='cite'> [<a href='mainli5.html#XLambright:1996en'>199</a>, <a href='mainli5.html#XNoel:1993dc'>200</a>]</span>, and removing these
residues promotes GDP release due to a reduced GDP-binding affinity<span class='cite'> [<a href='mainli5.html#XDenker:1992uj'>201</a>, <a href='mainli5.html#XEthanPMarin:2002fr'>202</a>]</span>. Taken
together, such evidence suggests that the last five residues of G<span class='cmmi-12'>α</span>q help stabilize the inactive
state and that removing them would accelerate activation. In support of this hypothesis, we
find that the energetic barrier to GDP release is lower in metadynamics simulations of the
truncated variant than for full-length G<span class='cmmi-12'>α</span>q (Figure <a href='mainap2.html#x15-140001r1'>B.1<!-- tex4ht:ref: fig:ch4-fig2-supp1  --></a>). These simulations, and those described
hereafter, were initiated from an X-ray structure of the G<span class='cmmi-12'>α</span>q heterotrimer bound to GDP and an
inhibitor of nucleotide exchange<span class='cite'> [<a href='mainli5.html#XNishimura:2010gt'>203</a>]</span>; G<span class='cmmi-12'>βγ </span>and the inhibitor were excluded from all
simulations.
</p><!-- l. 39 --><p class='noindent'>To observe G-protein activation, we developed a variant of adaptive seeding<span class='cite'> [<a href='mainli5.html#XHuang:2009bx'>204</a>]</span> capable of
capturing slow processes like ligand unbinding that are beyond reach of conventional simulation
methods. First, we use metadynamics<span class='cite'> [<a href='mainli5.html#XTiwary:2015hw'>185</a>, <a href='mainli5.html#XLaio:2002ft'>195</a>, <a href='mainli5.html#XDama:2014ef'>205</a>]</span> to facilitate broad sampling of conformational
                                                                                      
                                                                                      
space by biasing simulations to sample conformations with different distances between GDP and G<span class='cmmi-12'>α</span>q.
Doing so forces GDP release to occur but provides limited mechanistic information because adding a
biasing force can distort the system’s kinetics or cause the simulations to sample high-energy
conformations that are not representative of behavior at thermal equilibrium. To overcome
these limitations, we chose starting conformations along release pathways observed by
metadynamics as starting points for standard molecular dynamics simulations, together
yielding an aggregate simulation time of 122.6 <span class='cmmi-12'>μ</span>s. These simulations should quickly relax
away from high-energy conformations and provide more accurate kinetics. Then we use
these simulations to build an MSM (Source data 1). MSMs are adept at extracting both
thermodynamic and kinetic information from many standard simulations that, together,
cover larger regions of conformational space than any individual simulation<span class='cite'> [<a href='mainli5.html#XBowman:2013uj'>196</a>]</span>. Related
approaches have successfully captured the dynamics of small model systems<span class='cite'> [<a href='mainli5.html#XBiswas:2018hc'>206</a>]</span> and RNA
polymerase<span class='cite'> [<a href='mainli5.html#XZhang:2016ex'>207</a>]</span>.
</p><!-- l. 41 --><p class='noindent'>This protocol enabled us to capture the entire mechanism of G-protein activation, including GDP
release and the rate-limiting step for this process. Identifying the rate-limiting step for this process is
of great value because GDP release is the rate-limiting step for G-protein activation and downstream
signaling. Therefore, the key structural and dynamical changes responsible for activation
should be apparent from the rate-limiting conformational transition for this dissociation
process.
</p><!-- l. 43 --><p class='noindent'>To identify the rate-limiting step, we applied transition path theory<span class='cite'> [<a href='mainli5.html#XNoe:2009en'>208</a>, <a href='mainli5.html#XWeinan:2006vr'>209</a>]</span> to find the
highest flux paths from bound structures resembling the GDP-bound crystal structure to
fully dissociated conformations. Then, we identified the least probable steps along the 10
highest flux release pathways (Figure <a href='#x8-37001r2'>3.2<!-- tex4ht:ref: fig:ch4-fig2  --></a>A and Figure <a href='mainap2.html#x15-140003r2'>B.2<!-- tex4ht:ref: fig:ch4-fig2-supp2  --></a>), which represent the rate-limiting
step of release. By comparing the structures before and after the rate-limiting step, we
define the bound state as all conformations where the distance from the center of mass
of GDP’s phosphates to the center of mass of three residues on H1 that contact the GDP
<span class='cmmi-12'>β</span>-phosphate (Lys52<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>1</span></sup>, Ser53<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>2</span></sup>, and Thr54<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>3</span></sup>) is less than 8 Å. Consistent with this
                                                                                      
                                                                                      
definition, this distance remains less than 8 Å throughout the entirety of 35.3 <span class='cmmi-12'>μ</span>s of GDP-bound
simulations.
</p>
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x8-37001r2'></a>
                                                                                      
                                                                                      
<!-- l. 47 --><p class='noindent'><img width='339' height='339' src='Chapter4/ch4-fig2.png' alt='PIC' />
<a id='x8-37002'></a>
<br /> </p><div class='caption'><span class='id'>Figure 3.2: </span><span class='content'>Structural and dynamical changes during the rate limiting step for GDP release. <span class='ptmb7t-x-x-120'>A.</span>
Overlay of representative structures before (blue) and after (orange) the rate limiting step. <span class='ptmb7t-x-x-120'>B.</span>
Change in conformational disorder (Shannon entropy) across the rate-limiting step, according
to the color scale on the right. <span class='ptmb7t-x-x-120'>C. </span>Histograms of inter-domain distances before (top, blue) and
after (middle, orange) the rate limiting step, along with the inter-domain distance distribution
after  GDP  is  released  (bottom,  black).  Inter-domain  distance  is  measured  using  residues
analogous to those used in DEER experiments<span class='cite'> [<a href='mainli5.html#XDror:2015bg'>188</a>]</span>, Leu97<sup><span class='cmmi-8'>H.HA.</span><span class='cmr-8'>29</span></sup> in the helical domain and
Glu250<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>3</span><span class='cmmi-8'>.</span><span class='cmr-8'>4</span></sup>  on H3. <span class='ptmb7t-x-x-120'>D. </span>Distribution of distances between Glu49<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>1</span><span class='cmmi-8'>h</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>4</span></sup>  and Arg183<sup><span class='cmmi-8'>G.hfs</span><span class='cmr-8'>2</span><span class='cmmi-8'>.</span><span class='cmr-8'>2</span></sup>
(left) before (blue) and after (orange) the rate-limiting step, as well as after GDP release (black).
Representative structures of the interacting residues are also shown (right).</span></div><!-- tex4ht:label?: x8-37001r3.3  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<!-- l. 53 --><p class='noindent'>The conformational changes we observe during the rate-limiting step are consistent with previous
structural and biochemical work. For example, we observe that the Ras-like and helical domains
separate (Figure <a href='#x8-37001r2'>3.2<!-- tex4ht:ref: fig:ch4-fig2  --></a>C), as observed in DEER experiments<span class='cite'> [<a href='mainli5.html#XVanEps:2011hb'>210</a>]</span> and previous simulations<span class='cite'> [<a href='mainli5.html#XDror:2015bg'>188</a>]</span>. This
finding is consistent with the intuition that these domains must separate to sterically permit GDP
release, and that this separation is driven by the disruption of multiple inter-domain interactions. For
example, we note a disrupted salt bridge between K275<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>5</span><span class='cmmi-8'>hg.</span><span class='cmr-8'>1</span></sup> and D155<sup><span class='cmmi-8'>H.hdhe.</span><span class='cmr-8'>5</span></sup> (Figure <a href='mainap2.html#x15-140007r4'>B.4<!-- tex4ht:ref: fig:ch4-fig2-supp4  --></a>),
previously identified in structural studies<span class='cite'> [<a href='mainli5.html#XFlock:2015dj'>171</a>]</span>. Domain opening is accompanied by disruption of a
key salt bridge between Glu49<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>1</span><span class='cmmi-8'>h</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>4</span></sup> of the P-loop and Arg183<sup><span class='cmmi-8'>G.hfs</span><span class='cmr-8'>2</span><span class='cmmi-8'>.</span><span class='cmr-8'>2</span></sup> of switch 1 (Figure <a href='#x8-37001r2'>3.2<!-- tex4ht:ref: fig:ch4-fig2  --></a>D), as
well as an increase in the disorder of many of the surrounding residues (Figure <a href='#x8-37001r2'>3.2<!-- tex4ht:ref: fig:ch4-fig2  --></a>B and Figure
<a href='mainap2.html#x15-140005r3'>B.3<!-- tex4ht:ref: fig:ch4-fig2-supp3  --></a>A), consistent with the proposal that this salt bridge stabilizes the closed, GDP-bound
state<span class='cite'> [<a href='mainli5.html#XLiang:2017cz'>176</a>]</span>.
</p><!-- l. 55 --><p class='noindent'>While domain opening is necessary for GDP release, previous simulations suggest it is insufficient
for unbinding<span class='cite'> [<a href='mainli5.html#XDror:2015bg'>188</a>]</span>. Indeed, we also see that this opening is necessary but not sufficient
for GDP unbinding, as the Ras-like and helical domains often separate prior to release
(Figure <a href='#x8-37001r2'>3.2<!-- tex4ht:ref: fig:ch4-fig2  --></a>C). Notably, the Ras-like and helical domains only separate by <span class='cmsy-10x-x-120'>∼</span>10 Å during the
rate-limiting step. In contrast, these domains separate by 56 Å in the first structure of a
GPCR-G-protein complex. This result suggests that GDP release may have occurred long before a
G protein adopts the sort of widely opened conformations observed in crystallographic
structures<span class='cite'> [<a href='mainli5.html#XRasmussen:2011kp'>68</a>]</span>.
</p>
<h4 class='subsectionHead'><span class='titlemark'>3.3.2    </span> <a id='x8-380003.3.2'></a>Tilting of H5 helps induce GDP release</h4>
<!-- l. 58 --><p class='noindent'>We also observe less expected conformational changes associated with GDP release. The most striking
is tilting of H5 by about 26<span class='ptmr8c-x-x-120'>° </span>(Figure <a href='#x8-38001r3'>3.3<!-- tex4ht:ref: fig:ch4-fig3  --></a>A, and Figure <a href='mainap2.html#x15-140011r6'>B.6<!-- tex4ht:ref: fig:ch4-fig3-supp1  --></a>). We find that H5 tilting correlates strongly
                                                                                      
                                                                                      
with the distance between GDP and G<span class='cmmi-12'>α</span>q along the highest flux dissociation pathway (Figure <a href='#x8-38001r3'>3.3<!-- tex4ht:ref: fig:ch4-fig3  --></a>B and
<a href='mainap2.html#x15-140015r1'>B.1<!-- tex4ht:ref: tab:galpha-struc-compares  --></a>). In particular, substantial tilting of H5 is coincident with the rate-limiting step for GDP release.
This tilting contrasts with X-ray structures and the universal mechanism, in which H5 is
proposed to translate along its helical axis towards the GPCR, initiating the process of
GDP release (Figure <a href='#x8-38001r3'>3.3<!-- tex4ht:ref: fig:ch4-fig3  --></a>A). During our simulations we also observe translation of H5, but
it is not correlated with the rate-limiting step of GDP release (Figure <a href='#x8-38001r3'>3.3<!-- tex4ht:ref: fig:ch4-fig3  --></a>C, Figure <a href='mainap2.html#x15-140013r7'>B.7<!-- tex4ht:ref: fig:ch4-fig3-supp2  --></a>).
Therefore, we are not merely missing an important role for translation due to insufficient
sampling.
</p><!-- l. 60 --><p class='noindent'>The potential importance of H5 tilting is supported by other structural data. For example, a
crystal structure of rhodopsin<span class='cite'> [<a href='mainli5.html#XChoe:2011cb'>211</a>]</span> with a C-terminal fragment from H5 of G<span class='cmmi-12'>α</span>t shows a
similar degree of tilting (Figure <a href='#x8-38001r3'>3.3<!-- tex4ht:ref: fig:ch4-fig3  --></a>A). Also, the tilt of H5 varies in crystal structures of the
<span class='cmmi-12'>β</span>2AR-Gs complex <span class='cite'> [<a href='mainli5.html#XRasmussen:2011kp'>68</a>]</span>, two different GLP-1 receptor-Gs complexes<span class='cite'> [<a href='mainli5.html#XLiang:2017cz'>176</a>, <a href='mainli5.html#XZhang:2017gu'>212</a>]</span>, and an
A2AR-mini-Gs complex<span class='cite'> [<a href='mainli5.html#XCarpenter:2016gr'>198</a>]</span>. The potential relevance of tilting has also been acknowledged by a
number of recent works<span class='cite'> [<a href='mainli5.html#XFlock:2015dj'>171</a>, <a href='mainli5.html#XRasmussen:2011kp'>68</a>, <a href='mainli5.html#XOldham:2006kn'>213</a>]</span> including four recently published structures of
receptor-G-protein complexes across which H5 also shows a broad range of tilting and translational
motion<span class='cite'> [<a href='mainli5.html#XKoehl:2018fv'>20</a>, <a href='mainli5.html#XDraperJoyce:2018il'>21</a>, <a href='mainli5.html#XGarciaNafria:2018hu'>22</a>, <a href='mainli5.html#XKang:2018kq'>23</a>]</span>. Interestingly, the tilting and translation we observe falls within the
observed range of tilting and translational motions that H5 undergoes in available GPCR-G protein
complex structures<span class='cite'> [<a href='mainli5.html#XRasmussen:2011kp'>68</a>, <a href='mainli5.html#XKoehl:2018fv'>20</a>, <a href='mainli5.html#XDraperJoyce:2018il'>21</a>, <a href='mainli5.html#XGarciaNafria:2018hu'>22</a>, <a href='mainli5.html#XKang:2018kq'>23</a>]</span>, providing support that conformational selection plays an
important role (<a href='mainap2.html#x15-140015r1'>B.1<!-- tex4ht:ref: tab:galpha-struc-compares  --></a>). Finally, H5 is translated toward the GPCR in the A2AR-mini-Gs structure but
GDP remains bound<span class='cite'> [<a href='mainli5.html#XCarpenter:2016gr'>198</a>]</span>. The authors of that study originally suggested that one of the
mutations in mini-Gs decouples H5 translation from GDP release. However, given that we see
GDP release without H5 translation in our simulations, it is also possible that amino acid
substitutions required to create mini-Gs instead mitigate H5 tilting. Both of these models are
consistent with the fact that some of the mutations in mini-Gs stabilize the GDP-bound
state<span class='cite'> [<a href='mainli5.html#XSun:2015gj'>180</a>]</span>.
</p>
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x8-38001r3'></a>
                                                                                      
                                                                                      
<!-- l. 64 --><p class='noindent'><img width='289' height='289' src='Chapter4/ch4-fig3.png' alt='PIC' />
<a id='x8-38002'></a>
<br /> </p><div class='caption'><span class='id'>Figure 3.3:  </span><span class='content'>Tilting  of  H5  is  correlated  with  GDP  release  but  translation  of  H5  is  not.  <span class='ptmb7t-x-x-120'>A.</span>
Displacements  of  H5  relative  to  the  GDP-bound  crystal  structure  (blue).  The  three  other
orientations  of  H5  come  from  the  rate-limiting  step  in  our  model  (orange),  the  co-crystal
structure of G<span class='cmmi-12'>α</span>s and <span class='cmmi-12'>β</span>2AR (green, PDB ID 3SN6 <span class='cite'> [<a href='mainli5.html#XRasmussen:2011kp'>68</a>]</span>), and the co-crystal structure of a
C-terminal fragment of G<span class='cmmi-12'>α</span>t and rhodopsin (magenta, PDB ID 3PQR <span class='cite'> [<a href='mainli5.html#XChoe:2011cb'>211</a>]</span>). The black arrows
highlight the change in orientation of the long axis of each helix. A representative GPCR (gray)
and G<span class='cmmi-12'>α </span>(white) structure are shown for reference. <span class='ptmb7t-x-x-120'>B. </span>GDP release distance across the highest
flux pathway, defined as the distance from the GDP <span class='cmmi-12'>β</span>-phosphate to the center of mass between
residues Lys52<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>1</span></sup>, Ser53<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>2</span></sup>, and Thr54<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>3</span></sup> on H1. The state marking the rate-limiting
step  is  highlighted  by  the  blue  dashed  line.  <span class='ptmb7t-x-x-120'>C.  </span>H5  motion  across  the  highest  flux  pathway.
The  distances  measured  here  representing  H5  motion  are  taken  from  the  same  states  as  in
B. H5 tilting (orange) is measured by the distance between Leu349<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>5</span><span class='cmmi-8'>.</span><span class='cmr-8'>16</span></sup> and Tyr325<sup><span class='cmmi-8'>G.S</span><span class='cmr-8'>6</span><span class='cmmi-8'>.</span><span class='cmr-8'>2</span></sup>.
Likewise, H5 vertical motion (green) is measured by the distance between Thr334<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>5</span><span class='cmmi-8'>.</span><span class='cmr-8'>1</span></sup> and
Phe341<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>5</span><span class='cmmi-8'>.</span><span class='cmr-8'>8</span></sup>. The rate-limiting step is marked with the blue dashed line, extended down from
B.</span></div><!-- tex4ht:label?: x8-38001r3.3  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<!-- l. 70 --><p class='noindent'>We propose that tilting of H5 is an early step in the GDP release process, which is followed by upward
translation of this helix to form a GPCR-G protein complex primed to bind GTP. This hypothesis
stems from our observation that tilting of H5 is coincident with the rate-limiting step for GDP release,
while translation of H5 only becomes stable after GDP dissociates (Figure <a href='#x8-38001r3'>3.3<!-- tex4ht:ref: fig:ch4-fig3  --></a>C). This
model is consistent with previous suggestions that G-protein activation occurs through
a series of sequential interactions with a GPCR <span class='cite'> [<a href='mainli5.html#XOldham:2008cr'>169</a>, <a href='mainli5.html#XRasmussen:2011kp'>68</a>]</span>. Another possibility is that
any displacement of H5, whether tilting or translation, may be sufficient to trigger GDP
release.
</p>
<h4 class='subsectionHead'><span class='titlemark'>3.3.3    </span> <a id='x8-390003.3.3'></a>Identification of the allosteric network that triggers GDP release</h4>
<!-- l. 73 --><p class='noindent'>While conformational changes of H5 are important for G<span class='cmmi-12'>α </span>activation, other regions could also play a
role<span class='cite'> [<a href='mainli5.html#XHilger:2018en'>177</a>, <a href='mainli5.html#XSun:2015gj'>180</a>]</span>. However, it is not straightforward to determine what other structural elements
contribute to activation or their importance relative to H5. Our hypothesis that spontaneous motions of
a protein encode functionally relevant conformational changes suggests that the coupling between the
GPCR- and nucleotide-binding sites of G<span class='cmmi-12'>α </span>should be present in simulations of the inactive protein;
This provides a means to identify key elements of this allosteric network. To test this hypothesis, we
ran 35.3 <span class='cmmi-12'>μ</span>s of simulation of GDP-G<span class='cmmi-12'>α</span>q. Then we applied the CARDS method<span class='cite'> [<a href='mainli5.html#XSingh:2017hh'>90</a>]</span> to
detect correlations between both the structure and dynamical states of every pair of dihedral
angles. Structural states are determined by assigning dihedral angles to the three dominant
rotameric states for side-chains (gauche+, gauche-, and trans) and the two dominant rotameric
states for backbone dihedrals (cis and trans). Dynamical states are determined by whether a
dihedral angle remains in a single rotameric state (ordered) or rapidly transitions between
multiple rotameric states (disordered). These pairwise correlations serve as a basis for
                                                                                      
                                                                                      
quantifying the correlation of every residue to a target site, such as the GPCR-binding
site. Combining these correlations with structural and dynamical changes in our model of
GDP release provides a basis for inferring how perturbations to the GPCR-binding site
are transmitted to the nucleotide-binding site. We focus our analysis on the most direct
routes for communication between the GPCR- and nucleotide-binding sites by following
correlated motions that emanate from structural elements that directly contact GPCRs until
they reach the GDP-binding site. There are correlations between many other elements of
G<span class='cmmi-12'>α</span>q, including parts of the helical domain, that branch off of this allosteric network. Such
correlations may be important for aspects of G<span class='cmmi-12'>α </span>function besides activation, but are beyond
the scope of the present study, which focuses on how GPCR-binding impacts nucleotide
release.
</p><!-- l. 75 --><p class='noindent'>To understand how H5 tilting triggers GDP release, we first identified structural elements with strong
coupling to H5 and then worked our way outward in repeated iterations until we reached the
nucleotide-binding site (Figure <a href='#x8-39001r4'>3.4<!-- tex4ht:ref: fig:ch4-fig4  --></a>). This analysis reveals that tilting of H5 directly communicates
with and impacts the conformational preferences of the s6h5 loop, which contacts the nucleobase of
GDP (Figure <a href='#x8-39001r4'>3.4<!-- tex4ht:ref: fig:ch4-fig4  --></a> and Figure <a href='#x8-39003r5'>3.5<!-- tex4ht:ref: fig:ch4-fig5  --></a>). During the rate-limiting step, these contacts are broken and there is
increased disorder in the s6h5 loop, particularly Ala331 of the TCAT motif (Figure <a href='#x8-39003r5'>3.5<!-- tex4ht:ref: fig:ch4-fig5  --></a> and Figure
<a href='mainap2.html#x15-140005r3'>B.3<!-- tex4ht:ref: fig:ch4-fig2-supp3  --></a>B). The importance of the TCAT motif in our model is consistent with its conservation and the
fact that mutating it accelerates GDP release<span class='cite'> [<a href='mainli5.html#XIiri:1994kw'>214</a>, <a href='mainli5.html#XPosner:1998gj'>215</a>, <a href='mainli5.html#XThomas:1993wv'>216</a>]</span>. Based on our model, we
propose these mutations accelerate release by weakening shape complementarity with
GDP.
</p>
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x8-39001r4'></a>
                                                                                      
                                                                                      
<!-- l. 79 --><p class='noindent'><img width='452' height='453' src='Chapter4/ch4-fig4.png' alt='PIC' />
<a id='x8-39002'></a>
<br /> </p><div class='caption'><span class='id'>Figure 3.4: </span><span class='content'>Allosteric network connecting H5 motion to the nucleotide binding-site via s6h5.
CARDS data showing communication per residue to a target site (dashed box) is plotted (left)
and mapped onto the structure of G<span class='cmmi-12'>α</span>q (right) for <span class='ptmb7t-x-x-120'>(A) </span>H5, <span class='ptmb7t-x-x-120'>(B) </span>s6h5, <span class='ptmb7t-x-x-120'>(C) </span>HG, <span class='ptmb7t-x-x-120'>(D) </span>H1, and <span class='ptmb7t-x-x-120'>(E)</span>
the P-loop. Arrows indicate regions of importance with significant communication to the target
site.</span></div><!-- tex4ht:label?: x8-39001r3.3  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x8-39003r5'></a>
                                                                                      
                                                                                      
<!-- l. 87 --><p class='noindent'><img width='361' height='361' src='Chapter4/ch4-fig5.png' alt='PIC' />
<a id='x8-39004'></a>
<br /> </p><div class='caption'><span class='id'>Figure 3.5: </span><span class='content'>Change in the s6h5 loop conformation across the rate-limiting step. Distribution
of distances (left) from GDP’s <span class='cmmi-12'>β</span>-phosphate to Ala331<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>6</span><span class='cmmi-8'>h</span><span class='cmr-8'>5</span><span class='cmmi-8'>.</span><span class='cmr-8'>3</span></sup> on the s6h5 loop before (blue)
and after (orange) the rate-limiting step, as well as after GDP release (black). Representative
structures of the s6h5 loop (right) are shown for before (top right, blue) and after (bottom right,
orange) the rate limiting step.</span></div><!-- tex4ht:label?: x8-39003r3.3  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<!-- l. 93 --><p class='noindent'>We also observe an important role for communication from H5 to H1, consistent with the universal
mechanism. In particular, H1 is strongly coupled with the s6h5 loop (Figure <a href='#x8-39001r4'>3.4<!-- tex4ht:ref: fig:ch4-fig4  --></a>B), which is sensitive
to displacement of H5. In the rate-limiting step, s6h5 moves away from H1, contributing to an increase
in disorder of H1 and the P-loop (Figure <a href='mainap2.html#x15-140005r3'>B.3<!-- tex4ht:ref: fig:ch4-fig2-supp3  --></a>A and Figure <a href='mainap2.html#x15-140005r3'>B.3<!-- tex4ht:ref: fig:ch4-fig2-supp3  --></a>B). Increased disorder in a set of
residues that directly contact the GDP phosphates (Glu49<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>1</span><span class='cmmi-8'>h</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>4</span></sup>, Ser50<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>1</span><span class='cmmi-8'>h</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>5</span></sup>, Gly51<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>1</span><span class='cmmi-8'>h</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>6</span></sup>,
Lys52<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>1</span></sup>, and Ser53<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>2</span></sup>) likely contributes to a reduced affinity for GDP (Figure <a href='mainap2.html#x15-140005r3'>B.3<!-- tex4ht:ref: fig:ch4-fig2-supp3  --></a>A).
Increased disorder in these residues also contributes to disruption of the salt bridge between
Glu49<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>1</span><span class='cmmi-8'>h</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>4</span></sup> of the Ras-like domain and Arg183<sup><span class='cmmi-8'>G.hfs</span><span class='cmr-8'>2</span><span class='cmmi-8'>.</span><span class='cmr-8'>2</span></sup> of the helical domain, facilitating
inter-domain separation.
</p><!-- l. 95 --><p class='noindent'>We further note that the s6h5 loop impacts the nucleotide-binding site through allosteric coupling with
the HG helix, which also contacts GDP via Lys275<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>5</span><span class='cmmi-8'>hg.</span><span class='cmr-8'>1</span></sup> and Asp277<sup><span class='cmmi-8'>G.HG.</span><span class='cmr-8'>2</span></sup> (Figures <a href='#x8-39001r4'>3.4<!-- tex4ht:ref: fig:ch4-fig4  --></a>E and <a href='#x8-39005r6'>3.6<!-- tex4ht:ref: fig:ch4-fig6  --></a>).
The disorder of both of these residues increases during the rate-limiting step, consistent with
observations of increased mobility in HG upon receptor-mediated activation (Oldham and Hamm,
2008). There are also correlations between the P-loop and Lys275<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>5</span><span class='cmmi-8'>hg.</span><span class='cmr-8'>1</span></sup> on Helix G (Figure <a href='#x8-39001r4'>3.4<!-- tex4ht:ref: fig:ch4-fig4  --></a>E),
which result from the disruption of a key salt bridge between Lys275<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>5</span><span class='cmmi-8'>hg.</span><span class='cmr-8'>1</span></sup> and Glu49<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>1</span><span class='cmmi-8'>h</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>4</span></sup> on
the P-loop during the rate-limiting step (Figure <a href='#x8-39005r6'>3.6<!-- tex4ht:ref: fig:ch4-fig6  --></a>). Lys275<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>5</span><span class='cmmi-8'>hg.</span><span class='cmr-8'>1</span></sup> is conserved across
all G<span class='cmmi-12'>α </span>families, suggesting it plays an important role in the stability or function of the
protein. However, attempts to experimentally examine the role of this residue by mutating
Lys275<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>5</span><span class='cmmi-8'>hg.</span><span class='cmr-8'>1</span></sup> have resulted in aggregation<span class='cite'> [<a href='mainli5.html#XSun:2015gj'>180</a>]</span>. Our simulations suggest Lys275<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>5</span><span class='cmmi-8'>hg.</span><span class='cmr-8'>1</span></sup>
plays an important role in stabilizing the GDP-bound state and that breaking the salt bridge
with Glu49<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>1</span><span class='cmmi-8'>h</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>4</span></sup> facilitates GDP release. This finding demonstrates the utility of our
atomistic simulations, as we can examine the role of residues that are difficult to probe
experimentally.
</p>
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x8-39005r6'></a>
                                                                                      
                                                                                      
<!-- l. 99 --><p class='noindent'><img width='238' height='238' src='Chapter4/ch4-fig6.png' alt='PIC' />
<a id='x8-39006'></a>
<br /> </p><div class='caption'><span class='id'>Figure 3.6: </span><span class='content'>Probability distributions of the distance between the side-chains of Lys275<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>5</span><span class='cmmi-8'>hg.</span><span class='cmr-8'>1</span></sup>
and  Glu49<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>1</span><span class='cmmi-8'>h</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>4</span></sup>.  Distributions  were  computed  for  the  bound  (blue),  rate-limiting  step
(orange), and dissociated (black) states. Representative structures (above) for the bound (left,
blue) and rate-limiting step (right, orange) are included with residues as sticks.</span></div><!-- tex4ht:label?: x8-39005r3.3  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<h4 class='subsectionHead'><span class='titlemark'>3.3.4    </span> <a id='x8-400003.3.4'></a>H1 and <span class='cmmi-12'>β</span>-sheets are communication hubs</h4>
<!-- l. 106 --><p class='noindent'>To identify other important structural elements in the allosteric network underlying G protein
activation, we followed correlated motions emanating from other sites that are known to interact
directly with GPCRs, including the hNs1 loop, the h3s5 loop, and the h4s6 loop<span class='cite'> [<a href='mainli5.html#XOldham:2008cr'>169</a>]</span>. We find that
h3s5 and h4s6 are largely isolated, suggesting they play a role in forming a stable GPCR-G protein
complex but not in the allosteric mechanism that triggers GDP release. This finding is consistent with
sequence analysis suggesting these structural elements are important determinants of the specificity
of GPCR-G<span class='cmmi-12'>α </span>interactions<span class='cite'> [<a href='mainli5.html#XFlock:2017bm'>217</a>]</span>. In contrast, the hNs1 loop has strong correlations with
<span class='cmmi-12'>β</span>-strands S1-S3 (Figure <a href='#x8-40001r7'>3.7<!-- tex4ht:ref: fig:ch4-fig7  --></a>). These strands, in turn, communicate with H1, switch 1, and the
P-loop.
</p>
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x8-40001r7'></a>
                                                                                      
                                                                                      
<!-- l. 110 --><p class='noindent'><img width='404' height='404' src='Chapter4/ch4-fig7.png' alt='PIC' />
<a id='x8-40002'></a>
<br /> </p><div class='caption'><span class='id'>Figure 3.7: </span><span class='content'>CARDS data showing communication per residue to a target site (dashed box) is
plotted (left) and mapped onto the structure of G<span class='cmmi-12'>α</span>q (right) for <span class='ptmb7t-x-x-120'>(A) </span>hNs1, <span class='ptmb7t-x-x-120'>(B) </span>S1, <span class='ptmb7t-x-x-120'>(C) </span>S3, <span class='ptmb7t-x-x-120'>(D)</span>
S2, and <span class='ptmb7t-x-x-120'>(E) </span>H1. Arrows indicate regions of importance with significant communication to the
target site.</span></div><!-- tex4ht:label?: x8-40001r3.3  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<!-- l. 116 --><p class='noindent'>Integrating our correlation analysis with structural insight from the rate-limiting step described above
suggests an important role for S1-S3 in a complex allosteric network that couples the GPCR- and
nucleotide-binding sites (Figures <a href='#x8-40001r7'>3.7<!-- tex4ht:ref: fig:ch4-fig7  --></a> and <a href='#x8-40003r8'>3.8<!-- tex4ht:ref: fig:ch4-fig8  --></a>, Figure <a href='mainap2.html#x15-140017r8'>B.8<!-- tex4ht:ref: fig:ch4-fig7-supp1  --></a>). S2 and S3 twist relative to S1 and away
from H1 (Figures <a href='#x8-37001r2'>3.2<!-- tex4ht:ref: fig:ch4-fig2  --></a>A and <a href='#x8-40005r9'>3.9<!-- tex4ht:ref: fig:ch4-fig9  --></a>, Figure <a href='mainap2.html#x15-140021r10'>B.10<!-- tex4ht:ref: fig:ch4-fig9-supp1  --></a>). This twisting disrupts stacking between Phe194<sup><span class='cmmi-8'>G.S</span><span class='cmr-8'>2</span><span class='cmmi-8'>.</span><span class='cmr-8'>6</span></sup> on
S2 and His63<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>12</span></sup> on H1 and increases disorder of side-chains in H1 (Figures <a href='#x8-37001r2'>3.2<!-- tex4ht:ref: fig:ch4-fig2  --></a>B and <a href='#x8-40005r9'>3.9<!-- tex4ht:ref: fig:ch4-fig9  --></a>, Figure
<a href='mainap2.html#x15-140005r3'>B.3<!-- tex4ht:ref: fig:ch4-fig2-supp3  --></a>C). Increased disorder in H1 is also a crucial component of the proposed universal
mechanism, but in that model translation of H5 is the key trigger for changes in H1. The role
for the <span class='cmmi-12'>β</span>-sheets in our model is consistent with previous work identifying interactions
between S2 and H1<span class='cite'> [<a href='mainli5.html#XFlock:2015dj'>171</a>]</span>, NMR experiments showing chemical exchange in the methyls of
S1-S3 upon receptor binding<span class='cite'> [<a href='mainli5.html#XToyama:2017gy'>179</a>]</span>, and mutational data. In particular, Flock et al. have
previously noted the important interaction between residues Phe194<sup><span class='cmmi-8'>G.S</span><span class='cmr-8'>2</span><span class='cmmi-8'>.</span><span class='cmr-8'>6</span></sup> and His63<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>12</span></sup>
<span class='cite'> [<a href='mainli5.html#XFlock:2015dj'>171</a>]</span>.
</p>
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x8-40003r8'></a>
                                                                                      
                                                                                      
<!-- l. 120 --><p class='noindent'><img width='228' height='228' src='Chapter4/ch4-fig8.png' alt='PIC' />
<a id='x8-40004'></a>
<br /> </p><div class='caption'><span class='id'>Figure 3.8: </span><span class='content'>Allosteric network connecting the GPCR- and nucleotide-binding interfaces. The
coloring  scheme  corresponds  to  that  used  in  <a href='#x8-35001r1'>3.1<!-- tex4ht:ref: fig:ch4-fig1  --></a>,  highlighting  the  GPCR  binding  interface
(gray), GDP phosphate-binding regions (pink), GDP nucleotide-binding regions (green), and
the <span class='cmmi-12'>β</span>-sheets (blue).</span></div><!-- tex4ht:label?: x8-40003r3.3  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x8-40005r9'></a>
                                                                                      
                                                                                      
<!-- l. 128 --><p class='noindent'><img width='228' height='228' src='Chapter4/ch4-fig9.png' alt='PIC' />
<a id='x8-40006'></a>
<br /> </p><div class='caption'><span class='id'>Figure 3.9:  </span><span class='content'><span class='cmmi-12'>π</span>–stacking  between  S2  and  H1  is  disrupted  during  the  rate-limiting  step.
Distribution of the <span class='cmmi-12'>χ</span><sub><span class='cmr-8'>1</span></sub> angle (bottom) of Phe194<sup><span class='cmmi-8'>G.S</span><span class='cmr-8'>2</span><span class='cmmi-8'>.</span><span class='cmr-8'>6</span></sup> on S2 before (blue) and after (orange)
the  rate-limiting  step,  as  well  as  after  GDP  release  (black).  Representative  structures  of
Phe194<sup><span class='cmmi-8'>G.S</span><span class='cmr-8'>2</span><span class='cmmi-8'>.</span><span class='cmr-8'>6</span></sup> and His63<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>12</span></sup> (top) corresponding to before and after the rate-limiting step are
also shown.</span></div><!-- tex4ht:label?: x8-40005r3.3  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<!-- l. 134 --><p class='noindent'>The importance of H1 and <span class='cmmi-12'>β</span>-strands S1-S3 is underscored by mapping the global communication of
every residue onto a structure of G<span class='cmmi-12'>α </span>(Figure <a href='mainap2.html#x15-140019r9'>B.9<!-- tex4ht:ref: fig:ch4-fig8-supp1  --></a>). The global communication of a residue is the sum
of its correlations to every other residue and is a useful metric for identifying residues that are
important players in allosteric networks<span class='cite'> [<a href='mainli5.html#XSingh:2017hh'>90</a>]</span>. Interestingly, these <span class='cmmi-12'>β</span>-strands and H1 have higher global
communication than H5 and the s6h5 loop. This suggests that H1 and the <span class='cmmi-12'>β</span>-sheets integrate
conformational information from multiple sources, including the hNs1 loop, and not just H5. The
importance of the <span class='cmmi-12'>β</span>-sheets and H1 for allosteric communication is consistent with their
conservation<span class='cite'> [<a href='mainli5.html#XSun:2015gj'>180</a>]</span>, which may not simply reflect the role they play in protein folding and stability, as
had been suggested previously<span class='cite'> [<a href='mainli5.html#XSun:2015gj'>180</a>, <a href='mainli5.html#XHatley:2003co'>218</a>]</span>.
                                                                                      
                                                                                      
</p>
<h4 class='subsectionHead'><span class='titlemark'>3.3.5    </span> <a id='x8-410003.3.5'></a>GDP release alters the structure and dynamics of the G<span class='cmmi-12'>β</span>-binding site</h4>
<!-- l. 139 --><p class='noindent'>We also find that switch 2 has strong correlations with the nucleotide-binding site, especially switch 1
(Figure <a href='mainap2.html#x15-140017r8'>B.8<!-- tex4ht:ref: fig:ch4-fig7-supp1  --></a>). Given that switch 2 is a major component of the interface between G<span class='cmmi-12'>α </span>and G<span class='cmmi-12'>β</span>, this
communication could enable GDP release to trigger dissociation of G<span class='cmmi-12'>α </span>from G<span class='cmmi-12'>βγ</span>. Examining the
rate-limiting step for GDP release reveals that switch 2 shifts towards the nucleotide-binding site
(Figure <a href='#x8-41001r10'>3.10<!-- tex4ht:ref: fig:ch4-fig10  --></a>) and exhibits increased conformational disorder (Figure <a href='#x8-37001r2'>3.2<!-- tex4ht:ref: fig:ch4-fig2  --></a>B and Figure <a href='mainap2.html#x15-140005r3'>B.3<!-- tex4ht:ref: fig:ch4-fig2-supp3  --></a>). These
findings are consistent with previous kinetic studies postulating that switch 2 dynamics are impacted
prior to GDP release<span class='cite'> [<a href='mainli5.html#XHerrmann:2004gp'>219</a>]</span>.
</p>
<hr class='figure' /><div class='figure'>
                                                                                      
                                                                                      
<a id='x8-41001r10'></a>
                                                                                      
                                                                                      
<!-- l. 143 --><p class='noindent'><img width='289' height='289' src='Chapter4/ch4-fig10.png' alt='PIC' />
<a id='x8-41002'></a>
<br /> </p><div class='caption'><span class='id'>Figure 3.10: </span><span class='content'>Switch 2 moves towards GDP across the rate-limiting step. Distance distribution
(left) of Gly207<sup><span class='cmmi-8'>G.s</span><span class='cmr-8'>3</span><span class='cmmi-8'>h</span><span class='cmr-8'>2</span><span class='cmmi-8'>.</span><span class='cmr-8'>1</span></sup> to GDP before (blue) and after (orange) the rate-limiting step, as well
as after GDP release (black). Representative structures of Switch 2 motion are shown (right).</span></div><!-- tex4ht:label?: x8-41001r3.3  -->
                                                                                      
                                                                                      
</div><hr class='endfigure' />
<h3 class='sectionHead'><span class='titlemark'>3.4    </span> <a id='x8-420003.4'></a>Conclusion</h3>
<!-- l. 150 --><p class='noindent'>We have succeeded in simulating G protein activation, including both the allosteric coupling
between the GPCR- and nucleotide-binding sites of G<span class='cmmi-12'>α</span>q and consequent unbinding of GDP.
Our results reveal a previously unobserved intermediate that defines the rate-limiting step
for GDP release and, ultimately, G protein activation. Our model synthesizes a wealth of
experimental data and previous analyses. For example, we identify an important role for coupling
from H5 to the s6h5 loop and H1 that is consistent with a previously proposed universal
mechanism for G protein activation. However, we also find that this allosteric network
incorporates the hNs1 loop, <span class='cmmi-12'>β</span>-strands S1-S3, and the HG helix. Strands S1-S3 and H1
serve as hubs in this network, simultaneously integrating information from both H5 and the
hNs1 loop. Our observation is consistent with previous postulates that information flows
from H5 and hNs1 to H1<span class='cite'> [<a href='mainli5.html#XPreininger:2013if'>220</a>]</span>. It is important to note that our model was extracted using
simulations of G<span class='cmmi-12'>α</span>q, and so some correlations or changes in conformation and dynamics may
apply only to G<span class='cmmi-12'>α</span>q. However, by focusing our analysis on secondary structure elements and
residues that are shared across all G<span class='cmmi-12'>α </span>homologs, our model likely captures a universal
‘skeleton’ of changes involved in G<span class='cmmi-12'>α </span>activation, expanding upon a previously proposed
universal mechanism for G<span class='cmmi-12'>α </span>activation<span class='cite'> [<a href='mainli5.html#XFlock:2015dj'>171</a>]</span>. The consistency of our model with a wide
variety of structural and biochemical data suggests that it is a promising foundation for
future efforts to understand the determinants of GPCR-G<span class='cmmi-12'>α </span>interaction specificity, how
mutations cause aberrant signaling and disease, and how small molecule inhibitors modulate
G<span class='cmmi-12'>α </span>activation. Our model also adds weight to the growing appreciation for the fact that a
protein’s spontaneous fluctuations encode considerable information about its functional
dynamics<span class='cite'> [<a href='mainli5.html#XKoehl:2018fv'>20</a>, <a href='mainli5.html#XDraperJoyce:2018il'>21</a>, <a href='mainli5.html#XGarciaNafria:2018hu'>22</a>, <a href='mainli5.html#XKang:2018kq'>23</a>]</span>. Construction of our model was enabled by a powerful combination of
                                                                                      
                                                                                      
simulation methods, namely metadynamics and MSMs. In the future, we expect this methodology
will prove valuable for understanding other slow conformational changes and unbinding
processes.
</p><!-- l. 153 --><p class='noindent'>
</p>
<h3 class='sectionHead'><span class='titlemark'>3.5    </span> <a id='x8-430003.5'></a>Materials and Methods</h3>
<!-- l. 154 --><p class='noindent'>
</p>
<h4 class='subsectionHead'><span class='titlemark'>3.5.1    </span> <a id='x8-440003.5.1'></a>Molecular dynamics simulations of GDP unbinding</h4>
<!-- l. 155 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x8-450003.5.1'></a>System preparation</h5>
<!-- l. 156 --><p class='noindent'>We used the crystal structure of G<span class='cmmi-12'>α </span>from the heterotrimeric G<span class='cmmi-12'>α</span>q protein (PDB entry 3AH8) as
the initial structure to set up our simulations<span class='cite'> [<a href='mainli5.html#XNishimura:2010gt'>203</a>]</span>. The first 36 residues were removed
prior to simulation, as they come from G<span class='cmmi-12'>α</span>i, along with the G<span class='cmmi-12'>β </span>and G<span class='cmmi-12'>γ </span>subunits which
were removed to minimize the system size and maximize our chances of observing GDP
release<span class='cite'> [<a href='mainli5.html#XOldham:2008cr'>169</a>]</span>.
                                                                                      
                                                                                      
</p><!-- l. 158 --><p class='noindent'>The protein structure was solvated in a dodecahedron box of TIP3P water<span class='cite'> [<a href='mainli5.html#XJorgensen:1983fl'>143</a>]</span> that extended 1 nm
beyond the protein in every dimension. A single Mg2+ ion was added coordinating the phosphate
groups of GDP as its presence accelerates GDP release<span class='cite'> [<a href='mainli5.html#XZhang:2000cy'>221</a>]</span>. Thereafter, Na+ and Cl<span class='cmsy-10x-x-120'>− </span>were added to
produce a neutral system at 0.15 M NaCl. The final system consists of one GDP, one G<span class='cmmi-12'>α</span>q,
57 Cl<sup><span class='cmsy-8'>−</span><span class='cmr-8'>1</span></sup>, 64 Na<sup><span class='cmr-8'>+1</span></sup>, one Mg<sup><span class='cmr-8'>2+</span></sup>, and 18696 TIP3P water molecules, for a total of 61,893
atoms.
</p><!-- l. 160 --><p class='noindent'>Molecular dynamics (MD) simulations were carried out using Gromacs<span class='cite'> [<a href='mainli5.html#XAbraham:2015gj'>222</a>, <a href='mainli5.html#XVanDerSpoel:2005hz'>145</a>]</span> and the
AMBER03 force field<span class='cite'> [<a href='mainli5.html#XDuan:2003gt'>146</a>]</span>. The force field parameters of GDP were obtained from the
AMBER Parameter Database (http://research.bmh.manchester.ac.uk/bryce/amber)<span class='cite'> [<a href='mainli5.html#XMeagher:2003be'>223</a>]</span>.
The system was energy minimized with the steepest descent algorithm for 50,000 steps
until the maximum force fell below 100 kJ/mol/nm using a step size of 0.02 nm and a
cut-off distance of 0.9 nm for the neighbor list, electrostatic interactions and van der Waals
interactions. Afterward, the solvent was relaxed by a NVT simulation of 100 ps with the
constraint of 1000 kJ/mol/nm applied to the protein heavy atoms and 2 fs as the time step. In
this relaxation simulation, the temperature of the system is constrained to 300 K using
V-rescale thermostat (with a time constant of 0.1 ps)<span class='cite'> [<a href='mainli5.html#XBussi:2007cs'>224</a>]</span>. A cut-off distance of 1 nm
was used for the van der Waals, short-range electrostatic interactions. Periodic boundary
conditions are applied to x, y and z directions. The Particle-Mesh-Ewald method is employed
to recover the long-range electrostatic interactions with 0.16 nm as the grid spacing and
with a fourth order spline<span class='cite'> [<a href='mainli5.html#XKolafa:2006jh'>225</a>]</span>. All the bonds connecting to hydrogens are constrained
using LINCS algorithm<span class='cite'> [<a href='mainli5.html#XHess:2008fl'>226</a>]</span>. After the NVT relaxation, the system further underwent
an NPT simulation for one ns with the time step of 2 fs for equilibration. The simulation
parameters here are kept the same as the NVT relaxation except for the application of
Parrinello-Rahman barostat for pressure coupling<span class='cite'> [<a href='mainli5.html#XParrinello:1981it'>227</a>]</span>. After these equilibration runs, the constraint
on heavy atoms were removed for all subsequent production runs. Virtual sites were used to
allow for a 4 fs time-step<span class='cite'> [<a href='mainli5.html#XFeenstra:1999ue'>228</a>]</span>. We then applied a three-step protocol to simulate GDP
release.
                                                                                      
                                                                                      
</p>
<h5 class='subsubsectionHead'><a id='x8-460003.5.1'></a>Step one: MD simulations of the GDP-bound state</h5>
<!-- l. 162 --><p class='noindent'>We performed 100 parallel simulations of the GDP bound state on the Folding@home<span class='cite'> [<a href='mainli5.html#XShirts:2000du'>37</a>]</span> distributed
computing environment for an aggregate simulation time of 35.3 <span class='cmmi-12'>μ</span>s.
</p><!-- l. 164 --><p class='noindent'>
</p>
<h5 class='subsubsectionHead'><a id='x8-470003.5.1'></a>Step two: Metadynamics simulations</h5>
<!-- l. 165 --><p class='noindent'>We subsequently ran metadynamics simulations<span class='cite'> [<a href='mainli5.html#XLaio:2002ft'>195</a>, <a href='mainli5.html#XTribello:2014eb'>229</a>]</span> initiated from conformations generated in
step one to actively promote GDP release. Starting conformations were selected from step one
by clustering protein conformations into 625 states using a hybrid K-center/K-medoids
method<span class='cite'> [<a href='mainli5.html#XBeauchamp:2011he'>230</a>]</span> with a 2 Åcutoff. The 10 most populated states included conformations
with large inter-domain separation distances between the Ras-like and AH domains as
measured by the angle formed between the <span class='cmmi-12'>α</span>-carbon atoms of Leu97<sup><span class='cmmi-8'>H.HA.</span><span class='cmr-8'>29</span></sup>, Asn82<sup><span class='cmmi-8'>H.HA.</span><span class='cmr-8'>14</span></sup>,
Ile258<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>3</span><span class='cmmi-8'>.</span><span class='cmr-8'>12</span></sup>, and Glu250<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>3</span><span class='cmmi-8'>.</span><span class='cmr-8'>4</span></sup>. This inter-domain angle ranged from <span class='cmsy-10x-x-120'>−</span>30<span class='ptmr8c-x-x-120'>° </span>to 0<span class='ptmr8c-x-x-120'>°</span>. From
these states, three representative structures were chosen with inter-domain angles of <span class='cmsy-10x-x-120'>−</span>6<span class='ptmr8c-x-x-120'>°</span>
(Conf. 1),<span class='cmsy-10x-x-120'>−</span>10<span class='ptmr8c-x-x-120'>° </span>(Conf. 2), and <span class='cmsy-10x-x-120'>−</span>26.6<span class='ptmr8c-x-x-120'>° </span>(Conf. 3) as starting conformations for metadynamics
simulations, which were run on PLUMED<span class='cite'> [<a href='mainli5.html#XTribello:2014eb'>229</a>]</span>. We defined two collective variables for our
metadynamics simulations: 1) the distance between GDP’s phosphate groups and the backbone of
Lys52<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>1</span></sup>-Thr54<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>3</span></sup> in G<span class='cmmi-12'>α</span>q subunit (CV1), and 2) the RMSD of GDP to the starting
conformation (CV2).
</p><!-- l. 167 --><p class='noindent'>In metadynamics, a history-dependent biased potential VG(S,t) is added to the two selected
CVs.
                                                                                      
                                                                                      
</p>
<table class='equation'><tr><td><a id='x8-47001r1'></a>
<center class='math-display'>
<img src='main10x.png' alt='           ∫ t            ∑ d (S (R ) − S (R (t′)))2
VG (S,t) =    (dt′ωexp (−     --i--------i2--------))
            0              i=1         2σ i
' class='math-display' /></center></td><td class='equation-label'>(3.1)</td></tr></table>
<!-- l. 171 --><p class='nopar'>
</p><!-- l. 173 --><p class='noindent'>where <span class='cmmi-12'>t </span>represents time, <span class='cmmi-12'>S </span>are collective variables, <span class='cmmi-12'>ω </span>is the energy rate and <span class='cmmi-12'>σ</span><sub><span class='cmmi-8'>i</span></sub> controls the width of the
Gaussian for the ith collective variable. Summing up the Gaussians allows us to obtain the biased
potential <span class='cmmi-12'>V</span> <sub><span class='cmmi-8'>G</span></sub>. The free energy <span class='cmsy-10x-x-120'>−</span><span class='cmmi-12'>F</span><span class='cmr-12'>(</span><span class='cmmi-12'>S</span><span class='cmr-12'>) </span>is derived by the assumption,
</p>
<table class='equation'><tr><td><a id='x8-47002r2'></a>
<center class='math-display'>
<img src='main11x.png' alt=' lim  (S,t) ∼ − F (S )
t→ ∞
' class='math-display' /></center></td><td class='equation-label'>(3.2)</td></tr></table>
<!-- l. 177 --><p class='nopar'>
</p><!-- l. 179 --><p class='noindent'>The metadynamics simulations were repeated three times for each selected representative structure
using different Gaussian widths (<a href='#x8-47003r1'>3.1<!-- tex4ht:ref: tab:metadyn-table  --></a>). We set the Gaussian height to 1.5 kJ/mol.
</p>
<div class='table'>
                                                                                      
                                                                                      
<!-- l. 181 --><p class='noindent'><a id='x8-47003r1'></a></p><hr class='float' /><div class='float'>
                                                                                      
                                                                                      
<a id='x8-47004'></a>
 <div class='caption'><span class='id'>Table 3.1: </span><span class='content'>Details of metadynamics simulations</span></div><!-- tex4ht:label?: x8-47003r3.5  -->
<div class='tabular'> <table class='tabular' cellspacing='0' cellpadding='0' id='TBL-1'><colgroup id='TBL-1-1g'><col id='TBL-1-1' /></colgroup><colgroup id='TBL-1-2g'><col id='TBL-1-2' /></colgroup><colgroup id='TBL-1-3g'><col id='TBL-1-3' /></colgroup><colgroup id='TBL-1-4g'><col id='TBL-1-4' /></colgroup><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-1-1-'><td style='white-space:nowrap; text-align:center;' id='TBL-1-1-1' class='td11'> <div style='white-space:nowrap; text-align:center;' class='multicolumn'><span class='ptmb7t-x-x-120'>Starting conformation</span></div> </td><td style='white-space:nowrap; text-align:center;' id='TBL-1-1-2' class='td11'> <div style='white-space:nowrap; text-align:center;' class='multicolumn'><span class='ptmb7t-x-x-120'>Width of CV1 (Å)</span></div> </td><td style='white-space:nowrap; text-align:center;' id='TBL-1-1-3' class='td11'> <div style='white-space:nowrap; text-align:center;' class='multicolumn'><span class='ptmb7t-x-x-120'>Width of CV2 (Å)</span></div> </td><td style='white-space:nowrap; text-align:center;' id='TBL-1-1-4' class='td11'> <div style='white-space:nowrap; text-align:center;' class='multicolumn'><span class='ptmb7t-x-x-120'>Number of conformations selected</span></div>
</td></tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-1-2-'><td style='white-space:nowrap; text-align:left;' id='TBL-1-2-1' class='td11'> <span class='ptmb7t-x-x-120'>1                                      </span></td><td style='white-space:nowrap; text-align:left;' id='TBL-1-2-2' class='td11'> 0.1                            </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-2-3' class='td11'> 0.1                            </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-2-4' class='td11'> 171                                                      </td></tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-1-3-'><td style='white-space:nowrap; text-align:left;' id='TBL-1-3-1' class='td11'> <span class='ptmb7t-x-x-120'>1 </span></td><td style='white-space:nowrap; text-align:left;' id='TBL-1-3-2' class='td11'> 0.08 </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-3-3' class='td11'> 0.03 </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-3-4' class='td11'> 132</td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-1-4-'><td style='white-space:nowrap; text-align:left;' id='TBL-1-4-1' class='td11'> <span class='ptmb7t-x-x-120'>1                                      </span></td><td style='white-space:nowrap; text-align:left;' id='TBL-1-4-2' class='td11'> 0.03                          </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-4-3' class='td11'> 0.01                          </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-4-4' class='td11'> 504                                                     </td></tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-1-5-'><td style='white-space:nowrap; text-align:left;' id='TBL-1-5-1' class='td11'> <span class='ptmb7t-x-x-120'>2 </span></td><td style='white-space:nowrap; text-align:left;' id='TBL-1-5-2' class='td11'> 0.1 </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-5-3' class='td11'> 0.1 </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-5-4' class='td11'> 145</td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-1-6-'><td style='white-space:nowrap; text-align:left;' id='TBL-1-6-1' class='td11'> <span class='ptmb7t-x-x-120'>2                                      </span></td><td style='white-space:nowrap; text-align:left;' id='TBL-1-6-2' class='td11'> 0.08                          </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-6-3' class='td11'> 0.03                          </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-6-4' class='td11'> 141                                                     </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-1-7-'><td style='white-space:nowrap; text-align:left;' id='TBL-1-7-1' class='td11'> <span class='ptmb7t-x-x-120'>2                                      </span></td><td style='white-space:nowrap; text-align:left;' id='TBL-1-7-2' class='td11'> 0.03                          </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-7-3' class='td11'> 0.01                          </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-7-4' class='td11'> 320                                                     </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-1-8-'><td style='white-space:nowrap; text-align:left;' id='TBL-1-8-1' class='td11'> <span class='ptmb7t-x-x-120'>3                                      </span></td><td style='white-space:nowrap; text-align:left;' id='TBL-1-8-2' class='td11'> 0.1                            </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-8-3' class='td11'> 0.1                            </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-8-4' class='td11'> 198                                                     </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-1-9-'><td style='white-space:nowrap; text-align:left;' id='TBL-1-9-1' class='td11'> <span class='ptmb7t-x-x-120'>3                                      </span></td><td style='white-space:nowrap; text-align:left;' id='TBL-1-9-2' class='td11'> 0.08                          </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-9-3' class='td11'> 0.03                          </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-9-4' class='td11'> 186                                                     </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr style='vertical-align:baseline;' id='TBL-1-10-'><td style='white-space:nowrap; text-align:left;' id='TBL-1-10-1' class='td11'> <span class='ptmb7t-x-x-120'>3                                      </span></td><td style='white-space:nowrap; text-align:left;' id='TBL-1-10-2' class='td11'> 0.03                          </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-10-3' class='td11'> 0.01                          </td><td style='white-space:nowrap; text-align:left;' id='TBL-1-10-4' class='td11'> 288                                                     </td>
</tr><tr class='hline'><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr></table></div>
                                                                                      
                                                                                      
</div><hr class='endfloat' />
</div>
<!-- l. 212 --><p class='noindent'>We observed the release of GDP from G<span class='cmmi-12'>α </span>in the metadynamics simulations and obtained the
free-energy landscape for the release. We then applied the string method<span class='cite'> [<a href='mainli5.html#XMaragliano:2006kg'>231</a>]</span> to detect potential
release pathways and use these conformations as the seeds for simulations in step 3. We can use the
function <span class='cmmi-12'>χ</span><span class='cmr-12'>(</span><span class='cmmi-12'>α</span><span class='cmr-12'>) </span>to represent <span class='cmmi-12'>χ</span><span class='cmr-12'>(</span><span class='cmmi-12'>C</span><span class='cmr-12'>1</span><span class='cmmi-12'>,C</span><span class='cmr-12'>2) </span>which showing the minimum free-energy path. We thus
have
</p>
<table class='equation'><tr><td><a id='x8-47005r3'></a>
<center class='math-display'>
<img src='main12x.png' alt='          n
dZ-(α-)   ∑   ∂𝜃i(χ(α))-dχk-
 d α   =        ∂χ     dα
         k=1      k
' class='math-display' /></center></td><td class='equation-label'>(3.3)</td></tr></table>
<!-- l. 215 --><p class='nopar'>
which is parallel to
</p>
<table class='equation'><tr><td><a id='x8-47006r4'></a>
                                                                                      
                                                                                      
<center class='math-display'>
<img src='main13x.png' alt='∑ n ∂ 𝜃(χ(α)) dF(χk )   ∑n   ∂𝜃i(χ (α))∂ 𝜃j(χ (α))dF (z(α))
    --------- -------=       ----------------------------
 k=1   ∂χk     dχk      j,k=1   ∂ χk      ∂ χk     dZ (α )
' class='math-display' /></center></td><td class='equation-label'>(3.4)</td></tr></table>
<!-- l. 219 --><p class='nopar'>
The average of the tensor
</p>
<table class='equation'><tr><td><a id='x8-47007r5'></a>
<center class='math-display'>
<img src='main14x.png' alt=' n
∑    ∂𝜃i(χ-(α))∂-𝜃j(χ-(α))
       ∂ χk      ∂ χk
j,k=1
' class='math-display' /></center></td><td class='equation-label'>(3.5)</td></tr></table>
<!-- l. 223 --><p class='nopar'>
can be represented as
</p>
<table class='equation'><tr><td><a id='x8-47008r6'></a>
                                                                                      
                                                                                      
<center class='math-display'>
<img src='main15x.png' alt='                   ∫    ∑n  ∂𝜃i(χ(α))∂ 𝜃j(χ (α))
Mij (z ) = Ω −1eβF (z)        -------------------e−βF (z)
                    IRN k=1   ∂ χk      ∂ χk
' class='math-display' /></center></td><td class='equation-label'>(3.6)</td></tr></table>
<!-- l. 227 --><p class='nopar'>
</p>
<table class='equation'><tr><td><a id='x8-47009r7'></a>
<center class='math-display'>
<img src='main16x.png' alt='N
∏
   δ(zv − 𝜃v(χ))dχ
v=1
' class='math-display' /></center></td><td class='equation-label'>(3.7)</td></tr></table>
<!-- l. 230 --><p class='nopar'>
The points determining the minimum free-energy path along the surface satisfy
</p>
<table class='equation'><tr><td><a id='x8-47010r8'></a>
                                                                                      
                                                                                      
<center class='math-display'>
<img src='main17x.png' alt='0 = (Mij (z)ΔF (z(α )))⊥
' class='math-display' /></center></td><td class='equation-label'>(3.8)</td></tr></table>
<!-- l. 234 --><p class='nopar'>
</p><!-- l. 236 --><p class='noindent'>We applied this method<span class='cite'> [<a href='mainli5.html#XMaragliano:2007bz'>232</a>, <a href='mainli5.html#XMaragliano:2014jj'>233</a>]</span> to obtain a minimum free energy path, extracting 2085
conformations along potential GDP release pathways. Notably, we only observed the transition from
the bound state and the unbound state for one time in a single metadynamics simulation. This implies
that the predicted free-energy surface from metadynamics cannot be used to describe the release
events accurately. In spite of this, we can still use the conformations along the release pathway
explored by metadynamics to seed unbiased parallel MD simulations.
</p>
<h5 class='subsubsectionHead'><a id='x8-480003.5.1'></a>Step three: Metadynamics-seeded MD simulation of GDP release</h5>
<!-- l. 239 --><p class='noindent'>Lastly, we carried out unbiased MD simulations initiated from the 2085 conformations
obtained from metadynamics using the Folding@home platform <span class='cite'> [<a href='mainli5.html#XShirts:2000du'>37</a>]</span>. A total of 122.6 <span class='cmmi-12'>μ</span>s of
simulation was generated in this step. All the following analyses are based on unbiased MD
simulations.
</p><!-- l. 241 --><p class='noindent'>
                                                                                      
                                                                                      
</p>
<h4 class='subsectionHead'><span class='titlemark'>3.5.2    </span> <a id='x8-490003.5.2'></a>Identifying the allosteric network with CARDS</h4>
<!-- l. 242 --><p class='noindent'>To determine how the GPCR- and GDP-binding regions communicate with one another, we applied
the CARDS<span class='cite'> [<a href='mainli5.html#XSingh:2017hh'>90</a>]</span> methodology to simulations of the GDP-bound state of G<span class='cmmi-12'>α</span>q. CARDS measures
communication between every pair of dihedrals via both correlated changes in structural
motions and dynamical behavior. Structural states are captured by discretizing backbone <span class='cmmi-12'>ϕ</span>
and <span class='cmmi-12'>ψ </span>dihedrals into two structural states (cis and trans), while side-chain <span class='cmmi-12'>χ </span>angles are
placed into three states (gauche+, gauche-, and trans). Every dihedral is also parsed into
dynamical states, capturing whether the dihedral is stable in a single state (ordered), or rapidly
transitioning between multiple states (disordered). These dynamical states are identified using two
kinetic signatures of dihedral motion: the average time a dihedral persists in a structural
state (an ordered timescale), and the typical timescale for transitions between structural
states (a disordered timescale). Parsing into dynamical states utilizes a two-step process by
(i) calculating the distribution or ordered and disordered times from the simulations and
(ii) assigning each period of time between two consecutive transitions into ordered and
disordered states based on which distribution the time between two transitions is most consistent
with.
</p><!-- l. 244 --><p class='noindent'>From these states, a holistic communication (<span class='cmmi-12'>I</span><sub><span class='cmmi-8'>H</span></sub><span class='cmr-12'>(</span><span class='cmmi-12'>X,Y </span><span class='cmr-12'>)</span>) is computed for every pair of dihedrals <span class='cmmi-12'>X </span>and
<span class='cmmi-12'>Y </span>:
</p>
<table class='equation'><tr><td><a id='x8-49001r9'></a>
<center class='math-display'>
                                                                                      
                                                                                      
<img src='main18x.png' alt='            ---------- ----------  ----------  ----------
IH(X, Y ) = Iss(X, Y ) + Idd(X, Y ) + Ids(X, Y ) + Isd(X, Y)
' class='math-display' /></center></td><td class='equation-label'>(3.9)</td></tr></table>
<!-- l. 248 --><p class='nopar'>
where <span class='overline'><span class='cmmi-12'>I</span><sub><span class='cmmi-8'>ss</span></sub><span class='cmr-12'>(</span><span class='cmmi-12'>X,Y </span><span class='cmr-12'>)</span></span>is the normalized mutual information between the structure (i.e., rotameric state) of
dihedral <span class='cmmi-12'>X </span>and the structure of dihedral <span class='cmmi-12'>Y </span>, <span class='overline'><span class='cmmi-12'>I</span><sub><span class='cmmi-8'>ds</span></sub><span class='cmr-12'>(</span><span class='cmmi-12'>X,Y </span><span class='cmr-12'>)</span></span> is the normalized mutual information
between the structure of dihedral <span class='cmmi-12'>X </span>and the dynamical state of dihedral <span class='cmmi-12'>Y </span>, <span class='overline'><span class='cmmi-12'>I</span><sub><span class='cmmi-8'>s</span></sub><span class='cmmi-12'>d</span></span><span class='cmr-12'>(</span><span class='cmmi-12'>X,Y </span><span class='cmr-12'>) </span>is the
normalized mutual information between the dynamical state of dihedral <span class='cmmi-12'>X </span>and the structure of
dihedral <span class='cmmi-12'>Y </span>, and <span class='overline'><span class='cmmi-12'>I</span><sub><span class='cmmi-8'>dd</span></sub><span class='cmr-12'>(</span><span class='cmmi-12'>X,Y </span><span class='cmr-12'>)</span></span> is the normalized mutual information between the dynamical
state of dihedral <span class='cmmi-12'>X </span>and the dynamical state of dihedral <span class='cmmi-12'>Y </span>. The Mutual Information (<span class='cmmi-12'>I</span>)
is
</p>
<table class='equation'><tr><td><a id='x8-49002r10'></a>
<center class='math-display'>
<img src='main19x.png' alt='             ∑  ∑     P(x, y)
I(X, Y ) = −        ----------
             x∈X y∈Y P (x)P (y)
' class='math-display' /></center></td><td class='equation-label'>(3.10)</td></tr></table>
<!-- l. 252 --><p class='nopar'>
where <span class='cmmi-12'>x </span><span class='cmsy-10x-x-120'>∈ </span><span class='cmmi-12'>X </span>refers to the set of possible states that dihedral <span class='cmmi-12'>X </span>can adopt, <span class='cmmi-12'>p</span><span class='cmr-12'>(</span><span class='cmmi-12'>x</span><span class='cmr-12'>) </span>is the probability that
dihedral <span class='cmmi-12'>X </span>adopts state <span class='cmmi-12'>x</span>, and <span class='cmmi-12'>p</span><span class='cmr-12'>(</span><span class='cmmi-12'>x,y</span><span class='cmr-12'>) </span>is the joint probability that dihedral <span class='cmmi-12'>X </span>adopts state <span class='cmmi-12'>x </span>and
                                                                                      
                                                                                      
dihedral <span class='cmmi-12'>Y </span>adopts state <span class='cmmi-12'>y</span>. Normalized mutual information is computed using the maximum possible
mutual information for any specific mode of communication.
</p><!-- l. 255 --><p class='noindent'>From the pairwise correlation for every dihedral-pair, we extracted how much each individual residue
communicates with a target site of interest via bootstrapping with 10 random samples with
replacement. After locating the group of residues communicating most strongly with a specific target
site, we set this newly identified group as the new target site; The iteration of this process
allows us to identify a pathway of communication from one region of interest to another.
Here, we set the GPCR contact sites as our initial target sites. We then iteratively used this
approach to identify pathways connecting these contact sites with the GDP-binding site of
G<span class='cmmi-12'>α</span>q.
</p><!-- l. 257 --><p class='noindent'>
</p>
<h4 class='subsectionHead'><span class='titlemark'>3.5.3    </span> <a id='x8-500003.5.3'></a>Markov state model construction</h4>
<!-- l. 258 --><p class='noindent'>We clustered G<span class='cmmi-12'>α</span>q conformations and GDP binding states separately and combined the assignments to
build a Markov State Model using MSMbuilder <span class='cite'> [<a href='mainli5.html#XBeauchamp:2011he'>230</a>, <a href='mainli5.html#XBowman:2009cb'>234</a>]</span> and enspara<span class='cite'> [<a href='mainli5.html#Xporter2018enspara'>39</a>]</span>. First, we clustered
protein conformations into 5040 states using a hybrid k-center/k-medoids method with 1.8 Å cutoff.
Then we clustered the GDP-binding state into 321 states using the automatic partitioning algorithm
(APM)<span class='cite'> [<a href='mainli5.html#XSheong:2015jd'>235</a>]</span> with a residence time of 2 ns. By combining the assignments from protein
conformations and the GDP-binding states, we obtained a total of 221,965 states. The
implied timescales of this MSM show Markovian behavior with a lag time of 5 ns<span class='cite'> [<a href='mainli5.html#XSwope:2004gx'>236</a>]</span>.
(Figure <a href='mainap2.html#x15-140009r5'>B.5<!-- tex4ht:ref: fig:ch4-fig2-supp5  --></a>). Analyses of distances, angles, and dihedrals of interest were carried out using
bootstrapping with ten random samples, with replacement. Results were insensitive to varying the
number of bootstrapped samples between 5 and 30. Histograms were generated using 100
bins.
                                                                                      
                                                                                      
</p><!-- l. 260 --><p class='noindent'>
</p>
<h4 class='subsectionHead'><span class='titlemark'>3.5.4    </span> <a id='x8-510003.5.4'></a>Quantifying conformational disorder</h4>
<!-- l. 261 --><p class='noindent'>The disorder of every residue was measured by computing Shannon entropy<span class='cite'> [<a href='mainli5.html#XShannon:1948iy'>237</a>]</span> of each dihedral as
they are natural degrees of freedom for describing protein dynamics. Shannon entropy (<span class='cmmi-12'>H</span>) is defined
as
</p>
<table class='equation'><tr><td><a id='x8-51001r11'></a>
<center class='math-display'>
<img src='main20x.png' alt='           ∑
H (X ) = −     p(x)log(p(x))
           x∈X
' class='math-display' /></center></td><td class='equation-label'>(3.11)</td></tr></table>
<!-- l. 265 --><p class='nopar'>
where <span class='cmmi-12'>x </span><span class='cmsy-10x-x-120'>∈ </span><span class='cmmi-12'>X </span>refers to the set of possible states that dihedral <span class='cmmi-12'>X </span>can adopt and <span class='cmmi-12'>p</span><span class='cmr-12'>(</span><span class='cmmi-12'>x</span><span class='cmr-12'>) </span>is the probability
that dihedral <span class='cmmi-12'>X </span>adopts state <span class='cmmi-12'>x</span>. Dihedral angles were calculated using MDTraj (McGibbon et al., 2015)
and assigned to discrete rotameric states as described above using CARDS. The entropy of a single
residue was computed by summing up the entropies of its dihedrals, and normalized by the residue’s
maximum possible Shannon entropy. This maximum possible Shannon entropy, using a flat
                                                                                      
                                                                                      
distribution for the appropriate number of bins, is referred to as the ’channel capacity’ and has been
used to normalize other information-theoretic metrics<span class='cite'> [<a href='mainli5.html#XSingh:2017hh'>90</a>]</span>. Summing entropies within a residue
establishes an upper bound on the degree of motion for a single residue, while ignoring intra-residue
correlations between dihedrals.
</p><!-- l. 268 --><p class='noindent'>
</p>
<h4 class='subsectionHead'><span class='titlemark'>3.5.5    </span> <a id='x8-520003.5.5'></a>Identification of the rate-limiting step for GDP release</h4>
<!-- l. 269 --><p class='noindent'>We used transition path theory (TPT) <span class='cite'> [<a href='mainli5.html#XNoe:2009en'>208</a>, <a href='mainli5.html#XWeinan:2006vr'>209</a>]</span> to find the highest flux paths from the bound state
to the unbound state<span class='cite'> [<a href='mainli5.html#XMetzner:2009jl'>238</a>]</span>. The bound state was defined as all clusters that satisfied two criteria: (i)
GDP is within 6 Å of the backbone atoms of Lys52<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>1</span></sup>-Thr54<sup><span class='cmmi-8'>G.H</span><span class='cmr-8'>1</span><span class='cmmi-8'>.</span><span class='cmr-8'>1</span></sup>, and (ii) GDP has an RMSD
<span class='cmmi-12'>&lt; </span><span class='cmr-12'>0</span><span class='cmmi-12'>.</span><span class='cmr-12'>5</span><span class='cmmi-12'>Å</span> to its crystallographic conformation. The unbound state was defined as all clusters with GDP
<span class='cmmi-12'>&gt; </span><span class='cmr-12'>55</span><span class='cmmi-12'>Å</span> from the binding pocket. The rate-limiting step was identified by finding the bottleneck in the
highest flux paths. To obtain this, we first calculate the flux between states <span class='cmmi-12'>i </span>and <span class='cmmi-12'>j </span>along any possible
unbinding path using
</p>
<table class='equation-star'><tr><td>
                            <span class='cmmi-12'>f</span><span class='cmr-12'>(</span><span class='cmmi-12'>x</span><span class='cmr-12'>) = </span><img src='main21x.png' alt='(|
|{ πiq− Tijq+ ,  if i ⁄= j
     i    j
||(
  0,           if i = j' class='left' align='middle' />
                                                                                      
                                                                                      
</td></tr></table>
<!-- l. 276 --><p class='nopar'>
where <span class='cmmi-12'>q</span><sup><span class='cmr-8'>+</span></sup> is the committor probability from the bound to the unbound state, and <span class='cmmi-12'>q</span><sub>
<span class='cmmi-8'>i</span></sub><sup><span class='cmsy-8'>−</span></sup> is <span class='cmr-12'>1 </span><span class='cmsy-10x-x-120'>− </span><span class='cmmi-12'>q</span><sub>
<span class='cmmi-8'>i</span></sub><sup><span class='cmr-8'>+</span></sup>; <span class='cmmi-12'>π</span><sub>
<span class='cmmi-8'>i</span></sub>
is the weighted probability, and <span class='cmmi-12'>T</span><sub><span class='cmmi-8'>ij</span></sub> is the transition matrix. The highest flux paths can
be identified by maximizing the fluxes between the bound states and the unbound states
using
</p>
<table class='equation'><tr><td><a id='x8-52001r12'></a>
<center class='math-display'>
<img src='main22x.png' alt='c(w ) = min (f +ilil+1|l = 1...nw  − 1)
' class='math-display' /></center></td><td class='equation-label'>(3.12)</td></tr></table>
<!-- l. 281 --><p class='nopar'>
where <span class='cmmi-12'>i</span><sub><span class='cmmi-8'>l</span></sub> are intermediate states. From this, the slowest step was extracted as the minimum flux step of
the highest flux release pathway.
</p><!-- l. 284 --><p class='noindent'>
</p>
                                                                                      
                                                                                      
<h3 class='sectionHead'><span class='titlemark'>3.6    </span> <a id='x8-530003.6'></a>Acknowledgements</h3>
<!-- l. 285 --><p class='noindent'>We thank TE Frederick and TD Todd for their helpful discussion and insight. We are grateful to the
Folding@home users for computing resources.
                                                                                      
                                                                                      
</p>
<!-- l. 3 --><div class='crosslinks'><p class='noindent'></p></div>
 
</body></html>