<?xml version="1.0" encoding="iso-8859-1" ?> 
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1 plus MathML 2.0//EN" 
"http://www.w3.org/Math/DTD/mathml2/xhtml-math11-f.dtd" > 
<html xmlns="http://www.w3.org/1999/xhtml"  
> 
<head><title>8 Conclusions</title> 
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" /> 
<meta name="generator" content="TeX4ht (https://tug.org/tex4ht/)" /> 
<meta name="originator" content="TeX4ht (https://tug.org/tex4ht/)" /> 
<!-- html,xhtml,2,mathml --> 
<meta name="src" content="main.tex" /> 
<link rel="stylesheet" type="text/css" href="main.css" /> 
 
<script type="text/javascript" 
src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=MML_HTMLorMML" 
></script> 
 
<script async defer src="http://hypothes.is/embed.js"></script><link rel="stylesheet" type="text/css" href="css/scale.css" /> 
<script type="text/javascript" src="js/fw.js"></script> 
<script type="text/javascript" src="js/sidebar.js"></script> 
<script type="text/javascript" src="js/bib.js"></script></head><body 
>
<!--l. 3--><div class="crosslinks"><p class="noindent">[<a 
href="mainap1.html" >next</a>] [<a 
href="mainch7.html" >prev</a>] [<a 
href="mainch7.html#tailmainch7.html" >prev-tail</a>] [<a 
href="#tailmainch8.html">tail</a>] [<a 
href="main.html#mainch8.html" >up</a>] </p></div>
<h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;8</span><br /><a 
 id="x12-1300008"></a>Conclusions</h2>
<div class="chapterTOCS">
&#x00A0;<span class="sectionToc" >8.1 <a 
href="mainch8.html#x12-1310008.1">Main findings</a></span>
&#x00A0;<span class="sectionToc" >8.2 <a 
href="mainch8.html#x12-1320008.2">Future directions</a></span>
</div>
<h3 class="sectionHead"><span class="titlemark">8.1    </span> <a 
 id="x12-1310008.1"></a>Main findings</h3>
<!--l. 11--><p class="noindent" >This thesis demonstrates how to measure allosteric communication in proteins from long-timescale
MD simulations, and illustrates how insight from allosteric networks can explain fundamental protein
behaviors and open up new therapeutic opportunities. This work also highlights the utility of protein
conformational ensembles and the tremendous scientific value encoded within protein energy
landscapes. The data presented provides further support to the hypothesis that a protein&#8217;s thermal
fluctuations contain all of its functionally relevant states.
</p><!--l. 13--><p class="noindent" >In <span 
class="ptmb7t-x-x-120">Chapter </span><a 
href="mainch2.html#x6-140002"><span 
class="ptmb7t-x-x-120">2</span><!--tex4ht:ref: ch:CARDS --></a>, the CARDS methodology is described as a means to integrate concerted structural and
disorder-mediated correlations into a holistic view of allosteric communication. Applications of
CARDS to the Catabolite Activator Protein (CAP), shown experimentally to undergo allosteric
coupling via conformational entropy, are described. Specifically, we showed that examining the
coupling of every residue to a known cAMP binding site naturally highlights regions of the protein
that are known to be impacted by cAMP binding. Decomposing correlations into disorder-mediated
and purely structural components demonstrates an important role for disorder-mediated coupling in
the absence of concerted structural changes. Our global communication metric also provides a means
to identify important functional sites without foreknowledge of their existence and locations. I
expect CARDS to be of great utility for understanding allostery in systems where it is
already known to occur, as well as for predicting allostery in systems where it has yet to be
observed.
</p><!--l. 15--><p class="noindent" >In <span 
class="ptmb7t-x-x-120">Chapter </span><a 
href="mainch3.html#x7-320003"><span 
class="ptmb7t-x-x-120">3</span><!--tex4ht:ref: ch:GDPrelease --></a>, we describe the complete process of G protein activation and GDP release, specifically focusing
on G<!--l. 15--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B1;</mi></math>q,
which has the slowest dissociation rate. These results reveal a previously unobserved intermediate that
defines the rate-limiting step for GDP release and, ultimately, G protein activation. The model
                                                                                      
                                                                                      
synthesizes a wealth of experimental data and previous analyses, and builds upon decades of literature
to create a single, unified model. We highlight roles in the allosteric network for regions of known
importance and identify new regions of value in the allosteric network coupling the receptor- and
nucleotide-binding sites. The consistency of our model with a wide variety of structural and biochemical
data suggests that it is a promising foundation for future efforts to understand the determinants of
GPCR-G<!--l. 15--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B1;</mi></math>
interaction specificity, how mutations cause aberrant signaling and disease, and how small molecule inhibitors
modulate G<!--l. 15--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B1;</mi></math>
activation. The model also adds weight to the growing appreciation for the fact that a protein&#8217;s
spontaneous fluctuations encode considerable information about its functional dynamics. I hope this
approach and combination of analyses will prove valuable for understanding other slow
conformational changes and unbinding processes.
</p><!--l. 17--><p class="noindent" >Described in <span 
class="ptmb7t-x-x-120">Chapter </span><a 
href="mainch4.html#x8-530004"><span 
class="ptmb7t-x-x-120">4</span><!--tex4ht:ref: ch:vp35pocket --></a> is a cryptic allosteric site in the IID of the Ebola VP35 protein that provides a
new therapeutic opportunity against this essential viral protein. We used adaptive sampling
simulations to access more of the ensemble of conformations that VP35 adopts, uncovering an
unanticipated cryptic pocket. Application of CARDS to these simulations suggested that the cryptic
pocket is allosterically coupled to the blunt end-binding interface and, therefore, could modulate
biologically-important interactions. Subsequent experiments highlighted that fluctuations within the
folded state of the IID expose two buried cysteines that line the proposed cryptic pocket to solvent.
Moreover, covalently modifying these cysteines to stabilize the open form of the cryptic pocket
allosterically disrupts binding to dsRNA blunt ends by at least 5-fold. Therefore, it may be
possible to attenuate the impact of viral replication and restrict pathogenicity by designing
small molecules to target the cryptic allosteric site we report here. More generally, these
results speak to the power of simulations to provide simultaneous access to both hidden
conformations and dynamics with atomic resolution. Thus I hope this demonstrate the potential
of simulations as a means to uncover unanticipated features of proteins&#8217; conformational
ensembles, such as cryptic pockets and allostery, providing a foundation for the design of further
                                                                                      
                                                                                      
experiments. We anticipate such simulations will enable the discovery of cryptic pockets and
cryptic allosteric sites in other proteins, particularly those that are currently considered
undruggable.
</p><!--l. 19--><p class="noindent" >Chapter <a 
href="mainch5.html#x9-680005">5<!--tex4ht:ref: ch:chapter6 --></a> and <a 
href="mainch6.html#x10-880006">6<!--tex4ht:ref: ch:covid19pockets --></a> highlight the power of Folding@home to tackle a myriad of biological problems, both
fundamental and with potential relevance to therapeutic design. Work in <span 
class="ptmb7t-x-x-120">Chapter </span><a 
href="mainch5.html#x9-680005"><span 
class="ptmb7t-x-x-120">5</span><!--tex4ht:ref: ch:chapter6 --></a> revealed that
SARS-CoV-2 N protein undergoes phase separation with RNA when reconstituted in vitro. In this
work we propose a model where a single-genome condensate forms through N protein gRNA
interaction, driven by a small number of high-affinity sites. This (meta)-stable single-genome
condensate undergoes subsequent maturation, leading to virion assembly. In this model,
condensate-associated N proteins are in exchange with a bulk pool of soluble N protein, such that the
interactions that drive compaction are heterogeneous and dynamic. The model provides a
physical mechanism in good empirical agreement with data for N protein oligomerization and
assembly.
</p><!--l. 21--><p class="noindent" >In <span 
class="ptmb7t-x-x-120">Chapter </span><a 
href="mainch6.html#x10-880006"><span 
class="ptmb7t-x-x-120">6</span><!--tex4ht:ref: ch:covid19pockets --></a> we describe how the excascale power of Folding@home allows us to hunt for
druggable opportunities throughout the entire SARS-CoV-2 proteome. The pandemic caused by
SARS-CoV-2 necessitated a call-to-arms; a call that over a million citizen-scientists answered to
generate 0.1 seconds of simulation data. We find that spike proteins have a strong trade-off
between making ACE2 binding interfaces accessible to infiltrate cells and conformationally
masking epitopes to subvert immune responses. These simulations also provide an atomically
detailed roadmap for targeting proteins for vaccines and antivirals. We describe a number of
cryptic pockets that we identify throughout the proteome of SARS-CoV-2, with more to be
described as they are discovered. For each protein system in Table <a 
href="mainch6.html#x10-94003r1">6.1<!--tex4ht:ref: tab:covid-systems-table1 --></a>, an extraordinary
amount of sampling has led to the generation of a quantitative map of its conformational
landscape.
</p><!--l. 23--><p class="noindent" >Finally, in <span 
class="ptmb7t-x-x-120">Chapter </span><a 
href="mainch7.html#x11-1060007"><span 
class="ptmb7t-x-x-120">7</span><!--tex4ht:ref: ch:chapter2 --></a> we demonstrate how molecular simulations, when integrated with experiments, can be used
to study the biophysical impact of mutations, such as those that grant Ceftazidime Resistance to the CTX-M
                                                                                      
                                                                                      
<!--l. 23--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
(P167S and D240G). However, these mutations actually undergo negative epistasis with one
another. Here we show that while P167S and D240G individually grant CTX-M ceftazidime
resistance, the double mutant P167S/D240G displays a wild-type behavior. The results
presented here suggest that conformational heterogeneity, particularly in active-site
loops, plays an important role in the substrate specificity and evolutionary capacity of
<!--l. 23--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamases.
The P167S and D240G mutants readily sample multiple conformations and yet are
more stable than P167S/D240G, which samples fewer conformations. Crystallography
and molecular dynamics results show that the CTX-M acyl-enzyme complex exists in
equilibrium between inactive and active conformations and that the P167S and D240G
variants have a higher probability of adopting active conformations. Taken together, our data
suggests that the P167S and D240G substitutions promote an open conformation of the
<!--l. 23--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
that creates access for ceftazidime and allows Glu166 to sample conformations
consistent with deacylation, whereas the WT and P167S/D240G mutant exhibit a closed
<!--l. 23--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
conformation that constrains access for ceftazidime and prevents Glu166 from efficiently coordinating
water for deacylation.
</p><!--l. 25--><p class="noindent" >
</p>
<h3 class="sectionHead"><span class="titlemark">8.2    </span> <a 
 id="x12-1320008.2"></a>Future directions</h3>
<!--l. 26--><p class="noindent" >Altogether, this thesis highlights the importance of understanding allostery in biological systems. Both
in scale and approach, we are entering a new phase of using molecular dynamics simulations to
                                                                                      
                                                                                      
understand biophysics; the complex task of simulating an organism&#8217;s entire proteome could become
more commonplace. It is worth speculating what the future holds for studies of protein dynamics,
allosteric communication, and the role of conformational entropy in biological function and
disease.
</p><!--l. 28--><p class="noindent" >Methodologically, CARDS is already able to measure allosteric communication in a holistic manner,
but there are many new methodological enhancements through which we can obtain an increased
understanding of allosteric communication. As previously discussed, using a rotameric library might
improve upon identifying significant allosteric communication beyond noise, and using a different a
multi-exponential distribution to compute probabilities of ordered and disordered regimes might
improve detection of dynamical states.
</p><!--l. 30--><p class="noindent" >Beyond previously discussed avenues of improvements, construction of our allosteric network in
chapter <a 
href="mainch3.html#x7-320003">3<!--tex4ht:ref: ch:GDPrelease --></a> is aided by the knowledge that the GPCR-binding site sends a signal to the GDP-binding
site<span class="cite">&#x00A0;[<a 
href="mainli4.html#XOldham:2006ui">168</a>,&#x00A0;<a 
href="mainli4.html#XOldham:2008cr">169</a>,&#x00A0;<a 
href="mainli4.html#XJohnston:2007ir">170</a>]</span>. While ample evidence exists to indicate that coupling occurs in both directions,
CARDS does not explicitly describe directionality in pairwise communication. This kind of
directional information has immense value in less well-studied systems. Hence, there is value in
identifying an alternative metric to mutual information that specifically measures a directional transfer
of information. Additionally, CARDS extracts data directly from simulation datasets, rather than
an MSM. As such, CARDS is inherently limited by the sampling bias that may exist in
a dataset, measuring stronger correlations between residues in states that are closer to a
trajectory&#8217;s starting configuration. While computing structural-correlations is already possible
using MSMs, it is difficult to assign dynamical states to MSMs. Therefore it is valuable to
develop frameworks to extract kinetic signatures of disorder from MSMs that can be used to
assign dynamical states. Then correlated motions will be able to be extracted from MSMs
trivially.
</p><!--l. 32--><p class="noindent" >Along with developments with CARDS, there are a number of questions surrounding allostery in
G protein signalling that remain unexplored. While we highlight the potential utility of
                                                                                      
                                                                                      
our allosteric network in chapter <a 
href="mainch3.html#x7-320003">3<!--tex4ht:ref: ch:GDPrelease --></a>, it is worth noting that an allosteric inhibitor of
G<!--l. 32--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B1;</mi></math>q
already exists, YM-254890 (YM)<span class="cite">&#x00A0;[<a 
href="mainli4.html#XNishimura:2010gt">203</a>]</span>. This depsipeptide binds at the hinge between the two
domains and prevents G protein activation by trapping the protein in a GDP bound state, preventing
GDP dissociation. However, the mechanism of inhibition, efficacious even in cells<span class="cite">&#x00A0;[<a 
href="mainli4.html#XOnken:2018gz">492</a>]</span>,
remains unclear. Structural models suggest that the domain-opening motion of G proteins is
inhibited by YM, but simulation of large peptide-protein complexes, particularly at the scale of
heterotrimeric G proteins, remains a challenge. Learning the allosteric mechanism of G protein
inhibition by YM might inform chemical strategies to design a simplified peptide analog,
as well as inform design principles for the general design of highly-selective G protein
inhibitors.
</p><!--l. 34--><p class="noindent" >Building off the allosteric networks described in chapters <a 
href="mainch3.html#x7-320003">3<!--tex4ht:ref: ch:GDPrelease --></a> and <a 
href="mainch4.html#x8-530004">4<!--tex4ht:ref: ch:vp35pocket --></a>, it is worth exploring if protein
homologs have conserved allosteric networks. For example, many G protein isoforms have similar
structures<span class="cite">&#x00A0;[<a 
href="mainli4.html#XFlock:2015dj">171</a>]</span>, but different behaviors ranging from their effector targets to their GDP dissociation
kinetics. Given previous success studying different isoforms of a protein family<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xporter_conformational_2020">10</a>]</span>, it may be that
the differences in conformational landscapes encode similarities and differences in allosteric networks.
Assuming a &#8220;core&#8221; allosteric network exists across all G protein isoforms, the differences between
them may be all the more interesting as they might provide a basis for regions that govern
isoform-specific behaviors.
</p><!--l. 36--><p class="noindent" >Going beyond primarily-simulation studies, it is exciting to enter an era where microsecond-to-millisecond
dynamics can be explored and integrated with experiments. While simulations already provide
predictive models for experimental measurements (see chapter <a 
href="mainch3.html#x7-320003">3<!--tex4ht:ref: ch:GDPrelease --></a>, <a 
href="mainch4.html#x8-530004">4<!--tex4ht:ref: ch:vp35pocket --></a>, <a 
href="mainch5.html#x9-680005">5<!--tex4ht:ref: ch:chapter6 --></a> and <a 
href="mainch7.html#x11-1060007">7<!--tex4ht:ref: ch:chapter2 --></a>), there are opportunities to
further integrate with experiments that directly measure conformational dynamics, such as
single-molecule studies or Nuclear Magnetic Resonance (NMR) spectroscopy. NMR is able to observe
major and minor conformational states and obtain thermodynamic and kinetic information at
residue-level resolution. It provides a powerful methodology to identify and characterize excited states
in a protein&#8217;s conformational landscape. However, one limitation of NMR is that it remains difficult to
                                                                                      
                                                                                      
characterize these excited states or to identify the motions that may drive the equilibrium
motions (called &#8220;exchange processes&#8221;) observed in NMR spectra. MD provides a powerful
complement to these limitations, with the potential to characterize the motions underlying
conformational equilibria and the participating residues. NMR and MD together may be
a powerful means by which protein conformational landscapes can be more completely
characterized. As we enter an era of exascale computing, it is exciting to consider the ways in
these kinds of integrated biophysical approaches may help comprehend complex biological
phenomena.
                                                                                      
                                                                                      
</p>
<!--l. 227--><div class="crosslinks"><p class="noindent">[<a 
href="mainap1.html" >next</a>] [<a 
href="mainch7.html" >prev</a>] [<a 
href="mainch7.html#tailmainch7.html" >prev-tail</a>] [<a 
href="mainch8.html" >front</a>] [<a 
href="main.html#mainch8.html" >up</a>] </p></div>
<!--l. 227--><p class="noindent" ><a 
 id="tailmainch8.html"></a></p> 
</body></html> 
