<?xml version="1.0" encoding="iso-8859-1" ?> 
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1 plus MathML 2.0//EN" 
"http://www.w3.org/Math/DTD/mathml2/xhtml-math11-f.dtd" > 
<html xmlns="http://www.w3.org/1999/xhtml"  
> 
<head><title>7 Antagonism between substitutions in &#x03B2;-lactamase explains a path not taken in the evolution of
bacterial drug resistance</title> 
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" /> 
<meta name="generator" content="TeX4ht (https://tug.org/tex4ht/)" /> 
<meta name="originator" content="TeX4ht (https://tug.org/tex4ht/)" /> 
<!-- html,xhtml,2,mathml --> 
<meta name="src" content="main.tex" /> 
<link rel="stylesheet" type="text/css" href="main.css" /> 
 
<script type="text/javascript" 
src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=MML_HTMLorMML" 
></script> 
 
<script async defer src="http://hypothes.is/embed.js"></script><link rel="stylesheet" type="text/css" href="css/scale.css" /> 
<script type="text/javascript" src="js/fw.js"></script> 
<script type="text/javascript" src="js/sidebar.js"></script> 
<script type="text/javascript" src="js/bib.js"></script></head><body 
>
<!--l. 3--><div class="crosslinks"><p class="noindent">[<a 
href="mainch8.html" >next</a>] [<a 
href="mainch6.html" >prev</a>] [<a 
href="mainch6.html#tailmainch6.html" >prev-tail</a>] [<a 
href="#tailmainch7.html">tail</a>] [<a 
href="main.html#mainch7.html" >up</a>] </p></div>
<h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;7</span><br /><a 
 id="x11-1060007"></a>Antagonism  between  substitutions  in
<!--l. 3--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
explains a path not taken in the evolution of bacterial drug resistance</h2>

<div class="chapterTOCS">

<br />&#x00A0;<span class="sectionToc" >7.1 <a 
href="mainch7.html#x11-1070007.1">Abstract</a></span>

<br />&#x00A0;<span class="sectionToc" >7.2 <a 
href="mainch7.html#x11-1080007.2">Introduction</a></span>

<br />&#x00A0;<span class="sectionToc" >7.3 <a 
href="mainch7.html#x11-1090007.3">Results</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >7.3.1 <a 
href="mainch7.html#x11-1100007.3.1" id="QQ2-9-87">Ceftazidime resistance levels of P167S/D240G double mutant are reduced compared with single mutants.</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >7.3.2 <a 
href="mainch7.html#x11-1110007.3.2" id="QQ2-9-87">Antibiotic hydrolysis by the P167S/D240G double mutant is reduced compared with single mutants.</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >7.3.3 <a 
href="mainch7.html#x11-1120007.3.3" id="QQ2-9-87">P167S/D240G double mutant exhibits reduced stability compared with single mutants.</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >7.3.4 <a 
href="mainch7.html#x11-1130007.3.4" id="QQ2-9-87">Steady-state levels of the P167S/D240G enzyme in E. coli are reduced compared with single mutants.</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >7.3.5 <a 
href="mainch7.html#x11-1140007.3.5" id="QQ2-9-87">X-ray structures of P167S/D240G apo, E166A/D240G/CAZ, and E166A/P167S/D240G/CAZ acyl-enzyme complexes reveal alternate conformations of the Ω-loop.</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >7.3.6 <a 
href="mainch7.html#x11-1150007.3.6" id="QQ2-9-87">Molecular dynamics simulations reveal that conformational heterogeneity of the Ω-loop is greater in the single mutants than in the WT or double mutant.</a></span>

<br />&#x00A0;<span class="sectionToc" >7.4 <a 
href="mainch7.html#x11-1160007.4">Discussion</a></span>

<br />&#x00A0;<span class="sectionToc" >7.5 <a 
href="mainch7.html#x11-1170007.5">Methods</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >7.5.1 <a 
href="mainch7.html#x11-1180007.5.1" id="QQ2-9-87">Bacterial strains and plasmids.</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >7.5.2 <a 
href="mainch7.html#x11-1190007.5.3" id="QQ2-9-87">Site-directed mutagenesis.</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >7.5.3 <a 
href="mainch7.html#x11-1200007.5.3" id="QQ2-9-87">Minimum inhibitory concentration determinations.</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >7.5.4 <a 
href="mainch7.html#x11-1210007.5.4" id="QQ2-9-87">Immunoblotting.</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >7.5.5 <a 
href="mainch7.html#x11-1220007.5.5" id="QQ2-9-87">Protein purification.</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >7.5.6 <a 
href="mainch7.html#x11-1230007.5.6" id="QQ2-9-87">Determination of thermal stabilities.</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >7.5.7 <a 
href="mainch7.html#x11-1240007.5.7" id="QQ2-9-87">Steady-state enzyme kinetic parameters.</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >7.5.8 <a 
href="mainch7.html#x11-1250007.5.8" id="QQ2-9-87">Protein crystallization and structure determination.</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >7.5.9 <a 
href="mainch7.html#x11-1260007.5.9" id="QQ2-9-87">Molecular dynamics simulations.</a></span>

<br />&#x00A0;<span class="sectionToc" >7.6 <a 
href="mainch7.html#x11-1270007.6">Author contributions</a></span>

<br />&#x00A0;<span class="sectionToc" >7.7 <a 
href="mainch7.html#x11-1280007.7">Acknowledgments</a></span>

<br />&#x00A0;<span class="sectionToc" >7.8 <a 
href="mainch7.html#x11-1290007.8">Additional information.</a></span>
</div>
<!--l. 8--><p class="noindent" ><span 
class="ptmri7t-x-x-120">This chapter is adapted from the following publication:</span>
</p><!--l. 10--><p class="noindent" ><span 
class="ptmri7t-x-x-120">Brown, C.A., Hu, L., Sun, Z., Patel, M.P., Singh, S., Porter, J.R., Sankaran, B., Venkataram</span>
<span 
class="ptmri7t-x-x-120">Prasad, B.V.V., Bowman, G.R., Palzkill, T., Antagonism between substitutions in</span>
<!--l. 10--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math><span 
class="ptmri7t-x-x-120">-lactamase</span>
<span 
class="ptmri7t-x-x-120">explains a path not taken in the evolution of bacterial drug resistance, J.Biol., Chem., 2020,</span>
<span 
class="ptmri7t-x-x-120">doi:10.1074/jbc.RA119.012489</span><span class="cite">&#x00A0;[<a 
href="mainli4.html#Xbrown_antagonism_2020">437</a>]</span>
</p><!--l. 12--><p class="noindent" ><span 
class="ptmri7t-x-x-120">My role in parameterizing and running molecular simulations of the crystal structures in apo and</span>
<span 
class="ptmri7t-x-x-120">acyl-enzyme forms led to the data presented in figures </span><a 
href="#x11-115001r9"><span 
class="ptmri7t-x-x-120">7.9</span><!--tex4ht:ref: fig:ch2-fig9 --></a> <span 
class="ptmri7t-x-x-120">and </span><a 
href="#x11-115003r10"><span 
class="ptmri7t-x-x-120">7.10</span><!--tex4ht:ref: fig:ch2-fig10 --></a><span 
class="ptmri7t-x-x-120">, and appendix figures </span><a 
href="mainap5.html#x17-170001r1"><span 
class="ptmri7t-x-x-120">E.1</span><!--tex4ht:ref: fig:ch2-suppfig1 --></a><span 
class="ptmri7t-x-x-120">, </span><a 
href="mainap5.html#x17-170003r2"><span 
class="ptmri7t-x-x-120">E.2</span><!--tex4ht:ref: fig:ch2-suppfig2 --></a><span 
class="ptmri7t-x-x-120">,</span>
<a 
href="mainap5.html#x17-170005r3"><span 
class="ptmri7t-x-x-120">E.3</span><!--tex4ht:ref: fig:ch2-suppfig3 --></a><span 
class="ptmri7t-x-x-120">.</span>
</p>
<h3 class="sectionHead"><span class="titlemark">7.1    </span> <a 
 id="x11-1070007.1"></a>Abstract</h3>
<!--l. 18--><p class="noindent" >CTX-M <!--l. 18--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamases
are widespread in Gram-negative bacterial pathogens and provide resistance to the cephalosporin
cefotaxime but not to the related antibiotic ceftazidime. Nevertheless, variants have emerged that
confer resistance to ceftazidime. Two natural mutations, causing P167S and D240G substitutions in
the CTX-M enzyme, result in 10-fold increased hydrolysis of ceftazidime. Although the combination
of these mutations would be predicted to increase ceftazidime hydrolysis further, the P167S/D240G
combination has not been observed in a naturally occurring CTX-M variant. Here, using
recombinantly expressed enzymes, minimum inhibitory concentration measurements, steady-state
enzyme kinetics, and X-ray crystallography, we show that the P167S/D240G double mutant enzyme
exhibits decreased ceftazidime hydrolysis, lower thermostability, and decreased protein expression
levels compared with each of the single mutants, indicating negative epistasis. X-ray structures of
                                                                                      
                                                                                      
mutant enzymes with covalently trapped ceftazidime suggested that a change of an active-site
<!--l. 18--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
to an open conformation accommodates ceftazidime leading to enhanced catalysis.
10-<!--l. 18--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>s
molecular dynamics simulations further correlated -loop opening with catalytic activity. We
observed that the WT and P167S/D240G variant with acylated ceftazidime both favor a closed
conformation not conducive for catalysis. In contrast, the single substitutions dramatically
increased the probability of open conformations. We conclude that the antagonism is due to
restricting the conformation of the -loop. These results reveal the importance of conformational
heterogeneity of active-site loops in controlling catalytic activity and directing evolutionary
trajectories.
</p><!--l. 21--><p class="noindent" >
</p>
<h3 class="sectionHead"><span class="titlemark">7.2    </span> <a 
 id="x11-1080007.2"></a>Introduction</h3>
<!--l. 22--><p class="noindent" >Enzymes have evolved to catalyze reactions critical to the functioning of the cell<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xknowles_enzyme_1991">438</a>]</span>. Evolution of
enzyme function proceeds through the accumulation of amino acid substitutions that shape stability,
solubility, and catalytic activity, among other properties. How substitutions interact when combined
plays a key role in the trajectory of mutations that accumulate during evolution<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xbreen_epistasis_2012">439</a>,&#x00A0;<a 
href="mainli4.html#Xde_visser_empirical_2014">440</a>]</span>. For
example, amino acid substitutions can act additively on catalysis whereupon each substitution
increases activity, and upon combination, the increase in activity in the double mutant is the product of
the fold changes of the individual mutations<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xwells_additivity_1990">441</a>]</span>. Alternatively, combinations of substitutions are
often nonadditive where the double mutant has a greater activity or less activity than expected based
on the activity of the single mutants. Such nonadditive effects are termed epistasis and can
strongly influence the mutational pathways that are possible in the evolution of enzyme
                                                                                      
                                                                                      
function<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xweinreich_darwinian_2006">442</a>]</span>.
</p><!--l. 24--><p class="noindent" >Enzymes act by binding substrates and stabilizing transition states of reactions<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xknowles_enzyme_1991">438</a>]</span>. Toward this
end, conformational changes are often important, and flexible loops in the active site are a common
feature involved in enzyme function<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xdellus-gur_what_2013">443</a>,&#x00A0;<a 
href="mainli4.html#Xdoucet_flexibility_2009">444</a>,&#x00A0;<a 
href="mainli4.html#Xwhittier_conformational_2013">445</a>]</span>. Moreover, conformational dynamics have been
proposed to play an important role in protein evolvability<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xjames_conformational_2003">446</a>,&#x00A0;<a 
href="mainli4.html#Xtokuriki_protein_2009">447</a>]</span>. By this view, conformational
fluctuations can result in an enzyme adopting multiple structures, some of which have properties that
allow interactions with alternate ligands. These conformations may be rare in the ensemble of WT
structures, but mutations may shift the distribution toward alternate conformations that become
dominant in an evolved enzyme, thereby allowing for altered substrate specificity or new enzyme
functions to emerge on an enzyme scaffold<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xtokuriki_protein_2009">447</a>,&#x00A0;<a 
href="mainli4.html#Xpetrovic_conformational_2018">448</a>]</span>.
</p><!--l. 26--><p class="noindent" >Here, we address the role of epistasis and conformational diversity of active-site loops in the evolution of variants of the CTX-M
<!--l. 26--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase with a broadened
substrate specificity for <!--l. 26--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactam
antibiotics. <!--l. 26--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-Lactams
are the most frequently prescribed class of antibiotic worldwide, making up 65% of all
use<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xlivermore_-lactamase_2006">449</a>]</span>. However, bacterial resistance to these drugs is a growing problem, and
the most common mechanism of resistance is enzyme-mediated hydrolysis of the
<!--l. 26--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactam ring<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xfisher_bacterial_2005">450</a>]</span>. This hydrolysis
is catalyzed by various <!--l. 26--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamases,
which are divided into classes A&#8211;D based on primary amino acid sequence homology<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xfisher_bacterial_2005">450</a>,&#x00A0;<a 
href="mainli4.html#Xambler_standard_1991">451</a>]</span>.
</p><!--l. 28--><p class="noindent" >Class A <!--l. 28--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamases,
such as CTX-M, are widespread in Gram-negative bacteria and share a similar mechanism of catalysis but can
differ widely in substrate profile<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xbush_epidemiological_2011">452</a>,&#x00A0;<a 
href="mainli4.html#Xpalzkill_structural_2018">453</a>]</span>. These enzymes are serine hydrolases that hydrolyze the amide bond
in the <!--l. 28--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactam
ring via sequential acylation and deacylation steps. The conserved catalytic Ser70
residue is activated by Lys73 and Glu166 for attack on the carbonyl carbon to form an
                                                                                      
                                                                                      
acyl-enzyme intermediate. A catalytic water molecule is then activated by Glu166
for attack on the carbonyl of the covalent complex to deacylate the enzyme and release
the product (Fig. <a 
href="#x11-108001r1">7.1<!--tex4ht:ref: fig:ch2-fig1 --></a>)<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpalzkill_structural_2018">453</a>,&#x00A0;<a 
href="mainli4.html#Xmobashery_three_2009">454</a>,&#x00A0;<a 
href="mainli4.html#Xbonomo_kinetics_2007">455</a>]</span>. The reaction scheme and mechanism of serine
<!--l. 28--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamases
is shown in Fig. <a 
href="#x11-108001r1">7.1<!--tex4ht:ref: fig:ch2-fig1 --></a>.
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                      
                                                                                      
<a 
 id="x11-108001r1"></a>
                                                                                      
                                                                                      
<!--l. 32--><p class="noindent" ><img 
src="Chapter2/ch2-fig1.jpg" alt="PIC"  
width="252" height="252"  />
<a 
 id="x11-108002"></a>
<br /> </p><div class="caption" 
><span class="id">Figure&#x00A0;7.1: </span><span  
class="content"><!--l. 34--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-Lactamase
mechanism. A, reaction scheme for <!--l. 34--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi><mo 
class="MathClass-bin">-</mo></math>lactamase
where E represents <!--l. 34--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi><mo 
class="MathClass-bin">-</mo></math>lactamase,
ES    represents    the    enzyme-substrate    complex,    EAc    represents    the    acyl-enzyme
complex,  and  P  represents  product.  k1  and  k-1  are  the  rate  constants  for  association
and   dissociation   of   the   enzyme   substrate   complex,   and   k2   and   k3   are   the   rate
constants   for   acylation   and   deacylation,   respectively.    B,   schematic   illustration   of
<!--l. 34--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi><mo 
class="MathClass-bin">-</mo></math>lactamase
mechanism.       The       catalytic       Ser70       hydroxyl       group       is       activated       for
nucleophilic     attack     on     the     carbonyl     oxygen     of     the     amide     bond     of     the
<!--l. 34--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi><mo 
class="MathClass-bin">-</mo></math>lactam
by  an  active-site  residue  serving  as  a  general  base.  This  residue  is  viewed  as  either  Lys73
or  Glu166  acting  through  a  water  molecule.  This  leads  to  formation  of  the  acyl  enzyme
intermediate, which is subsequently deacylated by a water that is activated by Glu166 acting
as a base and resulting in free enzyme and the hydrolyzed product.</span></div><!--tex4ht:label?: x11-108001r7.2 -->
                                                                                      
                                                                                      
</div><hr class="endfigure" />
<!--l. 38--><p class="noindent" >CTX-M <!--l. 38--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamases are a family of
class A extended-spectrum <!--l. 38--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamases
that are so named because they efficiently hydrolyze the oxyimino-cephalosporin cefotaxime<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xbonnet_growing_2004">456</a>]</span> (Fig.
<a 
href="#x11-108003r2">7.2<!--tex4ht:ref: fig:ch2-fig2 --></a>). To date, more than 140 variants of the CTX-M enzymes have been identified<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xdandrea_ctx-m-type_2013">457</a>]</span>. CTX-M-14
<!--l. 38--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
has become a model system for studies of the structure and function of CTX-M
enzymes<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xchen_atomic_2005">458</a>,&#x00A0;<a 
href="mainli4.html#Xadamski_molecular_2015">459</a>,&#x00A0;<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>,&#x00A0;<a 
href="mainli4.html#Xpatel_drug-resistant_2017">461</a>]</span>.
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                      
                                                                                      
<a 
 id="x11-108003r2"></a>
                                                                                      
                                                                                      
<!--l. 42--><p class="noindent" ><img 
src="Chapter2/ch2-fig2.jpg" alt="PIC"  
width="252" height="252"  />
<a 
 id="x11-108004"></a>
<br />       </p><div class="caption" 
><span class="id">Figure&#x00A0;7.2:            </span><span  
class="content">Structures            of            antibiotics            and            CTX-M-14
<!--l. 44--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase.
A,      cephalothin.      R1      and      R2      denote      the      groups      that      differ      between
cephalosporins.     B,     cefotaxime.     C,     ceftazidime.     D,     structure     of     CTX-M-14
<!--l. 44--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
(PDB code 1YLT). The <!--l. 44--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
is       colored       green.       The       catalytic       Ser70       is       colored       orange.       Pro167
and     Asp240     are     colored     gray.     Note     that     Pro167     is     located     on     the
<!--l. 44--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop.</span></div><!--tex4ht:label?: x11-108003r7.2 -->
                                                                                      
                                                                                      
</div><hr class="endfigure" />
<!--l. 48--><p class="noindent" >CTX-M enzymes efficiently hydrolyze cefotaxime but not another commonly used
oxyimino-cephalosporin, ceftazidime (Fig. <a 
href="#x11-108003r2">7.2<!--tex4ht:ref: fig:ch2-fig2 --></a>A). Natural variants containing either the
P167S or D240G substitutions have emerged, however, that more efficiently hydrolyze
ceftazidime<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xdandrea_ctx-m-type_2013">457</a>,&#x00A0;<a 
href="mainli4.html#Xbonnet_effect_2003">462</a>,&#x00A0;<a 
href="mainli4.html#Xkimura_role_2004">463</a>,&#x00A0;<a 
href="mainli4.html#Xcanton_ctx-m_2012">464</a>]</span>. These two substitutions, when present, individually increase
k<!--l. 48--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>/K<!--l. 48--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math>
for ceftazidime hydrolysis by 10-fold, resulting in increased ceftazidime resistance for bacteria
containing the mutants<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xchen_atomic_2005">458</a>,&#x00A0;<a 
href="mainli4.html#Xpatel_drug-resistant_2017">461</a>]</span>. Multiple natural variants in the CTX-M family possess one of
these substitutions<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xdandrea_ctx-m-type_2013">457</a>]</span>.
</p><!--l. 50--><p class="noindent" >Pro167 resides in the <!--l. 50--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
that  forms  the  bottom  of  the  active  site  in  class  A
<!--l. 50--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamases
including CTX-M enzymes<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xchen_atomic_2005">458</a>,&#x00A0;<a 
href="mainli4.html#Xstrynadka_molecular_1992">465</a>]</span> (Fig. <a 
href="#x11-108003r2">7.2<!--tex4ht:ref: fig:ch2-fig2 --></a>D). It is adjacent to Glu166, which is
conserved and serves as a general base to activate a water molecule for deacylation of
<!--l. 50--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactam
substrates<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xstrynadka_molecular_1992">465</a>]</span>. The peptide bond preceding Pro167 is in a cis conformation
in CTX-M enzymes, which strongly influences the conformation of the
<!--l. 50--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
and the positioning of the Asn170 residue that hydrogen bonds to Glu166 and the deacylation water.
We previously used the CTX-M-14 enzyme as a model system to examine the structural changes
caused by the P167S substitution<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_drug-resistant_2017">461</a>]</span>. These studies revealed a large conformational change of the
<!--l. 50--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
that results in a larger active-site cavity to accommodate ceftazidime. This conformational change
required both the P167S substitution and the presence of acylated ceftazidime<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_drug-resistant_2017">461</a>]</span> In addition, the
structures showed that the conformational change is associated with a shift in the peptide bond
preceding residue 167 from cis to trans and that the P167S substitution was required for this shift.
Thus, the P167S substitution appears to cause increased ceftazidime hydrolysis through promoting a
                                                                                      
                                                                                      
conformational change to relieve steric restraints on catalysis.
</p><!--l. 52--><p class="noindent" >Chen et al.<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xchen_atomic_2005">458</a>]</span> previously determined the X-ray structure of the D240G mutant
enzyme, and anisotropic B-factor analysis revealed increased flexibility of the B3
<!--l. 52--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-strand
that forms one side of the CTX-M active site. The increased flexibility of the B3
<!--l. 52--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-stand
was proposed to allow access for the bulky side chain of ceftazidime.
</p><!--l. 54--><p class="noindent" >Despite the increase in ceftazidime hydrolysis and bacterial resistance resulting from each of the
substitutions, there has yet to be a CTX-M enzyme identified in clinical isolates that harbors both the
P167S and D240G mutations. Based on simple additivity, the combination of substitutions that each
increase hydrolysis by 10-fold would be expected to increased hydrolysis 100-fold relative to the WT
enzyme<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xwells_additivity_1990">441</a>]</span>. However, a P167S/D240G double mutant created by site-directed mutagenesis in a
CTX-M-3 enzyme background exhibited a loss of ceftazidime resistance, indicating an antagonist
effect and negative epistasis<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xnovais_mutational_2008">466</a>]</span>. The mechanism of this antagonism, however, was not
examined.
</p><!--l. 56--><p class="noindent" >Here, we show that the P167S/D240G double mutant displays decreased ceftazidime hydrolysis compared
with either of the single mutants, indicating antagonism. Further, X-ray structures of single and double
mutants as apoenzymes and acylated with ceftazidime show alternate open and closed conformations
of the <!--l. 56--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
that are associated with high and low activity. Finally, molecular dynamics simulations of the WT,
P167S, D240G, and P167S/D240G enzymes acylated with ceftazidime indicate that the
single substitutions dramatically increase the probability of open conformations of the
<!--l. 56--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop,
whereas the WT and P167S/D240G variant both favor a well-defined closed conformation not
favorable for catalysis. Taken together, the results suggest that the P167S/D240G double mutant has
not been observed in resistant clinical isolates because the combination results in decreased catalysis,
                                                                                      
                                                                                      
decreased stability, and therefore decreased fitness in the presence of ceftazidime for bacteria
containing this enzyme.
</p>
<h3 class="sectionHead"><span class="titlemark">7.3    </span> <a 
 id="x11-1090007.3"></a>Results</h3>
<!--l. 60--><p class="noindent" >
</p>
<h4 class="subsectionHead"><span class="titlemark">7.3.1    </span> <a 
 id="x11-1100007.3.1"></a>Ceftazidime resistance levels of P167S/D240G double mutant are reduced compared with
single mutants.</h4>
<!--l. 62--><p class="noindent" >The P167S and D240G substitutions have been observed in multiple CTX-M
<!--l. 62--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
variants and are associated with 10-fold increased ceftazidime hydrolysis<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xchen_atomic_2005">458</a>,&#x00A0;<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>]</span>.
Further, introduction of the P167S and D240G substitutions into the CTX-M-3
<!--l. 62--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
results in lower ceftazidime resistance than either of the single mutants<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xnovais_mutational_2008">466</a>]</span>. We extended these
findings to the CTX-M-14 model system by determining minimum inhibitory concentrations
(MICs)<a 
href="#x11-110001r1">7.1<!--tex4ht:ref: tab:ch2-tab1 --></a> for ceftazidime, cefotaxime, and cephalothin for Escherichia coli harboring WT and the
mutants (Table <a 
href="#x11-110001r1">7.1<!--tex4ht:ref: tab:ch2-tab1 --></a>). The results show that the P167S and D240G individual substitutions both result in
increased resistance to ceftazidime, whereas the P167S/D240G double mutant exhibits a loss of
ceftazidime resistance compared with either the P167S or D240G single mutants (Table
<a 
href="#x11-110001r1">7.1<!--tex4ht:ref: tab:ch2-tab1 --></a>)<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>,&#x00A0;<a 
href="mainli4.html#Xnovais_mutational_2008">466</a>]</span>. These data confirm the apparent incompatibility of the P167S and D240G
substitutions as first suggested by Novais et al. <span class="cite">&#x00A0;[<a 
href="mainli4.html#Xnovais_mutational_2008">466</a>]</span> and extends the findings to the CTX-M-14
                                                                                      
                                                                                      
enzyme background.
</p>
<div class="table">
                                                                                      
                                                                                      
<!--l. 64--><p class="noindent" ><a 
 id="x11-110001r1"></a></p><hr class="float" /><div class="float" 
>
                                                                                      
                                                                                      
<a 
 id="x11-110002"></a>
  <div class="caption" 
><span class="id">Table&#x00A0;7.1:   </span><span  
class="content">MICs   for   E.   coli   containing   CTX-M-14   wild   type,   mutants,   and   no
<!--l. 66--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi><mo 
class="MathClass-bin">-</mo></math>lactamase
control</span></div><!--tex4ht:label?: x11-110001r7.3 -->
<div class="tabular"> <table id="TBL-4" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
><colgroup id="TBL-4-1g"><col 
id="TBL-4-1" /></colgroup><colgroup id="TBL-4-2g"><col 
id="TBL-4-2" /></colgroup><colgroup id="TBL-4-3g"><col 
id="TBL-4-3" /></colgroup><colgroup id="TBL-4-4g"><col 
id="TBL-4-4" /></colgroup><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-1-"><td  style="text-align:left; white-space:nowrap;" id="TBL-4-1-1"  
class="td11">           </td><td colspan="3" style="text-align:left; white-space:nowrap;" id="TBL-4-1-2"  
class="td11"> <div class="multicolumn"  style="text-align:left; white-space:nowrap;">MIC (<!--l. 70--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>g/ml)</div>
</td></tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-2-"><td  style="text-align:left; white-space:nowrap;" id="TBL-4-2-1"  
class="td11">           </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-2-2"  
class="td11"> Cephalothin  </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-2-3"  
class="td11"> Cefotaxime  </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-2-4"  
class="td11"> Ceftazidime                                                                                                </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-3-"><td  style="text-align:left; white-space:nowrap;" id="TBL-4-3-1"  
class="td11"> pTP123             </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-3-2"  
class="td11"> 12                  </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-3-3"  
class="td11"> 0.0625          </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-3-4"  
class="td11"> 0.19                                                                                                             </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-4-"><td  style="text-align:left; white-space:nowrap;" id="TBL-4-4-1"  
class="td11"> CTX-M-14 wt  </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-4-2"  
class="td11"> <span 
class="cmmi-12">&#x003E;</span>256             </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-4-3"  
class="td11"> 1.5                </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-4-4"  
class="td11"> 0.75                                                                                                             </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-5-"><td  style="text-align:left; white-space:nowrap;" id="TBL-4-5-1"  
class="td11"> P167S               </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-5-2"  
class="td11"> <span 
class="cmmi-12">&#x003E;</span>256             </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-5-3"  
class="td11"> 0.375            </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-5-4"  
class="td11"> 12                                                                                                                </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-6-"><td  style="text-align:left; white-space:nowrap;" id="TBL-4-6-1"  
class="td11"> D240G              </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-6-2"  
class="td11"> <span 
class="cmmi-12">&#x003E;</span>256             </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-6-3"  
class="td11"> 1                   </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-6-4"  
class="td11"> 1.5                                                                                                               </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-7-"><td  style="text-align:left; white-space:nowrap;" id="TBL-4-7-1"  
class="td11"> P167S/D240G  </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-7-2"  
class="td11"> <span 
class="cmmi-12">&#x003E;</span>256             </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-7-3"  
class="td11"> 0.19              </td><td  style="text-align:left; white-space:nowrap;" id="TBL-4-7-4"  
class="td11"> 0.75                                                                                                             </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-8-"><td  style="text-align:left; white-space:nowrap;" id="TBL-4-8-1"  
class="td11">           </td></tr></table></div>
                                                                                      
                                                                                      
</div><hr class="endfloat" />
</div>
<h4 class="subsectionHead"><span class="titlemark">7.3.2    </span> <a 
 id="x11-1110007.3.2"></a>Antibiotic hydrolysis by the P167S/D240G double mutant is reduced compared with single
mutants.</h4>
<!--l. 82--><p class="noindent" >Although the P167S and D240G substitutions increase the catalytic efficiency
(k<!--l. 82--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>/K<!--l. 82--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math>)
for ceftazidime hydrolysis by <!--l. 82--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" > <mo 
class="MathClass-rel">~</mo></math>10-fold,
the activity of the P167S/D240G double mutant enzyme has not been examined <span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>,&#x00A0;<a 
href="mainli4.html#Xbonnet_effect_2003">462</a>,&#x00A0;<a 
href="mainli4.html#Xkimura_role_2004">463</a>,&#x00A0;<a 
href="mainli4.html#Xishii_biochemical_2007">467</a>]</span>.
Therefore, both WT and the double mutant CTX-M-14 enzymes were purified, and their kinetic
parameters were determined for hydrolysis of the oxyimino-cephalosporins cefotaxime and
ceftazidime, as well as cephalothin (Table <a 
href="#x11-111001r2">7.2<!--tex4ht:ref: tab:ch2-tab2 --></a>).
</p>
<div class="table">
                                                                                      
                                                                                      
<!--l. 84--><p class="noindent" ><a 
 id="x11-111001r2"></a></p><hr class="float" /><div class="float" 
>
                                                                                      
                                                                                      
<a 
 id="x11-111002"></a>
       <div class="caption" 
><span class="id">Table&#x00A0;7.2:            </span><span  
class="content">Enzyme            kinetic            parameters            of            CTX-M-14
<!--l. 86--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
and mutant enzymes</span></div><!--tex4ht:label?: x11-111001r7.3 -->
<div class="tabular"> <table id="TBL-5" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
><colgroup id="TBL-5-1g"><col 
id="TBL-5-1" /></colgroup><colgroup id="TBL-5-2g"><col 
id="TBL-5-2" /></colgroup><colgroup id="TBL-5-3g"><col 
id="TBL-5-3" /></colgroup><colgroup id="TBL-5-4g"><col 
id="TBL-5-4" /></colgroup><colgroup id="TBL-5-5g"><col 
id="TBL-5-5" /></colgroup><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-1-"><td  style="text-align:left; white-space:nowrap;" id="TBL-5-1-1"  
class="td11"> Enzyme           </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-1-2"  
class="td11"> Parameter                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-1-3"  
class="td11"> Cephalothin                                                                                                                                                     </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-1-4"  
class="td11"> Cefotaxime                                                                                                                                                      </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-1-5"  
class="td11"> Ceftazidime                                                                                                                                                               </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-2-"><td  style="text-align:left; white-space:nowrap;" id="TBL-5-2-1"  
class="td11"> CTX-M-14       </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-2-2"  
class="td11"> k<!--l. 91--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math> (s<!--l. 91--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">-</mo><mn>1</mn></mrow></msup 
></math>)                                                                                                                                                                                                                                                                                                                                                                                                                                                                               </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-2-3"  
class="td11"> 1400 <!--l. 91--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 38    </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-2-4"  
class="td11"> 161 <!--l. 91--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 9        </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-2-5"  
class="td11"> NDa                                                                                                                                                                           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-3-"><td  style="text-align:left; white-space:nowrap;" id="TBL-5-3-1"  
class="td11"> &#x00A0;                       </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-3-2"  
class="td11"> K<!--l. 92--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math> (<!--l. 92--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>M)                                                                                                                                                                                                                                                                                                                                                                                                                                                                            </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-3-3"  
class="td11"> 83 <!--l. 92--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 6          </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-3-4"  
class="td11"> 60 <!--l. 92--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 7          </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-3-5"  
class="td11"> <span 
class="cmmi-12">&#x003E;</span>500                                                                                                                                                                          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-4-"><td  style="text-align:left; white-space:nowrap;" id="TBL-5-4-1"  
class="td11">           </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-4-2"  
class="td11"> k<!--l. 93--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>/K<!--l. 93--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math> (<!--l. 93--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>M<!--l. 93--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">-</mo><mn>1</mn></mrow></msup 
></math>s<!--l. 93--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" > <mo 
class="MathClass-bin">-</mo> <mn>1</mn></math>)  </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-4-3"  
class="td11"> 17.0 <!--l. 93--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 0.7    </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-4-4"  
class="td11"> 2.71 <!--l. 93--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 0.16  </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-4-5"  
class="td11"> 0.0011 <!--l. 93--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 0.00007  </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-5-"><td  style="text-align:left; white-space:nowrap;" id="TBL-5-5-1"  
class="td11"> P167Sb             </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-5-2"  
class="td11"> k<!--l. 94--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math> (s<!--l. 94--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">-</mo><mn>1</mn></mrow></msup 
></math>)                                                                                                                                                                                                                                                                                                                                                                                                                                                                               </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-5-3"  
class="td11"> 681 <!--l. 94--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 36.8   </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-5-4"  
class="td11"> 297 <!--l. 94--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 29.7   </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-5-5"  
class="td11"> ND                                                                                                                                                                             </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-6-"><td  style="text-align:left; white-space:nowrap;" id="TBL-5-6-1"  
class="td11">           </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-6-2"  
class="td11"> K<!--l. 95--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math> (<!--l. 95--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>M)                                                                                                                                                                                                                                                                                                                                                                                                                                                                            </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-6-3"  
class="td11"> 32 <!--l. 95--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 0.8       </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-6-4"  
class="td11"> 37 <!--l. 95--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 6.3       </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-6-5"  
class="td11"> <span 
class="cmmi-12">&#x003E;</span>500                                                                                                                                                                          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-7-"><td  style="text-align:left; white-space:nowrap;" id="TBL-5-7-1"  
class="td11">           </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-7-2"  
class="td11"> k<!--l. 96--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>/K<!--l. 96--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math> (<!--l. 96--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>M<!--l. 96--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">-</mo><mn>1</mn></mrow></msup 
></math>s<!--l. 96--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" > <mo 
class="MathClass-bin">-</mo> <mn>1</mn></math>)  </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-7-3"  
class="td11"> 21.1 <!--l. 96--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 1.3    </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-7-4"  
class="td11"> 8.0 <!--l. 96--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 1.6      </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-7-5"  
class="td11"> 0.011 <!--l. 96--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 0.0002      </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-8-"><td  style="text-align:left; white-space:nowrap;" id="TBL-5-8-1"  
class="td11"> D240Gb            </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-8-2"  
class="td11"> k<!--l. 97--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math> (s<!--l. 97--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">-</mo><mn>1</mn></mrow></msup 
></math>)                                                                                                                                                                                                                                                                                                                                                                                                                                                                               </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-8-3"  
class="td11"> 471 <!--l. 97--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 10.9   </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-8-4"  
class="td11"> 321 <!--l. 97--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 46.2   </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-8-5"  
class="td11"> ND                                                                                                                                                                             </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-9-"><td  style="text-align:left; white-space:nowrap;" id="TBL-5-9-1"  
class="td11">           </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-9-2"  
class="td11"> K<!--l. 98--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math> (<!--l. 98--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>M)                                                                                                                                                                                                                                                                                                                                                                                                                                                                            </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-9-3"  
class="td11"> 47.1 <!--l. 98--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 41.7  </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-9-4"  
class="td11"> 52 <!--l. 98--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 8.6       </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-9-5"  
class="td11"> <span 
class="cmmi-12">&#x003E;</span>500                                                                                                                                                                          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-10-"><td  style="text-align:left; white-space:nowrap;" id="TBL-5-10-1"  
class="td11">           </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-10-2"  
class="td11"> k<!--l. 99--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>/K<!--l. 99--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math> (<!--l. 99--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>M<!--l. 99--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">-</mo><mn>1</mn></mrow></msup 
></math>s<!--l. 99--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" > <mo 
class="MathClass-bin">-</mo> <mn>1</mn></math>)  </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-10-3"  
class="td11"> 10.0 <!--l. 99--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 2.5    </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-10-4"  
class="td11"> 6.2 <!--l. 99--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 1.4      </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-10-5"  
class="td11"> 0.013 <!--l. 99--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 0.0007      </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-11-"><td  style="text-align:left; white-space:nowrap;" id="TBL-5-11-1"  
class="td11"> P167S/D240G  </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-11-2"  
class="td11"> k<!--l. 100--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math> (s<!--l. 100--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">-</mo><mn>1</mn></mrow></msup 
></math>)                                                                                                                                                                                                                                                                                                                                                                                                                                                                               </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-11-3"  
class="td11"> 165 <!--l. 100--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 11      </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-11-4"  
class="td11"> 139 <!--l. 100--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 3        </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-11-5"  
class="td11"> ND                                                                                                                                                                             </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-12-"><td  style="text-align:left; white-space:nowrap;" id="TBL-5-12-1"  
class="td11">           </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-12-2"  
class="td11"> K<!--l. 101--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math> (<!--l. 101--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>M)                                                                                                                                                                                                                                                                                                                                                                                                                                                                            </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-12-3"  
class="td11"> 15 <!--l. 101--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 3          </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-12-4"  
class="td11"> 42 <!--l. 101--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 0.5       </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-12-5"  
class="td11"> <span 
class="cmmi-12">&#x003E;</span>500                                                                                                                                                                          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-13-"><td  style="text-align:left; white-space:nowrap;" id="TBL-5-13-1"  
class="td11">           </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-13-2"  
class="td11"> k<!--l. 102--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>/K<!--l. 102--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math> (<!--l. 102--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>M<!--l. 102--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">-</mo><mn>1</mn></mrow></msup 
></math>s<!--l. 102--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" > <mo 
class="MathClass-bin">-</mo> <mn>1</mn></math>)  </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-13-3"  
class="td11"> 10.8 <!--l. 102--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 1.4    </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-13-4"  
class="td11"> 3.27 <!--l. 102--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 0.1    </td><td  style="text-align:left; white-space:nowrap;" id="TBL-5-13-5"  
class="td11"> 0.0060 <!--l. 102--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math> 0.00004  </td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-14-"><td  style="text-align:left; white-space:nowrap;" id="TBL-5-14-1"  
class="td11">           </td></tr></table>
</div>
                                                                                      
                                                                                      
</div><hr class="endfloat" />
</div>
<!--l. 106--><p class="noindent" >Ceftazidime hydrolysis by the WT, P167S, and D240G enzymes exhibits high
K<!--l. 106--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math> values (>500
<!--l. 106--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>M), which precluded determination
of k<!--l. 106--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math> values <span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>]</span>. Nevertheless,
k<!--l. 106--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>/K<!--l. 106--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math>
values for the P167S and D240G enzymes were 10-fold higher than that observed
for WT CTX-M-14. If the P167S and D240G substitutions act additively,
k<!--l. 106--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>/K<!--l. 106--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math>
for ceftazidime by the double mutant should be a further 10-fold
higher than that observed for the single mutants<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xwells_additivity_1990">441</a>]</span>. However,
k<!--l. 106--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>/K<!--l. 106--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math>
for  ceftazidime  hydrolysis  by  the  double  mutant  was
<!--l. 106--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" > <mo 
class="MathClass-rel">~</mo></math>2-fold
lower than that observed for the P167S and D240G single mutants (Table <a 
href="#x11-111001r2">7.2<!--tex4ht:ref: tab:ch2-tab2 --></a>). Therefore, the P167S
and D240G substitutions are antagonistic with respect to ceftazidime hydrolysis. This suggests that the
presence of one substitution alters the environment of the other to perturb its contribution to
catalysis<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xwells_additivity_1990">441</a>]</span>.
</p><!--l. 108--><p class="noindent" >The P167S and D240G substitutions were previously observed to modestly increase
k<!--l. 108--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>/K<!--l. 108--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math>
for cefotaxime hydrolysis (<!--l. 108--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" > <mo 
class="MathClass-rel">~</mo></math>2-fold)
compared with the WT enzyme<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>]</span>. The P167S/D240G double mutant exhibited a
k<!--l. 108--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>/K<!--l. 108--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math>
value similar to WT and 2-fold lower than the single mutants indicating possible antagonism, as found
for ceftazidime hydrolysis (Table <a 
href="#x11-111001r2">7.2<!--tex4ht:ref: tab:ch2-tab2 --></a>).
                                                                                      
                                                                                      
</p><!--l. 110--><p class="noindent" >The second-generation cephalosporin cephalothin is an excellent substrate for the WT
CTX-M-14 enzyme (Table <a 
href="#x11-111001r2">7.2<!--tex4ht:ref: tab:ch2-tab2 --></a>)<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>]</span>. The P167S and D240G substitutions reduce both
k<!--l. 110--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math> and
K<!--l. 110--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math> values
for cephalothin hydrolysis (Table <a 
href="#x11-111001r2">7.2<!--tex4ht:ref: tab:ch2-tab2 --></a>). The P167S/D240G double mutant exhibited a further reduction
in k<!--l. 110--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math> and
K<!--l. 110--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math>
compared with the single mutants. Interestingly, the P167S and D240G substitutions act additively in
the double mutant for cephalothin hydrolysis. Therefore, the additivity relationship between the P167S
and D240G substitutions is substrate-dependent, with simple additivity observed for cephalothin and
antagonism observed for ceftazidime hydrolysis, suggesting that the effects are mediated through
interaction with the substrates.
</p>
<h4 class="subsectionHead"><span class="titlemark">7.3.3    </span> <a 
 id="x11-1120007.3.3"></a>P167S/D240G double mutant exhibits reduced stability compared with single mutants</h4>
<!--l. 114--><p class="noindent" >Amino acid substitutions can affect catalysis, as shown above, but also can impact protein stability. It
was previously shown that the P167S and D240G single mutants destabilize CTX-M-14<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>]</span>. We
extended this finding to the P167S/D240G enzyme using CD spectroscopy to monitor
<!--l. 114--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B1;</mi></math>-helix
ellipticity with increasing temperature (Fig. <a 
href="#x11-112001r3">7.3<!--tex4ht:ref: fig:ch2-fig3 --></a>). Previous studies showed that the WT CTX-M-14
exhibited a melting temperature (Tm) of 54.6 &#8451;, and the single mutants D240G and P167S decreased
the Tm by 0.4 and 2.8 &#8451;, respectively<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>]</span>. The P167S/D240G enzyme exhibited a Tm of 50.5
&#8451;, indicating that the double mutant is less stable than WT and the single mutants (Fig.
<a 
href="#x11-112001r3">7.3<!--tex4ht:ref: fig:ch2-fig3 --></a>).
</p>
                                                                                      
                                                                                      
<hr class="figure" /><div class="figure" 
>
                                                                                      
                                                                                      
<a 
 id="x11-112001r3"></a>
                                                                                      
                                                                                      
<!--l. 118--><p class="noindent" ><img 
src="Chapter2/ch2-fig3.jpg" alt="PIC"  
width="216" height="216"  />
<a 
 id="x11-112002"></a>
<br /> </p><div class="caption" 
><span class="id">Figure&#x00A0;7.3: </span><span  
class="content">Thermal stability of WT and mutant <!--l. 120--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamases,
as  measured  by  CD.  Mean  ellipsivity  is  normalized  and  fit  to  a  Boltzmann  sigmoidal
function.   Tm,   as   determined   by   the   Boltzmann   equation,   is   also   plotted,   as   is
<!--l. 120--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x0394;</mi></math>Tm,
which is the change from the WT Tm. The Tm indicates that each single mutant is less stable
than  the  WT  CTX-M-14  enzyme,  and  this  instability  has  an  additive  effect  in  the  double
mutant,  P167S/D240G  CTX-M-14.  The  data  for  CTX-M-14,  D240G,  and  P167S  are  from
Patel et al.<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>]</span>.</span></div><!--tex4ht:label?: x11-112001r7.3 -->
                                                                                      
                                                                                      
</div><hr class="endfigure" />
<h4 class="subsectionHead"><span class="titlemark">7.3.4    </span> <a 
 id="x11-1130007.3.4"></a>Steady-state levels of the P167S/D240G enzyme in E. coli are reduced compared with single
mutants.</h4>
<!--l. 126--><p class="noindent" >The level of antibiotic resistance conferred to bacteria by a
<!--l. 126--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase depends
on the rate of hydrolysis, as well as the steady-state levels of enzyme expression<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xhuang_natural_1997">468</a>]</span>. A correlation has been shown
between <!--l. 126--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
stability and expression levels in E. coli caused by increased proteolysis and aggregation of unstable
proteins<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xhuang_natural_1997">468</a>,&#x00A0;<a 
href="mainli4.html#Xbrown_multiple_2010">469</a>,&#x00A0;<a 
href="mainli4.html#Xmayer_correlation_2007">470</a>]</span>. Because the P167S and D240G substitutions decrease enzyme stability
and the double mutant decreases stability further, we hypothesized that the double mutant
would display lower expression levels. Immunoblot analysis of whole cell lysates using
<!--l. 126--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B1;</mi></math>-CTX-M-14
<!--l. 126--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
polyclonal antibody showed that the P167S mutant did not significantly decrease expression
levels relative to WT, consistent with previous studies (Fig. <a 
href="#x11-113001r4">7.4<!--tex4ht:ref: fig:ch2-fig4 --></a>)<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>]</span>. The D240G mutant,
which shows only a 0.4 &#8451; decrease in stability relative to WT, displayed lower expression
levels. Thus, although D240G has higher thermal stability than P167S, it displays lower
expression levels, indicating that thermal stability does not fully correlate with expression levels.
However, the P167S/D240G enzyme exhibited lower expression levels than either WT
or the P167S and D240G single mutants, consistent with the lower thermal stability of
this mutant. Taken together, these findings provide a rationale for why the P167S/D240G
double mutant has not been observed in resistant clinical isolates in that it is compromised
for catalysis, stability, and expression levels compared with the P167S and D240G single
mutants.
</p>
                                                                                      
                                                                                      
<hr class="figure" /><div class="figure" 
>
                                                                                      
                                                                                      
<a 
 id="x11-113001r4"></a>
                                                                                      
                                                                                      
<!--l. 130--><p class="noindent" ><img 
src="Chapter2/ch2-fig4.jpg" alt="PIC"  
width="216" height="216"  />
<a 
 id="x11-113002"></a>
<br />   </p><div class="caption" 
><span class="id">Figure&#x00A0;7.4:     </span><span  
class="content">Steady-state     protein     levels     of     WT     CTX-M-14     and     mutant
<!--l. 132--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamases.
Western   blotting   analysis   with   an   anti-CTX-M-14   polyclonal   antibody   shows   protein
expression  levels  of  WT  CTX-M-14  and  the  P167S,  D240G,  and  P167S/D240G  mutants
in  the  periplasm  of  recombinant  E.  coli  cells.  Analysis  with  a  polyclonal  antibody  to  the
periplasmic  protein  MBP  was  used  as  a  loading  control.  The  hybridization  signal  was
visualized by chemiluminescence and quantified by densitometry. The signal for CTX-M-14
<!--l. 132--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
was normalized to that for MBP in the same sample. The protein levels of mutant CTX-M-14
<!--l. 132--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
are expressed relative to that of WT CTX-M-14 <!--l. 132--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
in the bar graph. The quantification data in the bar graph are the averages of two independent
experiments, and one representative immunoblot result is shown above the bar graph.</span></div><!--tex4ht:label?: x11-113001r7.3 -->
                                                                                      
                                                                                      
</div><hr class="endfigure" />
<h4 class="subsectionHead"><span class="titlemark">7.3.5    </span> <a 
 id="x11-1140007.3.5"></a>X-ray structures of P167S/D240G apo, E166A/D240G/CAZ, and
E166A/P167S/D240G/CAZ acyl-enzyme complexes reveal alternate conformations of the
<!--l. 136--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop.</h4>
<!--l. 138--><p class="noindent" >We previously determined the X-ray structure of the P167S enzyme, which had a very similar overall structure as WT
<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_drug-resistant_2017">461</a>]</span>. The <!--l. 138--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop,
which forms the bottom of the active site, was in a folded, closed conformation with the peptide bond
preceding Ser167 in a cis configuration (Fig. <a 
href="#x11-114001r5">7.5<!--tex4ht:ref: fig:ch2-fig5 --></a>, A&#8211;C). The structure of the D240G enzyme
was previously determined, and it also is highly similar to the WT structure<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xchen_atomic_2005">458</a>]</span> (Fig.
<a 
href="#x11-114001r5">7.5<!--tex4ht:ref: fig:ch2-fig5 --></a>D). We next determined the structure of the P167S/D240G enzyme, which exhibits lower
ceftazidime hydrolysis than either of the single mutants. The structure includes a boronic
acid from the crystallization buffer in complex with Ser70 and is very similar to the WT,
P167S, and D240G structures, with the Ser167 peptide bond in the cis configuration and the
<!--l. 138--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop in
a folded, closed conformation (Fig. <a 
href="#x11-114001r5">7.5<!--tex4ht:ref: fig:ch2-fig5 --></a>, E&#8211;H, and Table <a 
href="mainap5.html#x17-169001r1">E.1<!--tex4ht:ref: tab:ch2-supptable1 --></a>). A difference was noted, however, in the
B-factors in the active-site 103&#8211;106 loop, suggesting increased disorder. B-factors reflect the
degree to which electron density is scattered and therefore indicate how ordered an atom is
in the structure<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xyuan_prediction_2005">471</a>]</span>. The B-factors for residues in the 103&#8211;106 loop and the 164&#8211;179
<!--l. 138--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
were normalized to the overall B-factor of each structure to facilitate comparison across structures
(Fig. <a 
href="#x11-114003r6">7.6<!--tex4ht:ref: fig:ch2-fig6 --></a>). The normalized B-factors for the P167S/D240G structure for residues Val103 and Asn104
were higher than in the WT, P167S, and D240G structures. These findings suggest increased disorder
for residues 103&#8211;104 in the P167S/D240G structure. We have previously shown that Asn104 is
important for cefotaxime and ceftazidime hydrolysis, and therefore increased disorder of this residue
in the P167S/D240G enzyme could result in the observed lower activity for ceftazidime
                                                                                      
                                                                                      
hydrolysis<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_synergistic_2018">472</a>]</span>.
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                      
                                                                                      
<a 
 id="x11-114001r5"></a>
                                                                                      
                                                                                      
<!--l. 142--><p class="noindent" ><img 
src="Chapter2/ch2-fig5.jpg" alt="PIC"  
width="361" height="361"  />
<a 
 id="x11-114002"></a>
<br />   </p><div class="caption" 
><span class="id">Figure&#x00A0;7.5:      </span><span  
class="content">Structures      of      the      active-site      region      of      WT      CTX-M14
<!--l. 144--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
(PDB  code  1YLT),  as  well  as  P167S  (PDB  code  5TWD),  D240G  (PDB  code  1YLP),  and
P167S/D240G  mutant  enzymes  in  the  apo  form.  A,  ribbon  diagram  showing  structural
alignment of the CTX-M-14 WT (salmon), P167S (gray), D240G (blue), and P167S/D240G
(cyan)  enzymes.  The  active-site  region  shown  in  B&#8211;H  is  boxed.  B&#8211;E,  active-site  regions
of  CTX-M-14  WT  (B),  P167S  (C),  D240G  (D),  and  P167S/D240G  (E).  Note  that  the
P167S/D240G structure has a boronic acid in complex with Ser70, which is not shown for
clarity.  F,  structural  alignment  of  active-site  region  of  P167S  apo  enzyme  (gray)  with  the
P167S/D240G  double  mutant  (cyan).  G,  structural  alignment  of  D240G  apo  enzyme  (blue)
with  P167S/D240G  (cyan).  H,  structural  alignment  of  CTX-M-14  WT  apo  enzyme  with
P167S/D240G (cyan). In all panels, oxygen is shown in red, and nitrogen is in blue.</span></div><!--tex4ht:label?: x11-114001r7.3 -->
                                                                                      
                                                                                      
</div><hr class="endfigure" />
<!--l. 148--><p class="noindent" >We next determined the structures of the mutant enzymes in complex with ceftazidime to evaluate
whether the presence of substrate influences active-site structure (Table <a 
href="mainap5.html#x17-169001r1">E.1<!--tex4ht:ref: tab:ch2-supptable1 --></a>). The E166A mutation
blocks deacylation and allows for crystallization of the acyl-enzyme complex<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xstrynadka_molecular_1992">465</a>]</span>. The previously
determined structure of the acyl-enzyme complex of the CTX-M-14 pseudo WT E166A enzyme with
ceftazidime (E166A/CAZ) shows the Pro167 peptide bond in the cis configuration and the
<!--l. 148--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop in
the folded, closed conformation (Fig. <a 
href="#x11-114005r7">7.7<!--tex4ht:ref: fig:ch2-fig7 --></a>A) <span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_drug-resistant_2017">461</a>]</span>. Contacts between ceftazidime and the
enzyme include hydrogen bonds between the side chains of Asn132 and Asn104 with
the carbonyl oxygen of the acylamide of the ceftazidime R-2 group, as well as hydrogen
bonds between the hydroxyls of Thr235 and Ser237 with the C4 carboxylate of the
dihydrothiazine ring (Figs. <a 
href="#x11-108003r2">7.2<!--tex4ht:ref: fig:ch2-fig2 --></a>C and <a 
href="#x11-114005r7">7.7<!--tex4ht:ref: fig:ch2-fig7 --></a>A). The imino group of ceftazidime is pointed to
solvent and does not interact with the enzyme. The previously determined structure of
E166A/P167S/CAZ (Fig. 7B) shows the Ser167 peptide bond in the trans configuration and the
<!--l. 148--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
in an unraveled, open conformation, which widens the floor of the active site by
<!--l. 148--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" > <mo 
class="MathClass-rel">~</mo></math>5 &#197; to
accommodate ceftazidime<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_drug-resistant_2017">461</a>]</span>. This leads to a change in conformation of ceftazidime in the
acyl-enzyme with the aminothiazole ring assuming a buried position (Fig. <a 
href="#x11-114005r7">7.7<!--tex4ht:ref: fig:ch2-fig7 --></a>B)<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_drug-resistant_2017">461</a>]</span>. In addition,
there are hydrogen bonds between the C4 carboxylate of the dihydrothiazine ring and the side chains
of Thr235 and Ser237, as well as between the side chains of Asn132 and Asn104 with the
carbonyl oxygen of the acylamide group (Fig. <a 
href="#x11-114005r7">7.7<!--tex4ht:ref: fig:ch2-fig7 --></a>B). Further, there is a hydrogen bond
between Asn104 and the carboxyl group of the imino side chain of ceftazidime. These
interactions are consistent with tighter binding of ceftazidime and enhanced catalysis<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_drug-resistant_2017">461</a>]</span>. In
addition, the normalized B-factors of Val103 and Asn104 are not increased relative to WT
CTX-M-14, suggesting that the Asn104 residue is well-ordered for interaction with ceftazidime
(Fig. <a 
href="#x11-114003r6">7.6<!--tex4ht:ref: fig:ch2-fig6 --></a>A). Residues 168&#8211;170, however, show elevated B-factors, suggesting that the
                                                                                      
                                                                                      
<!--l. 148--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
has increased flexibility, consistent with its unfolded structure (Fig. <a 
href="#x11-114003r6">7.6<!--tex4ht:ref: fig:ch2-fig6 --></a>B).
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                      
                                                                                      
<a 
 id="x11-114003r6"></a>
                                                                                      
                                                                                      
<!--l. 152--><p class="noindent" ><img 
src="Chapter2/ch2-fig6.jpg" alt="PIC"  
width="339" height="339"  />
<a 
 id="x11-114004"></a>
<br />  </p><div class="caption" 
><span class="id">Figure&#x00A0;7.6:    </span><span  
class="content">Normalized    B-factors    for    the    103&#8211;106    loop    and    the    164&#8211;179
<!--l. 154--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
in the CTX-M enzyme structures. The B-factors were normalized by dividing the B-factor for
a given residue by the average B-factor for the entire structure. Thus, a normalized B-factor
of 1 means the B-factor at that residue is the same as the average B-factor of the structure.
A,  normalized  B-factors  for  the  103&#8211;106  loop.  B,  normalized  B-factors  for  the  164&#8211;179
<!--l. 154--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop.
Black     circle,     E166A/CAZ;     blue     square,     E166A/P167S/CAZ;     black     triangle,
E166A/D240G/CAZ;  inverted  red  triangle,  E166A/P167S/D240G/CAZ-1;  green  diamond,
E166A/P167S/D240G/CAZ-2;  open  circle,  P167S/D240G  apo;  open  square,  P167S  apo;
open triangle, D240G apo; inverted open triangle, E166A/P167S/D240G apo; open diamond,
CTX-M-14 WT.</span></div><!--tex4ht:label?: x11-114003r7.3 -->
                                                                                      
                                                                                      
</div><hr class="endfigure" />
<!--l. 158--><p class="noindent" >The D240G substitution is also associated with increased ceftazidime hydrolysis<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xbonnet_effect_2003">462</a>,&#x00A0;<a 
href="mainli4.html#Xcanton_ctx-m_2012">464</a>]</span>.
We therefore determined the structure of the E166A/D240G enzyme in complex with
ceftazidime for comparison with the E166A and E166A/P167S acyl-enzyme structures.
It was found that the peptide bond preceding Pro167 is in the cis configuration, and the
<!--l. 158--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop is
in the folded, closed conformation similar to the D240G apo enzyme structure and the E166A
structure with ceftazidime (Fig. <a 
href="#x11-114005r7">7.7<!--tex4ht:ref: fig:ch2-fig7 --></a>C). In contrast to the E166A/CAZ structure, however, the
E166A/D240G/CAZ structure has the side chain of Ser237 rotated away from the C4 carboxylate and
instead forms hydrogen bonds to the carboxylate of the imino side chain, which may facilitate
substrate binding and catalysis (Figs. <a 
href="#x11-108003r2">7.2<!--tex4ht:ref: fig:ch2-fig2 --></a>C and <a 
href="#x11-114005r7">7.7<!--tex4ht:ref: fig:ch2-fig7 --></a>C).
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                      
                                                                                      
<a 
 id="x11-114005r7"></a>
                                                                                      
                                                                                      
<!--l. 162--><p class="noindent" ><img 
src="Chapter2/ch2-fig7.jpg" alt="PIC"  
width="339" height="339"  />
<a 
 id="x11-114006"></a>
<br />   </p><div class="caption" 
><span class="id">Figure&#x00A0;7.7:     </span><span  
class="content">Structures     of     the     active-site     region     of     CTX-M-14     mutant
<!--l. 164--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
acyl-enzyme   complexes   with   ceftazidime.   A,   structure   of   the   E166A   mutant   (tan)
with   ceftazidime   (dark   gray)   trapped   in   the   acyl-enzyme   form   (PDB   code   5U53).
Oxygen  is  shown  in  red,  and  nitrogen  is  in  blue.  Hydrogen  bonds  are  indicated  by
thin  black  lines.  Active-site  residues  are  labeled.  B,  structure  of  the  E166A/P167S/CAZ
acyl-enzyme   (light   blue)   (PDB   code   5TW6).   C,   structure   of   the   E166A/D240G/CAZ
acyl-enzyme     (light     green).     D,     structure     of     the     E166A/P167S/D240G/CAZ-1
acyl-enzyme    (pink).    E,    structure    of    the    E166A/P167S/D240G/CAZ-2    acyl-enzyme
(white).   F,   structural   alignment   of   the   E166A/CAZ   (tan)   and   E166A/D240G/CAZ
(green)    acyl-enzyme    complexes.    The    ceftazidime    from    the    E166A/CAZ    structure
is   shown   in   dark   gray,   and   that   from   E166A/D240G/CAZ   is   shown   in   white.   The
<!--l. 164--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
region remains folded in the closed form and the ceftazidime occupies a similar position with
the aminothiazole ring surface exposed in these structures.</span></div><!--tex4ht:label?: x11-114005r7.3 -->
                                                                                      
                                                                                      
</div><hr class="endfigure" />
<!--l. 168--><p class="noindent" >The P167S/D240G enzyme displays lower catalytic activity toward ceftazidime than either single
mutant (Table <a 
href="#x11-111001r2">7.2<!--tex4ht:ref: tab:ch2-tab2 --></a>). To better understand the basis of this antagonism, we determined the structure of
the E166A/P167S/D240G enzyme in complex with ceftazidime. Two structures from different space
groups were obtained, and interestingly, they show different conformations of ceftazidime and the
<!--l. 168--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop.
In the first structure, the peptide bond preceding Ser167 is in the trans configuration, which is in
contrast to the cis bond found in the P167S/D240G apo structure (Fig. <a 
href="#x11-114001r5">7.5<!--tex4ht:ref: fig:ch2-fig5 --></a>, D and E). However, the
<!--l. 168--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop in
the E166A/P167S/D240G/CAZ-1 structure remains in the folded, closed conformation with
ceftazidime located in a similar position as that in the E166A/D240G/CAZ structure (Fig. <a 
href="#x11-114005r7">7.7<!--tex4ht:ref: fig:ch2-fig7 --></a>D).
There are differences, however, between these structures. First, the carboxylate group of the imino
side chain of ceftazidime in the E166A/P167S/D240G/CAZ-1 structure does not contact the enzyme,
in contrast to the E166A/D240G/CAZ structure (Fig. <a 
href="#x11-114005r7">7.7<!--tex4ht:ref: fig:ch2-fig7 --></a>, C and D). More importantly, the positioning
of the active-site 103&#8211;106 loop is altered, and the side chain of Asn104 is shifted out of the active site
in the E166A/P167S/D240G/CAZ-1 structure (Fig. <a 
href="#x11-114005r7">7.7<!--tex4ht:ref: fig:ch2-fig7 --></a>D). In addition, the normalized B-factors for
residues Val103 and Asn104 are elevated compared with the E166A/CAZ, E166A/P167S/CAZ, and
E166A/D240G/CAZ structures, suggesting that Val103 and Asn104 are disordered (Fig. <a 
href="#x11-114003r6">7.6<!--tex4ht:ref: fig:ch2-fig6 --></a>A).
We have previously shown that the hydrogen bond between Asn104 and the acyl-amide
of cefotaxime and ceftazidime is important, and a N104A mutant exhibits 10-fold lower
k<!--l. 168--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>/K<!--l. 168--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math>
for both substrates<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_synergistic_2018">472</a>]</span>. These observations suggest that the conformation of the enzyme and
ceftazidime observed in the E166A/P167S/D240G/CAZ-1 structure is not consistent with
hydrolysis.
</p><!--l. 170--><p class="noindent" >It is noteworthy that the E166A/P167S/D240G/CAZ-1 structure described above was
obtained by soaking a crystal with ceftazidime. Another crystal was also soaked, and the
structure was determined with the same space group, but ceftazidime was not present in
                                                                                      
                                                                                      
the active site. Interestingly, this apo structure is very similar to the structure with bound
ceftazidime. The peptide bond preceding Ser167 is in the trans configuration and the
<!--l. 170--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop is
in the folded, closed conformation (Fig. <a 
href="#x11-114007r8">7.8<!--tex4ht:ref: fig:ch2-fig8 --></a>, C and D). In addition, the 103&#8211;106 loop is in a similar
position as in the ceftazidime-bound structure with the side chain of Asn104 pointed out of the active
site and with elevated B-factors for Val103 and Asn104 (Fig. <a 
href="#x11-114003r6">7.6<!--tex4ht:ref: fig:ch2-fig6 --></a>A). Thus, this conformation of the
enzyme, and particularly the 103&#8211;106 loop, occurs in the absence of ceftazidime, in contrast to
the different conformations of the E166A/P167S apo and E166A/P167S/CAZ structures
where the presence of ceftazidime is apparently required to produce the conformational
change.
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                      
                                                                                      
<a 
 id="x11-114007r8"></a>
                                                                                      
                                                                                      
<!--l. 174--><p class="noindent" ><img 
src="Chapter2/ch2-fig8.jpg" alt="PIC"  
width="361" height="361"  />
<a 
 id="x11-114008"></a>
<br />   </p><div class="caption" 
><span class="id">Figure&#x00A0;7.8:     </span><span  
class="content">Structures     of     the     active-site     region     of     CTX-M-14     mutant
<!--l. 176--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
acyl-enzyme   complexes   with   ceftazidime.   A,   structure   of   the   E166A   mutant   (tan)
with   ceftazidime   (dark   gray)   trapped   in   the   acyl-enzyme   form   (PDB   code   5U53).
Oxygen  is  shown  in  red,  and  nitrogen  is  in  blue.  Hydrogen  bonds  are  indicated  by
thin  black  lines.  Active-site  residues  are  labeled.  B,  structure  of  the  E166A/P167S/CAZ
acyl-enzyme   (light   blue)   (PDB   code   5TW6).   C,   structure   of   the   E166A/D240G/CAZ
acyl-enzyme     (light     green).     D,     structure     of     the     E166A/P167S/D240G/CAZ-1
acyl-enzyme    (pink).    E,    structure    of    the    E166A/P167S/D240G/CAZ-2    acyl-enzyme
(white).   F,   structural   alignment   of   the   E166A/CAZ   (tan)   and   E166A/D240G/CAZ
(green)    acyl-enzyme    complexes.    The    ceftazidime    from    the    E166A/CAZ    structure
is   shown   in   dark   gray,   and   that   from   E166A/D240G/CAZ   is   shown   in   white.   The
<!--l. 176--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
region remains folded in the closed form and the ceftazidime occupies a similar position with
the aminothiazole ring surface exposed in these structures.</span></div><!--tex4ht:label?: x11-114007r7.3 -->
                                                                                      
                                                                                      
</div><hr class="endfigure" />
<!--l. 180--><p class="noindent" >The second E166A/P167S/D240G/CAZ structure is superimposable with that of the E166A/P167S/CAZ
structure where the peptide bond preceding Ser167 is in the trans configuration, and the
<!--l. 180--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop is
in an unfolded, open conformation to accommodate ceftazidime (Fig. <a 
href="#x11-114005r7">7.7<!--tex4ht:ref: fig:ch2-fig7 --></a>E). Because the P167S
enzyme exhibits enhanced ceftazidime hydrolysis, these findings suggest that the conformation
of the enzyme in the E166A/P167S/D240G/CAZ-2 structure is competent to hydrolyze
ceftazidime.
</p><!--l. 182--><p class="noindent" >The two structures of E166A/P167S/D240G/CAZ with distinct conformations of the
enzyme and ceftazidime suggests there are at least two conformational substates of the
P167S/D240G enzyme in the presence of ceftazidime. We suggest that the form with the closed
<!--l. 182--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop and altered
103&#8211;106 loop with Asn104 pointed out of the active site does not efficiently hydrolyze ceftazidime, whereas the form
with the open <!--l. 182--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
is catalytically competent.
</p>
<h4 class="subsectionHead"><span class="titlemark">7.3.6    </span> <a 
 id="x11-1150007.3.6"></a>Molecular dynamics simulations reveal that conformational heterogeneity of the
<!--l. 184--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop is
greater in the single mutants than in the WT or double mutant.</h4>
<!--l. 186--><p class="noindent" >To directly probe the conformational heterogeneity of the
<!--l. 186--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
and acyl-enzyme complex, we conducted molecular dynamics simulations of the acylated forms of
WT, D240G, P167S, and P167S/D240G. In addition to providing atomically detailed models of the
distribution of structures that CTX-M adopts, the fact that no chemical reactions occur in these
                                                                                      
                                                                                      
simulations allowed us to include Glu166 and interrogate its interactions with ceftazidime and
CTX-M. Simulations of WT were initiated from a crystal structure of the acyl-enzyme complex (PDB
code 5U53)<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_drug-resistant_2017">461</a>]</span>, and simulations of P167S were initiated based on a previous model of the
E166A/P167S/CAZ structure (PDB code 5TW6)<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_drug-resistant_2017">461</a>]</span>. Simulations of D240G were initiated from
the E166A/D240G/CAZ crystal structure presented in this work. The closed-conformation crystal
structure of E166A/P167S/D240G/CAZ-1 was the initial starting structure for simulations of
P167S/D240G/CAZ. In all structures, Ala166 was mutated back to a glutamic acid, and a total of 2.5
<!--l. 186--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>s of
simulation was run for each variant.
</p><!--l. 188--><p class="noindent" >The distribution of <!--l. 188--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
conformations observed in our simulations suggests a correlation between
<!--l. 188--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
opening and ceftazidime hydrolysis activity. The WT and P167S/D240G variant with acylated
ceftazidime both favor a well-defined closed conformation (Fig. <a 
href="#x11-115001r9">7.9<!--tex4ht:ref: fig:ch2-fig9 --></a>). However, we note that the
P167S/D240G variant with acylated ceftazidime sparsely samples open conformations of the
<!--l. 188--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop,
some of which are very similar to the crystallographic structure capturing the open
state (Fig. <a 
href="#x11-114005r7">7.7<!--tex4ht:ref: fig:ch2-fig7 --></a>, D and E, and Fig. <a 
href="mainap5.html#x17-170005r3">E.3<!--tex4ht:ref: fig:ch2-suppfig3 --></a>). A previous combination of simulations and
experiments have also demonstrated that the WT has a sparsely populated state with an open
<!--l. 188--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop<span class="cite">&#x00A0;[<a 
href="mainli4.html#XPorter:2019hv">49</a>]</span>.
In contrast, the P167S and D240G substitutions dramatically increase the probability of a
diversity of open conformations. The conformational heterogeneity of P167S is consistent
with the open structure and elevated B-factors observed in the E166A/P167S/CAZ crystal
structure (Figs. <a 
href="#x11-114003r6">7.6<!--tex4ht:ref: fig:ch2-fig6 --></a> and <a 
href="#x11-114005r7">7.7<!--tex4ht:ref: fig:ch2-fig7 --></a>B). Although D240G also displays substantial conformational
heterogeneity, it has a deeper minima for the closed state than P167S, potentially explaining
why only the closed state of D240G has been observed crystallographically so far (Fig.
<a 
href="#x11-114005r7">7.7<!--tex4ht:ref: fig:ch2-fig7 --></a>C).
                                                                                      
                                                                                      
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                      
                                                                                      
<a 
 id="x11-115001r9"></a>
                                                                                      
                                                                                      
<!--l. 192--><p class="noindent" ><img 
src="Chapter2/ch2-fig9.png" alt="PIC"  
width="231" height="231"  />
<a 
 id="x11-115002"></a>
<br />       </p><div class="caption" 
><span class="id">Figure&#x00A0;7.9:            </span><span  
class="content">The            conformational            heterogeneity            of            the
<!--l. 194--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
is greater in the single mutants than in the WT or double mutant. A, joint distributions of two
C<!--l. 194--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B1;</mi></math>&#8211;C<!--l. 194--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B1;</mi></math>
distances        that        capture        the        conformational        heterogeneity        of        the
<!--l. 194--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-Loop:
(i) Asn104 to position 167 on one side of the <!--l. 194--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
and    (ii)    Thr171    to    position    240    on    the    other    side.    Distributions    are    shown
for   WT   (top   left   panel),   D240G   (top   right   panel),   P167S   (bottom   left   panel),
and     P167S/D240G     (bottom     right     panel).     Each     point     represents     a     snapshot
from    the    molecular    dynamics    simulations    colored    according    to    its    probability
based   on   a   2D   histogram.    B,   cumulative   distribution   of   distances   between   the
C<!--l. 194--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B3;</mi></math>
atom of Glu166 and the N<!--l. 194--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B4;</mi></math>
atom   of   Asn170   for   WT   (black),   D240G   (blue),   P167S   (red),   and   P167S/D240G
(purple),     capturing     the     loss     of     interaction     between     Glu166     and     Asn170
in     the     open     state.     C,     distribution     of     distances     between     the     sidechain
N<!--l. 194--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B4;</mi></math>
atom of Asn132 and the C<!--l. 194--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B3;</mi></math>
of Glu166 for WT (black), D240G (blue), P167S (red), and P167S/D240G (purple), capturing
the newly formed interaction between Glu166 and Asn132 in the open state.</span></div><!--tex4ht:label?: x11-115001r7.3 -->
                                                                                      
                                                                                      
</div><hr class="endfigure" />
<!--l. 198--><p class="noindent" >These simulations suggest that the closed conformation inhibits catalysis by favoring a conformation
of Glu166 that is incompatible with a deacylation reaction, whereas the open conformation of the
<!--l. 198--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
allows Glu166 to adopt a wider range of conformations, at least some of which are
compatible with the requirements for deacylation. Previous work has established that Glu166
coordinates a water that plays a role in the deacylation reaction<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xchen_acylation_2007">473</a>]</span> and that mutation of
Glu166 traps the acyl intermediate by inhibiting deacylation<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xstrynadka_molecular_1992">465</a>]</span>. Examining closed
structures preferentially adopted by WT and P167S/D240G reveals that the carboxyl
group of Glu166 tends to hydrogen bond with Asn170, trapping Glu166 under the
<!--l. 198--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
and preventing it from coordinating the water required for deacylation (Figs. <a 
href="#x11-115001r9">7.9<!--tex4ht:ref: fig:ch2-fig9 --></a>, B and C, and <a 
href="#x11-115003r10">7.10<!--tex4ht:ref: fig:ch2-fig10 --></a>).
In the open conformations preferentially sampled by the D240G and P167S variants, the hydrogen
bond between Asn170 and Glu166 is disrupted. This open conformation is stabilized by a
rearrangement of the hydrogen-bonding network in the active site where Asn132 hydrogen bonds with
Glu166 (Figs. <a 
href="#x11-115001r9">7.9<!--tex4ht:ref: fig:ch2-fig9 --></a>C and <a 
href="#x11-115003r10">7.10<!--tex4ht:ref: fig:ch2-fig10 --></a>).
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                      
                                                                                      
<a 
 id="x11-115003r10"></a>
                                                                                      
                                                                                      
<!--l. 202--><p class="noindent" ><img 
src="Chapter2/ch2-fig10.png" alt="PIC"  
width="361" height="361"  />
<a 
 id="x11-115004"></a>
<br />   </p><div class="caption" 
><span class="id">Figure&#x00A0;7.10:     </span><span  
class="content">Conformational     changes     between     inactive     and     active     forms
of     the     acyl-enzyme.     Representative     structures     of     the     CTX-M     acyl-enzyme
complex    with    ceftazidime    (labeled    CAZ,    green)    highlighting    inactive    (blue,    left
panels)    and    active    (orange,    right    panels)    conformations.    Conformations    of    the
<!--l. 204--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
(top  panels)  and  residues  contacting  ceftazidime  (bottom  panels)  are  shown,  with  relevant
residues labeled. Key hydrogen bond interactions are depicted in dashed lines (black).</span></div><!--tex4ht:label?: x11-115003r7.3 -->
                                                                                      
                                                                                      
</div><hr class="endfigure" />
<!--l. 208--><p class="noindent" >The opening and closing of the <!--l. 208--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
is also associated with a rearrangement of ceftazidime in the acyl-enzyme complex
(Fig. <a 
href="#x11-115003r10">7.10<!--tex4ht:ref: fig:ch2-fig10 --></a> and Fig. <a 
href="mainap5.html#x17-170001r1">E.1<!--tex4ht:ref: fig:ch2-suppfig1 --></a>). One major feature observed in both crystal structures
and simulations is that the aminothiazole ring of ceftazidime is buried under the
<!--l. 208--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop when it is
open (Figs. <a 
href="#x11-114005r7">7.7<!--tex4ht:ref: fig:ch2-fig7 --></a>, B and E, and <a 
href="#x11-115003r10">7.10<!--tex4ht:ref: fig:ch2-fig10 --></a>). Consistent with the crystal structures of the single-mutant variants, this
rearrangement of ceftazidime in the acyl-enzyme complex is facilitated by new interactions with Asn104
and the <!--l. 208--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>3
loop. In the D240G and P167S constructs, Ser237 and Asn104 form interactions with ceftazidime
(Fig. <a 
href="#x11-115003r10">7.10<!--tex4ht:ref: fig:ch2-fig10 --></a> and Figs. <a 
href="mainap5.html#x17-170001r1">E.1<!--tex4ht:ref: fig:ch2-suppfig1 --></a> and <a 
href="mainap5.html#x17-170003r2">E.2<!--tex4ht:ref: fig:ch2-suppfig2 --></a>). We also note an additional interaction between the imino group of
ceftazidime and Ser237 that is present in the more open, active variants but not the more closed,
inactive variants (Fig. <a 
href="mainap5.html#x17-170003r2">E.2<!--tex4ht:ref: fig:ch2-suppfig2 --></a>). These interactions with ceftazidime are rare in the WT and
P167S/D240G simulations (Figs. <a 
href="mainap5.html#x17-170001r1">E.1<!--tex4ht:ref: fig:ch2-suppfig1 --></a> and <a 
href="mainap5.html#x17-170003r2">E.2<!--tex4ht:ref: fig:ch2-suppfig2 --></a>). Furthermore, Asn104 appears to point
outward toward the solvent in the closed configuration (Fig. <a 
href="#x11-115003r10">7.10<!--tex4ht:ref: fig:ch2-fig10 --></a>), similar to what is seen in
the crystal structure of the closed configuration of the P167S/D240G/CAZ variant (Fig.
<a 
href="#x11-114005r7">7.7<!--tex4ht:ref: fig:ch2-fig7 --></a>D).
</p><!--l. 210--><p class="noindent" >Overall, our simulations suggest that the CTX-M acyl-enzyme complex is in equilibrium between
inactive and active conformations and that the P167S and D240G variants have a higher
probability of adopting active conformations (Fig. <a 
href="#x11-115003r10">7.10<!--tex4ht:ref: fig:ch2-fig10 --></a>). In the inactive conformation the
<!--l. 210--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop is
closed, burying the Asn170&#8211;Glu166 hydrogen bond under the aminothiazole ring (Fig. <a 
href="#x11-115003r10">7.10<!--tex4ht:ref: fig:ch2-fig10 --></a>). Opening of
the <!--l. 210--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop and
rearrangement of ceftazidime via burial of the aminothiazole ring and coordination between the imino group and
Asn104 and the <!--l. 210--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>3
loop likely transitions CTX-M into a catalytically competent state. These rearrangements of the
<!--l. 210--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
                                                                                      
                                                                                      
and ceftazidime allow Glu166 to coordinate a water molecule that can access the
ester bond of the ceftazidime&#8211;acyl-enzyme complex, facilitating the deacylation
reaction. Taken together, the crystallography and molecular dynamics results indicate
that the P167S and D240G substitutions promote an open conformation of the
<!--l. 210--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
that creates access for ceftazidime and allows Glu166 to sample conformations
consistent with deacylation, whereas the WT and P167S/D240G mutant exhibit a closed
<!--l. 210--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
conformation that constrains access for ceftazidime and prevents Glu166 from efficiently coordinating
water for deacylation.
</p>
<h3 class="sectionHead"><span class="titlemark">7.4    </span> <a 
 id="x11-1160007.4"></a>Discussion</h3>
<!--l. 214--><p class="noindent" >The CTX-M <!--l. 214--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamases
emerged in the late 1980s and are characterized by their ability to efficiently hydrolyze cephalosporins,
particularly the oxyimino-cephalosporin cefotaxime<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xbonnet_growing_2004">456</a>,&#x00A0;<a 
href="mainli4.html#Xdandrea_ctx-m-type_2013">457</a>]</span>. For example, the catalytic efficiency for
cefotaxime hydrolysis by CTX-M enzymes is 1500-fold higher than that exhibited by the common TEM-1
<!--l. 214--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpalzkill_structural_2018">453</a>]</span>.
Nevertheless, the related oxyimino-cephalosporin ceftazidime is poorly hydrolyzed by CTX-M enzymes with
k<!--l. 214--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>/K<!--l. 214--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math>
values 1000-fold lower than those observed for cefotaxime<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpalzkill_structural_2018">453</a>]</span>. Natural variants
of the TEM-1 enzyme have evolved through mutations that provide increased rates
of ceftazidime catalysis. Similarly, the P167S and D240G substitutions have been
found in multiple variants of CTX-M enzymes that exhibit increased ceftazidime
hydrolysis<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xdandrea_ctx-m-type_2013">457</a>,&#x00A0;<a 
href="mainli4.html#Xbonnet_effect_2003">462</a>,&#x00A0;<a 
href="mainli4.html#Xkimura_role_2004">463</a>,&#x00A0;<a 
href="mainli4.html#Xcanton_ctx-m_2012">464</a>,&#x00A0;<a 
href="mainli4.html#Xcartelle_high-level_2004">474</a>]</span>. Each of these substitutions results in a 10-fold increased
                                                                                      
                                                                                      
k<!--l. 214--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>/K<!--l. 214--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math>
value for ceftazidime hydrolysis<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xchen_atomic_2005">458</a>,&#x00A0;<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>]</span>. However, variants containing both substitutions have not
been observed, despite the prediction that such variants would exhibit increased hydrolysis. Here we
have shown that the failure of the double mutant to emerge in natural isolates is due to
epistasis resulting from decreased stability and lower bacterial expression levels of the
P167S/D240G enzyme, as well as antagonism between the substitutions with respect to
catalysis.
</p><!--l. 216--><p class="noindent" >The combination of amino acid substitutions that each increase catalytic activity can display simple
additivity or cooperativity when introduced together into an enzyme<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xwells_additivity_1990">441</a>]</span>. For additive
interactions, the fold change of the double mutant is expected to be the product of the fold
changes of the single mutants. Such additive combinations indicate that the substitutions have
independent effects on catalysis<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xwells_additivity_1990">441</a>]</span>. However, not all substitutions act additively. The CTX-M
P167S and D240G substitutions are antagonizing in the double mutant (Table <a 
href="#x11-111001r2">7.2<!--tex4ht:ref: tab:ch2-tab2 --></a>). This
negative cooperativity suggests that the substitutions interact, either directly or indirectly, and
that the interaction has a negative effect on ceftazidime catalysis<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xwells_additivity_1990">441</a>]</span>. In the case of
the P167S and D240G substitutions, the interaction is not via direct contact, because the
<!--l. 216--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B1;</mi></math>-carbons
are located 10.7 &#197; apart.
</p><!--l. 218--><p class="noindent" >X-ray crystallography and molecular dynamics simulations of the P167S, D240G, and P167S/D240G
mutants provide a rationale for the increased ceftazidime hydrolysis by the single mutants and the negative
cooperativity observed for the double mutant. The acyl-enzyme complex of the E166A/P167S/CAZ
X-ray structure shows a trans configuration peptide bond preceding residue 167 and an unfolded
<!--l. 218--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop in
an open conformation with the aminothiazole ring of the antibiotic in a buried position. This
interaction increases van der Waals contacts and hydrogen bonds between the enzyme and ceftazidime
and is consistent with enhanced catalytic efficiency toward ceftazidime. In contrast, the
E166A/D240G/CAZ structure revealed a closed conformation. Two structures were obtained for the
                                                                                      
                                                                                      
E166A/P167S/D240G ceftazidime acyl-enzyme, one of which is superimposable with the open
<!--l. 218--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
structure of E166A/P167S/CAZ and, based on this similarity, is proposed
to hydrolyze ceftazidime. The second structure, however, displays a closed
<!--l. 218--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
and an altered conformation of the 103&#8211;106 loop such that the critical Asn104 residue is turned out
of the active site and does not interact with ceftazidime, suggesting reduced ceftazidime
catalysis.
</p><!--l. 220--><p class="noindent" >Although the E166A/P167S/CAZ and E166A/D240G/CAZ structures suggest that the substitutions act by different
mechanisms, 2.5-<!--l. 220--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>s
molecular dynamics simulations of each variant indicate that both the
P167S and D240G substitutions promote an open conformation of the
<!--l. 220--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
to accommodate ceftazidime and that the WT and P167S/D240G enzymes
exhibit reduced ceftazidime hydrolysis because open conformations of the
<!--l. 220--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
are less probable.
</p><!--l. 222--><p class="noindent" >The TEM-1 <!--l. 222--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
is <!--l. 222--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" > <mo 
class="MathClass-rel">~</mo></math>35%
identical in amino acid sequence to CTX-M enzymes and efficiently hydrolyzes penicillins and many
cephalosporins, but oxyimino-cephalosporins are poor substrates<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpalzkill_structural_2018">453</a>]</span>. Nevertheless, natural
variants of TEM-1 have evolved that exhibit increased catalytic efficiency for cefotaxime
and ceftazidime hydrolysis<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpetrosino_-lactamases:_1998">475</a>,&#x00A0;<a 
href="mainli4.html#Xsalverda_natural_2010">476</a>]</span>. These variants, termed extended-spectrum
<!--l. 222--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamases
(ESBLs), contain 1&#8211;5 amino acid substitutions, and multiply substituted enzymes are common.
Many of the substitutions found in these variants act additively when combined<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpalzkill_structural_2018">453</a>]</span>.
Some combinations of mutations, however, are not additive. A well-studied example is the
                                                                                      
                                                                                      
combination of the R164S and G238S substitutions. The Gly238 residue is on the
<!--l. 222--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>3
strand (analogous to Gly238 in CTX-M-14), and Arg164 is at the base of the
<!--l. 222--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop. Each
of these substitutions results in increased enzyme activity toward cefotaxime and ceftazidime, yet the double
mutant has reduced activity<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xdellus-gur_negative_2015">477</a>,&#x00A0;<a 
href="mainli4.html#Xgiakkoupi_detrimental_2000">478</a>]</span>. Dellus-Gur et al. <span class="cite">&#x00A0;[<a 
href="mainli4.html#Xdellus-gur_negative_2015">477</a>]</span> determined the structure of the TEM-1
G238S and R164S enzymes. The G238S-containing enzyme exhibited two dominant conformations of
the G238 loop, whereas the R164S substitution induced an ensemble of conformations of the
<!--l. 222--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop. The
structure of the R164S/G238S double mutant, however, exhibited a wider ensemble of conformations
of the <!--l. 222--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
than the single mutants<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xdellus-gur_negative_2015">477</a>]</span>. Based on these results, it was hypothesized that the entropic cost of the
substrate selecting suitable conformations among many alternatives results in the low activity for
cefotaxime hydrolysis by the double mutant, accounting for the negative epistasis observed for the
combination<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xdellus-gur_negative_2015">477</a>]</span>.
</p><!--l. 224--><p class="noindent" >In the case of the negative epistasis observed with the P167S and D240G combination in CTX-M
<!--l. 224--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase,
multiple conformations of the enzyme also appear to play a role. Based on our molecular dynamics
simulation results, the P167S and D240G substitutions are analogous to the R164S substitution in
TEM-1, where the substitutions induce an ensemble of conformations, some of which are predicted to
be capable of hydrolyzing ceftazidime. The CTX-M P167S and D240G substitutions antagonize
each other in the double mutant, similar to the TEM-1 R164S and G238S substitutions.
The mechanism of antagonism, however, is different, with the CTX-M P167S/D240G
double mutant showing a reduced probability of sampling multiple conformations of the
<!--l. 224--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop, whereas the TEM-1
R164S/G238S double mutant <!--l. 224--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
samples an excess of conformations<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xdellus-gur_negative_2015">477</a>]</span>. Nevertheless, both cases demonstrate the importance of
                                                                                      
                                                                                      
conformational heterogeneity of active-site loops in controlling catalytic activity and evolutionary
trajectories.
</p><!--l. 226--><p class="noindent" >Several studies have provided evidence that the conformation of the
<!--l. 226--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
is an important determinant of substrate specificity of class A
<!--l. 226--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamases,
particularly with regard to the hydrolysis of oxyimino-cephalosporins. As
described above, the TEM ESBL mutation R164S is thought to broaden the
specificity of the enzyme by increasing the conformational heterogeneity of the
<!--l. 226--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xdellus-gur_negative_2015">477</a>]</span>.
In addition, the structure of an apo enzyme form of a triple mutant of the TEM enzyme
containing the substitutions W165Y/E166Y/P167G that hydrolyzes ceftazidime shows the
<!--l. 226--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop in
an unfolded, open conformation similar to that observed for the CTX-M E166A/P167S/CAZ and
E166A/P167S/D240G/CAZ-2 structures<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xstojanoski_triple_2015">479</a>]</span>. Further, computational studies predict that TEM ESBL
substitutions that broaden the specificity of the enzyme to include cefotaxime stabilize conformations
of the <!--l. 226--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop
that facilitate substrate binding<span class="cite">&#x00A0;[<a 
href="mainli4.html#XHart:2016kb">9</a>]</span>.
</p><!--l. 228--><p class="noindent" >The results also suggest an important role for the active-site 103&#8211;106 loop in cefotaxime and
ceftazidime hydrolysis. We recently showed that an N106S mutation in the 103&#8211;106 loop that is found
in CTX-M enzymes from clinical isolates lowers cefotaxime and ceftazidime hydrolysis because of a
change in conformation of the loop<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_synergistic_2018">472</a>]</span>. Asn106 is at the base of the loop and not in the active site.
However, the N106S substitution changes the hydrogen-bonding network connectivity in the loop such
that the side chain of Asn104 rotates out of the active site, thereby eliminating a hydrogen bond with
substrate. Further experiments showed that an N104A mutant exhibits 10-fold reduced catalytic
efficiency for oxyimino cephalosporin hydrolysis, suggesting that the hydrogen bond is important
for catalysis. Thus, the conformation of the 103&#8211;106 loop is a determinant of substrate
                                                                                      
                                                                                      
specificity<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_synergistic_2018">472</a>]</span>. In this study, it was found that the P167S/D240G enzyme, which exhibits
reduced ceftazidime hydrolysis, has increased B-factors for the 103&#8211;106 loop, suggesting
disorder in Asn104 that is consistent with reduced activity. In addition, in the structure of the
E166A/P167S/D240 apo enzyme the B-factors of the 103&#8211;106 loop are increased, and in one of the
structures of E166A/P167S/D240G in complex with ceftazidime, the side chain of Asn104 is
rotated out of the active site, again consistent with decreased ceftazidime hydrolysis. Thus,
although the P167S and D240G substitutions are not in the 103&#8211;106 loop, the antagonism
between the substitutions is at least partially reflected in changes in the conformation of the
loop.
</p><!--l. 230--><p class="noindent" >Thermal stability studies of the WT, P167S, D240G, and P167S/D240G enzymes show that the single
mutants are less stable than WT, whereas the double mutant is less stable than either single mutant.
There is some correlation between stability and protein expression levels in that the P167S/D240G
mutant is the least stable and is also expressed at the lowest levels among the mutants. However, also
note the P167S mutant is less stable than D240G but is expressed at higher levels, suggesting there are
exceptions to the stability/protein expression level correlation. Some recent studies have shown that
lower enzyme stability correlates with increased flexibility and increased cephalosporin hydrolysis in
<!--l. 230--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamases<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xfrohlich_oxa-48-mediated_2019">480</a>,&#x00A0;<a 
href="mainli4.html#Xbarnes_deciphering_2018">481</a>]</span>.
Here, we do not observe a correlation between stability and flexibility in that the P167S and D240G
mutants readily sample multiple conformations and yet are more stable than P167S/D240G, which
samples fewer conformations. Further, we do not observe a correlation between stability and
catalytic activity toward ceftazidime because P167S/D240G has low stability but also low
activity.
</p><!--l. 232--><p class="noindent" >Taken together, the results presented here suggest that active-site loops play
an important role in the substrate specificity and evolutionary capacity of
<!--l. 232--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamases. Class
A <!--l. 232--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamases
such as TEM and CTX-M can evolve altered substrate specificity by mutations that change the
                                                                                      
                                                                                      
conformation of active-site loops. An active site with flexible loops loosely associated with a
highly ordered, stable scaffold structure has been described as fold polarity, and there is
evidence that such an organization facilitates the evolution of new functions because of a
tolerance to changes in the loops without drastically destabilizing the enzyme<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xdellus-gur_what_2013">443</a>]</span>. Such an
organization is clearly advantageous for antibiotic resistance enzymes such as CTX-M
<!--l. 232--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamases
that are under selective pressure for altered substrate specificity.
</p><!--l. 234--><p class="noindent" >
</p>
<h3 class="sectionHead"><span class="titlemark">7.5    </span> <a 
 id="x11-1170007.5"></a>Methods</h3>
<!--l. 235--><p class="noindent" >
</p>
<h4 class="subsectionHead"><span class="titlemark">7.5.1    </span> <a 
 id="x11-1180007.5.1"></a>Bacterial strains and plasmids.</h4>
<!--l. 236--><p class="noindent" >The CTX-M-14-pTP123 plasmid was used for site-directed mutagenesis, MIC determinations, and
immunoblotting. This plasmid was constructed by inserting the blaCTX-M-14 gene into the previously
described pTP123 plasmid. CTX-M-14-pTP123 has a chloramphenicol (CMP) resistance marker and
<!--l. 236--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi><mo 
class="MathClass-bin">-</mo></math>-lactamase expression is controlled
by the isopropyl<!--l. 236--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" > <mo 
class="MathClass-bin">-</mo> <mi 
>&#x03B2;</mi><mo 
class="MathClass-bin">-</mo></math>-D-galactopyranoside
(IPTG)&#8211;inducible P<!--l. 236--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>t</mi><mi 
>r</mi><mi 
>c</mi></mrow></msub 
></math>
promoter<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpetrosino_contributions_1999">482</a>]</span>. Under conditions without IPTG, protein expression is maintained at a basal level;
                                                                                      
                                                                                      
these were the conditions under which MIC determination and immunoblotting were performed. The
E. coli strain XL1-Blue (recA1 endA1 gyrA96 thi-1 hsdR17 supE44 relA1 lac [F9 proAB
lacI<!--l. 236--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mi 
>q</mi></mrow></msup 
></math>ZM15
Tn10 (Tet<!--l. 236--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mi 
>r</mi></mrow></msup 
></math>)])
(Stratagene, Inc., La Jolla, CA) was used as the host for the construction of the P167S/D240G CTX-M-14
mutant via site-directed mutagenesis, as well as MIC determination. The E. coli strain RB791 (W3110
lacIqL8) was used as the host for the determination of protein expression levels of WT CTX-M-14
<!--l. 236--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
and its mutants<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xamann_vectors_1983">483</a>]</span>. For protein purification, WT CTX-M-14 and the mutants were expressed
in the pET28a plasmid using the protocol outlined by Patel et al.<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>]</span>. CTX-M-14 and
mutants in the pET28a plasmid were expressed in BL21(DE3) [fhuA2 (lon) ompT gal
(<!--l. 236--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BB;</mi></math> DE3)
(dcm) <!--l. 236--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x0394;</mi></math>hsdS
<!--l. 236--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BB;</mi></math> DE3 =
<!--l. 236--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BB;</mi></math>sBamHIo
<!--l. 236--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x0394;</mi></math>EcoRI-B int::(lacI::PlacUV5::T7
gene1) i21 <!--l. 236--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x0394;</mi></math>nin5]<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xstudier_use_1986">484</a>]</span>.
</p><!--l. 238--><p class="noindent" >
</p>
<h4 class="subsectionHead"><span class="titlemark">7.5.2    </span> <a 
 id="x11-1190007.5.2"></a>Site-directed mutagenesis.</h4>
<!--l. 239--><p class="noindent" >The CTX-M-14 P167S/D240G mutant was constructed using the D240G mutant in pTP123 as template for
QuikChange mutagenesis using 1 unit of Phusion DNA polymerase (New England Biolabs, Ipswich, MA)
and 0.4 <!--l. 239--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>M
P167S primer (5&#8217;-CTGGATCGCACTGAAAGCACGCTGAATACCGCC-3&#8217;)<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>]</span>.
                                                                                      
                                                                                      
Primers were obtained from Integrated DNA Technologies (Coralville, IA). Thermocycler
products were digested with DpnI (New England Biolabs) and transformed into
electrocompetent E. coli XL1-Blue cells and selected on LB agar supplemented with 12.5
<!--l. 239--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>g/ml
chloramphenicol. The DNA sequence of the resulting mutant was confirmed by DNA sequencing (Genewiz,
Plainfield, NJ). The E166A/P167S/D240G mutant was constructed by QuikChange mutagenesis with
a primer encoding the E166A/P167S mutations (5&#8217;-GATCGCACTGCTCCTACGCTGAAT-3&#8217;) using
CTX-M-14 P167S/D240G pTP123 as template and confirmed using DNA sequencing.
</p><!--l. 241--><p class="noindent" >
</p>
<h4 class="subsectionHead"><span class="titlemark">7.5.3    </span> <a 
 id="x11-1200007.5.3"></a>Minimum inhibitory concentration determinations.</h4>
<!--l. 242--><p class="noindent" >MICs for cephalothin were determined by Etest strip (BioMérieux, Marcy-l&#8217;Étoile, France). This
was performed by growing a single colony of E. coli XL1-Blue harboring the pTP123 plasmid with
either WT, mutant CTX-M-14, or empty vector overnight in LB supplemented with 12.5
<!--l. 242--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>g/ml
CMP in a shaking incubator at 37 &#8451;. The overnight culture was diluted
10<!--l. 242--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mn>2</mn></mrow></msup 
></math> and
spread onto LB agar containing CMP, an Etest strip was placed on the agar, and the MIC was
determined based on the zone of inhibition.
</p><!--l. 244--><p class="noindent" >MIC determinations for cefotaxime were performed by broth dilution. Again, a single colony of E.
coli XL1-Blue harboring pTP123 with WT or mutant CTX-M-14 or empty vector was
grown overnight in LB with CMP in a shaking incubator at 37 &#8451;. The cultures were diluted
10<!--l. 244--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mn>4</mn></mrow></msup 
></math>, and
100 <!--l. 244--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>l
                                                                                      
                                                                                      
of culture was used to inoculate 2 ml of LB supplemented with increasing
concentrations of cefotaxime in 14-ml test tubes. Concentrations of cefotaxime (in
<!--l. 244--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>g/ml)
used for WT and D240G were 0, 1, 1.5, 2, and 3. The concentrations used for the P167S and
P167S/D240G mutants were 0, 0.1875, 0.25, 0.375, and 0.5, and for pTP123 empty vector control, the
concentrations were 0, 0.03, 0.045, and 0.06. The cultures were incubated with shaking for 18 h at
37 &#8451;. The concentration at which no visible growth was observed was reported as the
MIC.
</p><!--l. 246--><p class="noindent" >MIC determinations for ceftazidime were performed by broth dilution, with a single colony of E.
coli XL1-Blue harboring pTP123 with WT or mutant CTX-M-14 or empty vector being
grown overnight in LB with CMP, as above. Saturated cultures were diluted 104, and 25
<!--l. 246--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>l was used to
inoculate 500 <!--l. 246--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>l of
LB supplemented with increasing concentrations of ceftazidime in a deep-well 96-well plate. Concentrations of
ceftazidime (in <!--l. 246--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>g/ml)
used for WT, D240G, P167S/D240G, and pTP123 empty vector were 0, 0.12, 0.19, 0.25, 0.38, 0.5,
0.75, 1, 1.5, 2, 3, and 4; for P167S, the concentrations were 0, 0.38, 0.5, 0.75, 1, 1.5, 2, 4, 6, 8, and 12.
The 96-well plate was covered with a sterile, breathable seal (Excel Scientific, Victorville, CA) and
incubated shaking at 37 &#8451; for 18 h. The concentration at which no growth was observed
(A<!--l. 246--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mn>6</mn><mn>0</mn><mn>0</mn></mrow></msub 
> <mo 
class="MathClass-rel">&#x003C;</mo></math> 0.1)
was recorded as the MIC.
</p><!--l. 248--><p class="noindent" >
</p>
<h4 class="subsectionHead"><span class="titlemark">7.5.4    </span> <a 
 id="x11-1210007.5.4"></a>Immunoblotting.</h4>
                                                                                      
                                                                                      
<!--l. 249--><p class="noindent" >To determine the effects of the P167S/D240G mutation on steady-state protein expression, single colonies
of E. coli RB791 harboring pTP123 or the recombinant pTP123 encoding WT or mutant CTX-M-14
<!--l. 249--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
were incubated overnight with shaking in 2x YT medium supplemented with 12.5
<!--l. 249--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>g/ml
CMP at 37 &#8451;. A total of 10 ml of 2x YT medium with CMP was inoculated with 100
<!--l. 249--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>l of
overnight culture and incubated at 37 &#8451; while shaking until the A600 reached between 0.9.
The cells were pelleted, and the periplasmic proteins were extracted by osmotic shock as
described previously<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_synergistic_2018">472</a>]</span>. The proteins were fractionated by SDS-PAGE and transferred
onto a nitrocellulose membrane (GE Healthcare). The membrane was probed with a rabbit
serum raised against CTX-M-14 protein and a rabbit serum raised against maltose-binding
protein (MBP) (a gift from Dr. Anna Konovalova, University of Texas Health Science
Center at Houston), which functions as a loading control. Then the membrane was probed
with a donkey anti-rabbit secondary antibody conjugated with horseradish peroxidase (GE
Healthcare). After development of the immunoblot with the SuperSignal West Pico
chemiluminescent substrate (Thermo Fisher Scientific), the hybridization signals of CTX-M-14
<!--l. 249--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase and MBP
were quantified by densitometry using ImageJ software (National Institutes of Health). The signal for WT and mutant
CTX-M-14 <!--l. 249--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
was normalized to that for MBP.
</p><!--l. 251--><p class="noindent" >
</p>
<h4 class="subsectionHead"><span class="titlemark">7.5.5    </span> <a 
 id="x11-1220007.5.5"></a>Protein purification.</h4>
                                                                                      
                                                                                      
<!--l. 252--><p class="noindent" >WT CTX-M-14 <!--l. 252--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
and the P167S/D240G mutant were expressed from a pET28a vector in E. coli BL21(DE3) cells.
Proteins expressed in this plasmid have an N-terminal His tag with a TEV protease cleavage site. E.
coli BL21(DE3) cells harboring the CTX-M-14 plasmid were used to inoculate LB supplemented with
25 <!--l. 252--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>g/ml
kanamycin and incubated at 37 &#8451; in a shaking incubator until the A600 reached
<!--l. 252--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" > <mo 
class="MathClass-rel">~</mo></math>0.9, at
which time IPTG was added to yield a final concentration of 0.2 mM to induce protein expression.
The culture was then grown for 20 h at 23 &#8451; in a shaking incubator. The culture was centrifuged at
8000 rpm at 4 &#8451;, and the pellet was stored overnight at -80 &#8451;. The pellet was then thawed on ice
and resuspended in 30 ml of lysis buffer (20 mM HEPES, pH 7.4, 300 mM NaCl, 20 mM
imidazole). The cells were lysed using a French Press at 1250 p.s.i. and a probe sonicator,
followed by centrifugation for 50 min at 10,000 rpm and filtration of the supernatant with a
0.45-<!--l. 252--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>m
filter (EMD Millipore, Billerica, MA). Filtered lysate was then bound to a HisTrap FF column (GE
Healthcare) equilibrated with the lysis buffer. The CTX-M-14 enzyme was eluted using 20&#8211;500 mM
imidazole gradient in the lysis buffer. Pure fractions containing His-CTX-M-14 protein were pooled,
concentrated, and buffer-exchanged to the lysis buffer using 10-kDa molecular mass cutoff centrifugal
filters (EMD Millipore). 0.25 mg of TEV protease was added to the His-tagged enzyme and incubated
overnight at 4 &#8451;. TEV protease and uncleaved His-CTX-M-14 protein were removed by incubation
with nickel&#8211;Sepharose Hi-Performance beads (GE Healthcare). CTX-M-14 proteins were further
purified by gel-filtration chromatography with Superdex 75 Increase column using (20
mM HEPES, pH 7.4, 150 mM NaCl) as running buffer. The fractions corresponding to
monomer of CTX-M-14 WT or P167S/D240G mutant protein were pooled and concentrated
with 10-kDa molecular mass cutoff centrifugal filters. The purity of purified proteins was
<!--l. 252--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" > <mo 
class="MathClass-rel">&#x003E;</mo></math>95%
determined by SDS-PAGE. Their concentrations were determined by
measuring the absorbance at 280 nm with DU800 spectrophotometer
                                                                                      
                                                                                      
(BecK<!--l. 252--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math>an Coulter) and using an
extinction coefficient of 23, 950 M<!--l. 252--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">-</mo><mn>1</mn></mrow></msup 
></math>
cm<!--l. 252--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">-</mo><mn>1</mn></mrow></msup 
></math>.
</p><!--l. 254--><p class="noindent" >
</p>
<h4 class="subsectionHead"><span class="titlemark">7.5.6    </span> <a 
 id="x11-1230007.5.6"></a>Determination of thermal stabilities.</h4>
<!--l. 255--><p class="noindent" >Thermal stabilities of the WT and mutant enzymes were determined as previously described<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>]</span>. In
short, the fraction of folded protein was measured with a spectropolarimeter at 222 nm, while the
temperature was increased from 30 to 70 &#8451; at a rate of 0.01 &#8451;/s. The melting temperature
(T<!--l. 255--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math>), the
temperature midpoint of protein unfolding, was determined by fitting the data to a single Boltzmann
two-state model using GraphPad Prism 6 (San Diego, CA)<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xbrown_multiple_2010">469</a>]</span>.
</p><!--l. 257--><p class="noindent" >
</p>
<h4 class="subsectionHead"><span class="titlemark">7.5.7    </span> <a 
 id="x11-1240007.5.7"></a>Steady-state enzyme kinetic parameters.</h4>
<!--l. 258--><p class="noindent" >Michaelis&#8211;Menten steady-state kinetic parameters were measured as previously described<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>,&#x00A0;<a 
href="mainli4.html#Xmarciano_genetic_2008">485</a>]</span>.
The kinetic parameters for CTX-M-14 P167S and D240G are from Patel et al.<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>]</span>. Antibiotic
hydrolysis was measured at 30 &#8451; in 50 mM phosphate buffer (pH 7.0) containing 1
<!--l. 258--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>g/ml BSA. BSA was added
to stabilize <!--l. 258--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
                                                                                      
                                                                                      
when it is diluted to low concentration for kinetic assays. Cephalothin, cefotaxime, and
ceftazidime hydrolysis were measured at 262, 264, and 260 nm, respectively<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>]</span>.
K<!--l. 258--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math> and
k<!--l. 258--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>
were determined by fitting the initial velocities of increasing substrate concentrations to
the Michaelis&#8211;Menten equation using GraphPad Prism 6. For ceftazidime, which has a
K<!--l. 258--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math>
<!--l. 258--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" > <mo 
class="MathClass-rel">&#x003E;</mo></math> 500
<!--l. 258--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>M,
k<!--l. 258--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>/K<!--l. 258--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math>
was  estimated  using  the  equation,  v  =
k<!--l. 258--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>/K<!--l. 258--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math>[E][S],
where [S] <!--l. 258--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" > <mo 
class="MathClass-rel">&#x226A;</mo></math>
K<!--l. 258--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math>. All measurements were
performed at least in duplicate. k<!--l. 258--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>
and K<!--l. 258--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math> values
from each run were averaged, and the standard deviations reported are the sums of the percent standard
deviations of k<!--l. 258--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>c</mi><mi 
>a</mi><mi 
>t</mi></mrow></msub 
></math>
and K<!--l. 258--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>m</mi></mrow></msub 
></math><span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpatel_characterization_2015">460</a>]</span>.
</p><!--l. 260--><p class="noindent" >
</p>
<h4 class="subsectionHead"><span class="titlemark">7.5.8    </span> <a 
 id="x11-1250007.5.8"></a>Protein crystallization and structure determination.</h4>
<!--l. 261--><p class="noindent" >Crystallization conditions were screened based on previously solved crystal structures
for CTX-M-14. Purified P167S/D240G enzyme in 50 mM phosphate buffer, pH 7.0, was
                                                                                      
                                                                                      
concentrated to 40 mg/ml, and protein was mixed with mother liquor 1:1 in a 200-nl drop
and grown by hanging-drop vapor diffusion. Diffraction-quality crystals were obtained in
0.1 M MIB buffer (malonic acid, imidazole, and boric acid buffer), pH 4.0, 25% (w/v)
PEG 1500, and were harvested and cryoprotected in 25% glycerol: 75% mother liquor.
The crystals were plunged in liquid nitrogen and sent to Beamline 5.0.2 at the Advanced
Light Source (Berkeley, CA) for data collection. Because the first data set appeared to
show high twinning, a second data set was collected on the same crystal. This data set was
processed at 1.5 &#197; in the space group P41212 using HKL200 and the Phaser program
from the CCP4 suite was used for molecular replacement. CTX-M-14 (PDB code 1YLT)
was used as a phasing model<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xchen_atomic_2005">458</a>]</span>. Refinement was performed using REFMAC5 and
phenix.refine, as part of the Phenix program suite<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xadams_phenix_2010">486</a>]</span>. The model was built manually using
COOT<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xemsley_features_2010">487</a>]</span>.
</p><!--l. 263--><p class="noindent" >E166A/P167S/D240G was crystallized by concentrating the protein in 50 mM phosphate buffer, pH
7.0, to 40 mg/ml and mixing with mother liquor 1:1 in a 200-nl drop and grown by hanging-drop
vapor diffusion. Crystals from the condition containing 0.1 M PCB buffer, pH 6, 25% (w/v) PEG 1500
were soaked for 24 h in 25 mM ceftazidime, 20% glycerol:80% mother liquor. Structure determination
indicated an acyl-enzyme complex with ceftazidime. Crystals grown in the condition containing 0.2 M
CaCl, 0.1 M sodium acetate, pH 5, 20% (w/v) PEG 6000 were also soaked for 24 h in 25 mM
ceftazidime, 20% glycerol:80% mother liquor but were not in complex with ceftazidime, resulting in
the apoenzyme. The data were collected on Beamline ALS 501 and was processed as described above.
The E166A/P167S/D240G enzyme was also crystallized by concentrating the protein in 50 mM
phosphate buffer, pH 7.0, to 36 mg/ml, mixing with mother liquor 1:1 in a 200-nl drop, and grown by
hanging-drop vapor diffusion. Crystals obtained in the condition 0.1 M MMT (1:2:2 ratio of
DL-malic acid:MES:Tris base), pH 6.0, 25% (w/v) PEG 1500 were soaked for 24 h in 25 mM
ceftazidime, 25% glycerol:75% mother liquor. The data were collected on Beamline ALS
821. Structure determination revealed an acyl-enzyme complex with ceftazidime with the
<!--l. 263--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03A9;</mi></math>-loop in
                                                                                      
                                                                                      
an open conformation.
</p><!--l. 265--><p class="noindent" >E166A/D240G was crystallized by concentrating the protein in 50 mM phosphate buffer,
pH 7.0, to 36 mg/ml and mixing with mother liquor 1:1 in a 200-nl drop and grown by
hanging-drop vapor diffusion. Crystals grown in 0.1 M Tris-HCl, pH 8.5, 25% (w/v) PEG
3000 were soaked for 24 h in 25 mM ceftazidime, 25% glycerol:75% mother liquor. The
data were collected on Beamline ALS 822. However, structure determination revealed this
to be the apo enzyme. Crystals grown in the condition 0.2 M NaCl, 0.1 M Tris, pH 8.0,
20% (w/v) PEG 6000 were soaked for 24 h in 15 mM ceftazidime, 25% glycerol:75%
mother liquor, the data were collected on Beamline ALS 822, and structure determination
revealed an acyl-enzyme complex with ceftazidime. The data set was processed as described
above for the P167S/D240G enzyme. X-ray crystallography statistics are listed in Table
<a 
href="mainap5.html#x17-169001r1">E.1<!--tex4ht:ref: tab:ch2-supptable1 --></a>.
</p><!--l. 267--><p class="noindent" >
</p>
<h4 class="subsectionHead"><span class="titlemark">7.5.9    </span> <a 
 id="x11-1260007.5.9"></a>Molecular dynamics simulations.</h4>
<!--l. 268--><p class="noindent" >As described previously<span class="cite">&#x00A0;[<a 
href="mainli4.html#XHart:2016kb">9</a>]</span>, simulations were run at 300 K with the GROMACS software
package<span class="cite">&#x00A0;[<a 
href="mainli4.html#XHart:2016kb">9</a>,&#x00A0;<a 
href="mainli4.html#XBerendsen:1995dd">488</a>,&#x00A0;<a 
href="mainli4.html#XAbraham:2015gj">222</a>,&#x00A0;<a 
href="mainli4.html#XVanDerSpoel:2005hz">145</a>]</span> using the Amber03 force field<span class="cite">&#x00A0;[<a 
href="mainli4.html#XDuan:2003gt">146</a>]</span> and TIP3P explicit
solvent<span class="cite">&#x00A0;[<a 
href="mainli4.html#XJorgensen:1983fl">143</a>]</span>. Mutations were introduced in PyMOL<span class="cite">&#x00A0;[<a 
href="mainli4.html#XDeLano:2010wf">144</a>]</span>, and parameters for the acyl group
were generated with the generalized amber force field<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xsousa_da_silva_acpype_2012">489</a>,&#x00A0;<a 
href="mainli4.html#Xwang_development_2004">490</a>,&#x00A0;<a 
href="mainli4.html#Xwang_automatic_2006">491</a>]</span>. A total of 2.5
<!--l. 268--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math> s of
simulation were run for each variant.
                                                                                      
                                                                                      
</p><!--l. 273--><p class="noindent" >
</p>
<h3 class="sectionHead"><span class="titlemark">7.6    </span> <a 
 id="x11-1270007.6"></a>Author contributions</h3>
<!--l. 274--><p class="noindent" >C. A. B., B. V. V. P., G. R. B., and T. P. conceptualization; C. A. B., L.H., Z.S., M.P.P., S.S., J.R.P., and
B.S. investigation; C. A. B., L.H., Z.S., M.P.P., S.S., J.R.P., and B.S. methodology; C. A. B. and T. P.
writing-original draft; C. A. B., L.H., Z.S., M.P.P., S.S., J.R.P., B.S., B. V. V. P., G. R. B., and T. P.
writing-review and editing; L.H., Z.S., S.S., J.R.P., B.S., B. V. V. P., G. R. B., and T. P. formal analysis;
Z.S. data curation; S.S. visualization; B. V. V. P., G. R. B., and T. P. supervision; B. V. V. P., G. R. B.,
and T. P. funding acquisition.
</p><!--l. 278--><p class="noindent" >
</p>
<h3 class="sectionHead"><span class="titlemark">7.7    </span> <a 
 id="x11-1280007.7"></a>Acknowledgments</h3>
<!--l. 279--><p class="noindent" >We thank Hiram Gilbert for discussions and comments on the manuscript. The ALS-ENABLE
Beamlines are supported in part by National Institutes of Health, NIGMS Grant P30 GM124169-01.
The Advanced Light Source is a Department of Energy Office of Science User Facility under Contract
DE-AC02-05CH11231.
</p><!--l. 281--><p class="noindent" >This work was supported by National Institutes of Health Grants R01 AI32956 (to T. P.) and R01
GM12400701 (to G. R. B.) and by Robert Welch Foundation Grant Q1279 (to B. V. V. P.). The authors
declare that they have no conflicts of interest with the contents of this article. The content is solely the
responsibility of the authors and does not necessarily represent the official views of the National
Institutes of Health.
                                                                                      
                                                                                      
</p><!--l. 283--><p class="noindent" >
</p>
<h3 class="sectionHead"><span class="titlemark">7.8    </span> <a 
 id="x11-1290007.8"></a>Additional information.</h3>
<!--l. 284--><p class="noindent" >The atomic coordinates and structure factors (codes 6V5E, 6V6G, 6V6P, 6V7T, 6V8V, and 6V83)
have been deposited in the Protein Data Bank (http://wwpdb.org/).
</p>
<div class="table">
                                                                                      
                                                                                      
<!--l. 286--><p class="noindent" ><a 
 id="x11-129001r3"></a></p><hr class="float" /><div class="float" 
>
                                                                                      
                                                                                      
<a 
 id="x11-129002"></a>
 <div class="caption" 
><span class="id">Table&#x00A0;7.3: </span><span  
class="content">The abbreviations used are:</span></div><!--tex4ht:label?: x11-129001r7.8 -->
<div class="tabular"> <table id="TBL-6" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-6-1g"><col 
id="TBL-6-1" /><col 
id="TBL-6-2" /></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-6-1-"><td  style="text-align:left; white-space:nowrap;" id="TBL-6-1-1"  
class="td11"> <span 
class="ptmb7t-x-x-120">Abbreviation  </span></td><td  style="text-align:left; white-space:nowrap;" id="TBL-6-1-2"  
class="td11"> <span 
class="ptmb7t-x-x-120">Expanded name                                                                                                                                                                                 </span></td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-6-2-"><td  style="text-align:left; white-space:nowrap;" id="TBL-6-2-1"  
class="td11"> <span 
class="ptmb7t-x-x-120">MIC                 </span></td><td  style="text-align:left; white-space:nowrap;" id="TBL-6-2-2"  
class="td11"> <span 
class="ptmri7t-x-x-120">Minimum inhibitory concentration                                                                                                                                                     </span></td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-6-3-"><td  style="text-align:left; white-space:nowrap;" id="TBL-6-3-1"  
class="td11"> <span 
class="ptmb7t-x-x-120">PDB                 </span></td><td  style="text-align:left; white-space:nowrap;" id="TBL-6-3-2"  
class="td11"> <span 
class="ptmri7t-x-x-120">Protein data bank                                                                                                                                                                                </span></td></tr><tr 
class="hline"><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-6-4-"><td  style="text-align:left; white-space:nowrap;" id="TBL-6-4-1"  
class="td11"> <span 
class="ptmb7t-x-x-120">ESBL </span></td><td  style="text-align:left; white-space:nowrap;" id="TBL-6-4-2"  
class="td11"> <span 
class="ptmri7t-x-x-120">Extended spectrum </span><!--l. 294--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math><span 
class="ptmri7t-x-x-120">-lactamase</span></td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-6-5-"><td  style="text-align:left; white-space:nowrap;" id="TBL-6-5-1"  
class="td11"> <span 
class="ptmb7t-x-x-120">CMP                </span></td><td  style="text-align:left; white-space:nowrap;" id="TBL-6-5-2"  
class="td11"> <span 
class="ptmri7t-x-x-120">chloramphenicol                                                                                                                                                                                  </span></td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-6-6-"><td  style="text-align:left; white-space:nowrap;" id="TBL-6-6-1"  
class="td11"> <span 
class="ptmb7t-x-x-120">IPTG                </span></td><td  style="text-align:left; white-space:nowrap;" id="TBL-6-6-2"  
class="td11"> <span 
class="ptmri7t-x-x-120">isopropyl-</span><!--l. 296--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math><span 
class="ptmri7t-x-x-120">-D-galactopyranoside  </span></td>
</tr><tr 
class="hline"><td><hr /></td><td><hr /></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-6-7-"><td  style="text-align:left; white-space:nowrap;" id="TBL-6-7-1"  
class="td11"> <span 
class="ptmb7t-x-x-120">MBP                 </span></td><td  style="text-align:left; white-space:nowrap;" id="TBL-6-7-2"  
class="td11"> <span 
class="ptmri7t-x-x-120">maltose-binding protein                                                                                                                                                                      </span></td></tr></table>
</div>
                                                                                      
                                                                                      
</div><hr class="endfloat" />
</div>
                                                                                      
                                                                                      
<!--l. 3--><div class="crosslinks"><p class="noindent">[<a 
href="mainch8.html" >next</a>] [<a 
href="mainch6.html" >prev</a>] [<a 
href="mainch6.html#tailmainch6.html" >prev-tail</a>] [<a 
href="mainch7.html" >front</a>] [<a 
href="main.html#mainch7.html" >up</a>] </p></div>
<!--l. 3--><p class="noindent" ><a 
 id="tailmainch7.html"></a></p> 
</body></html> 
