<?xml version="1.0" encoding="iso-8859-1" ?> 
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1 plus MathML 2.0//EN" 
"http://www.w3.org/Math/DTD/mathml2/xhtml-math11-f.dtd" > 
<html xmlns="http://www.w3.org/1999/xhtml"  
> 
<head><title>4 Discovery of a cryptic allosteric site in Ebola&#8217;s &#8216;undruggable&#8217; VP35 protein using simulations and
experiments</title> 
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" /> 
<meta name="generator" content="TeX4ht (https://tug.org/tex4ht/)" /> 
<meta name="originator" content="TeX4ht (https://tug.org/tex4ht/)" /> 
<!-- html,xhtml,2,mathml --> 
<meta name="src" content="main.tex" /> 
<link rel="stylesheet" type="text/css" href="main.css" /> 
 
<script type="text/javascript" 
src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=MML_HTMLorMML" 
></script> 

<script src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
 
<script async defer src="http://hypothes.is/embed.js"></script><link rel="stylesheet" type="text/css" href="css/scale.css" /> 
<script type="text/javascript" src="js/fw.js"></script> 
<script type="text/javascript" src="js/sidebar.js"></script> 
<script type="text/javascript" src="js/bib.js"></script></head><body 
>
<!--l. 3--><div class="crosslinks"><p class="noindent">[<a 
href="mainch5.html" >next</a>] [<a 
href="mainch3.html" >prev</a>] [<a 
href="mainch3.html#tailmainch3.html" >prev-tail</a>] [<a 
href="#tailmainch4.html">tail</a>] [<a 
href="main.html#mainch4.html" >up</a>] </p></div>
<h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;4</span><br /><a 
 id="x8-530004"></a>Discovery of a cryptic allosteric site in Ebola&#8217;s &#8216;undruggable&#8217; VP35 protein using
simulations and experiments</h2>
<div class="chapterTOCS">
&#x00A0;&#x00A0;<span class="subsectionToc" >3.3.4                                           <a 
href="mainch3.html#x7-390003.3.4" id="QQ2-7-43">H1                                           and
<!--l. 59--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-sheets
are communication hubs</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >3.3.5  <a 
href="mainch3.html#x7-400003.3.5" id="QQ2-7-44">GDP  release  alters  the  structure  and  dynamics  of  the
G<!--l. 63--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-binding site</a></span>
<br />&#x00A0;<span class="sectionToc" >3.4 <a 
href="mainch3.html#x7-410003.4">Conclusion</a></span>
<br />&#x00A0;<span class="sectionToc" >3.5 <a 
href="mainch3.html#x7-420003.5">Materials and Methods</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >3.5.1 <a 
href="mainch3.html#x7-430003.5.1" id="QQ2-7-47">Molecular dynamics simulations of GDP unbinding</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >3.5.2 <a 
href="mainch3.html#x7-480003.5.2" id="QQ2-7-52">Identifying the allosteric network with CARDS</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >3.5.3 <a 
href="mainch3.html#x7-490003.5.3" id="QQ2-7-53">Markov state model construction</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >3.5.4 <a 
href="mainch3.html#x7-500003.5.4" id="QQ2-7-54">Quantifying conformational disorder</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >3.5.5 <a 
href="mainch3.html#x7-510003.5.5" id="QQ2-7-55">Identification of the rate-limiting step for GDP release</a></span>
<br />&#x00A0;<span class="sectionToc" >3.6 <a 
href="mainch3.html#x7-520003.6">Acknowledgements</a></span>
                                                                                      
                                                                                      
</div>
<!--l. 8--><p class="noindent" ><span 
class="ptmri7t-x-x-120">This chapter is adapted from the following publication:</span>
</p><!--l. 10--><p class="noindent" ><span 
class="ptmri7t-x-x-120">Cruz, M.A.</span><!--l. 10--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">*</mo></mrow></msup 
></math><span 
class="ptmri7t-x-x-120">and</span>
<span 
class="ptmri7t-x-x-120">Frederick, T.E.</span><!--l. 10--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">*</mo></mrow></msup 
></math><span 
class="ptmri7t-x-x-120">,</span>
<span 
class="ptmri7t-x-x-120">Singh, S., Vithani, N., Zimmerman, M.I., Porter, J.R., Moeder, K.E., Amarasinghe, G.K., and Bowman,</span>
<span 
class="ptmri7t-x-x-120">G.R., Discovery of a cryptic allosteric site in Ebola&#8217;s &#8217;undruggable&#8217; VP35 protein using simulations</span>
<span 
class="ptmri7t-x-x-120">and experiments. Preprint on BioRxiv https://doi.org/10.1101/2020.02.09.940510</span><span class="cite">&#x00A0;[<a 
href="mainli4.html#XCruz2020vp35">54</a>]</span>
</p><!--l. 12--><p class="noindent" ><!--l. 12--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">*</mo></mrow></msup 
></math><span 
class="ptmri7t-x-x-120">Authors</span>
<span 
class="ptmri7t-x-x-120">contributed equally to this work</span>
</p><!--l. 14--><p class="noindent" ><span 
class="ptmri7t-x-x-120">In this work, my work analyzing the allostery in VP35 and the coupling between the cryptic pocket</span>
<span 
class="ptmri7t-x-x-120">opening and binding interface is presented in figure </span><a 
href="#x8-58001r3"><span 
class="ptmri7t-x-x-120">4.3</span><!--tex4ht:ref: fig:ch5-fig3 --></a> <span 
class="ptmri7t-x-x-120">and appendix figure </span><a 
href="mainap3.html#x15-141003r2"><span 
class="ptmri7t-x-x-120">C.2</span><!--tex4ht:ref: fig:ch5-suppfig2 --></a><span 
class="ptmri7t-x-x-120">.</span>
</p>
<h3 class="sectionHead"><span class="titlemark">4.1    </span> <a 
 id="x8-540004.1"></a>Abstract</h3>
<!--l. 17--><p class="noindent" >Many proteins are classified as &#8216;undruggable,&#8217; especially those that engage in protein-protein and
protein-nucleic acid interactions. Discovering &#8216;cryptic&#8217; pockets that are absent in available structures
but open due to protein dynamics could provide new druggable sites. Here, we integrate simulations
and experiments to search for cryptic pockets in Ebola viral protein 35 (VP35). VP35 plays
essential roles in Ebola&#8217;s replication cycle, including binding the viral RNA genome to block a
host&#8217;s innate immunity. However, VP35 has so far proved undruggable. Using adaptive
sampling simulations and allosteric network detection algorithms, we uncover a cryptic pocket
that is allosterically coupled to VP35&#8217;s key RNA-binding interface. Experimental tests
corroborate the predicted pocket and confirm that stabilizing the open form allosterically
disrupts RNA binding. These results demonstrate simulations&#8217; power to characterize hidden
                                                                                      
                                                                                      
conformations and dynamics, uncovering cryptic pockets and allostery that present new therapeutic
opportunities.
</p><!--l. 19--><p class="noindent" >
</p>
<h3 class="sectionHead"><span class="titlemark">4.2    </span> <a 
 id="x8-550004.2"></a>Introduction</h3>
<!--l. 20--><p class="noindent" >Many proteins have proved so difficult to target with small molecule drugs that they are often
classified as undruggable, greatly limiting the scope of drug design efforts. In fact, up to 85% of
human proteins have been classified as undruggable because their folds are thought to lack binding
pockets where small molecules can bind with the affinity and specificity required for drug
design<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xhopkins_druggable_2002">239</a>]</span>. Many undruggable proteins predominantly participate in protein-protein interactions
(PPIs) and protein-nucleic acid interactions (PNIs)<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xarkin_small-molecule_2004">72</a>,&#x00A0;<a 
href="mainli4.html#Xjohnson_computational_2017">240</a>]</span>. In contrast to the binding pockets that
many enzymes and receptors use to bind their small molecule ligands, the large flat interfaces involved
in PPIs and PNIs do not lend themselves to forming many favorable interactions with small drug-like
molecules. As a result, PPIs and PNIs are often considered intractable targets even when
there is strong evidence that disrupting these interactions would be of great therapeutic
value.
</p><!--l. 22--><p class="noindent" >Cryptic pockets could provide new opportunities to target undruggable proteins<span class="cite">&#x00A0;[<a 
href="mainli4.html#XKnoverek:2018kk">41</a>,&#x00A0;<a 
href="mainli4.html#Xvajda_cryptic_2018">241</a>]</span>, but
realizing this potential remains challenging. Such pockets are absent in available experimental
structures because they only form in a subset of excited states that arise due to protein dynamics.
Cryptic sites can serve as valuable drug targets if they coincide with key functional sites, as can
cryptic allosteric sites that are coupled to distant functional sites. However, identifying cryptic pockets
remains difficult. Most known cryptic sites were only identified after the serendipitous
discovery of a small molecule that binds and stabilizes the open form of the pocket<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xvajda_cryptic_2018">241</a>,&#x00A0;<a 
href="mainli4.html#Xhardy_searching_2004">74</a>]</span>.
                                                                                      
                                                                                      
Experimental techniques for intentionally identifying and targeting cryptic pockets show great
promise<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xerlanson_site-directed_2000">242</a>,&#x00A0;<a 
href="mainli4.html#Xostrem_k-rasg12c_2013">77</a>,&#x00A0;<a 
href="mainli4.html#Xkeedy_expanded_2018">243</a>]</span>, but they still leverage the simultaneous discovery of ligands that bind and
stabilize the open pocket. To overcome this limitation, a number of computational methods have
been developed to identify cryptic pockets without requiring the simultaneous discovery of
small molecules that bind them<span class="cite">&#x00A0;[<a 
href="mainli4.html#XBowman:2012jd">47</a>,&#x00A0;<a 
href="mainli4.html#XPorter:2019hv">49</a>,&#x00A0;<a 
href="mainli4.html#Xjohnson_druggable_2013">244</a>,&#x00A0;<a 
href="mainli4.html#Xoleinikovas_understanding_2016">245</a>,&#x00A0;<a 
href="mainli4.html#Xcimermancic_cryptosite:_2016">246</a>,&#x00A0;<a 
href="mainli4.html#Xschmidt_cosolvent-enhanced_2019">247</a>,&#x00A0;<a 
href="mainli4.html#Xcuchillo_collective_2015">248</a>,&#x00A0;<a 
href="mainli4.html#Xghanakota_moving_2016">249</a>]</span>. These methods
have proved capable of retrodicting a number of previously identified cryptic pockets.
More importantly, applications to a few established drug targets and other enzymes have
successfully identified novel cryptic pockets that have been corroborated by subsequent
experiments<span class="cite">&#x00A0;[<a 
href="mainli4.html#XPorter:2019hv">49</a>,&#x00A0;<a 
href="mainli4.html#Xwassman_computational_2013">250</a>,&#x00A0;<a 
href="mainli4.html#Xschames_discovery_2004">79</a>]</span>.
</p><!--l. 24--><p class="noindent" >Here, we integrate atomically-detailed computer simulations and biophysical experiments to explore
the potential utility of cryptic pockets in a target that has so far proved undruggable: the interferon
inhibitory domain (IID) of Ebola viral protein 35 (VP35). Ebola virus causes a hemorrhagic fever that
is often lethal, with case fatality rates approaching 90% in past outbreaks<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xkeshwara_toward_2017">251</a>,&#x00A0;<a 
href="mainli4.html#Xmulangu_randomized_2019">252</a>]</span>.
While recent progress in vaccine development and use of biologics, such as antibodies, for
therapeutic and prophylactic purposes show promise<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xmulangu_randomized_2019">252</a>]</span>, small molecule drugs still
offer many advantages, including ease of delivery, lower cost, and longer shelf life. The
1&#x0303;20 residue IID of VP35 is a particularly appealing drug target for combating Ebola and
other viruses in the Filoviridae family given that it has a well-conserved sequence and
plays multiple essential roles in the viral lifecycle<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xmessaoudi_filovirus_2015">253</a>]</span>. One of its primary functions is to
antagonize a host&#8217;s innate immunity, particularly RIG-I-like receptor (RLR)-mediated
detection of viral nucleic acids, to prevent an interferon (IFN) response and signaling of
neighboring cells to heighten their antiviral defenses<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xcardenas_ebola_2006">254</a>,&#x00A0;<a 
href="mainli4.html#Xbasler_ebola_2003">255</a>,&#x00A0;<a 
href="mainli4.html#Xhartman_c-terminal_2004">256</a>]</span>. Crystal structures have
provided a foundation for understanding much about the mechanism of VP35-mediated IFN
antagonism<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xleung_structure_2009">257</a>,&#x00A0;<a 
href="mainli4.html#Xleung_structural_2010">258</a>]</span>. For example, they have revealed that VP35&#8217;s IID binds both the blunt
ends and backbone of double-stranded RNA (dsRNA), and that there is a PPI between
these dsRNA-binding modes (Fig. <a 
href="#x8-55001r1">4.1<!--tex4ht:ref: fig:ch5-fig1 --></a>)<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xleung_structural_2010">258</a>]</span>. Disrupting any of these interactions could
potentially render Ebola susceptible to a host&#8217;s innate immunity. In particular, binding
                                                                                      
                                                                                      
to dsRNA blunt ends plays a dominant role in IFN suppression by Ebola<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xedwards_differential_2016">259</a>]</span>. Indeed,
mutations that reduce the IID&#8217;s affinity for dsRNA blunt ends are sufficient to mitigate
IFN antagonism, ultimately attenuating Ebola&#8217;s pathogenicity<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xedwards_differential_2016">259</a>,&#x00A0;<a 
href="mainli4.html#Xhartman_reverse_2006">260</a>,&#x00A0;<a 
href="mainli4.html#Xprins_mutations_2010">261</a>,&#x00A0;<a 
href="mainli4.html#Xprins_basic_2010">262</a>]</span>. So,
disrupting this single binding mode could dramatically reduce the impact of an Ebola infection
on the host and potentially reduce deleterious effects, including lethality. However, both
dsRNA-binding interfaces are large flat surfaces that are difficult for small molecules to bind tightly
(Fig. <a 
href="#x8-55001r1">4.1<!--tex4ht:ref: fig:ch5-fig1 --></a>). As a result, [there are no approved therapies targeting VP35]. Both docking and
screening attempts to discover small molecules that bind these interfaces have not yielded
sufficiently strong leads to warrant clinical development<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xbrown_silico_2014">263</a>,&#x00A0;<a 
href="mainli4.html#Xglanzer_silico_2016">264</a>]</span>. The discovery of
cryptic pockets in VP35 could provide new opportunities for drugging this essential viral
component.
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                      
                                                                                      
<a 
 id="x8-55001r1"></a>
                                                                                      
                                                                                      
<!--l. 28--><p class="noindent" ><img 
src="Chapter5/ch5-fig1.png" alt="PIC"  
width="180" height="180"  />
<a 
 id="x8-55002"></a>
<br /> </p><div class="caption" 
><span class="id">Figure&#x00A0;4.1: </span><span  
class="content">Crystal structure of two copies of VP35&#8217;s IID (dark gray) bound to dsRNA (light
gray) via two flat interfaces (PDB ID 3L25). The backbone-binding interface (blue) and blunt
end-binding interface (orange) are shown as spheres to highlight that they lack deep pockets
amenable to binding small molecules.</span></div><!--tex4ht:label?: x8-55001r4.2 -->
                                                                                      
                                                                                      
</div><hr class="endfigure" />
<h3 class="sectionHead"><span class="titlemark">4.3    </span> <a 
 id="x8-560004.3"></a>Results</h3>
<!--l. 36--><p class="noindent" >
</p>
<h4 class="subsectionHead"><span class="titlemark">4.3.1    </span> <a 
 id="x8-570004.3.1"></a>Computer simulations reveal a potentially druggable cryptic pocket.</h4>
<!--l. 37--><p class="noindent" >We applied our fluctuation amplification of specific traits (FAST) simulation algorithm<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xzimmerman_fast_2015">35</a>]</span> to
enhance sampling of structures with large pocket volumes that may harbor cryptic pockets. FAST is a
goal-oriented adaptive sampling algorithm that exploits Markov state model (MSM) methods to focus
computational resources on exploring regions of conformational space with user-specified structural
features. An MSM is a network model of a protein&#8217;s energy landscape which consists of a set of
structural states the protein adopts and the rates of hopping between them<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xpande_everything_2010">265</a>,&#x00A0;<a 
href="mainli4.html#Xprinz_markov_2011">53</a>]</span>. Adaptive
sampling algorithms enable efficient construction of MSMs by iteratively 1) running a batch of
simulations, 2) building an MSM, and 3) selecting a subset of the states that have been
identified so far as starting points for the next batch of simulations to maximize the chances
of improving the model<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xbowman_network_2010">266</a>,&#x00A0;<a 
href="mainli4.html#XHruska:2018ec">267</a>]</span>. FAST selects which states to further simulate in a
manner that balances exploration/exploitation tradeoffs by considering 1) how well each state
optimizes a user defined structural criterion (in this case maximizing the total pocket volume)
and 2) the likelihood of discovering new conformational states<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xzimmerman_fast_2015">35</a>]</span>. After running FAST,
we collected additional simulation data by launching each state on the Folding@home
distributed computing environment, which brings together the computing resources of tens of
thousands of citizen scientists who volunteer to run simulations on their personal computers.
                                                                                      
                                                                                      
Our final model has 11,891 conformational states, providing a detailed characterization
of the different structures the IID adopts but making manual interpretation of the model
difficult.
</p><!--l. 39--><p class="noindent" >To identify cryptic pockets within the large ensemble captured by our MSM, we applied our exposons
analysis pipeline<span class="cite">&#x00A0;[<a 
href="mainli4.html#XPorter:2019hv">49</a>]</span>. An exposon is a cluster of residues that undergo cooperative changes in their
solvent exposure. Coupling between the solvent exposure of every pair of residues is quantified using
a mutual information metric, as described in Methods. Exposons are often associated with cryptic sites
because the opening/closing of such pockets gives rise to cooperative increases/decreases in the
solvent exposure of surrounding residues. Importantly, once an exposon has been identified, our MSM
framework provides a facile means to identify the conformational changes that give rise to that
exposon.
</p><!--l. 41--><p class="noindent" >The IID has two significant exposons, one of which corresponds to a large cryptic pocket. The blue
exposon (Fig. <a 
href="#x8-57001r2">4.2<!--tex4ht:ref: fig:ch5-fig2 --></a>A and <a 
href="#x8-57001r2">4.2<!--tex4ht:ref: fig:ch5-fig2 --></a>B) consists of a set of strongly-coupled residues in helix 7 and adjacent
loops and secondary structure elements. Visualizing the conformational change that gives rise to
this exposon reveals a substantial displacement of helix 7, creating a large cryptic pocket
between it and the helical domain (Fig. <a 
href="#x8-57001r2">4.2<!--tex4ht:ref: fig:ch5-fig2 --></a>C). A number of residues that are displaced along
with helix 7 (i.e. A306, K309, and S310) make van der Waals contacts with the dsRNA
backbone in the dsRNA-bound crystal structure<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xleung_structural_2010">258</a>]</span>, so targeting this cryptic pocket could
directly disrupt this binding mode. Retrospective analysis of other validated drug targets
suggests cryptic sites created by the movement of secondary structure elements, such as the
displacement of helix 7, are often druggable<span class="cite">&#x00A0;[<a 
href="mainli4.html#XBeglov:2018dm">268</a>]</span>. The potential druggability of this cryptic
site is also supported by application of the FTMap algorithm<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xngan_ftmap:_2012">269</a>,&#x00A0;<a 
href="mainli4.html#Xkozakov_ftmap_2015">270</a>]</span>, which predicts
a number of hotspots within the pocket where small molecules could form a variety of
energetically-favorable interactions (<a 
href="mainap3.html#x15-141001r1">C.1<!--tex4ht:ref: fig:ch5-suppfig1 --></a>). Unfortunately, disrupting backbone binding is of less
therapeutic utility than disrupting blunt end binding and it is unknown whether the contacts
between A306, K309, and S310 are essential for backbone binding. Therefore, it is unclear
from this analysis alone whether drugging this newly discovered cryptic pocket would be
                                                                                      
                                                                                      
useful.
</p><!--l. 43--><p class="noindent" >The second exposon (orange in Fig. <a 
href="#x8-57001r2">4.2<!--tex4ht:ref: fig:ch5-fig2 --></a>) encompasses portions of both dsRNA-binding interfaces, but
it does not correspond to a cryptic pocket. This exposon includes residues that bind dsRNA&#8217;s
backbone (i.e. S272) and residues that interact with both the blunt ends and backbone of dsRNA (i.e.
F239, Q274, and I340)<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xleung_structural_2010">258</a>]</span>. Therefore, altering the conformational preferences of the second
exposon could potentially disrupt the blunt end-binding mode and its crucial role in Ebola&#8217;s
ability to evade an immune response. However, the largest conformational change involved
in the formation of this exposon is a displacement of the loop between helices 3 and 4
(Fig. <a 
href="#x8-57001r2">4.2<!--tex4ht:ref: fig:ch5-fig2 --></a>D). This rearrangement does not create a cryptic pocket that is large enough to
accommodate drug-like molecules, so it is not obvious how to directly manipulate the orange
exposon.
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                      
                                                                                      
<a 
 id="x8-57001r2"></a>
                                                                                      
                                                                                      
<!--l. 47--><p class="noindent" ><img 
src="Chapter5/ch5-fig2.png" alt="PIC"  
width="216" height="316"  />
<a 
 id="x8-57002"></a>
<br /> </p><div class="caption" 
><span class="id">Figure&#x00A0;4.2: </span><span  
class="content">Exposons identify a large cryptic pocket and suggest potential allosteric coupling.
<span 
class="ptmb7t-x-x-120">A.  </span>Structure  of  VP35&#8217;s  IID  highlighting  residues  in  two  exposons  (blue  and  orange),  the
N-terminus  (N-term),  and  C-terminus  (I340).  <span 
class="ptmb7t-x-x-120">B.  </span>Network  representation  of  the  coupling
between the solvent exposure of residues in the two exposons. The edge width between residues
is proportional to the mutual information between them. <span 
class="ptmb7t-x-x-120">C. </span>Structure highlighting the opening
of  a  cryptic  pocket  via  the  displacement  of  helix  7  that  gives  rise  to  the  blue  exposon.  <span 
class="ptmb7t-x-x-120">D.</span>
Structure highlighting the conformational change that gives rise to the orange exposon overlaid
on the crystal structure (gray) to highlight that the rearrangements are subtler than in the blue
exposon.</span></div><!--tex4ht:label?: x8-57001r4.3 -->
                                                                                      
                                                                                      
</div><hr class="endfigure" />
<h4 class="subsectionHead"><span class="titlemark">4.3.2    </span> <a 
 id="x8-580004.3.2"></a>The cryptic pocket is allosterically coupled to the blunt end-binding interface.</h4>
<!--l. 54--><p class="noindent" >Even though the cryptic pocket does not coincide with the interface of VP35&#8217;s IID that binds dsRNA
blunt ends, it could still serve as a cryptic allosteric site that allosterically controls RNA
binding. Indeed, the physical proximity of the two exposons and the coupling between them
both hint at the possibility for allosteric coupling. Furthermore, our exposons analysis
could easily underestimate this coupling given that it focuses on correlated transitions of
residues between solvent exposed and completely buried states, leaving it blind to more subtle
conformational fluctuations and the coupling of residues that are always buried (or always
exposed).
</p><!--l. 56--><p class="noindent" >To explore the potential for a broader allosteric network, we applied the correlation of all rotameric
and dynamical states (CARDS) algorithm<span class="cite">&#x00A0;[<a 
href="mainli4.html#XSingh:2017hh">90</a>]</span>. CARDS classifies each dihedral in each
snapshot of a simulation as being in one of three rotameric states (gauche+, gauche-, or
trans) and one of two dynamical states (ordered or disordered). A dihedral is said to be
disordered if it is rapidly hopping between different structural states, and it is classified as
ordered if it appears to be locked into a single rotameric state for a prolonged time. The
mutual information metric is then used to quantify how strongly coupled the structural and
dynamical states of each pair of dihedrals are, enabling CARDS to capture the roles of both
concerted structural changes and conformational entropy in allosteric communication.
Importantly, CARDS accounts for the potential role of residues that are always buried or always
exposed to solvent and subtle conformational changes that do not alter the solvent exposure of
residues.
</p><!--l. 58--><p class="noindent" >CARDS reveals a broader allosteric network than that identified by our exposons analysis and
suggests strong coupling between the cryptic pocket and blunt end-binding interface (Fig. <a 
href="#x8-58001r3">4.3<!--tex4ht:ref: fig:ch5-fig3 --></a>). This
                                                                                      
                                                                                      
network consists of five communities of strongly coupled residues, four of which coincide with large
portions of the two dsRNA-binding interfaces. One of these communities (orange) is a hub in the
network, having significant coupling to all the other communities. It encompasses part of the
orange exposon, particularly residues around the loop between helices 3 and 4. The orange
CARDS community and exposon both capture Q274, which engages in both dsRNA-binding
interfaces, and S272, which contacts the backbone<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xleung_structural_2010">258</a>]</span>. However, the CARDS community
includes many additional residues not captured by exposons analysis. Examples include
I278, which engages in both dsRNA-binding interfaces, and D271, which is part of the PPI
between the two binding modes<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xleung_structural_2010">258</a>]</span>. One of the orange community&#8217;s strongest allosteric
connections is to the green community. This community encompasses the rest of the residues
in the orange exposon, including F239 and I340, which are part of both dsRNA-binding
interfaces<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xleung_structural_2010">258</a>]</span>. The green community also captures additional residues, reaching deep into the
helical domain. The orange community is also strongly coupled to the blue community,
which includes much of helix 7 and nearby residues that move to give rise to the cryptic
pocket that was captured by the blue exposon. Notably, the orange and blue communities are
both coupled to a cyan cluster that was not hinted at by our exposons analysis because the
residues involved are always solvent exposed. It includes R322, which is part of the blunt
end-binding interface and the PPI between the two binding modes, and K282, which also
contacts dsRNA blunt ends<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xleung_structural_2010">258</a>]</span>. In addition, this community includes K339, which is an
important determinant of the electrostatic favorability of dsRNA binding<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xleung_structural_2010">258</a>]</span>. Together,
these results suggest that opening of the cryptic pocket could strongly impact residues
involved in both dsRNA-binding interfaces, as well as the PPI between the two binding
modes.
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                      
                                                                                      
<a 
 id="x8-58001r3"></a>
                                                                                      
                                                                                      
<!--l. 62--><p class="noindent" ><img 
src="Chapter5/ch5-fig3.png" alt="PIC"  
width="289" height="392"  />
<a 
 id="x8-58002"></a>
<br /> </p><div class="caption" 
><span class="id">Figure&#x00A0;4.3:  </span><span  
class="content">eVP35  allosteric  network  revealed  by  the  CARDS  algorithm.  <span 
class="ptmb7t-x-x-120">A.  </span>Structure  of
VP35&#8217;s  IID  with  residues  in  the  allosteric  network  shown  in  sticks  and  colored  according
to  which  of  five  communities  they  belong  to.  Substitution  of  residues  labeled  in  red  with
alanine disrupts binding to dsRNA blunt ends and results in a dramatic reduction in immune
suppression.  <span 
class="ptmb7t-x-x-120">B.  </span>Network  representation  of  the  coupling  between  communities  of  residues,
colored  as  in  A.  Node  size  is  proportional  to  the  strength  of  coupling  between  residues  in
the  community,  and  edge  widths  are  proportional  to  the  strength  of  coupling  between  the
communities.  <span 
class="ptmb7t-x-x-120">C.  </span>Free  energy  landscape  of  the  orange  exposon  projected  onto  the  first  two
principal components, PC1 and PC2, highlighting the centroid structures of the free energy
minimum (gray circle) and excited state (orange circle). <span 
class="ptmb7t-x-x-120">D. </span>Structures of the centroids (colored
as in panel C) capture opening of the cryptic pocket and rearrangements involving key residues
for PPIs and PNIs.</span></div><!--tex4ht:label?: x8-58001r4.3 -->
                                                                                      
                                                                                      
</div><hr class="endfigure" />
<!--l. 69--><p class="noindent" >To determine the relative importance of the structural and dynamical preferences of this
community, we compared the magnitudes of the structural and dynamical components of
CARDS. This analysis revealed that concerted structural changes are the dominant mode of
allosteric communication in the IID, rather than conformational entropy and dynamical
allostery (<a 
href="mainap3.html#x15-141003r2">C.2<!--tex4ht:ref: fig:ch5-suppfig2 --></a>). Therefore, examining structures where the orange community undergoes large
conformational changes might reveal the perturbations these motions induce elsewhere in the
protein.
</p><!--l. 71--><p class="noindent" >To understand the potential impact of targeting the cryptic pocket on the blunt end-binding mode, we
performed a dimensionality reduction based on the orange community. Since the orange
community is a hub in the allosteric network, we reasoned that performing a dimensionality
reduction based on the structural preferences of this community and examining representative
structures would report on what is happening throughout the protein. To understand what sort
of conformational changes are present, we performed a dimensionality reduction on our
simulation dataset by applying principal component analysis (PCA) to the distances between the
C<!--l. 71--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>
atoms of every pair of residues in the orange community. Projecting our MSM onto the first two
principal components (PC1 and PC2) reveals one dominant free energy minimum and a broad excited
state (Fig. <a 
href="#x8-58001r3">4.3<!--tex4ht:ref: fig:ch5-fig3 --></a>C).
</p><!--l. 73--><p class="noindent" >Comparing representative structures for the orange community&#8217;s two dominant states suggests the
cryptic pocket is indeed a cryptic allosteric site, targeting of which could allosterically disrupt binding
of VP35&#8217;s IID to dsRNA blunt ends. Most importantly, conformational changes of the orange
community are associated with opening of the cryptic pocket (Fig. <a 
href="#x8-58001r3">4.3<!--tex4ht:ref: fig:ch5-fig3 --></a>D). Therefore, targeting
the cryptic pocket could modulate the entire allosteric network in addition to its potential
direct effect on the backbone-binding mode. Comparing the structures also reveals that
the end of helix 4 frays and the preceding loop, which sits at the PPI between the two
                                                                                      
                                                                                      
dsRNA-binding modes, is displaced. So, targeting the cryptic pocket could allosterically
modulate this PPI. Finally, we note a substantial reshuffling of residues F239, H231, and
P273 and modest displacements of R322 and K339. Previous work has demonstrated that
F239A, R322A, and K339A substitutions are each sufficient to disrupt dsRNA binding
and IFN suppression<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xleung_structural_2010">258</a>]</span>. CARDS analysis suggests targeting the cryptic pocket could
allosterically alter the structures of these residues and have a similar impact on dsRNA
binding.
</p>
<h4 class="subsectionHead"><span class="titlemark">4.3.3    </span> <a 
 id="x8-590004.3.3"></a>Thiol labeling experiments corroborate the predicted cryptic pocket.</h4>
<!--l. 76--><p class="noindent" >One way to experimentally test our prediction of a cryptic pocket is to probe for solvent exposure of
residues that are buried in available crystal structures but become exposed to solvent upon pocket
opening. Cysteines are particularly appealing candidates for such experiments because 1) they have a
low abundance and 2) their thiol groups are highly reactive, so it is straightforward to detect
exposed cysteines by introducing labeling reagents that covalently bind accessible thiols.
Fortuitously, VP35&#8217;s IID has two cysteines (C307 and C326) that are buried in available
crystal structures but become exposed to solvent when the cryptic pocket opens (Fig. <a 
href="#x8-59001r4">4.4<!--tex4ht:ref: fig:ch5-fig4 --></a>A).
There is also a cysteine (C275) that is on the surface of the apo crystal structure<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xleung_structure_2009">257</a>]</span>
and a fourth cysteine (C247) that is buried in the helical bundle. C275 is typically solvent
exposed in our simulations, as expected based on the crystallographic data. Examining
the solvent exposure of C247 revealed it is sometimes exposed to solvent via an opening
of helix 1 relative to the rest of the helical bundle (<a 
href="mainap3.html#x15-141005r3">C.3<!--tex4ht:ref: fig:ch5-suppfig3 --></a>), but FTMap did not identify any
hotspots that are likely to bind drug-like molecules in this region. Therefore, we expect to
observe labeling of all four cysteines on a timescale that is faster than global unfolding of the
protein.
                                                                                      
                                                                                      
</p><!--l. 78--><p class="noindent" >To experimentally test our predicted pocket, we applied a thiol labeling technique that probes the
solvent exposure of cysteine residues<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xbernstein_structural_2011">271</a>]</span>. For these experiments, 5,5&#8217;-Dithiobis-(2-Nitrobenzoic
Acid) (also known as DTNB or Ellman&#8217;s reagent, Fig. <a 
href="#x8-59001r4">4.4<!--tex4ht:ref: fig:ch5-fig4 --></a>B) is added to a protein sample.
Upon reaction with the thiol group of an exposed cysteine, DTNB breaks into two TNB
molecules, one of which remains covalently bound to the cysteine while the other is released
into solution. The accumulation of free TNB can be quantified based on the increased
absorbance at 412 nm. We have previously applied this technique to test predicted pockets in
<!--l. 78--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
enzymes<span class="cite">&#x00A0;[<a 
href="mainli4.html#XPorter:2019hv">49</a>,&#x00A0;<a 
href="mainli4.html#XBowman:2015dh">147</a>]</span>.
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                      
                                                                                      
<a 
 id="x8-59001r4"></a>
                                                                                      
                                                                                      
<!--l. 82--><p class="noindent" ><img 
src="Chapter5/ch5-fig4.png" alt="PIC"  
width="278" height="260"  />
<a 
 id="x8-59002"></a>
<br /> </p><div class="caption" 
><span class="id">Figure&#x00A0;4.4: </span><span  
class="content">Thiol labeling supports the existence of the predicted cryptic pocket. Structure
of  VP35&#8217;s  IID  highlighting  the  locations  of  the  four  native  cysteines  (sticks).  C307  and
C326 are both buried and point into the proposed cryptic pocket. B) Structure of the DTNB
labeling  reagent.  C)  Observed  labeling  rates  (circles)  at  a  range  of  DTNB  concentrations.
Fits  to  the  Linderstr&#248;m-Lang  model  are  shown  in  dashed  colored  lines  and  the  expected
labeling rate from the unfolded state is shown as black dotted lines. The mean and standard
deviation from three replicates are shown but error bars are generally smaller than the symbols.
Labeling  for  C275  is  not  shown  because  it  is  surface  exposed  in  both  the  available  crystal
structures  and  our  simulations,  and  it  behaves  as  expected  (labeling  rate  greater  than  1
s<!--l. 84--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">-</mo><mn>1</mn></mrow></msup 
></math>
with a linear dependence on [DTNB]).</span></div><!--tex4ht:label?: x8-59001r4.3 -->
                                                                                      
                                                                                      
</div><hr class="endfigure" />
<!--l. 88--><p class="noindent" >As expected from our computational model, the observed signal from our thiol labeling experiments is
consistent with opening of the cryptic pocket (Fig. <a 
href="#x8-59001r4">4.4<!--tex4ht:ref: fig:ch5-fig4 --></a>C). Absorbance curves are best fit by four
exponentials, each with an approximately equivalent amplitude that is consistent with expectations
based on the extinction coefficient for DTNB (<a 
href="mainap3.html#x15-141007r4">C.4<!--tex4ht:ref: fig:ch5-suppfig4 --></a>). To assign these labeling rates to individual
cysteines, we systematically mutated the cysteines to serines, performed thiol labeling experiments,
and assessed which rates disappeared and which remained (<a 
href="mainap3.html#x15-141011r6">C.6<!--tex4ht:ref: fig:ch5-suppfig6 --></a>). For example, labeling of the C275S
variant lacks the very fastest rate for wild-type, consistent with the intuition that a residue that is
surfaced exposed in the crystal structure (i.e. C275) should label faster than residues that are generally
buried. To test whether the observed labeling could be due to an alternative process, such as
global unfolding, we determined the population of the unfolded state and unfolding rate of
VP35&#8217;s IID under native conditions (<a 
href="mainap3.html#x15-141017r7">C.7<!--tex4ht:ref: fig:ch5-suppfig7 --></a>) and the intrinsic labeling rate for each cysteine
(<a 
href="mainap3.html#x15-141011r6">C.6<!--tex4ht:ref: fig:ch5-suppfig6 --></a>). As shown in Fig. <a 
href="#x8-59001r4">4.4<!--tex4ht:ref: fig:ch5-fig4 --></a>C, the observed labeling rates are all considerably faster than the
expected labeling rate from the unfolded state at a range of DTNB concentrations. This result
confirms that labeling of all four cysteines arises from fluctuations within the native state,
consistent with our computational predictions. Furthermore, the exposure of C247 is far
rarer than C307 or C326 (equilibrium constants for the exposure of C247 and C307 are
5.4<span 
class="ptmr8c-x-x-120">&#x00D7;</span>10<!--l. 88--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">-</mo><mn>4</mn></mrow></msup 
></math><!--l. 88--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math>8.1<!--l. 88--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#x00D7;</mo></math>10<!--l. 88--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">-</mo><mn>6</mn></mrow></msup 
></math>
and 8.5<!--l. 88--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#x00D7;</mo></math>10<!--l. 88--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">-</mo><mn>2</mn></mrow></msup 
></math><!--l. 88--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math>2.8<!--l. 88--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#x00D7;</mo></math>10<!--l. 88--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msup><mrow 
></mrow><mrow 
><mo 
class="MathClass-bin">-</mo><mn>3</mn></mrow></msup 
></math>,
respectively). Therefore, a ligand would have to pay a greater energetic cost to stabilize the
conformational change that exposes C247 than to stabilize the open state of the cryptic allosteric site
created by the motion of helix 7.
</p>
<h4 class="subsectionHead"><span class="titlemark">4.3.4    </span> <a 
 id="x8-600004.3.4"></a>Stabilizing the open cryptic pocket allosterically disrupts binding to dsRNA blunt
ends.</h4>
                                                                                      
                                                                                      
<!--l. 91--><p class="noindent" >We reasoned that covalent attachment of TNB to C307 and C326 would provide a means to capture
the open pocket and assess the impact of stabilizing this state on dsRNA binding. Addition of TNB to
these cysteines is sterically incompatible with the closed conformation of VP35&#8217;s IID that has been
observed crystallographically. TNB&#8217;s mass of 1&#x0303;98 Da is also similar to many drug fragments used in
screening campaigns, making it a reasonable surrogate for the type of effect one might achieve with a
fragment hit. Given that we already know DTNB labels the IID&#8217;s cysteines, a TNB-labeled sample is
easily obtainable by waiting until the labeling reaction goes to completion. Finally, we have previously
used this same strategy to identify cryptic pockets that exert allosteric control over the activity of
<!--l. 91--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-lactamase
enzymes<span class="cite">&#x00A0;[<a 
href="mainli4.html#XPorter:2019hv">49</a>,&#x00A0;<a 
href="mainli4.html#XBowman:2015dh">147</a>]</span>. To ensure that we primarily capture the effect of labeling on pocket
opening, we used a C247S/C275S variant of VP35&#8217;s IID that only has cysteines pointing
into the cryptic pocket. As with the wild-type protein, thiol labeling of the C247S/C275S
variant is consistent with the formation of the proposed cryptic pocket (Supplementary Fig.
<a 
href="mainap3.html#x15-141009r5">C.5<!--tex4ht:ref: fig:ch5-suppfig5 --></a>).
</p><!--l. 93--><p class="noindent" >To measure the effect of TNB labeling on the IID&#8217;s interaction with dsRNA, we developed a
fluorescence polarization (FP) assay for monitoring dsRNA binding. Paralleling our past work on
VP35-peptide interactions<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xliu_sensitive_2017">272</a>]</span>, we added varying concentrations of C247S/C275S IID to a fixed
concentration of 25-bp dsRNA with a fluorescein isothiocyanate (FITC) conjugation at one end (<a 
href="mainap3.html#x15-141017r7">C.7<!--tex4ht:ref: fig:ch5-suppfig7 --></a>).
Free FITC-dsRNA emits depolarized light upon excitation with polarized light because of the
molecule&#8217;s fast rotation. Binding of one or more VP35 molecules restricts the motion of FITC-dsRNA,
resulting in greater emission of polarized light, which is best monitored by the change in
anisotropy<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xkeck_ssbdna_2012">273</a>]</span>.
</p><!--l. 95--><p class="noindent" >Monitoring the binding of unlabeled protein to 25-bp dsRNA with either blunt ends
or 3&#8217; overhangs demonstrates that our FP assay is sensitive to both dsRNA-binding
modes and gives affinities that are consistent with past work. Past work using a
dot-blot assay to measure binding reported an apparent dissociation constant
(K<!--l. 95--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>d</mi></mrow></msub 
></math>) for blunt-ended
                                                                                      
                                                                                      
dsRNA of 3.4<!--l. 95--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math>0.07
<!--l. 95--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math>M<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xedwards_differential_2016">259</a>]</span>.
Furthermore, sterically hindering binding of the IID to dsRNA blunt ends by adding 2-nucleotide
overhangs to the 3&#8217; of the RNA reduces the apparent dsRNA-binding affinity by 10-fold<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xramanan_structural_2012">274</a>]</span>. This
weaker interaction was attributed to the backbone-binding mode since it is still available to VP35&#8217;s IID
even when the presence of an overhang inhibits blunt end binding. Similarly, our FP assay gives an
apparent K<!--l. 95--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>d</mi></mrow></msub 
></math>
of 3.6<!--l. 95--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math>0.34
<!--l. 95--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>M for
blunt-ended dsRNA (Fig. 5A). Addition of 3&#8217; overhangs results in a strong rightwards shift of the
binding curve, consistent with at least a 5-fold reduction in the apparent binding affinity (apparent
K<!--l. 95--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
></mrow><mrow 
><mi 
>d</mi></mrow></msub 
></math> of
20.4<!--l. 95--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math>1.1
<!--l. 95--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>M).
However, an upper baseline could not be captured due to limitations in the protein&#8217;s solubility, so
this apparent Kd is a lower bound. The data are also fit well assuming an apparent Kd of
30.1<!--l. 95--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mo 
class="MathClass-bin">&#177;</mo></math>7.2
<!--l. 95--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>M that
was reported previously<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xramanan_structural_2012">274</a>]</span>.
</p>
<hr class="figure" /><div class="figure" 
>
                                                                                      
                                                                                      
<a 
 id="x8-60001r5"></a>
                                                                                      
                                                                                      
<!--l. 99--><p class="noindent" ><img 
src="Chapter5/ch5-fig5.png" alt="PIC"  
width="228" height="333"  />
<a 
 id="x8-60002"></a>
<br /> </p><div class="caption" 
><span class="id">Figure&#x00A0;4.5: </span><span  
class="content">Stabilizing the open cryptic pocket in VP35&#8217;s IID disrupts dsRNA binding. <span 
class="ptmb7t-x-x-120">A.</span>
Binding of both TNB-labeled and unlabeled C247S/C275S variants of the IID to two different
dsRNA  constructs.  This  protein  variant  only  has  cysteines  in  the  cryptic  pocket.  The  RNA
constructs both have a 25-bp double-stranded segment, and one has 2 nucleotide overhangs on
the 3&#8217; ends. The anisotropy was measured via a fluorescence polarization assay and fit to a
single-site binding model (black lines). The mean and standard deviation from three replicates
are shown but error bars are generally smaller than the symbols. <span 
class="ptmb7t-x-x-120">B. </span>Circular dichroism (CD)
spectra of labeled and unlabeled protein demonstrate that labeling does not unfold the protein.
The opaque and semi-transparent lines represent the mean and standard deviation, respectively,
from three replicates.</span></div><!--tex4ht:label?: x8-60001r4.3 -->
                                                                                      
                                                                                      
</div><hr class="endfigure" />
<!--l. 105--><p class="noindent" >Repeating our FP assay with TNB-labeled protein reveals that labeling allosterically reduces the
affinity for blunt-ended dsRNA by at least 5-fold (Fig. <a 
href="#x8-60001r5">4.5<!--tex4ht:ref: fig:ch5-fig5 --></a>A). Solubility limitations again prevented us
from observing complete binding curves for labeled protein, but the data are sufficient to demonstrate
that TNB-labeling has at least as strong an effect on binding as addition of a 3&#8217; overhang. As a control
to ensure that labeling does not disrupt binding by simply unfolding the protein, we measured the
circular dichroism (CD) spectra of labeled and unlabeled protein. The similarity between the CD
spectra (Fig. <a 
href="#x8-60001r5">4.5<!--tex4ht:ref: fig:ch5-fig5 --></a>B) demonstrates that the IID&#8217;s overall fold is not grossly perturbed. The slight
decrease in helicity at 220 nm can be attributed to the covalent modification disrupting the
stability of helix 5 potentially causing local unfolding of this motif. Since the cryptic pocket
does not coincide with the blunt end-binding interface, our results suggests the impact
on dsRNA binding is allosteric. Furthermore, past work demonstrated that reducing the
blunt end-binding affinity by as little as 3-fold is sufficient to allow a host to mount an
effective immune response<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xleung_structural_2010">258</a>]</span>, so targeting our cryptic pocket could be of great therapeutic
value.
</p>
<h3 class="sectionHead"><span class="titlemark">4.4    </span> <a 
 id="x8-610004.4"></a>Discussion.</h3>
<!--l. 108--><p class="noindent" >We have identified a cryptic allosteric site in the IID of the Ebola VP35 protein that provides a new
opportunity to target this essential viral component. Past work identified several sites within the VP35
IID that are critical for immune evasion and viral replication<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xmessaoudi_filovirus_2015">253</a>,&#x00A0;<a 
href="mainli4.html#Xhartman_c-terminal_2004">256</a>,&#x00A0;<a 
href="mainli4.html#Xprins_mutations_2010">261</a>,&#x00A0;<a 
href="mainli4.html#Xprins_basic_2010">262</a>]</span>, but structural
snapshots captured crystallographically lacked druggable pockets<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xleung_structure_2009">257</a>,&#x00A0;<a 
href="mainli4.html#Xleung_structural_2010">258</a>]</span>. We used adaptive
sampling simulations to access more of the ensemble of conformations that VP35 adopts, uncovering
an unanticipated cryptic pocket. While the pocket directly coincides with the interface that binds the
backbone of dsRNA, it was not clearly of therapeutic relevance since binding dsRNA&#8217;s blunt ends is
                                                                                      
                                                                                      
more important for Ebola&#8217;s immune evasion mechanism<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xedwards_differential_2016">259</a>]</span>. However, our simulations also
suggested the cryptic pocket is allosterically coupled to the blunt end-binding interface and,
therefore, could modulate this biologically-important interaction. Subsequent experiments
confirmed that fluctuations within the folded state of the IID expose two buried cysteines
that line the proposed cryptic pocket to solvent. Moreover, covalently modifying these
cysteines to stabilize the open form of the cryptic pocket allosterically disrupts binding to
dsRNA blunt ends by at least 5-fold. Previous work demonstrated that reducing the binding
affinity by as little as 3-fold is sufficient to allow a host to mount an effective immune
response<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xleung_structural_2010">258</a>]</span>. Therefore, it may be possible to attenuate the impact of viral replication and restrict
pathogenicity by designing small molecules to target the cryptic allosteric site we report
here.
</p><!--l. 110--><p class="noindent" >More generally, our results speak to the power of simulations to provide simultaneous access to both
hidden conformations and dynamics with atomic resolution. Such information is extremely difficult to
obtain from single structural snapshots or powerful techniques that report on dynamics without
directly yielding structures, such as NMR and hydrogen deuterium exchange. As a result, simulations
are a powerful means to uncover unanticipated features of proteins&#8217; conformational ensembles, such
as cryptic pockets and allostery, providing a foundation for the design of further experiments. We
anticipate such simulations will enable the discovery of cryptic pockets and cryptic allosteric sites in
other proteins, particularly those that are currently considered undruggable. Furthermore, the detailed
structural insight from simulations will facilitate the design of small molecule drugs that target these
sites.
</p><!--l. 112--><p class="noindent" >
</p>
<h3 class="sectionHead"><span class="titlemark">4.5    </span> <a 
 id="x8-620004.5"></a>Methods</h3>
                                                                                      
                                                                                      
<!--l. 113--><p class="noindent" >
</p>
<h4 class="subsectionHead"><span class="titlemark">4.5.1    </span> <a 
 id="x8-630004.5.1"></a>Molecular dynamics simulations and analysis</h4>
<!--l. 114--><p class="noindent" >Simulations were initiated from chain B of PDB 3L25<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xleung_structural_2010">258</a>]</span> and run with Gromacs<span class="cite">&#x00A0;[<a 
href="mainli4.html#XVanDerSpoel:2005hz">145</a>]</span> using the
amber03 force field<span class="cite">&#x00A0;[<a 
href="mainli4.html#XDuan:2003gt">146</a>]</span> and TIP3P explicit solvent<span class="cite">&#x00A0;[<a 
href="mainli4.html#XJorgensen:1983fl">143</a>]</span> at a temperature of 300 K and 1 bar
pressure, as described previously<span class="cite">&#x00A0;[<a 
href="mainli4.html#XHart:2016kb">9</a>]</span>. We first applied our FAST-pockets algorithm<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xzimmerman_fast_2015">35</a>]</span> to balance 1)
preferentially simulating structures with large pocket volumes that may harbor cryptic
pockets with 2) broad exploration of conformational space. For FAST, we performed 10
rounds of simulations with 10 simulations/round and 80 ns/simulation. To acquire better
statistics across the landscape, we performed an RMSD-based clustering using a hybrid
k-centers/k-medoids algorithm<span class="cite">&#x00A0;[<a 
href="mainli4.html#XBeauchamp:2011he">230</a>]</span> implemented in Enspara<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xporter2018enspara">39</a>]</span> to divide the data into
1,000 clusters. Then we ran three simulations initiated from each cluster center on the
Folding@home distributed computing environment, resulting in an aggregate simulation time of 122
<!--l. 114--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03BC;</mi></math>s.
</p><!--l. 116--><p class="noindent" >Exposons were identified using our previously described protocols,11 as implemented in
Enspara<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xporter2018enspara">39</a>]</span>. Briefly, the solvent accessible surface area (SASA) of each residue&#8217;s side-chain was
calculated using the Shrake-Rupley algorithm<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xshrake_environment_1973">275</a>]</span> implemented in MDTraj<span class="cite">&#x00A0;[<a 
href="mainli4.html#XMcGibbon:2015fv">148</a>]</span> using a
drug-sized probe (2.8 &#197; sphere). Conformations were clustered based on the SASA of each residue
using a hybrid k-centers/k-medoids algorithm, using a 2.5 &#197;2 distance cutoff and 5 rounds of
k-medoids updates. A Markov time of 6 ns was selected based on the implied timescales
test (<a 
href="mainap3.html#x15-141019r8">C.8<!--tex4ht:ref: fig:ch5-suppfig8 --></a>). The center of each cluster was taken as an exemplar of that conformational
state, and residues were classified as exposed if their SASA exceeded 2.0 &#197;2 and buried
otherwise. The mutual information between the burial/exposure of each pair of residues
was then calculated based on the MSM (i.e. treating the centers as samples and weighting
                                                                                      
                                                                                      
them by the equilibrium probability of the state they represent). Finally, exposons were
identified by clustering the matrix of pairwise mutual information values using affinity
propagation<span class="cite">&#x00A0;[<a 
href="mainli4.html#XFrey:2007hs">276</a>]</span>.
</p><!--l. 118--><p class="noindent" >The CARDS algorithm<span class="cite">&#x00A0;[<a 
href="mainli4.html#XSingh:2017hh">90</a>]</span> was applied to identify allosteric coupling using our established
protocols<span class="cite">&#x00A0;[<a 
href="mainli4.html#XSun:2018kx">48</a>]</span>, as implemented in Enspara<span class="cite">&#x00A0;[<a 
href="mainli4.html#XSingh:2017hh">90</a>]</span>. Briefly, each dihedral angle in each snapshot of the
simulations was assigned to one of three rotameric states (gauche+, gauche-, or trans) and one of two
dynamical states (ordered or disordered). The total coupling between each pair of dihedrals
<!--l. 118--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>X</mi></math> and
<!--l. 118--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>Y</mi> </math> was
then calculated as </p><table class="equation"><tr><td> <a 
 id="x8-63001r1"></a>
<!--l. 119--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="block" class="equation">
          <msub><mrow 
><mi 
>I</mi></mrow><mrow 
><mi 
>H</mi></mrow></msub 
><mrow ><mo 
class="MathClass-open">(</mo><mrow><msub><mrow 
><mi 
>X</mi></mrow><mrow 
><mi 
>R</mi></mrow></msub 
><mo 
class="MathClass-punc">,</mo><msub><mrow 
><mi 
>Y</mi> </mrow><mrow 
><mi 
>R</mi></mrow></msub 
></mrow><mo 
class="MathClass-close">)</mo></mrow> <mo 
class="MathClass-rel">=</mo> <mi 
>I</mi><mrow ><mo 
class="MathClass-open">(</mo><mrow><msub><mrow 
><mi 
>X</mi></mrow><mrow 
><mi 
>R</mi></mrow></msub 
><mo 
class="MathClass-punc">,</mo><msub><mrow 
><mi 
>Y</mi> </mrow><mrow 
><mi 
>D</mi></mrow></msub 
></mrow><mo 
class="MathClass-close">)</mo></mrow> <mo 
class="MathClass-bin">+</mo> <mi 
>I</mi><mrow ><mo 
class="MathClass-open">(</mo><mrow><msub><mrow 
><mi 
>X</mi></mrow><mrow 
><mi 
>R</mi></mrow></msub 
><mo 
class="MathClass-punc">,</mo><msub><mrow 
><mi 
>Y</mi> </mrow><mrow 
><mi 
>D</mi></mrow></msub 
></mrow><mo 
class="MathClass-close">)</mo></mrow> <mo 
class="MathClass-bin">+</mo> <mi 
>I</mi><mrow ><mo 
class="MathClass-open">(</mo><mrow><msub><mrow 
><mi 
>X</mi></mrow><mrow 
><mi 
>D</mi></mrow></msub 
><mo 
class="MathClass-punc">,</mo><msub><mrow 
><mi 
>Y</mi> </mrow><mrow 
><mi 
>R</mi></mrow></msub 
></mrow><mo 
class="MathClass-close">)</mo></mrow> <mo 
class="MathClass-bin">+</mo> <mi 
>I</mi><mrow ><mo 
class="MathClass-open">(</mo><mrow><msub><mrow 
><mi 
>X</mi></mrow><mrow 
><mi 
>D</mi></mrow></msub 
><mo 
class="MathClass-punc">,</mo><msub><mrow 
><mi 
>Y</mi> </mrow><mrow 
><mi 
>D</mi></mrow></msub 
></mrow><mo 
class="MathClass-close">)</mo></mrow>
</math></td><td class="eq-no">(4.1)</td></tr></table>
<!--l. 123--><p class="noindent" >where <!--l. 123--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>I</mi></math> is the mutual
information metric, <!--l. 123--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
><mi 
>X</mi></mrow><mrow 
><mi 
>R</mi></mrow></msub 
></math> is the
rotameric state of dihedral <!--l. 123--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>X</mi></math>,
and <!--l. 123--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><msub><mrow 
><mi 
>X</mi></mrow><mrow 
><mi 
>D</mi></mrow></msub 
></math> is the dynamical
state of dihedral <!--l. 123--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>X</mi></math>.
The term <!--l. 123--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>I</mi><mrow ><mo 
class="MathClass-open">(</mo><mrow><msub><mrow 
><mi 
>X</mi></mrow><mrow 
><mi 
>R</mi></mrow></msub 
><mo 
class="MathClass-punc">,</mo><msub><mrow 
><mi 
>Y</mi> </mrow><mrow 
><mi 
>R</mi></mrow></msub 
></mrow><mo 
class="MathClass-close">)</mo></mrow></math>
is the purely structural coupling, while the sum of the other three terms is referred to as the
disorder-mediated coupling. The dihedral level couplings were coarse-grained into residue-level
coupling by summing the total coupling between all the relevant dihedrals. Communities of coupled
                                                                                      
                                                                                      
residues were identified by clustering the residue-level matrix of total couplings using affinity
propagation<span class="cite">&#x00A0;[<a 
href="mainli4.html#XFrey:2007hs">276</a>]</span>. The constructed network was subsequently filtered to only retain significant
edges<span class="cite">&#x00A0;[<a 
href="mainli4.html#XDianati:2016bt">277</a>]</span>.These algorithms are available at github.com/bowman-lab.
</p><!--l. 125--><p class="noindent" >
</p>
<h4 class="subsectionHead"><span class="titlemark">4.5.2    </span> <a 
 id="x8-640004.5.2"></a>Protein expression and purification</h4>
<!--l. 126--><p class="noindent" >All variants of VP35&#8217;s IID were purified from the cytoplasm of E. coli BL21(DE3) Gold
cells (Agilent Technologies). Variants were generated using the site directed mutagenesis
method and confirmed by DNA sequencing. Transformed cells were grown at 37&#8451; until OD
0.3 then grown at 18&#8451; until induction at OD 0.6 with 1 mM IPTG (Gold Biotechnology,
Olivette, MO). Cells were grown for 15 hours then centrifuged after which the pellet was
resuspended in 20 mM Sodium Phosphate pH 8, 1 M sodium chloride, with 5.1 mM
<!--l. 126--><math 
 xmlns="http://www.w3.org/1998/Math/MathML"  
display="inline" ><mi 
>&#x03B2;</mi></math>-mercaptoethanol.
Resuspended cells were subjected to sonication at 4<&#8451; followed by centrifugation. The supernatant
was then subjected to Ni-NTA affinity, TEV digestion, cation exchange (BioRad UNOsphere Rapid S
column), and size exclusion chromatography (BioRad Enrich SEC 70 column) into 10 mM Hepes pH
7, 150 mM NaCl, 1 mM MgCl2, 2 mM TCEP.
</p><!--l. 128--><p class="noindent" >
</p>
<h4 class="subsectionHead"><span class="titlemark">4.5.3    </span> <a 
 id="x8-650004.5.3"></a>Thiol labeling</h4>
                                                                                      
                                                                                      
<!--l. 129--><p class="noindent" >We monitored the change in absorbance over time of 5,5&#8217;-dithiobis-(2-nitrobenzoic acid) (DTNB,
Ellman&#8217;s reagent, Thermo Fisher Scientific). Various concentrations of DTNB were added to protein
and change in absorbance was measured in either an SX-20 Stopped Flow instrument (Applied
Photophysics, Leatherhead, UK), or an Agilent Cary60 UV-vis spectrophotometer at 412 nm until the
reaction reached steady state (3&#x0303;00 s). Data were fit with a Linderstr&#248;m-Lang model to extract the
thermodynamics and/or kinetics of pocket opening, as described in detail previously<span class="cite">&#x00A0;[<a 
href="mainli4.html#XPorter:2019hv">49</a>]</span>. As a
control, the equilibrium constant for folding and the unfolding rate were measured (<a 
href="mainap3.html#x15-141013r1">C.1<!--tex4ht:ref: tab:ch5-supptab1 --></a>)
and used to predict the expected labeling rate from the unfolded state. The equilibrium
constant was inferred from a two-state fit to urea melts monitored by fluorescence and
unfolding rates were inferred from single exponential fits to unfolding curves monitored by
fluorescence after the addition of urea, as described previously<span class="cite">&#x00A0;[<a 
href="mainli4.html#XPorter:2019hv">49</a>,&#x00A0;<a 
href="mainli4.html#XBowman:2015dh">147</a>,&#x00A0;<a 
href="mainli4.html#Xzimmerman_prediction_2017">278</a>]</span>. Fluorescence
data were collected using a Photon Technology International Quanta- Master 800 rapid
excitation spectrofluorometer with Quantum Northwest Inc. TC-125 Peltier-controlled cuvette
holder.
</p><!--l. 131--><p class="noindent" >
</p>
<h4 class="subsectionHead"><span class="titlemark">4.5.4    </span> <a 
 id="x8-660004.5.4"></a>Fluorescence polarization binding assay</h4>
<!--l. 132--><p class="noindent" >Binding affinities between variants of VP35&#8217;s IID and dsRNA were measured using fluorescence
polarization in 10 mM Hepes pH 7, 150 mM NaCl, 1 mM MgCl2. A 25 base pair FITC-dsRNA
(Integrated DNA Technologies) substrate with and without a 2 nucleotide 3&#8217; overhang was included at
100 nM. The sample was equilibrated for one hour before data collection. Data were collected on a
BioTek Synergy2 Multi-Mode Reader as polarization and were converted to anisotropy as described
previously<span class="cite">&#x00A0;[<a 
href="mainli4.html#Xkeck_ssbdna_2012">273</a>]</span>. TNB-labeled samples were generated by allowing DTNB and VP35&#8217;s
IID to react for 3 minutes and then removing excess DTNB with a Zeba spin desalting
                                                                                      
                                                                                      
columns (Thermo Fisher Scientific). A single-site binding model was sufficient to fit the
data.
</p><!--l. 134--><p class="noindent" >
</p>
<h3 class="sectionHead"><span class="titlemark">4.6    </span> <a 
 id="x8-670004.6"></a>Acknowledgements</h3>
<!--l. 135--><p class="noindent" >We are grateful to the citizen scientists who participate in Folding@home for volunteering to
run simulations on their personal computers. This work was funded by NSF CAREER
Award MCB-1552471 and NIH grant R01 GM124007 (Bowman), as well as NIH grants
R01AI123926, P01AI120943, and R01AI143292 (Amarasinghe). GRB holds a Career Award at
the Scientific Interface from the Burroughs Wellcome Fund and a Packard Fellowship
for Science and Engineering from The David &amp; Lucile Packard Foundation. MAC was
supported by the 5R25GM103757 IMSD program and SS was supported by a MilliporeSigma
Fellowship. We thank Drs. Timothy M. Lohman and Alexander G. Kozlov for advice on FP
assays.
                                                                                      
                                                                                      
</p>
<!--l. 3--><div class="crosslinks"><p class="noindent">[<a 
href="mainch5.html" >next</a>] [<a 
href="mainch3.html" >prev</a>] [<a 
href="mainch3.html#tailmainch3.html" >prev-tail</a>] [<a 
href="mainch4.html" >front</a>] [<a 
href="main.html#mainch4.html" >up</a>] </p></div>
<!--l. 3--><p class="noindent" ><a 
 id="tailmainch4.html"></a></p> 
</body></html> 
